{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "machine_shape": "hm",
      "gpuType": "A100",
      "toc_visible": true,
      "authorship_tag": "ABX9TyPTTGla8dKub0o7kSZedDFJ",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/arjan-hada/nuclease-design-vae/blob/main/01_ESM_IF1_nucB.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Generating protein sequences from their backbone atom coordinates with ESM-IF1"
      ],
      "metadata": {
        "id": "3v7MIe7RMbaa"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "In this notebook, we will use ESM-IF1 inverse folding model for predicting protein sequences from their backbone atom coordinates.\n",
        "\n",
        "**Goals**\n",
        "- Sample sequence designs for a given structure.\n",
        "- Score the conditional log-likelihoods for sequences conditioned on a given structure.\n",
        "\n",
        "**Rationale**\n",
        "\n",
        "In the next notebook, we will be building a generative unsupervised latent variable model for protein sequences - Variational AutoEncoder (VAE). We want to generate test data set for model validation. We will do so by creating our own simulated homologous set of sequences. The premise is that homologous sequences are characterized by a shared protein structure backbone. By leveraging this concept in reverse, we can start with a protein structure backbone and computationally generate sequences predicted to fold into this predetermined structure backbone. The ESM-IF1 inverse folding model generates sequences that are likely to fold into a predetermined structure backbone along with a log likelihood score of doing so.\n",
        "\n",
        "We will examine the correlation between the log likelihood scores of the simulated homology sequences as generated by our VAE and those produced by the inverse folding model. The VAE itself will be trained on multiple sequence alignment of natural homologs.\n",
        "\n"
      ],
      "metadata": {
        "id": "1OeCVLvJg5xu"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Setup"
      ],
      "metadata": {
        "id": "tR85ON3bCX2L"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "z6ObmljMMaKr",
        "outputId": "71b640a7-c717-40fe-aa1e-f365db72a4d2"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  Installing build dependencies ... \u001b[?25l\u001b[?25hdone\n",
            "  Getting requirements to build wheel ... \u001b[?25l\u001b[?25hdone\n",
            "  Preparing metadata (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m35.8/35.8 MB\u001b[0m \u001b[31m54.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h"
          ]
        }
      ],
      "source": [
        "# Colab environment setup\n",
        "\n",
        "# Install the correct version of Pytorch Geometric.\n",
        "import torch\n",
        "\n",
        "def format_pytorch_version(version):\n",
        "  return version.split('+')[0]\n",
        "\n",
        "TORCH_version = torch.__version__\n",
        "TORCH = format_pytorch_version(TORCH_version)\n",
        "\n",
        "def format_cuda_version(version):\n",
        "  return 'cu' + version.replace('.', '')\n",
        "\n",
        "CUDA_version = torch.version.cuda\n",
        "CUDA = format_cuda_version(CUDA_version)\n",
        "\n",
        "!pip install -q torch-scatter -f https://data.pyg.org/whl/torch-{TORCH}+{CUDA}.html\n",
        "!pip install -q torch-sparse -f https://data.pyg.org/whl/torch-{TORCH}+{CUDA}.html\n",
        "!pip install -q torch-cluster -f https://data.pyg.org/whl/torch-{TORCH}+{CUDA}.html\n",
        "!pip install -q torch-spline-conv -f https://data.pyg.org/whl/torch-{TORCH}+{CUDA}.html\n",
        "!pip install -q torch-geometric\n",
        "\n",
        "# Install esm\n",
        "!pip install -q git+https://github.com/facebookresearch/esm.git\n",
        "\n",
        "# Install biotite\n",
        "!pip install -q biotite==0.41.2"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import biotite\n",
        "print(biotite.__version__)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "wUs10apwEzae",
        "outputId": "c3472dcc-0998-4816-ec1b-0605bb6bf391"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "0.41.2\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install -q py3Dmol biopython Levenshtein"
      ],
      "metadata": {
        "id": "JtUnk6T1uPg7"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "## Verify that pytorch-geometric is correctly installed\n",
        "import torch_geometric\n",
        "import torch_sparse\n",
        "from torch_geometric.nn import MessagePassing"
      ],
      "metadata": {
        "id": "Cxb-ogfsvxjR"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "import matplotlib.pyplot as plt\n",
        "import seaborn as sns\n",
        "sns.set_context(context=\"paper\", font_scale=1.5)"
      ],
      "metadata": {
        "id": "6Fmr7qE41tMk"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "Inverse folding code from [Inverse folding with ESM-IF1]( https://github.com/facebookresearch/esm/blob/main/examples/inverse_folding/notebook.ipynb)."
      ],
      "metadata": {
        "id": "2st8Tq1_o2V-"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Load structure from PDB files"
      ],
      "metadata": {
        "id": "DhfZFd19C2rn"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!wget https://files.rcsb.org/download/5OMT.pdb -P data/"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "GMiChDe_C3_N",
        "outputId": "a82ebdf6-8ec8-482a-8ae9-911e5a5e91b2"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-09-12 16:42:52--  https://files.rcsb.org/download/5OMT.pdb\n",
            "Resolving files.rcsb.org (files.rcsb.org)... 128.6.159.245\n",
            "Connecting to files.rcsb.org (files.rcsb.org)|128.6.159.245|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: unspecified [application/octet-stream]\n",
            "Saving to: ‘data/5OMT.pdb.3’\n",
            "\n",
            "5OMT.pdb.3              [    <=>             ] 179.64K   280KB/s    in 0.6s    \n",
            "\n",
            "2024-09-12 16:42:53 (280 KB/s) - ‘data/5OMT.pdb.3’ saved [183951]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Look at first 10 lines of pdb file\n",
        "!head -n 10 data/5OMT.pdb"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Bs_hWNE3Gzlw",
        "outputId": "83a72222-e64d-453d-e65e-2b7eb1f2a3b7"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "HEADER    HYDROLASE                               01-AUG-17   5OMT              \n",
            "TITLE     ENDONUCLEASE NUCB                                                     \n",
            "COMPND    MOL_ID: 1;                                                            \n",
            "COMPND   2 MOLECULE: NUCB;                                                      \n",
            "COMPND   3 CHAIN: A;                                                            \n",
            "COMPND   4 ENGINEERED: YES                                                      \n",
            "SOURCE    MOL_ID: 1;                                                            \n",
            "SOURCE   2 ORGANISM_SCIENTIFIC: BACILLUS LICHENIFORMIS;                         \n",
            "SOURCE   3 ORGANISM_TAXID: 1402;                                                \n",
            "SOURCE   4 GENE: NUCB;                                                          \n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import py3Dmol\n",
        "\n",
        "def view_pdb(fpath, chain_id):\n",
        "    with open(fpath) as ifile:\n",
        "        system = \"\".join([x for x in ifile])\n",
        "\n",
        "    view = py3Dmol.view(width=600, height=400)\n",
        "    view.addModelsAsFrames(system)\n",
        "    view.setStyle({'model': -1, 'chain': chain_id}, {\"cartoon\": {'color': 'spectrum'}})\n",
        "    view.zoomTo()\n",
        "    view.show()"
      ],
      "metadata": {
        "id": "iEyPfKtFGzi6"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "fpath = 'data/5OMT.pdb'\n",
        "chain_id = ['A', 'B']\n",
        "view_pdb(fpath, chain_id)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 417
        },
        "id": "gvAje0V4Gzf6",
        "outputId": "6beee96f-9ad8-470d-a937-7a0d3075da26"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17261593772982423\"  style=\"position: relative; width: 600px; height: 400px;\">\n        <p id=\"3dmolwarning_17261593772982423\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.0/3Dmol-min.js');\n}\n\nvar viewer_17261593772982423 = null;\nvar warn = document.getElementById(\"3dmolwarning_17261593772982423\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17261593772982423 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17261593772982423\"),{backgroundColor:\"white\"});\nviewer_17261593772982423.zoomTo();\n\tviewer_17261593772982423.addModelsAsFrames(\"HEADER    HYDROLASE                               01-AUG-17   5OMT              \\nTITLE     ENDONUCLEASE NUCB                                                     \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: NUCB;                                                      \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: BACILLUS LICHENIFORMIS;                         \\nSOURCE   3 ORGANISM_TAXID: 1402;                                                \\nSOURCE   4 GENE: NUCB;                                                          \\nSOURCE   5 EXPRESSION_SYSTEM: BACILLUS SUBTILIS;                                \\nSOURCE   6 EXPRESSION_SYSTEM_TAXID: 1423                                        \\nKEYWDS    ENDONUCLEASE, BIOFILM, BACILLUS SUBTILIS, DNA, HYDROLASE              \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    A.BASLE,R.J.LEWIS                                                     \\nREVDAT   3   24-JAN-18 5OMT    1       JRNL                                     \\nREVDAT   2   06-DEC-17 5OMT    1       JRNL                                     \\nREVDAT   1   15-NOV-17 5OMT    0                                                \\nJRNL        AUTH   A.BASLE,L.HEWITT,A.KOH,H.K.LAMB,P.THOMPSON,J.G.BURGESS,      \\nJRNL        AUTH 2 M.J.HALL,A.R.HAWKINS,H.MURRAY,R.J.LEWIS                      \\nJRNL        TITL   CRYSTAL STRUCTURE OF NUCB, A BIOFILM-DEGRADING ENDONUCLEASE. \\nJRNL        REF    NUCLEIC ACIDS RES.            V.  46   473 2018              \\nJRNL        REFN                   ESSN 1362-4962                               \\nJRNL        PMID   29165717                                                     \\nJRNL        DOI    10.1093/NAR/GKX1170                                          \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : REFMAC 5.8.0158                                      \\nREMARK   3   AUTHORS     : MURSHUDOV,VAGIN,DODSON                               \\nREMARK   3                                                                      \\nREMARK   3    REFINEMENT TARGET : NULL                                          \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 44.56                          \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : 100.0                          \\nREMARK   3   NUMBER OF REFLECTIONS             : 25934                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : THROUGHOUT                      \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : RANDOM                          \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.150                           \\nREMARK   3   R VALUE            (WORKING SET) : 0.149                           \\nREMARK   3   FREE R VALUE                     : 0.186                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 5.000                           \\nREMARK   3   FREE R VALUE TEST SET COUNT      : 1369                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : 20                           \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : 1.35                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : 1.39                         \\nREMARK   3   REFLECTION IN BIN     (WORKING SET) : 1891                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : 100.0                        \\nREMARK   3   BIN R VALUE           (WORKING SET) : 0.1840                       \\nREMARK   3   BIN FREE R VALUE SET COUNT          : 85                           \\nREMARK   3   BIN FREE R VALUE                    : 0.2330                       \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 839                                     \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 0                                       \\nREMARK   3   SOLVENT ATOMS            : 104                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : 18.44                          \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : -0.79000                                             \\nREMARK   3    B22 (A**2) : -0.79000                                             \\nREMARK   3    B33 (A**2) : 1.58000                                              \\nREMARK   3    B12 (A**2) : 0.00000                                              \\nREMARK   3    B13 (A**2) : 0.00000                                              \\nREMARK   3    B23 (A**2) : 0.00000                                              \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED OVERALL COORDINATE ERROR.                                 \\nREMARK   3   ESU BASED ON R VALUE                            (A): 0.048         \\nREMARK   3   ESU BASED ON FREE R VALUE                       (A): 0.049         \\nREMARK   3   ESU BASED ON MAXIMUM LIKELIHOOD                 (A): 0.028         \\nREMARK   3   ESU FOR B VALUES BASED ON MAXIMUM LIKELIHOOD (A**2): 1.528         \\nREMARK   3                                                                      \\nREMARK   3 CORRELATION COEFFICIENTS.                                            \\nREMARK   3   CORRELATION COEFFICIENT FO-FC      : 0.969                         \\nREMARK   3   CORRELATION COEFFICIENT FO-FC FREE : 0.958                         \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES        COUNT    RMS    WEIGHT      \\nREMARK   3   BOND LENGTHS REFINED ATOMS        (A):   870 ; 0.014 ; 0.019       \\nREMARK   3   BOND LENGTHS OTHERS               (A):   780 ; 0.002 ; 0.020       \\nREMARK   3   BOND ANGLES REFINED ATOMS   (DEGREES):  1178 ; 1.676 ; 1.943       \\nREMARK   3   BOND ANGLES OTHERS          (DEGREES):  1808 ; 0.996 ; 3.000       \\nREMARK   3   TORSION ANGLES, PERIOD 1    (DEGREES):   114 ; 5.752 ; 5.000       \\nREMARK   3   TORSION ANGLES, PERIOD 2    (DEGREES):    42 ;35.634 ;22.381       \\nREMARK   3   TORSION ANGLES, PERIOD 3    (DEGREES):   140 ;12.034 ;15.000       \\nREMARK   3   TORSION ANGLES, PERIOD 4    (DEGREES):    10 ;10.784 ;15.000       \\nREMARK   3   CHIRAL-CENTER RESTRAINTS       (A**3):   119 ; 0.109 ; 0.200       \\nREMARK   3   GENERAL PLANES REFINED ATOMS      (A):  1005 ; 0.009 ; 0.020       \\nREMARK   3   GENERAL PLANES OTHERS             (A):   197 ; 0.002 ; 0.020       \\nREMARK   3   NON-BONDED CONTACTS REFINED ATOMS (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   NON-BONDED CONTACTS OTHERS        (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   NON-BONDED TORSION REFINED ATOMS  (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   NON-BONDED TORSION OTHERS         (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   H-BOND (X...Y) REFINED ATOMS      (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   H-BOND (X...Y) OTHERS             (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   POTENTIAL METAL-ION REFINED ATOMS (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   POTENTIAL METAL-ION OTHERS        (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY VDW REFINED ATOMS        (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY VDW OTHERS               (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY H-BOND REFINED ATOMS     (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY H-BOND OTHERS            (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY METAL-ION REFINED ATOMS  (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY METAL-ION OTHERS         (A):  NULL ;  NULL ;  NULL       \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.     COUNT   RMS    WEIGHT      \\nREMARK   3   MAIN-CHAIN BOND REFINED ATOMS  (A**2):   441 ; 2.302 ; 1.506       \\nREMARK   3   MAIN-CHAIN BOND OTHER ATOMS    (A**2):   440 ; 2.224 ; 1.494       \\nREMARK   3   MAIN-CHAIN ANGLE REFINED ATOMS (A**2):   551 ; 2.812 ; 2.261       \\nREMARK   3   MAIN-CHAIN ANGLE OTHER ATOMS   (A**2):   552 ; 2.835 ; 2.270       \\nREMARK   3   SIDE-CHAIN BOND REFINED ATOMS  (A**2):   429 ; 3.458 ; 1.949       \\nREMARK   3   SIDE-CHAIN BOND OTHER ATOMS    (A**2):   426 ; 3.343 ; 1.925       \\nREMARK   3   SIDE-CHAIN ANGLE REFINED ATOMS (A**2):  NULL ;  NULL ;  NULL       \\nREMARK   3   SIDE-CHAIN ANGLE OTHER ATOMS   (A**2):   623 ; 3.901 ; 2.757       \\nREMARK   3   LONG RANGE B REFINED ATOMS     (A**2):  1005 ; 4.521 ;19.441       \\nREMARK   3   LONG RANGE B OTHER ATOMS       (A**2):   984 ; 4.424 ;19.011       \\nREMARK   3                                                                      \\nREMARK   3 ANISOTROPIC THERMAL FACTOR RESTRAINTS.    COUNT   RMS   WEIGHT       \\nREMARK   3   RIGID-BOND RESTRAINTS          (A**2):  1650 ; 2.176 ; 3.000       \\nREMARK   3   SPHERICITY; FREE ATOMS         (A**2):    76 ;18.023 ; 5.000       \\nREMARK   3   SPHERICITY; BONDED ATOMS       (A**2):  1656 ;10.774 ; 5.000       \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS STATISTICS                                           \\nREMARK   3   NUMBER OF DIFFERENT NCS GROUPS : NULL                              \\nREMARK   3                                                                      \\nREMARK   3  TLS DETAILS                                                         \\nREMARK   3   NUMBER OF TLS GROUPS  : NULL                                       \\nREMARK   3                                                                      \\nREMARK   3  BULK SOLVENT MODELLING.                                             \\nREMARK   3   METHOD USED : NULL                                                 \\nREMARK   3   PARAMETERS FOR MASK CALCULATION                                    \\nREMARK   3   VDW PROBE RADIUS   : 1.20                                          \\nREMARK   3   ION PROBE RADIUS   : 0.80                                          \\nREMARK   3   SHRINKAGE RADIUS   : 0.80                                          \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: HYDROGENS HAVE BEEN ADDED IN THE RIDING   \\nREMARK   3  POSITIONS                                                           \\nREMARK   4                                                                      \\nREMARK   4 5OMT COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY PDBE ON 02-AUG-17.                  \\nREMARK 100 THE DEPOSITION ID IS D_1200006052.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 14-JUL-12                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \\nREMARK 200  RADIATION SOURCE               : DIAMOND                            \\nREMARK 200  BEAMLINE                       : I04                                \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : 0.9795                             \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : CCD                                \\nREMARK 200  DETECTOR MANUFACTURER          : ADSC QUANTUM 315                   \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : XDS, AIMLESS                       \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 27361                              \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 1.350                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : 44.560                             \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : 100.0                              \\nREMARK 200  DATA REDUNDANCY                : 7.800                              \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 16.1000                            \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 1.35                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 1.37                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : 100.0                              \\nREMARK 200  DATA REDUNDANCY IN SHELL       : 7.70                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : 3.600                              \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: SAD                          \\nREMARK 200 SOFTWARE USED: SHELXCD                                               \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 50.62                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.49                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: 100 MM MES PH6.5 12% PEG 20000, VAPOR    \\nREMARK 280  DIFFUSION, SITTING DROP, TEMPERATURE 293K                           \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 41 21 2                        \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X,-Y,Z+1/2                                             \\nREMARK 290       3555   -Y+1/2,X+1/2,Z+1/4                                      \\nREMARK 290       4555   Y+1/2,-X+1/2,Z+3/4                                      \\nREMARK 290       5555   -X+1/2,Y+1/2,-Z+1/4                                     \\nREMARK 290       6555   X+1/2,-Y+1/2,-Z+3/4                                     \\nREMARK 290       7555   Y,X,-Z                                                  \\nREMARK 290       8555   -Y,-X,-Z+1/2                                            \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       33.30200            \\nREMARK 290   SMTRY1   3  0.000000 -1.000000  0.000000       29.97750            \\nREMARK 290   SMTRY2   3  1.000000  0.000000  0.000000       29.97750            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       16.65100            \\nREMARK 290   SMTRY1   4  0.000000  1.000000  0.000000       29.97750            \\nREMARK 290   SMTRY2   4 -1.000000  0.000000  0.000000       29.97750            \\nREMARK 290   SMTRY3   4  0.000000  0.000000  1.000000       49.95300            \\nREMARK 290   SMTRY1   5 -1.000000  0.000000  0.000000       29.97750            \\nREMARK 290   SMTRY2   5  0.000000  1.000000  0.000000       29.97750            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       16.65100            \\nREMARK 290   SMTRY1   6  1.000000  0.000000  0.000000       29.97750            \\nREMARK 290   SMTRY2   6  0.000000 -1.000000  0.000000       29.97750            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000       49.95300            \\nREMARK 290   SMTRY1   7  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   8  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   8 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000       33.30200            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: MONOMERIC                  \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 0 ANGSTROM**2                             \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 6030 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: 0.0 KCAL/MOL                          \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 465                                                                      \\nREMARK 465 MISSING RESIDUES                                                     \\nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \\nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \\nREMARK 465                                                                      \\nREMARK 465   M RES C SSSEQI                                                     \\nREMARK 465     ALA A    33                                                      \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: CLOSE CONTACTS IN SAME ASYMMETRIC UNIT                     \\nREMARK 500                                                                      \\nREMARK 500 THE FOLLOWING ATOMS ARE IN CLOSE CONTACT.                            \\nREMARK 500                                                                      \\nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI           DISTANCE          \\nREMARK 500   NE   ARG A    77     O    HOH A   201              2.13            \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: CLOSE CONTACTS                                             \\nREMARK 500                                                                      \\nREMARK 500 THE FOLLOWING ATOMS THAT ARE RELATED BY CRYSTALLOGRAPHIC             \\nREMARK 500 SYMMETRY ARE IN CLOSE CONTACT.  AN ATOM LOCATED WITHIN 0.15          \\nREMARK 500 ANGSTROMS OF A SYMMETRY RELATED ATOM IS ASSUMED TO BE ON A           \\nREMARK 500 SPECIAL POSITION AND IS, THEREFORE, LISTED IN REMARK 375             \\nREMARK 500 INSTEAD OF REMARK 500.  ATOMS WITH NON-BLANK ALTERNATE               \\nREMARK 500 LOCATION INDICATORS ARE NOT INCLUDED IN THE CALCULATIONS.            \\nREMARK 500                                                                      \\nREMARK 500 DISTANCE CUTOFF:                                                     \\nREMARK 500 2.2 ANGSTROMS FOR CONTACTS NOT INVOLVING HYDROGEN ATOMS              \\nREMARK 500 1.6 ANGSTROMS FOR CONTACTS INVOLVING HYDROGEN ATOMS                  \\nREMARK 500                                                                      \\nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI  SSYMOP   DISTANCE          \\nREMARK 500   O    HOH A   224     O    HOH A   227     3555     1.94            \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    VAL A 141      -53.41   -130.99                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  5OMT A   33   142  UNP    F1BV52   F1BV52_BACLI    33    142             \\nSEQRES   1 A  110  ALA ALA ARG TYR ASP ASP VAL LEU TYR PHE PRO ALA SER          \\nSEQRES   2 A  110  ARG TYR PRO GLU THR GLY ALA HIS ILE SER ASP ALA ILE          \\nSEQRES   3 A  110  LYS ALA GLY HIS ALA ASP VAL CYS THR ILE GLU ARG SER          \\nSEQRES   4 A  110  GLY ALA ASP LYS ARG ARG GLN GLU SER LEU LYS GLY ILE          \\nSEQRES   5 A  110  PRO THR LYS PRO GLY PHE ASP ARG ASP GLU TRP PRO MET          \\nSEQRES   6 A  110  ALA MET CYS GLU GLU GLY GLY LYS GLY ALA SER VAL ARG          \\nSEQRES   7 A  110  TYR VAL SER SER SER ASP ASN ARG GLY ALA GLY SER TRP          \\nSEQRES   8 A  110  VAL GLY ASN ARG LEU ASN GLY TYR ALA ASP GLY THR ARG          \\nSEQRES   9 A  110  ILE LEU PHE ILE VAL GLN                                      \\nFORMUL   2  HOH   *104(H2 O)                                                    \\nHELIX    1 AA1 TYR A   47  ALA A   60  1                                  14    \\nHELIX    2 AA2 GLY A   72  LYS A   82  1                                  11    \\nHELIX    3 AA3 SER A  114  ASN A  129  1                                  16    \\nSHEET    1 AA1 3 ASP A  38  TYR A  41  0                                        \\nSHEET    2 AA1 3 ARG A 136  ILE A 140  1  O  LEU A 138   N  LEU A  40           \\nSHEET    3 AA1 3 VAL A  65  THR A  67 -1  N  CYS A  66   O  ILE A 137           \\nSHEET    1 AA2 2 PHE A  90  TRP A  95  0                                        \\nSHEET    2 AA2 2 SER A 108  SER A 113 -1  O  VAL A 112   N  ASP A  91           \\nSSBOND   1 CYS A   66    CYS A  100                          1555   1555  2.05  \\nCISPEP   1 TRP A   95    PRO A   96          0        -0.27                     \\nCRYST1   59.955   59.955   66.604  90.00  90.00  90.00 P 41 21 2     8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.016679  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.016679  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.015014        0.00000                         \\nATOM      1  N   ALA A  34      14.458  39.288   6.017  1.00 31.14           N  \\nANISOU    1  N   ALA A  34     4143   2682   5004   -276   1064   2656       N  \\nATOM      2  CA  ALA A  34      13.318  38.299   6.191  1.00 24.44           C  \\nANISOU    2  CA  ALA A  34     3739   2283   3262   -122     42   1126       C  \\nATOM      3  C   ALA A  34      13.580  37.130   7.127  1.00 25.95           C  \\nANISOU    3  C   ALA A  34     3588   2646   3624   -142    446   1610       C  \\nATOM      4  O   ALA A  34      14.736  36.794   7.371  1.00 25.95           O  \\nANISOU    4  O   ALA A  34     4013   2603   3241    411    660    350       O  \\nATOM      5  CB  ALA A  34      12.860  37.796   4.821  1.00 28.47           C  \\nANISOU    5  CB  ALA A  34     2807   4521   3486    255   -934    858       C  \\nATOM      6  N   ARG A  35      12.506  36.565   7.716  1.00 18.00           N  \\nANISOU    6  N   ARG A  35     3214   1382   2241    119    568    -72       N  \\nATOM      7  CA  ARG A  35      12.654  35.467   8.680  1.00 16.94           C  \\nANISOU    7  CA  ARG A  35     2667   1472   2295   -111   -178   -103       C  \\nATOM      8  C   ARG A  35      12.701  34.068   8.058  1.00 15.64           C  \\nANISOU    8  C   ARG A  35     2612   1649   1678    -14    -47    -75       C  \\nATOM      9  O   ARG A  35      13.272  33.117   8.613  1.00 16.57           O  \\nANISOU    9  O   ARG A  35     2498   1768   2028    -18   -133    255       O  \\nATOM     10  CB  ARG A  35      11.497  35.573   9.689  1.00 22.02           C  \\nANISOU   10  CB  ARG A  35     3437   2420   2506    -99    276   -503       C  \\nATOM     11  CG  ARG A  35      11.719  34.920  10.993  1.00 21.39           C  \\nANISOU   11  CG  ARG A  35     3098   2242   2787    187    192   -381       C  \\nATOM     12  CD  ARG A  35      10.696  35.325  12.058  1.00 20.47           C  \\nANISOU   12  CD  ARG A  35     3275   2455   2045   -196    230   -161       C  \\nATOM     13  NE  ARG A  35       9.381  34.700  11.909  1.00 18.84           N  \\nANISOU   13  NE  ARG A  35     3109   2243   1806    145     34   -249       N  \\nATOM     14  CZ  ARG A  35       9.044  33.472  12.332  1.00 19.49           C  \\nANISOU   14  CZ  ARG A  35     3214   2398   1791   -129    396   -415       C  \\nATOM     15  NH1 ARG A  35       9.925  32.690  12.910  1.00 20.87           N  \\nANISOU   15  NH1 ARG A  35     3561   2301   2066   -497    337    166       N  \\nATOM     16  NH2 ARG A  35       7.804  33.038  12.140  1.00 24.42           N  \\nANISOU   16  NH2 ARG A  35     3499   2915   2864   -424    412   -261       N  \\nATOM     17  N   TYR A  36      12.071  33.937   6.914  1.00 12.70           N  \\nANISOU   17  N   TYR A  36     2305   1146   1372    105    175     -8       N  \\nATOM     18  CA  TYR A  36      11.982  32.660   6.183  1.00 12.66           C  \\nANISOU   18  CA  TYR A  36     2240   1234   1335    364    267   -139       C  \\nATOM     19  C   TYR A  36      11.837  32.934   4.715  1.00 13.35           C  \\nANISOU   19  C   TYR A  36     2285   1338   1446    294    123    135       C  \\nATOM     20  O   TYR A  36      11.511  34.064   4.303  1.00 15.85           O  \\nANISOU   20  O   TYR A  36     3102   1346   1574    534    132     60       O  \\nATOM     21  CB  TYR A  36      10.830  31.796   6.721  1.00 14.81           C  \\nANISOU   21  CB  TYR A  36     2841   1274   1512    225    405    138       C  \\nATOM     22  CG  TYR A  36       9.509  32.488   6.731  1.00 14.38           C  \\nANISOU   22  CG  TYR A  36     2615   1488   1360    -95    564    134       C  \\nATOM     23  CD1 TYR A  36       8.685  32.490   5.630  1.00 17.53           C  \\nANISOU   23  CD1 TYR A  36     2934   1547   2178      9     20     11       C  \\nATOM     24  CD2 TYR A  36       9.081  33.149   7.878  1.00 14.85           C  \\nANISOU   24  CD2 TYR A  36     2462   1862   1316    -52    259   -128       C  \\nATOM     25  CE1 TYR A  36       7.463  33.160   5.661  1.00 19.34           C  \\nANISOU   25  CE1 TYR A  36     2511   2279   2556   -144      0    167       C  \\nATOM     26  CE2 TYR A  36       7.896  33.820   7.917  1.00 18.93           C  \\nANISOU   26  CE2 TYR A  36     2635   1791   2764    126    431   -436       C  \\nATOM     27  CZ  TYR A  36       7.094  33.845   6.803  1.00 21.32           C  \\nANISOU   27  CZ  TYR A  36     2442   2585   3071    230    234   -167       C  \\nATOM     28  OH  TYR A  36       5.866  34.508   6.901  1.00 27.25           O  \\nANISOU   28  OH  TYR A  36     2599   3687   4067    577    276   -895       O  \\nATOM     29  N   ASP A  37      12.113  31.908   3.927  1.00 11.65           N  \\nANISOU   29  N   ASP A  37     1743   1256   1425    140    149    146       N  \\nATOM     30  CA  ASP A  37      12.075  31.991   2.490  1.00 12.55           C  \\nANISOU   30  CA  ASP A  37     1894   1434   1438    229     46    273       C  \\nATOM     31  C   ASP A  37      10.817  31.403   1.857  1.00 13.91           C  \\nANISOU   31  C   ASP A  37     1829   1803   1653    152    129    267       C  \\nATOM     32  O   ASP A  37      10.380  31.858   0.801  1.00 17.76           O  \\nANISOU   32  O   ASP A  37     2052   2752   1944   -100   -369    513       O  \\nATOM     33  CB  ASP A  37      13.320  31.285   1.905  1.00 12.52           C  \\nANISOU   33  CB  ASP A  37     1866   1498   1392     22    203     84       C  \\nATOM     34  CG  ASP A  37      14.614  31.912   2.379  1.00 13.26           C  \\nANISOU   34  CG  ASP A  37     1894   1578   1565     33    134     71       C  \\nATOM     35  OD1 ASP A  37      14.776  33.141   2.167  1.00 15.66           O  \\nANISOU   35  OD1 ASP A  37     2210   1667   2072     84    -61    520       O  \\nATOM     36  OD2 ASP A  37      15.460  31.192   2.989  1.00 14.12           O  \\nANISOU   36  OD2 ASP A  37     1989   1361   2013    -13     83    267       O  \\nATOM     37  N   ASP A  38      10.249  30.389   2.481  1.00 13.41           N  \\nANISOU   37  N   ASP A  38     1722   1647   1724    164    -60    300       N  \\nATOM     38  CA  ASP A  38       9.029  29.737   2.038  1.00 13.80           C  \\nANISOU   38  CA  ASP A  38     1913   1822   1505    127   -180    156       C  \\nATOM     39  C   ASP A  38       8.311  29.177   3.223  1.00 11.72           C  \\nANISOU   39  C   ASP A  38     1635   1325   1493     88   -301     90       C  \\nATOM     40  O   ASP A  38       8.865  29.074   4.318  1.00 12.45           O  \\nANISOU   40  O   ASP A  38     1512   1815   1400    150   -140    288       O  \\nATOM     41  CB  ASP A  38       9.235  28.708   0.912  1.00 17.86           C  \\nANISOU   41  CB  ASP A  38     2980   2531   1273   -278     55   -112       C  \\nATOM     42  CG  ASP A  38       7.978  28.564  -0.064  1.00 22.48           C  \\nANISOU   42  CG  ASP A  38     2648   3317   2574     44   -160   -190       C  \\nATOM     43  OD1 ASP A  38       6.839  29.147   0.148  1.00 22.75           O  \\nANISOU   43  OD1 ASP A  38     3108   3830   1702    484     17    222       O  \\nATOM     44  OD2 ASP A  38       8.121  27.808  -1.077  1.00 26.00           O  \\nANISOU   44  OD2 ASP A  38     3351   3723   2803    742   -434   -438       O  \\nATOM     45  N   VAL A  39       7.064  28.776   2.988  1.00 12.94           N  \\nANISOU   45  N   VAL A  39     1656   1832   1429   -133   -381    257       N  \\nATOM     46  CA  VAL A  39       6.205  28.170   4.001  1.00 13.18           C  \\nANISOU   46  CA  VAL A  39     1706   1513   1788    -55   -198    328       C  \\nATOM     47  C   VAL A  39       5.850  26.760   3.579  1.00 12.90           C  \\nANISOU   47  C   VAL A  39     1620   1706   1575   -155   -257    183       C  \\nATOM     48  O   VAL A  39       5.751  26.465   2.388  1.00 17.73           O  \\nANISOU   48  O   VAL A  39     2728   2291   1717   -405   -210     11       O  \\nATOM     49  CB  VAL A  39       4.942  29.010   4.256  1.00 17.00           C  \\nANISOU   49  CB  VAL A  39     2008   2261   2191    197    115    258       C  \\nATOM     50  CG1 VAL A  39       5.284  30.440   4.537  1.00 18.59           C  \\nANISOU   50  CG1 VAL A  39     2311   2247   2503    291    181     71       C  \\nATOM     51  CG2 VAL A  39       3.899  28.876   3.171  1.00 20.54           C  \\nANISOU   51  CG2 VAL A  39     2543   2821   2438    135   -128    354       C  \\nATOM     52  N   LEU A  40       5.644  25.906   4.583  1.00 11.67           N  \\nANISOU   52  N   LEU A  40     1286   1643   1504   -114     37     41       N  \\nATOM     53  CA  LEU A  40       5.170  24.533   4.404  1.00 12.74           C  \\nANISOU   53  CA  LEU A  40     1337   1811   1690   -221     95   -153       C  \\nATOM     54  C   LEU A  40       4.005  24.329   5.383  1.00 10.91           C  \\nANISOU   54  C   LEU A  40     1221   1587   1335      8    -69   -155       C  \\nATOM     55  O   LEU A  40       4.148  24.504   6.585  1.00 12.22           O  \\nANISOU   55  O   LEU A  40     1347   1842   1454    -35   -148   -388       O  \\nATOM     56  CB  LEU A  40       6.299  23.578   4.709  1.00 15.21           C  \\nANISOU   56  CB  LEU A  40     1626   1998   2153     74    247   -227       C  \\nATOM     57  CG  LEU A  40       6.042  22.094   4.494  1.00 17.02           C  \\nANISOU   57  CG  LEU A  40     1947   2114   2405   -178    -18   -309       C  \\nATOM     58  CD1 LEU A  40       7.328  21.386   4.125  1.00 19.92           C  \\nANISOU   58  CD1 LEU A  40     2220   2396   2953    182    -73   -489       C  \\nATOM     59  CD2 LEU A  40       5.406  21.403   5.700  1.00 19.32           C  \\nANISOU   59  CD2 LEU A  40     2320   2470   2550    234    244   -228       C  \\nATOM     60  N   TYR A  41       2.854  23.930   4.852  1.00 11.26           N  \\nANISOU   60  N   TYR A  41     1212   1693   1373    -52   -195      4       N  \\nATOM     61  CA  TYR A  41       1.697  23.554   5.682  1.00 11.67           C  \\nANISOU   61  CA  TYR A  41     1217   1711   1504    -32   -168    191       C  \\nATOM     62  C   TYR A  41       1.803  22.046   5.928  1.00 12.04           C  \\nANISOU   62  C   TYR A  41     1457   1774   1342    134    -38    126       C  \\nATOM     63  O   TYR A  41       1.675  21.213   5.022  1.00 13.27           O  \\nANISOU   63  O   TYR A  41     1686   1788   1566     83   -321    120       O  \\nATOM     64  CB  TYR A  41       0.412  23.917   4.963  1.00 13.18           C  \\nANISOU   64  CB  TYR A  41     1276   1789   1941   -122   -395     98       C  \\nATOM     65  CG  TYR A  41       0.282  25.391   4.678  1.00 12.81           C  \\nANISOU   65  CG  TYR A  41     1115   1966   1784   -136   -342    277       C  \\nATOM     66  CD1 TYR A  41       0.026  26.279   5.709  1.00 15.19           C  \\nANISOU   66  CD1 TYR A  41     1624   2032   2113    269   -323    127       C  \\nATOM     67  CD2 TYR A  41       0.412  25.894   3.404  1.00 14.60           C  \\nANISOU   67  CD2 TYR A  41     1876   1825   1845    279    -69    307       C  \\nATOM     68  CE1 TYR A  41      -0.075  27.641   5.488  1.00 16.92           C  \\nANISOU   68  CE1 TYR A  41     1997   1998   2433    335   -390     61       C  \\nATOM     69  CE2 TYR A  41       0.330  27.246   3.205  1.00 16.10           C  \\nANISOU   69  CE2 TYR A  41     2459   1864   1793     68   -240    723       C  \\nATOM     70  CZ  TYR A  41       0.082  28.108   4.249  1.00 16.84           C  \\nANISOU   70  CZ  TYR A  41     2099   1866   2430    232   -692    302       C  \\nATOM     71  OH  TYR A  41      -0.028  29.475   3.995  1.00 21.92           O  \\nANISOU   71  OH  TYR A  41     2933   1727   3665    -66   -994    149       O  \\nATOM     72  N   PHE A  42       2.081  21.697   7.179  1.00 12.09           N  \\nANISOU   72  N   PHE A  42     1616   1561   1415    -70   -393     70       N  \\nATOM     73  CA  PHE A  42       2.342  20.304   7.482  1.00 11.25           C  \\nANISOU   73  CA  PHE A  42     1315   1587   1372    -54   -351    159       C  \\nATOM     74  C   PHE A  42       1.065  19.456   7.281  1.00 12.19           C  \\nANISOU   74  C   PHE A  42     1407   1632   1593   -131   -348    123       C  \\nATOM     75  O   PHE A  42      -0.004  19.879   7.679  1.00 12.37           O  \\nANISOU   75  O   PHE A  42     1512   1689   1500    -80   -317    169       O  \\nATOM     76  CB  PHE A  42       2.828  20.171   8.916  1.00 11.73           C  \\nANISOU   76  CB  PHE A  42     1550   1549   1356   -183   -326    213       C  \\nATOM     77  CG  PHE A  42       3.246  18.755   9.251  1.00 11.75           C  \\nANISOU   77  CG  PHE A  42     1659   1541   1265     16   -444    -14       C  \\nATOM     78  CD1 PHE A  42       4.510  18.308   8.930  1.00 13.02           C  \\nANISOU   78  CD1 PHE A  42     1548   1911   1486    -36   -466     96       C  \\nATOM     79  CD2 PHE A  42       2.341  17.865   9.767  1.00 11.29           C  \\nANISOU   79  CD2 PHE A  42     1353   1644   1291    180   -285     20       C  \\nATOM     80  CE1 PHE A  42       4.870  16.994   9.181  1.00 14.43           C  \\nANISOU   80  CE1 PHE A  42     1721   1969   1792    443    -77   -308       C  \\nATOM     81  CE2 PHE A  42       2.697  16.555   9.991  1.00 12.96           C  \\nANISOU   81  CE2 PHE A  42     1826   1711   1385    104   -484    202       C  \\nATOM     82  CZ  PHE A  42       3.971  16.138   9.700  1.00 14.58           C  \\nANISOU   82  CZ  PHE A  42     1970   1870   1699    299   -615     89       C  \\nATOM     83  N   PRO A  43       1.175  18.237   6.699  1.00 11.92           N  \\nANISOU   83  N   PRO A  43     1360   1620   1547    -11   -168    201       N  \\nATOM     84  CA  PRO A  43       0.035  17.374   6.405  1.00 12.93           C  \\nANISOU   84  CA  PRO A  43     1433   1727   1753      6   -248     52       C  \\nATOM     85  C   PRO A  43      -0.477  16.654   7.633  1.00 12.74           C  \\nANISOU   85  C   PRO A  43     1499   1650   1690     77    -43    -41       C  \\nATOM     86  O   PRO A  43      -0.318  15.424   7.791  1.00 12.88           O  \\nANISOU   86  O   PRO A  43     1518   1674   1701     76   -225     91       O  \\nATOM     87  CB  PRO A  43       0.595  16.419   5.336  1.00 14.05           C  \\nANISOU   87  CB  PRO A  43     1976   1788   1572     82   -496   -131       C  \\nATOM     88  CG  PRO A  43       2.033  16.292   5.735  1.00 14.13           C  \\nANISOU   88  CG  PRO A  43     1692   1984   1690    107     14   -115       C  \\nATOM     89  CD  PRO A  43       2.404  17.685   6.116  1.00 13.89           C  \\nANISOU   89  CD  PRO A  43     1595   1971   1711    117     -6    -26       C  \\nATOM     90  N   ALA A  44      -1.058  17.428   8.548  1.00 12.12           N  \\nANISOU   90  N   ALA A  44     1414   1556   1633    125    -76     83       N  \\nATOM     91  CA  ALA A  44      -1.484  16.884   9.813  1.00 14.63           C  \\nANISOU   91  CA  ALA A  44     1680   1971   1906    -72      7    325       C  \\nATOM     92  C   ALA A  44      -2.581  15.833   9.675  1.00 15.11           C  \\nANISOU   92  C   ALA A  44     1709   2059   1973   -143   -136    395       C  \\nATOM     93  O   ALA A  44      -2.648  14.919  10.489  1.00 18.33           O  \\nANISOU   93  O   ALA A  44     2358   2345   2258    -80   -530    673       O  \\nATOM     94  CB  ALA A  44      -1.945  18.015  10.712  1.00 18.87           C  \\nANISOU   94  CB  ALA A  44     2274   2422   2473    -56    625    -26       C  \\nATOM     95  N   SER A  45      -3.401  15.902   8.632  1.00 15.52           N  \\nANISOU   95  N   SER A  45     1582   2199   2114   -182   -142    676       N  \\nATOM     96  CA  SER A  45      -4.445  14.866   8.500  1.00 18.43           C  \\nANISOU   96  CA  SER A  45     1527   2481   2995   -227   -563    302       C  \\nATOM     97  C   SER A  45      -3.841  13.498   8.191  1.00 18.39           C  \\nANISOU   97  C   SER A  45     1875   2446   2663   -300   -743    242       C  \\nATOM     98  O   SER A  45      -4.376  12.456   8.589  1.00 23.63           O  \\nANISOU   98  O   SER A  45     2265   3028   3684   -664   -835    815       O  \\nATOM     99  CB  SER A  45      -5.476  15.237   7.423  1.00 20.57           C  \\nANISOU   99  CB  SER A  45     1740   3230   2845    197   -438    546       C  \\nATOM    100  OG  SER A  45      -4.934  15.192   6.083  1.00 27.19           O  \\nANISOU  100  OG  SER A  45     2773   4833   2725    113   -420   1297       O  \\nATOM    101  N   ARG A  46      -2.744  13.504   7.448  1.00 16.63           N  \\nANISOU  101  N   ARG A  46     1710   2283   2326   -115  -1051    164       N  \\nATOM    102  CA  ARG A  46      -2.094  12.269   7.022  1.00 16.41           C  \\nANISOU  102  CA  ARG A  46     2245   1826   2161   -328  -1192   -358       C  \\nATOM    103  C   ARG A  46      -1.213  11.696   8.159  1.00 17.24           C  \\nANISOU  103  C   ARG A  46     2081   2004   2464   -243  -1206   -268       C  \\nATOM    104  O   ARG A  46      -1.156  10.470   8.345  1.00 18.21           O  \\nANISOU  104  O   ARG A  46     2626   1988   2305   -211   -916   -231       O  \\nATOM    105  CB  ARG A  46      -1.219  12.477   5.777  1.00 23.50           C  \\nANISOU  105  CB  ARG A  46     2706   3579   2643   -488   -957    207       C  \\nATOM    106  CG  ARG A  46      -1.929  12.619   4.434  1.00 28.14           C  \\nANISOU  106  CG  ARG A  46     3804   3946   2940   -360  -1334    367       C  \\nATOM    107  CD  ARG A  46      -0.971  13.013   3.313  1.00 38.97           C  \\nANISOU  107  CD  ARG A  46     5066   5967   3771   -658     92   -647       C  \\nATOM    108  NE  ARG A  46       0.052  12.007   2.967  1.00 49.36           N  \\nANISOU  108  NE  ARG A  46     6649   6411   5692    571  -1323   -718       N  \\nATOM    109  CZ  ARG A  46      -0.164  10.902   2.246  1.00 51.28           C  \\nANISOU  109  CZ  ARG A  46     6713   6406   6364   1688   -425  -1407       C  \\nATOM    110  NH1 ARG A  46      -1.381  10.591   1.798  1.00 62.38           N  \\nANISOU  110  NH1 ARG A  46     9037   7342   7321   -391  -2428  -3363       N  \\nATOM    111  NH2 ARG A  46       0.840  10.085   1.964  1.00 50.85           N  \\nANISOU  111  NH2 ARG A  46     6176   7779   5364   2380   -751    -76       N  \\nATOM    112  N   TYR A  47      -0.545  12.578   8.912  1.00 12.74           N  \\nANISOU  112  N   TYR A  47     1833   1554   1452   -117   -621    -52       N  \\nATOM    113  CA  TYR A  47       0.444  12.162   9.919  1.00 12.44           C  \\nANISOU  113  CA  TYR A  47     1696   1485   1544    -79   -536     72       C  \\nATOM    114  C   TYR A  47       0.107  12.923  11.186  1.00 11.86           C  \\nANISOU  114  C   TYR A  47     1426   1447   1630   -236   -484     11       C  \\nATOM    115  O   TYR A  47       0.852  13.810  11.616  1.00 11.57           O  \\nANISOU  115  O   TYR A  47     1464   1435   1495   -285   -402    -20       O  \\nATOM    116  CB  TYR A  47       1.880  12.407   9.459  1.00 13.30           C  \\nANISOU  116  CB  TYR A  47     1694   1754   1604    -62   -556    -87       C  \\nATOM    117  CG  TYR A  47       2.138  11.885   8.087  1.00 14.23           C  \\nANISOU  117  CG  TYR A  47     1753   1880   1772    -41   -562   -238       C  \\nATOM    118  CD1 TYR A  47       2.112  10.535   7.841  1.00 16.56           C  \\nANISOU  118  CD1 TYR A  47     2532   2016   1741    300   -768   -374       C  \\nATOM    119  CD2 TYR A  47       2.401  12.743   7.032  1.00 15.43           C  \\nANISOU  119  CD2 TYR A  47     1834   2414   1615     52   -195   -232       C  \\nATOM    120  CE1 TYR A  47       2.257  10.081   6.552  1.00 19.33           C  \\nANISOU  120  CE1 TYR A  47     2485   2578   2280    486   -475  -1216       C  \\nATOM    121  CE2 TYR A  47       2.550  12.286   5.747  1.00 20.35           C  \\nANISOU  121  CE2 TYR A  47     2279   3618   1833    -66    -83   -629       C  \\nATOM    122  CZ  TYR A  47       2.512  10.955   5.504  1.00 22.07           C  \\nANISOU  122  CZ  TYR A  47     2654   3570   2162    262   -228   -963       C  \\nATOM    123  OH  TYR A  47       2.689  10.482   4.187  1.00 30.14           O  \\nANISOU  123  OH  TYR A  47     2786   5878   2786   -137    272  -2029       O  \\nATOM    124  N   PRO A  48      -1.035  12.594  11.806  1.00 11.98           N  \\nANISOU  124  N   PRO A  48     1526   1601   1423   -249   -405     27       N  \\nATOM    125  CA  PRO A  48      -1.496  13.404  12.900  1.00 13.00           C  \\nANISOU  125  CA  PRO A  48     1515   1742   1681     28   -394     58       C  \\nATOM    126  C   PRO A  48      -0.659  13.389  14.143  1.00 11.52           C  \\nANISOU  126  C   PRO A  48     1407   1406   1562   -107   -290    226       C  \\nATOM    127  O   PRO A  48      -0.632  14.389  14.868  1.00 13.19           O  \\nANISOU  127  O   PRO A  48     1703   1551   1755   -105   -207    100       O  \\nATOM    128  CB  PRO A  48      -2.908  12.851  13.184  1.00 15.18           C  \\nANISOU  128  CB  PRO A  48     1595   2378   1791   -216   -235    -11       C  \\nATOM    129  CG  PRO A  48      -2.880  11.477  12.676  1.00 15.97           C  \\nANISOU  129  CG  PRO A  48     1601   2199   2268   -578   -505     69       C  \\nATOM    130  CD  PRO A  48      -2.006  11.539  11.460  1.00 14.77           C  \\nANISOU  130  CD  PRO A  48     1835   1697   2079   -370   -548    -93       C  \\nATOM    131  N   GLU A  49      -0.012  12.281  14.444  1.00 12.58           N  \\nANISOU  131  N   GLU A  49     1671   1270   1836    -54   -276     62       N  \\nATOM    132  CA  GLU A  49       0.803  12.238  15.665  1.00 12.03           C  \\nANISOU  132  CA  GLU A  49     1568   1307   1694    -30   -141    -22       C  \\nATOM    133  C   GLU A  49       2.056  13.078  15.507  1.00 11.40           C  \\nANISOU  133  C   GLU A  49     1577   1270   1483     10   -169    102       C  \\nATOM    134  O   GLU A  49       2.442  13.818  16.436  1.00 12.12           O  \\nANISOU  134  O   GLU A  49     1653   1596   1355      2   -259      0       O  \\nATOM    135  CB  GLU A  49       1.196  10.806  16.029  1.00 14.74           C  \\nANISOU  135  CB  GLU A  49     2153   1339   2105    -52   -199    213       C  \\nATOM    136  CG  GLU A  49       0.115   9.950  16.550  1.00 17.40           C  \\nANISOU  136  CG  GLU A  49     2212   1793   2604    -26     26    203       C  \\nATOM    137  CD  GLU A  49       0.706   8.607  17.015  1.00 20.70           C  \\nANISOU  137  CD  GLU A  49     2275   1917   3670    -24    265    693       C  \\nATOM    138  OE1 GLU A  49       1.732   8.490  17.785  1.00 20.18           O  \\nANISOU  138  OE1 GLU A  49     2373   1852   3442    217    533    251       O  \\nATOM    139  OE2 GLU A  49       0.149   7.640  16.556  1.00 25.98           O  \\nANISOU  139  OE2 GLU A  49     3040   2366   4464    109   -626     72       O  \\nATOM    140  N   THR A  50       2.709  12.987  14.352  1.00 10.98           N  \\nANISOU  140  N   THR A  50     1433   1359   1379    -33   -261    -13       N  \\nATOM    141  CA  THR A  50       3.863  13.862  14.078  1.00 10.30           C  \\nANISOU  141  CA  THR A  50     1283   1314   1314     70   -229      2       C  \\nATOM    142  C   THR A  50       3.436  15.324  14.094  1.00  9.95           C  \\nANISOU  142  C   THR A  50     1309   1253   1216    -52   -384      0       C  \\nATOM    143  O   THR A  50       4.073  16.177  14.693  1.00 10.96           O  \\nANISOU  143  O   THR A  50     1420   1539   1204    -96   -333   -154       O  \\nATOM    144  CB  THR A  50       4.548  13.441  12.792  1.00 10.51           C  \\nANISOU  144  CB  THR A  50     1493   1088   1411     69   -136    -84       C  \\nATOM    145  OG1 THR A  50       4.958  12.066  12.896  1.00 11.72           O  \\nANISOU  145  OG1 THR A  50     1622   1183   1646    219   -240     -6       O  \\nATOM    146  CG2 THR A  50       5.755  14.255  12.518  1.00 11.56           C  \\nANISOU  146  CG2 THR A  50     1569   1426   1393      0    -52   -103       C  \\nATOM    147  N   GLY A  51       2.310  15.611  13.431  1.00 10.02           N  \\nANISOU  147  N   GLY A  51     1402   1174   1229    -56   -427    -53       N  \\nATOM    148  CA  GLY A  51       1.837  16.973  13.396  1.00  9.90           C  \\nANISOU  148  CA  GLY A  51     1394   1167   1199     -7   -356    -54       C  \\nATOM    149  C   GLY A  51       1.524  17.539  14.752  1.00  9.93           C  \\nANISOU  149  C   GLY A  51     1343   1144   1284     54   -104     31       C  \\nATOM    150  O   GLY A  51       1.802  18.708  15.023  1.00 10.97           O  \\nANISOU  150  O   GLY A  51     1557   1070   1540     23   -192    -72       O  \\nATOM    151  N   ALA A  52       0.898  16.735  15.617  1.00 10.43           N  \\nANISOU  151  N   ALA A  52     1358   1256   1349   -131    -66     -1       N  \\nATOM    152  CA  ALA A  52       0.595  17.195  16.961  1.00 11.37           C  \\nANISOU  152  CA  ALA A  52     1258   1537   1525    165    -16   -155       C  \\nATOM    153  C   ALA A  52       1.864  17.533  17.738  1.00 10.84           C  \\nANISOU  153  C   ALA A  52     1359   1475   1283    -22    -12    -76       C  \\nATOM    154  O   ALA A  52       1.898  18.497  18.516  1.00 12.56           O  \\nANISOU  154  O   ALA A  52     1823   1502   1448    -33    -23   -155       O  \\nATOM    155  CB  ALA A  52      -0.227  16.163  17.724  1.00 12.94           C  \\nANISOU  155  CB  ALA A  52     1530   1887   1498    -71     99   -108       C  \\nATOM    156  N   HIS A  53       2.903  16.721  17.566  1.00 10.29           N  \\nANISOU  156  N   HIS A  53     1330   1126   1454   -138    -92   -207       N  \\nATOM    157  CA  HIS A  53       4.177  17.033  18.189  1.00  9.80           C  \\nANISOU  157  CA  HIS A  53     1349   1296   1077   -104    -46    -95       C  \\nATOM    158  C   HIS A  53       4.722  18.368  17.714  1.00 10.19           C  \\nANISOU  158  C   HIS A  53     1524   1256   1089    -83   -177   -118       C  \\nATOM    159  O   HIS A  53       5.118  19.215  18.538  1.00 10.33           O  \\nANISOU  159  O   HIS A  53     1702   1279    943     17   -184   -178       O  \\nATOM    160  CB  HIS A  53       5.176  15.915  17.990  1.00 10.64           C  \\nANISOU  160  CB  HIS A  53     1571   1175   1296    -45   -190     21       C  \\nATOM    161  CG  HIS A  53       6.469  16.165  18.686  1.00 10.42           C  \\nANISOU  161  CG  HIS A  53     1489   1089   1381     -9   -143     26       C  \\nATOM    162  ND1 HIS A  53       7.681  15.705  18.204  1.00 11.81           N  \\nANISOU  162  ND1 HIS A  53     1478   1421   1586    -24   -126     55       N  \\nATOM    163  CD2 HIS A  53       6.742  16.842  19.832  1.00 11.10           C  \\nANISOU  163  CD2 HIS A  53     1469   1363   1382    125   -148    -62       C  \\nATOM    164  CE1 HIS A  53       8.642  16.113  19.022  1.00 11.59           C  \\nANISOU  164  CE1 HIS A  53     1451   1342   1607     63   -135    -69       C  \\nATOM    165  NE2 HIS A  53       8.096  16.797  20.011  1.00 11.06           N  \\nANISOU  165  NE2 HIS A  53     1498   1258   1444     31   -284    -15       N  \\nATOM    166  N   ILE A  54       4.769  18.555  16.393  1.00  9.71           N  \\nANISOU  166  N   ILE A  54     1475   1152   1062   -103   -140   -126       N  \\nATOM    167  CA  ILE A  54       5.320  19.790  15.859  1.00  9.20           C  \\nANISOU  167  CA  ILE A  54     1268   1265    963     14   -164     68       C  \\nATOM    168  C   ILE A  54       4.518  20.989  16.404  1.00  9.88           C  \\nANISOU  168  C   ILE A  54     1414   1185   1153    -14   -309    -92       C  \\nATOM    169  O   ILE A  54       5.096  21.987  16.840  1.00 10.12           O  \\nANISOU  169  O   ILE A  54     1541   1254   1050    -67   -206   -205       O  \\nATOM    170  CB  ILE A  54       5.320  19.775  14.331  1.00  9.72           C  \\nANISOU  170  CB  ILE A  54     1419   1277    997    -66   -243    -41       C  \\nATOM    171  CG1 ILE A  54       6.195  18.630  13.821  1.00 11.25           C  \\nANISOU  171  CG1 ILE A  54     1441   1289   1544   -150     -6    -92       C  \\nATOM    172  CG2 ILE A  54       5.745  21.116  13.832  1.00 11.39           C  \\nANISOU  172  CG2 ILE A  54     1623   1330   1374   -116   -169     37       C  \\nATOM    173  CD1 ILE A  54       6.080  18.414  12.336  1.00 13.38           C  \\nANISOU  173  CD1 ILE A  54     1765   1664   1654   -170    112   -356       C  \\nATOM    174  N   SER A  55       3.200  20.890  16.375  1.00 10.18           N  \\nANISOU  174  N   SER A  55     1418   1191   1256     96   -174   -145       N  \\nATOM    175  CA  SER A  55       2.353  21.984  16.849  1.00 10.89           C  \\nANISOU  175  CA  SER A  55     1457   1425   1255    244   -287   -187       C  \\nATOM    176  C   SER A  55       2.662  22.331  18.311  1.00 10.70           C  \\nANISOU  176  C   SER A  55     1555   1336   1174     25   -200    -66       C  \\nATOM    177  O   SER A  55       2.850  23.491  18.650  1.00 12.40           O  \\nANISOU  177  O   SER A  55     1673   1401   1636      1    157   -322       O  \\nATOM    178  CB  SER A  55       0.881  21.600  16.715  1.00 12.88           C  \\nANISOU  178  CB  SER A  55     1337   2021   1533    296   -266   -205       C  \\nATOM    179  OG  SER A  55       0.025  22.603  17.231  1.00 17.85           O  \\nANISOU  179  OG  SER A  55     2010   2208   2564    584   -264   -444       O  \\nATOM    180  N   ASP A  56       2.733  21.316  19.166  1.00 10.26           N  \\nANISOU  180  N   ASP A  56     1518   1360   1019    -70    -25    -56       N  \\nATOM    181  CA  ASP A  56       2.988  21.516  20.587  1.00 11.35           C  \\nANISOU  181  CA  ASP A  56     1673   1598   1040     19     31   -112       C  \\nATOM    182  C   ASP A  56       4.407  22.067  20.802  1.00 11.77           C  \\nANISOU  182  C   ASP A  56     1804   1570   1097   -138     -6   -186       C  \\nATOM    183  O   ASP A  56       4.624  22.914  21.654  1.00 12.21           O  \\nANISOU  183  O   ASP A  56     1783   1629   1225    -96    -83   -322       O  \\nATOM    184  CB  ASP A  56       2.805  20.208  21.368  1.00 13.08           C  \\nANISOU  184  CB  ASP A  56     2095   1677   1198    -50     21    -37       C  \\nATOM    185  CG  ASP A  56       1.362  19.807  21.569  1.00 17.02           C  \\nANISOU  185  CG  ASP A  56     2170   2162   2133     31    183     86       C  \\nATOM    186  OD1 ASP A  56       0.462  20.617  21.320  1.00 21.40           O  \\nANISOU  186  OD1 ASP A  56     2152   2819   3157    120    340    675       O  \\nATOM    187  OD2 ASP A  56       1.167  18.673  22.018  1.00 19.66           O  \\nANISOU  187  OD2 ASP A  56     2365   2452   2653    -63    187    366       O  \\nATOM    188  N   ALA A  57       5.371  21.538  20.059  1.00 10.48           N  \\nANISOU  188  N   ALA A  57     1581   1307   1091   -104    -73    -15       N  \\nATOM    189  CA  ALA A  57       6.753  21.961  20.233  1.00 11.12           C  \\nANISOU  189  CA  ALA A  57     1552   1384   1288    -28   -148   -178       C  \\nATOM    190  C   ALA A  57       6.873  23.441  19.868  1.00 10.47           C  \\nANISOU  190  C   ALA A  57     1473   1437   1068     -2   -274   -138       C  \\nATOM    191  O   ALA A  57       7.565  24.187  20.570  1.00 10.98           O  \\nANISOU  191  O   ALA A  57     1689   1376   1107    -22   -251   -326       O  \\nATOM    192  CB  ALA A  57       7.700  21.102  19.403  1.00 10.34           C  \\nANISOU  192  CB  ALA A  57     1485   1235   1209    -20   -204    -71       C  \\nATOM    193  N   ILE A  58       6.241  23.859  18.780  1.00  9.42           N  \\nANISOU  193  N   ILE A  58     1552   1117    909      9   -153   -142       N  \\nATOM    194  CA  ILE A  58       6.291  25.295  18.426  1.00  9.54           C  \\nANISOU  194  CA  ILE A  58     1477   1127   1017      7    -21   -148       C  \\nATOM    195  C   ILE A  58       5.635  26.136  19.540  1.00  9.46           C  \\nANISOU  195  C   ILE A  58     1533   1278    784    -67   -175   -276       C  \\nATOM    196  O   ILE A  58       6.162  27.215  19.880  1.00 11.33           O  \\nANISOU  196  O   ILE A  58     1816   1164   1325    -45      7   -419       O  \\nATOM    197  CB  ILE A  58       5.634  25.485  17.025  1.00  9.92           C  \\nANISOU  197  CB  ILE A  58     1510   1156   1101    -76   -136   -101       C  \\nATOM    198  CG1 ILE A  58       6.548  24.896  15.929  1.00 10.99           C  \\nANISOU  198  CG1 ILE A  58     1718   1238   1218    -57     28     -8       C  \\nATOM    199  CG2 ILE A  58       5.302  26.954  16.779  1.00 10.16           C  \\nANISOU  199  CG2 ILE A  58     1474   1218   1167     26    -98   -103       C  \\nATOM    200  CD1 ILE A  58       5.874  24.802  14.571  1.00 11.68           C  \\nANISOU  200  CD1 ILE A  58     2116   1126   1194     25     51    -33       C  \\nATOM    201  N   LYS A  59       4.538  25.672  20.117  1.00 10.95           N  \\nANISOU  201  N   LYS A  59     1660   1211   1286     -3    -28   -294       N  \\nATOM    202  CA  LYS A  59       3.902  26.422  21.224  1.00 11.52           C  \\nANISOU  202  CA  LYS A  59     1690   1290   1397      9     87   -310       C  \\nATOM    203  C   LYS A  59       4.717  26.379  22.540  1.00 11.95           C  \\nANISOU  203  C   LYS A  59     1723   1563   1252    188    248   -141       C  \\nATOM    204  O   LYS A  59       4.455  27.166  23.416  1.00 17.26           O  \\nANISOU  204  O   LYS A  59     2735   1898   1922    452    155   -687       O  \\nATOM    205  CB  LYS A  59       2.476  25.925  21.442  1.00 13.42           C  \\nANISOU  205  CB  LYS A  59     1858   1693   1547    -68    353   -494       C  \\nATOM    206  CG  LYS A  59       1.540  26.275  20.292  1.00 16.54           C  \\nANISOU  206  CG  LYS A  59     2046   1884   2351    291   -191   -658       C  \\nATOM    207  CD  LYS A  59       0.136  25.849  20.670  1.00 25.52           C  \\nANISOU  207  CD  LYS A  59     2435   3526   3732   -202    157   -407       C  \\nATOM    208  CE  LYS A  59      -0.884  26.259  19.640  1.00 35.12           C  \\nANISOU  208  CE  LYS A  59     4348   4676   4317   -161   -553    -27       C  \\nATOM    209  NZ  LYS A  59      -0.896  25.251  18.579  1.00 39.77           N  \\nANISOU  209  NZ  LYS A  59     5396   4901   4814   -150    376   -192       N  \\nATOM    210  N   ALA A  60       5.728  25.515  22.605  1.00 11.33           N  \\nANISOU  210  N   ALA A  60     1791   1234   1277    148    -17   -344       N  \\nATOM    211  CA  ALA A  60       6.710  25.459  23.672  1.00 11.93           C  \\nANISOU  211  CA  ALA A  60     1809   1512   1210    212     80   -388       C  \\nATOM    212  C   ALA A  60       7.990  26.240  23.375  1.00 11.71           C  \\nANISOU  212  C   ALA A  60     1883   1629    935     83    -57   -299       C  \\nATOM    213  O   ALA A  60       9.001  26.178  24.112  1.00 14.41           O  \\nANISOU  213  O   ALA A  60     1921   1861   1692    -73   -357   -206       O  \\nATOM    214  CB  ALA A  60       6.981  24.022  24.088  1.00 13.04           C  \\nANISOU  214  CB  ALA A  60     1968   1736   1251     11     30    -23       C  \\nATOM    215  N   GLY A  61       7.986  26.982  22.258  1.00 11.50           N  \\nANISOU  215  N   GLY A  61     1894   1551    922    -63     31   -294       N  \\nATOM    216  CA  GLY A  61       9.079  27.865  21.931  1.00 12.16           C  \\nANISOU  216  CA  GLY A  61     1751   1531   1336    -78   -135   -272       C  \\nATOM    217  C   GLY A  61      10.151  27.293  21.061  1.00 11.43           C  \\nANISOU  217  C   GLY A  61     1787   1209   1345    -30   -200   -229       C  \\nATOM    218  O   GLY A  61      11.162  27.951  20.816  1.00 13.18           O  \\nANISOU  218  O   GLY A  61     1964   1397   1645   -248   -124   -477       O  \\nATOM    219  N   HIS A  62       9.969  26.070  20.571  1.00 10.94           N  \\nANISOU  219  N   HIS A  62     1875   1132   1148    -79   -110   -124       N  \\nATOM    220  CA  HIS A  62      10.929  25.525  19.610  1.00 11.03           C  \\nANISOU  220  CA  HIS A  62     1643   1332   1213    -89   -181   -161       C  \\nATOM    221  C   HIS A  62      10.756  26.205  18.262  1.00 10.70           C  \\nANISOU  221  C   HIS A  62     1620   1389   1056    123    -36   -312       C  \\nATOM    222  O   HIS A  62       9.640  26.566  17.858  1.00 12.02           O  \\nANISOU  222  O   HIS A  62     1662   1770   1133    188   -144   -188       O  \\nATOM    223  CB  HIS A  62      10.759  24.010  19.506  1.00 11.18           C  \\nANISOU  223  CB  HIS A  62     1707   1326   1213     55   -269   -302       C  \\nATOM    224  CG  HIS A  62      11.055  23.313  20.781  1.00 11.13           C  \\nANISOU  224  CG  HIS A  62     1846   1185   1194     53   -236   -353       C  \\nATOM    225  ND1 HIS A  62      12.334  22.928  21.111  1.00 13.27           N  \\nANISOU  225  ND1 HIS A  62     2164   1389   1490    309   -567   -353       N  \\nATOM    226  CD2 HIS A  62      10.265  22.982  21.834  1.00 12.34           C  \\nANISOU  226  CD2 HIS A  62     1877   1621   1187     38   -297   -295       C  \\nATOM    227  CE1 HIS A  62      12.312  22.375  22.319  1.00 12.93           C  \\nANISOU  227  CE1 HIS A  62     1880   1693   1340    151   -496   -381       C  \\nATOM    228  NE2 HIS A  62      11.068  22.392  22.774  1.00 13.17           N  \\nANISOU  228  NE2 HIS A  62     2032   1605   1366    -29   -395   -150       N  \\nATOM    229  N   ALA A  63      11.858  26.320  17.534  1.00 11.39           N  \\nANISOU  229  N   ALA A  63     1698   1365   1264    102     26   -192       N  \\nATOM    230  CA  ALA A  63      11.856  27.070  16.256  1.00 10.78           C  \\nANISOU  230  CA  ALA A  63     1684   1193   1217     74     18   -268       C  \\nATOM    231  C   ALA A  63      10.781  26.564  15.309  1.00 10.72           C  \\nANISOU  231  C   ALA A  63     1882   1162   1028      0    -49    -53       C  \\nATOM    232  O   ALA A  63      10.693  25.358  15.054  1.00 11.29           O  \\nANISOU  232  O   ALA A  63     1898   1073   1319    -50   -217    -37       O  \\nATOM    233  CB  ALA A  63      13.216  26.984  15.623  1.00 12.62           C  \\nANISOU  233  CB  ALA A  63     1759   1543   1491    -31    129   -143       C  \\nATOM    234  N   ASP A  64      10.034  27.471  14.725  1.00 11.28           N  \\nANISOU  234  N   ASP A  64     1812   1140   1331    -76   -206    -53       N  \\nATOM    235  CA  ASP A  64       9.091  27.162  13.677  1.00 10.83           C  \\nANISOU  235  CA  ASP A  64     1682   1410   1023    -32    -99    168       C  \\nATOM    236  C   ASP A  64       9.692  27.226  12.279  1.00 10.45           C  \\nANISOU  236  C   ASP A  64     1484   1336   1148    -22    -48     72       C  \\nATOM    237  O   ASP A  64       9.081  26.710  11.328  1.00 12.17           O  \\nANISOU  237  O   ASP A  64     1599   1844   1178   -238    -96     -2       O  \\nATOM    238  CB  ASP A  64       7.817  28.036  13.756  1.00 13.95           C  \\nANISOU  238  CB  ASP A  64     1614   2239   1445    205    181    454       C  \\nATOM    239  CG  ASP A  64       8.071  29.586  13.722  1.00 17.09           C  \\nANISOU  239  CG  ASP A  64     2323   2111   2057    695    306    197       C  \\nATOM    240  OD1 ASP A  64       9.236  30.027  13.750  1.00 19.44           O  \\nANISOU  240  OD1 ASP A  64     2971   1514   2898    551    521    -65       O  \\nATOM    241  OD2 ASP A  64       7.086  30.352  13.671  1.00 23.24           O  \\nANISOU  241  OD2 ASP A  64     3044   2991   2794   1375    634    597       O  \\nATOM    242  N   VAL A  65      10.881  27.809  12.155  1.00 10.43           N  \\nANISOU  242  N   VAL A  65     1498   1333   1131    -83   -169    -29       N  \\nATOM    243  CA  VAL A  65      11.608  27.824  10.889  1.00 10.68           C  \\nANISOU  243  CA  VAL A  65     1488   1286   1284    -70    -81   -137       C  \\nATOM    244  C   VAL A  65      12.696  26.734  10.896  1.00 11.28           C  \\nANISOU  244  C   VAL A  65     1413   1543   1330    -30    -16    -14       C  \\nATOM    245  O   VAL A  65      13.458  26.641  11.865  1.00 13.03           O  \\nANISOU  245  O   VAL A  65     1590   1788   1572     24   -202   -148       O  \\nATOM    246  CB  VAL A  65      12.224  29.210  10.637  1.00 12.85           C  \\nANISOU  246  CB  VAL A  65     1843   1339   1700   -163     74   -136       C  \\nATOM    247  CG1 VAL A  65      13.120  29.179   9.404  1.00 14.99           C  \\nANISOU  247  CG1 VAL A  65     1899   1776   2018   -189    342     54       C  \\nATOM    248  CG2 VAL A  65      11.121  30.236  10.490  1.00 13.64           C  \\nANISOU  248  CG2 VAL A  65     2164   1418   1598     36    128     18       C  \\nATOM    249  N   CYS A  66      12.759  25.969   9.802  1.00 11.30           N  \\nANISOU  249  N   CYS A  66     1524   1491   1277    176   -146     48       N  \\nATOM    250  CA  CYS A  66      13.823  25.027   9.566  1.00 11.27           C  \\nANISOU  250  CA  CYS A  66     1457   1502   1320    190   -219    132       C  \\nATOM    251  C   CYS A  66      14.787  25.666   8.556  1.00 10.91           C  \\nANISOU  251  C   CYS A  66     1394   1205   1545    113   -103     25       C  \\nATOM    252  O   CYS A  66      14.408  25.837   7.391  1.00 11.68           O  \\nANISOU  252  O   CYS A  66     1339   1493   1606    130   -133     52       O  \\nATOM    253  CB  CYS A  66      13.284  23.728   8.966  1.00 13.28           C  \\nANISOU  253  CB  CYS A  66     1966   1382   1695    112   -146    150       C  \\nATOM    254  SG  CYS A  66      14.616  22.561   8.414  1.00 16.12           S  \\nANISOU  254  SG  CYS A  66     2350   1542   2230    -48    128   -244       S  \\nATOM    255  N   THR A  67      16.022  25.920   8.999  1.00 10.44           N  \\nANISOU  255  N   THR A  67     1422   1205   1337    125    -98     32       N  \\nATOM    256  CA  THR A  67      17.094  26.336   8.109  1.00 10.70           C  \\nANISOU  256  CA  THR A  67     1382   1142   1541     88    -57    -91       C  \\nATOM    257  C   THR A  67      17.772  25.025   7.686  1.00 10.34           C  \\nANISOU  257  C   THR A  67     1362   1119   1444    -33    -26   -125       C  \\nATOM    258  O   THR A  67      18.448  24.385   8.485  1.00 10.56           O  \\nANISOU  258  O   THR A  67     1347   1079   1585     26   -177   -213       O  \\nATOM    259  CB  THR A  67      18.089  27.286   8.785  1.00 12.01           C  \\nANISOU  259  CB  THR A  67     1565   1224   1774     23   -243    -34       C  \\nATOM    260  OG1 THR A  67      17.355  28.389   9.330  1.00 13.39           O  \\nANISOU  260  OG1 THR A  67     2089   1166   1830     15    -50    -89       O  \\nATOM    261  CG2 THR A  67      19.158  27.766   7.816  1.00 14.19           C  \\nANISOU  261  CG2 THR A  67     1782   1408   2199   -199    -58   -157       C  \\nATOM    262  N   ILE A  68      17.525  24.614   6.455  1.00  9.48           N  \\nANISOU  262  N   ILE A  68     1302    825   1474     13    -26   -161       N  \\nATOM    263  CA  ILE A  68      17.918  23.275   6.034  1.00  9.61           C  \\nANISOU  263  CA  ILE A  68     1367    867   1414    121    -51   -130       C  \\nATOM    264  C   ILE A  68      19.439  23.133   6.090  1.00 10.82           C  \\nANISOU  264  C   ILE A  68     1370   1104   1637     87     99     64       C  \\nATOM    265  O   ILE A  68      20.166  23.902   5.482  1.00 11.97           O  \\nANISOU  265  O   ILE A  68     1392   1252   1901    -63    -11    145       O  \\nATOM    266  CB  ILE A  68      17.399  22.962   4.626  1.00  9.85           C  \\nANISOU  266  CB  ILE A  68     1357   1094   1290     84    139   -163       C  \\nATOM    267  CG1 ILE A  68      15.867  23.009   4.575  1.00 10.87           C  \\nANISOU  267  CG1 ILE A  68     1349   1374   1407    144    -52   -224       C  \\nATOM    268  CG2 ILE A  68      17.881  21.590   4.174  1.00 10.80           C  \\nANISOU  268  CG2 ILE A  68     1397   1184   1521    140     82   -277       C  \\nATOM    269  CD1 ILE A  68      15.309  22.891   3.195  1.00 11.84           C  \\nANISOU  269  CD1 ILE A  68     1585   1420   1491     11   -280    -18       C  \\nATOM    270  N   GLU A  69      19.876  22.067   6.769  1.00 10.89           N  \\nANISOU  270  N   GLU A  69     1236   1150   1751    147     -2     26       N  \\nATOM    271  CA  GLU A  69      21.298  21.754   6.832  1.00 11.62           C  \\nANISOU  271  CA  GLU A  69     1200   1374   1840    121    109    -78       C  \\nATOM    272  C   GLU A  69      21.379  20.225   6.948  1.00 12.16           C  \\nANISOU  272  C   GLU A  69     1321   1400   1897    123    -56     48       C  \\nATOM    273  O   GLU A  69      21.375  19.675   8.032  1.00 13.82           O  \\nANISOU  273  O   GLU A  69     1856   1333   2060    204    127    137       O  \\nATOM    274  CB  GLU A  69      21.990  22.458   7.991  1.00 12.76           C  \\nANISOU  274  CB  GLU A  69     1366   1459   2020    -35   -137     99       C  \\nATOM    275  CG  GLU A  69      23.522  22.525   7.867  1.00 14.60           C  \\nANISOU  275  CG  GLU A  69     1354   1932   2260   -252   -348     64       C  \\nATOM    276  CD  GLU A  69      24.149  21.136   7.750  1.00 16.46           C  \\nANISOU  276  CD  GLU A  69     1326   2212   2716    104   -215    341       C  \\nATOM    277  OE1 GLU A  69      24.231  20.492   8.809  1.00 20.20           O  \\nANISOU  277  OE1 GLU A  69     1929   2742   3003    339     55    725       O  \\nATOM    278  OE2 GLU A  69      24.563  20.714   6.621  1.00 17.90           O  \\nANISOU  278  OE2 GLU A  69     1732   2191   2877    -90     75    283       O  \\nATOM    279  N   ARG A  70      21.459  19.572   5.829  1.00 12.05           N  \\nANISOU  279  N   ARG A  70     1339   1234   2002    162    166    158       N  \\nATOM    280  CA  ARG A  70      21.260  18.139   5.780  1.00 12.03           C  \\nANISOU  280  CA  ARG A  70     1363   1247   1960     71    144    -24       C  \\nATOM    281  C   ARG A  70      22.436  17.335   6.298  1.00 12.75           C  \\nANISOU  281  C   ARG A  70     1522   1299   2021    172     94     55       C  \\nATOM    282  O   ARG A  70      22.257  16.230   6.813  1.00 15.49           O  \\nANISOU  282  O   ARG A  70     1638   1377   2869    271    364    373       O  \\nATOM    283  CB  ARG A  70      20.972  17.738   4.339  1.00 11.97           C  \\nANISOU  283  CB  ARG A  70     1384   1280   1884     47    298     -8       C  \\nATOM    284  CG  ARG A  70      19.609  18.189   3.813  1.00 11.77           C  \\nANISOU  284  CG  ARG A  70     1344   1266   1860     17    271    -50       C  \\nATOM    285  CD  ARG A  70      19.436  17.996   2.317  1.00 12.61           C  \\nANISOU  285  CD  ARG A  70     1467   1325   1996     49    294   -232       C  \\nATOM    286  NE  ARG A  70      18.079  18.271   1.909  1.00 12.33           N  \\nANISOU  286  NE  ARG A  70     1475   1561   1647    -50    328   -531       N  \\nATOM    287  CZ  ARG A  70      17.645  19.364   1.289  1.00 12.42           C  \\nANISOU  287  CZ  ARG A  70     1288   1556   1873      5    257   -479       C  \\nATOM    288  NH1 ARG A  70      18.457  20.357   0.948  1.00 13.23           N  \\nANISOU  288  NH1 ARG A  70     1378   1335   2311    -36    268   -637       N  \\nATOM    289  NH2 ARG A  70      16.370  19.471   0.940  1.00 13.99           N  \\nANISOU  289  NH2 ARG A  70     1199   2132   1984   -186    227   -604       N  \\nATOM    290  N   SER A  71      23.639  17.868   6.169  1.00 14.67           N  \\nANISOU  290  N   SER A  71     1303   1595   2674    389    293    120       N  \\nATOM    291  CA  SER A  71      24.867  17.095   6.447  1.00 17.25           C  \\nANISOU  291  CA  SER A  71     1310   2263   2979    550    519    413       C  \\nATOM    292  C   SER A  71      25.043  16.782   7.938  1.00 17.94           C  \\nANISOU  292  C   SER A  71     1416   2210   3190    274    354    954       C  \\nATOM    293  O   SER A  71      25.630  15.782   8.332  1.00 24.25           O  \\nANISOU  293  O   SER A  71     1958   2678   4575    744    594   1573       O  \\nATOM    294  CB  SER A  71      26.050  17.836   5.847  1.00 19.82           C  \\nANISOU  294  CB  SER A  71      839   3352   3339    929    706    937       C  \\nATOM    295  OG  SER A  71      26.422  18.898   6.675  1.00 28.70           O  \\nANISOU  295  OG  SER A  71     2458   4350   4095   -136    360    560       O  \\nATOM    296  N   GLY A  72      24.525  17.649   8.769  1.00 17.08           N  \\nANISOU  296  N   GLY A  72     1159   2295   3034    231    145    715       N  \\nATOM    297  CA  GLY A  72      24.703  17.555  10.214  1.00 17.60           C  \\nANISOU  297  CA  GLY A  72     1644   2152   2892    103    232    711       C  \\nATOM    298  C   GLY A  72      23.704  16.677  10.947  1.00 16.43           C  \\nANISOU  298  C   GLY A  72     1478   2160   2604    188    338    463       C  \\nATOM    299  O   GLY A  72      23.699  16.659  12.169  1.00 17.97           O  \\nANISOU  299  O   GLY A  72     1600   2423   2804    339    339    955       O  \\nATOM    300  N   ALA A  73      22.801  16.020  10.213  1.00 15.51           N  \\nANISOU  300  N   ALA A  73     1644   1745   2504    244    609    432       N  \\nATOM    301  CA  ALA A  73      21.659  15.387  10.839  1.00 16.62           C  \\nANISOU  301  CA  ALA A  73     1665   1837   2811     55    557    457       C  \\nATOM    302  C   ALA A  73      22.030  14.254  11.795  1.00 17.19           C  \\nANISOU  302  C   ALA A  73     1915   1654   2959    270    801    387       C  \\nATOM    303  O   ALA A  73      21.434  14.120  12.859  1.00 19.38           O  \\nANISOU  303  O   ALA A  73     2165   1989   3210    202   1039    825       O  \\nATOM    304  CB  ALA A  73      20.683  14.932   9.790  1.00 17.21           C  \\nANISOU  304  CB  ALA A  73     2097   1584   2858    123    518     -9       C  \\nATOM    305  N   ASP A  74      23.043  13.452  11.458  1.00 22.28           N  \\nANISOU  305  N   ASP A  74     2504   2236   3724    793   1272    822       N  \\nATOM    306  CA  ASP A  74      23.475  12.400  12.402  1.00 25.73           C  \\nANISOU  306  CA  ASP A  74     3275   2258   4241   1182   1334    903       C  \\nATOM    307  C   ASP A  74      23.965  12.923  13.713  1.00 24.12           C  \\nANISOU  307  C   ASP A  74     2684   2619   3860   1259   1264   1440       C  \\nATOM    308  O   ASP A  74      23.589  12.368  14.760  1.00 30.09           O  \\nANISOU  308  O   ASP A  74     3643   3225   4562   1928   1780   2252       O  \\nATOM    309  CB  ASP A  74      24.572  11.553  11.796  1.00 29.13           C  \\nANISOU  309  CB  ASP A  74     3251   2596   5217   1345   1333    767       C  \\nATOM    310  CG  ASP A  74      24.055  10.665  10.711  1.00 34.40           C  \\nANISOU  310  CG  ASP A  74     5386   2289   5396   1778   1624    115       C  \\nATOM    311  OD1 ASP A  74      22.837  10.404  10.641  1.00 41.43           O  \\nANISOU  311  OD1 ASP A  74     6030   2977   6732    989   3329    -80       O  \\nATOM    312  OD2 ASP A  74      24.874  10.179   9.929  1.00 46.88           O  \\nANISOU  312  OD2 ASP A  74     6466   4307   7037   1646   3405    402       O  \\nATOM    313  N   LYS A  75      24.827  13.943  13.680  1.00 23.81           N  \\nANISOU  313  N   LYS A  75     2305   3248   3491   1147    852    551       N  \\nATOM    314  CA  LYS A  75      25.326  14.552  14.916  1.00 22.46           C  \\nANISOU  314  CA  LYS A  75     3866   2264   2404   1998    158   1459       C  \\nATOM    315  C   LYS A  75      24.122  15.105  15.679  1.00 24.80           C  \\nANISOU  315  C   LYS A  75     2621   3309   3491   1564   -236   1038       C  \\nATOM    316  O   LYS A  75      24.027  14.902  16.889  1.00 29.50           O  \\nANISOU  316  O   LYS A  75     2391   5104   3713   1405    139   2236       O  \\nATOM    317  CB  LYS A  75      26.287  15.737  14.646  1.00 30.01           C  \\nANISOU  317  CB  LYS A  75     3923   3542   3937    810    324    821       C  \\nATOM    318  CG  LYS A  75      27.731  15.371  14.317  1.00 40.91           C  \\nANISOU  318  CG  LYS A  75     4167   5912   5463    847    582    333       C  \\nATOM    319  CD  LYS A  75      28.624  16.616  14.208  1.00 47.87           C  \\nANISOU  319  CD  LYS A  75     6232   6097   5859   -139    247    628       C  \\nATOM    320  CE  LYS A  75      28.179  17.642  13.140  1.00 45.85           C  \\nANISOU  320  CE  LYS A  75     5743   6462   5214     29    241    389       C  \\nATOM    321  NZ  LYS A  75      28.111  17.176  11.704  1.00 34.24           N  \\nANISOU  321  NZ  LYS A  75     2393   6045   4572   -991   -473   1491       N  \\nATOM    322  N   ARG A  76      23.179  15.757  14.988  1.00 17.99           N  \\nANISOU  322  N   ARG A  76     1451   2638   2742    432    163   1261       N  \\nATOM    323  CA  ARG A  76      22.072  16.329  15.695  1.00 15.56           C  \\nANISOU  323  CA  ARG A  76     1824   1739   2348    206    -43    342       C  \\nATOM    324  C   ARG A  76      21.136  15.295  16.333  1.00 15.13           C  \\nANISOU  324  C   ARG A  76     1681   1745   2319    423     68    393       C  \\nATOM    325  O   ARG A  76      20.656  15.517  17.436  1.00 18.16           O  \\nANISOU  325  O   ARG A  76     2500   1889   2509    270    480    137       O  \\nATOM    326  CB  ARG A  76      21.266  17.267  14.790  1.00 16.28           C  \\nANISOU  326  CB  ARG A  76     1699   1751   2736     50   -431    205       C  \\nATOM    327  CG  ARG A  76      22.025  18.527  14.412  1.00 15.16           C  \\nANISOU  327  CG  ARG A  76     1628   1930   2200    -55   -444    297       C  \\nATOM    328  CD  ARG A  76      21.168  19.564  13.760  1.00 14.67           C  \\nANISOU  328  CD  ARG A  76     1730   1689   2154    -54   -414    211       C  \\nATOM    329  NE  ARG A  76      22.026  20.488  13.035  1.00 15.24           N  \\nANISOU  329  NE  ARG A  76     1332   2170   2287   -199   -484    258       N  \\nATOM    330  CZ  ARG A  76      22.323  20.448  11.753  1.00 14.54           C  \\nANISOU  330  CZ  ARG A  76     1385   1920   2218   -194   -412    356       C  \\nATOM    331  NH1 ARG A  76      21.763  19.576  10.912  1.00 14.55           N  \\nANISOU  331  NH1 ARG A  76     1588   1747   2191   -199   -118    170       N  \\nATOM    332  NH2 ARG A  76      23.199  21.338  11.272  1.00 17.41           N  \\nANISOU  332  NH2 ARG A  76     1617   2294   2701   -447   -221    356       N  \\nATOM    333  N   ARG A  77      20.884  14.172  15.676  1.00 18.60           N  \\nANISOU  333  N   ARG A  77     2562   1660   2843    298    934    177       N  \\nATOM    334  CA  ARG A  77      20.182  13.077  16.339  1.00 22.77           C  \\nANISOU  334  CA  ARG A  77     2956   2398   3296   -312   1256    110       C  \\nATOM    335  C   ARG A  77      20.908  12.549  17.550  1.00 25.58           C  \\nANISOU  335  C   ARG A  77     3621   2267   3830     -8   1244    807       C  \\nATOM    336  O   ARG A  77      20.246  12.195  18.552  1.00 27.36           O  \\nANISOU  336  O   ARG A  77     4617   2142   3637   -242   1726    187       O  \\nATOM    337  CB  ARG A  77      20.015  11.913  15.391  1.00 25.93           C  \\nANISOU  337  CB  ARG A  77     3712   2687   3454   -160    642   -123       C  \\nATOM    338  CG  ARG A  77      18.879  12.150  14.484  1.00 24.45           C  \\nANISOU  338  CG  ARG A  77     3589   2396   3303    705    476   -842       C  \\nATOM    339  CD  ARG A  77      18.265  10.889  13.850  1.00 30.79           C  \\nANISOU  339  CD  ARG A  77     4938   3620   3138   -144    495  -1652       C  \\nATOM    340  NE  ARG A  77      17.816  11.266  12.504  1.00 33.79           N  \\nANISOU  340  NE  ARG A  77     4150   4623   4065    146    349  -1360       N  \\nATOM    341  CZ  ARG A  77      18.654  11.407  11.479  1.00 30.20           C  \\nANISOU  341  CZ  ARG A  77     3270   5423   2781    765   -301  -1792       C  \\nATOM    342  NH1 ARG A  77      19.939  11.132  11.620  1.00 34.91           N  \\nANISOU  342  NH1 ARG A  77     3372   2732   7158    744  -1793  -1028       N  \\nATOM    343  NH2 ARG A  77      18.212  11.803  10.324  1.00 36.26           N  \\nANISOU  343  NH2 ARG A  77     4471   4747   4556    201   -255    750       N  \\nATOM    344  N   GLN A  78      22.242  12.435  17.465  1.00 28.13           N  \\nANISOU  344  N   GLN A  78     3867   2397   4421   1361   1259   1022       N  \\nATOM    345  CA  GLN A  78      23.023  11.925  18.636  1.00 33.97           C  \\nANISOU  345  CA  GLN A  78     4602   3519   4782   1564   1114   2410       C  \\nATOM    346  C   GLN A  78      22.686  12.800  19.822  1.00 33.47           C  \\nANISOU  346  C   GLN A  78     4271   4028   4416   1335    522   2311       C  \\nATOM    347  O   GLN A  78      22.396  12.314  20.901  1.00 42.10           O  \\nANISOU  347  O   GLN A  78     5175   6128   4691   2076   1075   3104       O  \\nATOM    348  CB  GLN A  78      24.557  12.020  18.451  1.00 39.30           C  \\nANISOU  348  CB  GLN A  78     4377   5545   5009   1420   -476   1819       C  \\nATOM    349  CG  GLN A  78      25.307  10.885  17.763  1.00 49.80           C  \\nANISOU  349  CG  GLN A  78     7001   5415   6502    800   -188    598       C  \\nATOM    350  CD  GLN A  78      25.063   9.518  18.374  1.00 50.32           C  \\nANISOU  350  CD  GLN A  78     7770   7188   4159     83  -1373   1923       C  \\nATOM    351  OE1 GLN A  78      24.083   8.837  18.049  1.00 62.39           O  \\nANISOU  351  OE1 GLN A  78     8342   9088   6275   -659   -825   -757       O  \\nATOM    352  NE2 GLN A  78      25.966   9.098  19.249  1.00 58.57           N  \\nANISOU  352  NE2 GLN A  78     6073   8565   7613   1102  -2420    222       N  \\nATOM    353  N   GLU A  79      22.690  14.109  19.587  1.00 27.21           N  \\nANISOU  353  N   GLU A  79     2382   3963   3992    707    503   2307       N  \\nATOM    354  CA  GLU A  79      22.455  15.081  20.621  1.00 30.44           C  \\nANISOU  354  CA  GLU A  79     2610   4599   4356    707   -671   1356       C  \\nATOM    355  C   GLU A  79      21.007  15.037  21.105  1.00 23.43           C  \\nANISOU  355  C   GLU A  79     2770   3280   2852    360   -658    784       C  \\nATOM    356  O   GLU A  79      20.763  14.951  22.296  1.00 25.22           O  \\nANISOU  356  O   GLU A  79     2767   4028   2785     -4   -709    971       O  \\nATOM    357  CB  GLU A  79      22.806  16.487  20.121  1.00 33.74           C  \\nANISOU  357  CB  GLU A  79     3863   5595   3360   -691  -2148   1776       C  \\nATOM    358  CG  GLU A  79      24.287  16.687  19.740  1.00 43.00           C  \\nANISOU  358  CG  GLU A  79     4356   6645   5336     61   -444    671       C  \\nATOM    359  CD  GLU A  79      25.284  16.603  20.899  1.00 56.18           C  \\nANISOU  359  CD  GLU A  79     5539   9068   6738    -99  -1520    583       C  \\nATOM    360  OE1 GLU A  79      24.888  16.357  22.063  1.00 71.67           O  \\nANISOU  360  OE1 GLU A  79     9744   9972   7512  -1021    -22    686       O  \\nATOM    361  OE2 GLU A  79      26.497  16.788  20.641  1.00 74.16           O  \\nANISOU  361  OE2 GLU A  79     6265  11159  10752   -577    600    318       O  \\nATOM    362  N   SER A  80      20.040  15.059  20.205  1.00 18.47           N  \\nANISOU  362  N   SER A  80     2492   2035   2490    145   -319    394       N  \\nATOM    363  CA  SER A  80      18.649  15.213  20.665  1.00 18.60           C  \\nANISOU  363  CA  SER A  80     2206   2292   2567   -109   -386    243       C  \\nATOM    364  C   SER A  80      18.162  13.992  21.383  1.00 16.37           C  \\nANISOU  364  C   SER A  80     2099   2128   1994     72   -523    106       C  \\nATOM    365  O   SER A  80      17.353  14.107  22.307  1.00 17.66           O  \\nANISOU  365  O   SER A  80     2551   2255   1902    -38   -456     86       O  \\nATOM    366  CB  SER A  80      17.705  15.476  19.522  1.00 19.06           C  \\nANISOU  366  CB  SER A  80     2375   2435   2430     79   -241    605       C  \\nATOM    367  OG  SER A  80      17.866  14.451  18.633  1.00 21.31           O  \\nANISOU  367  OG  SER A  80     2590   2902   2603    404   -306    507       O  \\nATOM    368  N   LEU A  81      18.668  12.817  20.971  1.00 16.04           N  \\nANISOU  368  N   LEU A  81     1919   2008   2166     64   -211    477       N  \\nATOM    369  CA  LEU A  81      18.158  11.532  21.481  1.00 17.24           C  \\nANISOU  369  CA  LEU A  81     2120   2182   2247    -13   -411    565       C  \\nATOM    370  C   LEU A  81      18.954  10.962  22.641  1.00 19.79           C  \\nANISOU  370  C   LEU A  81     2768   2207   2541    220   -600    755       C  \\nATOM    371  O   LEU A  81      18.628   9.881  23.118  1.00 20.53           O  \\nANISOU  371  O   LEU A  81     2878   2559   2362   -274   -618    959       O  \\nATOM    372  CB  LEU A  81      18.056  10.522  20.341  1.00 19.18           C  \\nANISOU  372  CB  LEU A  81     2538   2249   2500    460   -269    317       C  \\nATOM    373  CG  LEU A  81      17.161  10.944  19.178  1.00 20.40           C  \\nANISOU  373  CG  LEU A  81     2893   2317   2540    569   -508    121       C  \\nATOM    374  CD1 LEU A  81      17.027   9.781  18.206  1.00 25.01           C  \\nANISOU  374  CD1 LEU A  81     4191   2301   3010    729   -309   -137       C  \\nATOM    375  CD2 LEU A  81      15.775  11.364  19.615  1.00 21.17           C  \\nANISOU  375  CD2 LEU A  81     2321   2710   3013   -140   -677    170       C  \\nATOM    376  N   LYS A  82      20.006  11.647  23.061  1.00 21.79           N  \\nANISOU  376  N   LYS A  82     2933   2721   2622   -152   -930   1122       N  \\nATOM    377  CA  LYS A  82      20.885  11.130  24.114  1.00 26.48           C  \\nANISOU  377  CA  LYS A  82     3382   3396   3283    225  -1441   1130       C  \\nATOM    378  C   LYS A  82      20.052  10.746  25.331  1.00 24.96           C  \\nANISOU  378  C   LYS A  82     3499   3459   2525   -165  -1819    719       C  \\nATOM    379  O   LYS A  82      19.281  11.545  25.825  1.00 26.26           O  \\nANISOU  379  O   LYS A  82     3539   3430   3006   -558  -1381    862       O  \\nATOM    380  CB  LYS A  82      21.930  12.200  24.505  1.00 29.81           C  \\nANISOU  380  CB  LYS A  82     3555   3910   3859   -191  -1328   1162       C  \\nATOM    381  CG  LYS A  82      23.039  11.712  25.439  1.00 32.71           C  \\nANISOU  381  CG  LYS A  82     3945   5067   3414    397  -1173    999       C  \\nATOM    382  CD  LYS A  82      23.763  12.886  26.086  1.00 41.51           C  \\nANISOU  382  CD  LYS A  82     4784   5784   5203   -509   -677    264       C  \\nATOM    383  CE  LYS A  82      24.834  12.397  27.051  1.00 44.72           C  \\nANISOU  383  CE  LYS A  82     5308   6213   5469    348   -489    370       C  \\nATOM    384  NZ  LYS A  82      25.988  11.788  26.340  1.00 48.96           N  \\nANISOU  384  NZ  LYS A  82     5779   6414   6409    437    125     63       N  \\nATOM    385  N   GLY A  83      20.213   9.518  25.818  1.00 25.97           N  \\nANISOU  385  N   GLY A  83     3100   3734   3033    238  -1300   1012       N  \\nATOM    386  CA  GLY A  83      19.551   9.094  27.050  1.00 26.22           C  \\nANISOU  386  CA  GLY A  83     3176   3852   2931    -35  -1172    847       C  \\nATOM    387  C   GLY A  83      18.055   8.791  26.967  1.00 24.32           C  \\nANISOU  387  C   GLY A  83     2997   3842   2402    283   -721   1355       C  \\nATOM    388  O   GLY A  83      17.444   8.441  27.968  1.00 34.77           O  \\nANISOU  388  O   GLY A  83     5190   5425   2595  -1296   -205   1204       O  \\nATOM    389  N   ILE A  84      17.470   8.888  25.771  1.00 23.61           N  \\nANISOU  389  N   ILE A  84     3180   3221   2569    -45   -975    963       N  \\nATOM    390  CA  ILE A  84      16.050   8.622  25.575  1.00 21.65           C  \\nANISOU  390  CA  ILE A  84     3074   2619   2531     23   -409    445       C  \\nATOM    391  C   ILE A  84      15.903   7.170  25.095  1.00 21.50           C  \\nANISOU  391  C   ILE A  84     2850   2613   2706    182   -612    414       C  \\nATOM    392  O   ILE A  84      16.314   6.833  23.990  1.00 22.78           O  \\nANISOU  392  O   ILE A  84     2809   2998   2847    175   -609    -69       O  \\nATOM    393  CB  ILE A  84      15.422   9.595  24.579  1.00 21.87           C  \\nANISOU  393  CB  ILE A  84     2773   2797   2738    159   -298    600       C  \\nATOM    394  CG1 ILE A  84      15.500  11.008  25.147  1.00 23.17           C  \\nANISOU  394  CG1 ILE A  84     3035   2833   2934   -237   -533    710       C  \\nATOM    395  CG2 ILE A  84      13.962   9.221  24.328  1.00 20.45           C  \\nANISOU  395  CG2 ILE A  84     2937   2456   2375    -84   -431    491       C  \\nATOM    396  CD1 ILE A  84      14.953  12.075  24.247  1.00 22.91           C  \\nANISOU  396  CD1 ILE A  84     3418   2795   2492    148   -521    268       C  \\nATOM    397  N   PRO A  85      15.299   6.311  25.930  1.00 24.84           N  \\nANISOU  397  N   PRO A  85     3210   3419   2807     81   -529    895       N  \\nATOM    398  CA  PRO A  85      15.221   4.913  25.530  1.00 27.60           C  \\nANISOU  398  CA  PRO A  85     3247   3760   3477     15  -1292    814       C  \\nATOM    399  C   PRO A  85      14.257   4.697  24.383  1.00 25.12           C  \\nANISOU  399  C   PRO A  85     3090   2907   3546   -271  -1375   1383       C  \\nATOM    400  O   PRO A  85      13.374   5.516  24.139  1.00 27.39           O  \\nANISOU  400  O   PRO A  85     3102   3067   4238   -322  -1274   1861       O  \\nATOM    401  CB  PRO A  85      14.690   4.214  26.775  1.00 29.13           C  \\nANISOU  401  CB  PRO A  85     3924   3020   4124    107  -1037   1494       C  \\nATOM    402  CG  PRO A  85      13.952   5.267  27.529  1.00 30.25           C  \\nANISOU  402  CG  PRO A  85     4166   3556   3770   -147   -610   1308       C  \\nATOM    403  CD  PRO A  85      14.646   6.563  27.230  1.00 27.51           C  \\nANISOU  403  CD  PRO A  85     4106   3497   2850     23   -324   1280       C  \\nATOM    404  N   THR A  86      14.452   3.596  23.683  1.00 29.56           N  \\nANISOU  404  N   THR A  86     3575   3330   4324    782  -1475   1104       N  \\nATOM    405  CA  THR A  86      13.554   3.221  22.611  1.00 33.08           C  \\nANISOU  405  CA  THR A  86     5479   2083   5004    282  -2569   1573       C  \\nATOM    406  C   THR A  86      12.330   2.602  23.210  1.00 33.21           C  \\nANISOU  406  C   THR A  86     5243   3012   4360   -175  -2806   1246       C  \\nATOM    407  O   THR A  86      12.324   2.194  24.373  1.00 36.34           O  \\nANISOU  407  O   THR A  86     5467   3454   4885   -934  -3028   1708       O  \\nATOM    408  CB  THR A  86      14.197   2.295  21.592  1.00 34.29           C  \\nANISOU  408  CB  THR A  86     4892   2625   5509   -108  -1941    712       C  \\nATOM    409  OG1 THR A  86      14.518   1.079  22.229  1.00 42.69           O  \\nANISOU  409  OG1 THR A  86     6307   2134   7778    891  -2351    299       O  \\nATOM    410  CG2 THR A  86      15.476   2.948  21.018  1.00 35.37           C  \\nANISOU  410  CG2 THR A  86     4793   4111   4533    676   -765    272       C  \\nATOM    411  N   LYS A  87      11.254   2.632  22.432  1.00 29.53           N  \\nANISOU  411  N   LYS A  87     4295   2239   4684   -470  -2390   1546       N  \\nATOM    412  CA  LYS A  87       9.984   2.043  22.881  1.00 32.34           C  \\nANISOU  412  CA  LYS A  87     5470   2589   4228  -1024  -1989   1933       C  \\nATOM    413  C   LYS A  87       9.364   1.296  21.710  1.00 34.07           C  \\nANISOU  413  C   LYS A  87     5482   2359   5103   -757  -2240   1272       C  \\nATOM    414  O   LYS A  87       9.349   1.793  20.575  1.00 32.47           O  \\nANISOU  414  O   LYS A  87     4980   1678   5677   -833  -2704   1685       O  \\nATOM    415  CB  LYS A  87       9.106   3.168  23.395  1.00 33.39           C  \\nANISOU  415  CB  LYS A  87     6308   3135   3242   -651  -1542   2628       C  \\nATOM    416  CG  LYS A  87       7.703   2.860  23.883  1.00 43.91           C  \\nANISOU  416  CG  LYS A  87     6353   4579   5752  -1013   -656   1342       C  \\nATOM    417  CD  LYS A  87       7.144   4.140  24.515  1.00 50.08           C  \\nANISOU  417  CD  LYS A  87     6730   5554   6742   -613   -488    601       C  \\nATOM    418  CE  LYS A  87       5.636   4.251  24.439  1.00 52.22           C  \\nANISOU  418  CE  LYS A  87     6644   6224   6971   -515   -694    172       C  \\nATOM    419  NZ  LYS A  87       4.988   3.311  25.387  1.00 57.09           N  \\nANISOU  419  NZ  LYS A  87     7105   7115   7468   -999   -730    317       N  \\nATOM    420  N   PRO A  88       8.880   0.068  21.960  1.00 32.37           N  \\nANISOU  420  N   PRO A  88     4788   2594   4916   -757  -2425   1645       N  \\nATOM    421  CA  PRO A  88       8.492  -0.727  20.801  1.00 26.65           C  \\nANISOU  421  CA  PRO A  88     3716   1820   4589   -250  -1604   1316       C  \\nATOM    422  C   PRO A  88       7.238  -0.181  20.137  1.00 26.81           C  \\nANISOU  422  C   PRO A  88     3463   2409   4315   -631  -1661   1294       C  \\nATOM    423  O   PRO A  88       6.312   0.240  20.821  1.00 27.34           O  \\nANISOU  423  O   PRO A  88     4107   2181   4098   -708  -1446   1470       O  \\nATOM    424  CB  PRO A  88       8.295  -2.145  21.398  1.00 30.26           C  \\nANISOU  424  CB  PRO A  88     4383   1971   5144   -578   -784   1314       C  \\nATOM    425  CG  PRO A  88       8.031  -1.896  22.860  1.00 30.89           C  \\nANISOU  425  CG  PRO A  88     4296   2201   5239   -403    -10   1262       C  \\nATOM    426  CD  PRO A  88       8.934  -0.763  23.185  1.00 30.92           C  \\nANISOU  426  CD  PRO A  88     4303   2513   4931    -55  -2112   1617       C  \\nATOM    427  N   GLY A  89       7.235  -0.162  18.810  1.00 23.11           N  \\nANISOU  427  N   GLY A  89     3051   1294   4435   -476  -1012    878       N  \\nATOM    428  CA  GLY A  89       6.103   0.383  18.050  1.00 19.81           C  \\nANISOU  428  CA  GLY A  89     2875   1562   3087   -678   -843    771       C  \\nATOM    429  C   GLY A  89       6.183   1.898  17.797  1.00 17.43           C  \\nANISOU  429  C   GLY A  89     2147   1413   3060   -201   -448    451       C  \\nATOM    430  O   GLY A  89       5.324   2.442  17.097  1.00 17.38           O  \\nANISOU  430  O   GLY A  89     2149   1520   2933   -113   -353    591       O  \\nATOM    431  N   PHE A  90       7.162   2.575  18.429  1.00 17.35           N  \\nANISOU  431  N   PHE A  90     2202   1294   3096   -272   -670    857       N  \\nATOM    432  CA  PHE A  90       7.278   4.025  18.383  1.00 15.80           C  \\nANISOU  432  CA  PHE A  90     1956   1253   2795    -50   -235    720       C  \\nATOM    433  C   PHE A  90       8.653   4.417  17.929  1.00 16.98           C  \\nANISOU  433  C   PHE A  90     1978   1431   3042   -116   -260    872       C  \\nATOM    434  O   PHE A  90       9.650   3.746  18.233  1.00 22.11           O  \\nANISOU  434  O   PHE A  90     2071   1960   4367     58   -301   1314       O  \\nATOM    435  CB  PHE A  90       7.014   4.639  19.757  1.00 18.53           C  \\nANISOU  435  CB  PHE A  90     2334   2055   2651    184   -373    691       C  \\nATOM    436  CG  PHE A  90       5.623   4.480  20.219  1.00 19.08           C  \\nANISOU  436  CG  PHE A  90     2720   2147   2383   -224     80    763       C  \\nATOM    437  CD1 PHE A  90       5.213   3.306  20.792  1.00 27.96           C  \\nANISOU  437  CD1 PHE A  90     4095   2739   3788     99    860   1802       C  \\nATOM    438  CD2 PHE A  90       4.706   5.475  20.030  1.00 19.74           C  \\nANISOU  438  CD2 PHE A  90     2278   2393   2828   -354    -70   1140       C  \\nATOM    439  CE1 PHE A  90       3.898   3.122  21.192  1.00 37.34           C  \\nANISOU  439  CE1 PHE A  90     5002   2817   6366    270   2206   3101       C  \\nATOM    440  CE2 PHE A  90       3.399   5.330  20.413  1.00 22.15           C  \\nANISOU  440  CE2 PHE A  90     2406   2482   3526    -46    318    844       C  \\nATOM    441  CZ  PHE A  90       2.992   4.150  20.998  1.00 31.01           C  \\nANISOU  441  CZ  PHE A  90     3457   2850   5474   -431    788   1372       C  \\nATOM    442  N   ASP A  91       8.702   5.520  17.184  1.00 13.09           N  \\nANISOU  442  N   ASP A  91     1851   1202   1919   -108   -393    347       N  \\nATOM    443  CA  ASP A  91       9.954   6.185  16.916  1.00 13.60           C  \\nANISOU  443  CA  ASP A  91     1540   1473   2154    -16   -237   -141       C  \\nATOM    444  C   ASP A  91      10.054   7.442  17.785  1.00 12.49           C  \\nANISOU  444  C   ASP A  91     1716   1163   1865     63   -247    152       C  \\nATOM    445  O   ASP A  91       9.068   7.932  18.265  1.00 14.49           O  \\nANISOU  445  O   ASP A  91     1578   1633   2294    -56   -195   -102       O  \\nATOM    446  CB  ASP A  91      10.059   6.532  15.435  1.00 16.51           C  \\nANISOU  446  CB  ASP A  91     2147   1807   2317   -152   -115     -3       C  \\nATOM    447  CG  ASP A  91      10.183   5.284  14.537  1.00 17.56           C  \\nANISOU  447  CG  ASP A  91     2517   1558   2595    104   -163    103       C  \\nATOM    448  OD1 ASP A  91      10.466   4.151  15.022  1.00 19.63           O  \\nANISOU  448  OD1 ASP A  91     2849   1416   3191     89    -87    152       O  \\nATOM    449  OD2 ASP A  91       9.968   5.428  13.349  1.00 18.72           O  \\nANISOU  449  OD2 ASP A  91     2926   1348   2837     89   -264    205       O  \\nATOM    450  N   ARG A  92      11.274   7.920  17.992  1.00 13.92           N  \\nANISOU  450  N   ARG A  92     1679   1362   2245      2   -266    -38       N  \\nATOM    451  CA  ARG A  92      11.517   9.134  18.796  1.00 13.52           C  \\nANISOU  451  CA  ARG A  92     1749   1388   2000    -11    -28     40       C  \\nATOM    452  C   ARG A  92      11.585  10.319  17.864  1.00 14.72           C  \\nANISOU  452  C   ARG A  92     2037   1421   2136     49    -33    166       C  \\nATOM    453  O   ARG A  92      12.614  10.550  17.239  1.00 20.23           O  \\nANISOU  453  O   ARG A  92     2205   2067   3415     27    289    875       O  \\nATOM    454  CB  ARG A  92      12.823   8.985  19.568  1.00 14.16           C  \\nANISOU  454  CB  ARG A  92     1813   1262   2304      0   -198   -124       C  \\nATOM    455  CG  ARG A  92      12.767   7.879  20.618  1.00 16.67           C  \\nANISOU  455  CG  ARG A  92     2040   1529   2762   -101   -207    267       C  \\nATOM    456  CD  ARG A  92      14.092   7.444  21.159  1.00 18.43           C  \\nANISOU  456  CD  ARG A  92     2451   2052   2499     58   -391    252       C  \\nATOM    457  NE  ARG A  92      14.848   6.812  20.092  1.00 19.44           N  \\nANISOU  457  NE  ARG A  92     2482   1942   2959    251   -157    309       N  \\nATOM    458  CZ  ARG A  92      16.137   6.533  20.163  1.00 22.11           C  \\nANISOU  458  CZ  ARG A  92     2632   2478   3290    631   -135      5       C  \\nATOM    459  NH1 ARG A  92      16.834   6.788  21.269  1.00 25.51           N  \\nANISOU  459  NH1 ARG A  92     3060   2955   3676    777   -604    291       N  \\nATOM    460  NH2 ARG A  92      16.741   6.035  19.122  1.00 21.57           N  \\nANISOU  460  NH2 ARG A  92     2671   2681   2842    675   -660   -291       N  \\nATOM    461  N   ASP A  93      10.470  11.029  17.732  1.00 11.71           N  \\nANISOU  461  N   ASP A  93     1687   1173   1587   -287    -12    -32       N  \\nATOM    462  CA  ASP A  93      10.360  12.157  16.827  1.00 12.53           C  \\nANISOU  462  CA  ASP A  93     1996   1147   1615   -165   -117    -49       C  \\nATOM    463  C   ASP A  93      10.990  13.377  17.446  1.00 10.96           C  \\nANISOU  463  C   ASP A  93     1678   1108   1377     24   -229    -42       C  \\nATOM    464  O   ASP A  93      10.770  13.661  18.635  1.00 13.28           O  \\nANISOU  464  O   ASP A  93     2152   1480   1413   -269   -251   -209       O  \\nATOM    465  CB  ASP A  93       8.883  12.492  16.564  1.00 12.78           C  \\nANISOU  465  CB  ASP A  93     1841   1391   1624   -312   -212     17       C  \\nATOM    466  CG  ASP A  93       8.702  13.577  15.474  1.00 14.23           C  \\nANISOU  466  CG  ASP A  93     2152   1560   1693   -300   -277     88       C  \\nATOM    467  OD1 ASP A  93       9.212  13.366  14.329  1.00 16.83           O  \\nANISOU  467  OD1 ASP A  93     2631   1842   1921     59    -52   -189       O  \\nATOM    468  OD2 ASP A  93       8.098  14.632  15.788  1.00 15.24           O  \\nANISOU  468  OD2 ASP A  93     2283   2007   1500    -25   -327   -201       O  \\nATOM    469  N   GLU A  94      11.690  14.148  16.615  1.00 11.49           N  \\nANISOU  469  N   GLU A  94     1876   1193   1295    -55   -263   -157       N  \\nATOM    470  CA  GLU A  94      12.389  15.342  17.062  1.00 11.18           C  \\nANISOU  470  CA  GLU A  94     1752   1102   1394     46   -367   -261       C  \\nATOM    471  C   GLU A  94      11.832  16.580  16.369  1.00 10.09           C  \\nANISOU  471  C   GLU A  94     1416   1179   1236    -18   -102   -140       C  \\nATOM    472  O   GLU A  94      11.668  16.591  15.141  1.00 11.69           O  \\nANISOU  472  O   GLU A  94     1940   1233   1269    -19   -165   -127       O  \\nATOM    473  CB  GLU A  94      13.862  15.270  16.627  1.00 14.85           C  \\nANISOU  473  CB  GLU A  94     1858   1391   2392   -102   -286   -275       C  \\nATOM    474  CG  GLU A  94      14.665  14.085  17.018  1.00 16.79           C  \\nANISOU  474  CG  GLU A  94     2207   1821   2352    -37   -361    258       C  \\nATOM    475  CD  GLU A  94      15.868  13.874  16.102  1.00 21.15           C  \\nANISOU  475  CD  GLU A  94     2309   2580   3145    125   -183   -252       C  \\nATOM    476  OE1 GLU A  94      16.840  14.690  16.143  1.00 20.28           O  \\nANISOU  476  OE1 GLU A  94     2703   2039   2962     72   -604    206       O  \\nATOM    477  OE2 GLU A  94      15.787  12.952  15.261  1.00 28.95           O  \\nANISOU  477  OE2 GLU A  94     3623   3423   3954    140    405  -1159       O  \\nATOM    478  N   TRP A  95      11.629  17.644  17.143  1.00  9.84           N  \\nANISOU  478  N   TRP A  95     1612   1111   1015     11   -321    -32       N  \\nATOM    479  CA  TRP A  95      11.333  18.957  16.588  1.00  9.54           C  \\nANISOU  479  CA  TRP A  95     1523   1000   1101   -123   -160    -27       C  \\nATOM    480  C   TRP A  95      12.117  19.968  17.428  1.00  9.80           C  \\nANISOU  480  C   TRP A  95     1401   1077   1243      1   -120   -201       C  \\nATOM    481  O   TRP A  95      11.882  20.028  18.647  1.00 10.80           O  \\nANISOU  481  O   TRP A  95     1604   1313   1184    -18   -179   -107       O  \\nATOM    482  CB  TRP A  95       9.857  19.299  16.555  1.00  9.22           C  \\nANISOU  482  CB  TRP A  95     1559   1034    907     31   -168   -122       C  \\nATOM    483  CG  TRP A  95       9.661  20.600  15.879  1.00  9.81           C  \\nANISOU  483  CG  TRP A  95     1466   1042   1220     19    -52    -36       C  \\nATOM    484  CD1 TRP A  95       9.702  21.861  16.448  1.00 10.50           C  \\nANISOU  484  CD1 TRP A  95     1542   1088   1358     91     14    -71       C  \\nATOM    485  CD2 TRP A  95       9.546  20.792  14.476  1.00  9.53           C  \\nANISOU  485  CD2 TRP A  95     1268   1123   1228    155   -144   -202       C  \\nATOM    486  NE1 TRP A  95       9.585  22.813  15.461  1.00 10.90           N  \\nANISOU  486  NE1 TRP A  95     1701   1146   1292    126    113    -25       N  \\nATOM    487  CE2 TRP A  95       9.526  22.187  14.239  1.00  9.99           C  \\nANISOU  487  CE2 TRP A  95     1462   1226   1105    318    -59    113       C  \\nATOM    488  CE3 TRP A  95       9.511  19.910  13.377  1.00 10.03           C  \\nANISOU  488  CE3 TRP A  95     1418   1375   1017    191    -33   -205       C  \\nATOM    489  CZ2 TRP A  95       9.403  22.711  12.962  1.00 11.57           C  \\nANISOU  489  CZ2 TRP A  95     1644   1553   1198    218   -249    229       C  \\nATOM    490  CZ3 TRP A  95       9.409  20.444  12.113  1.00 12.14           C  \\nANISOU  490  CZ3 TRP A  95     1649   1830   1133    250   -236   -106       C  \\nATOM    491  CH2 TRP A  95       9.324  21.828  11.920  1.00 13.19           C  \\nANISOU  491  CH2 TRP A  95     1965   1853   1192    285   -287    125       C  \\nATOM    492  N   PRO A  96      12.965  20.806  16.840  1.00  9.33           N  \\nANISOU  492  N   PRO A  96     1500    995   1050     10   -163   -195       N  \\nATOM    493  CA  PRO A  96      13.261  20.882  15.415  1.00  9.35           C  \\nANISOU  493  CA  PRO A  96     1385   1065   1101    -26    -62   -183       C  \\nATOM    494  C   PRO A  96      13.885  19.588  14.891  1.00  9.49           C  \\nANISOU  494  C   PRO A  96     1296   1069   1241     31   -248   -150       C  \\nATOM    495  O   PRO A  96      14.501  18.835  15.647  1.00 10.41           O  \\nANISOU  495  O   PRO A  96     1370   1318   1268    107   -413    -95       O  \\nATOM    496  CB  PRO A  96      14.182  22.101  15.297  1.00  9.90           C  \\nANISOU  496  CB  PRO A  96     1364   1124   1273    -91   -213    -46       C  \\nATOM    497  CG  PRO A  96      13.901  22.890  16.566  1.00  9.96           C  \\nANISOU  497  CG  PRO A  96     1440    892   1451    -45   -230   -102       C  \\nATOM    498  CD  PRO A  96      13.648  21.869  17.602  1.00  9.77           C  \\nANISOU  498  CD  PRO A  96     1446   1024   1240   -140    -47   -217       C  \\nATOM    499  N   MET A  97      13.740  19.413  13.598  1.00  9.08           N  \\nANISOU  499  N   MET A  97     1331    977   1142    156    -52     -6       N  \\nATOM    500  CA  MET A  97      14.249  18.213  12.948  1.00  9.36           C  \\nANISOU  500  CA  MET A  97     1272    940   1343    126   -176    -97       C  \\nATOM    501  C   MET A  97      15.768  18.220  12.860  1.00  9.59           C  \\nANISOU  501  C   MET A  97     1225   1098   1319    -97   -153     26       C  \\nATOM    502  O   MET A  97      16.431  19.237  12.795  1.00 10.61           O  \\nANISOU  502  O   MET A  97     1446    989   1596    -83   -102   -162       O  \\nATOM    503  CB  MET A  97      13.706  18.115  11.511  1.00 10.06           C  \\nANISOU  503  CB  MET A  97     1372   1187   1261    113    -45   -320       C  \\nATOM    504  CG  MET A  97      12.191  18.017  11.480  1.00 11.12           C  \\nANISOU  504  CG  MET A  97     1449   1355   1419    -76    -28   -194       C  \\nATOM    505  SD  MET A  97      11.524  17.886   9.820  1.00 13.65           S  \\nANISOU  505  SD  MET A  97     1781   1673   1731     74   -393   -428       S  \\nATOM    506  CE  MET A  97      12.018  19.454   9.127  1.00 15.04           C  \\nANISOU  506  CE  MET A  97     2002   1868   1843    170   -363   -193       C  \\nATOM    507  N   ALA A  98      16.330  17.018  12.784  1.00 10.05           N  \\nANISOU  507  N   ALA A  98     1101   1170   1545    -32     95     38       N  \\nATOM    508  CA  ALA A  98      17.770  16.857  12.714  1.00 10.21           C  \\nANISOU  508  CA  ALA A  98     1170   1252   1454    119    -40   -164       C  \\nATOM    509  C   ALA A  98      18.353  17.422  11.410  1.00  9.81           C  \\nANISOU  509  C   ALA A  98     1124   1042   1561    107    -43      8       C  \\nATOM    510  O   ALA A  98      19.523  17.783  11.398  1.00 11.53           O  \\nANISOU  510  O   ALA A  98     1115   1317   1946     29    -60     52       O  \\nATOM    511  CB  ALA A  98      18.128  15.397  12.839  1.00 11.62           C  \\nANISOU  511  CB  ALA A  98     1485   1247   1682     88    -69    -19       C  \\nATOM    512  N   MET A  99      17.557  17.538  10.330  1.00 10.35           N  \\nANISOU  512  N   MET A  99     1237   1152   1541     62      2     96       N  \\nATOM    513  CA  MET A  99      17.998  18.125   9.072  1.00 10.92           C  \\nANISOU  513  CA  MET A  99     1408   1217   1522     25    146   -130       C  \\nATOM    514  C   MET A  99      17.973  19.635   9.087  1.00 10.93           C  \\nANISOU  514  C   MET A  99     1360   1251   1539    -16   -113     68       C  \\nATOM    515  O   MET A  99      18.169  20.230   8.023  1.00 12.02           O  \\nANISOU  515  O   MET A  99     1654   1318   1594   -187    -73    136       O  \\nATOM    516  CB  MET A  99      17.193  17.579   7.868  1.00 10.94           C  \\nANISOU  516  CB  MET A  99     1356   1320   1481    184    288   -255       C  \\nATOM    517  CG  MET A  99      15.744  17.952   7.903  1.00 11.76           C  \\nANISOU  517  CG  MET A  99     1244   1470   1752     11     -5   -299       C  \\nATOM    518  SD  MET A  99      14.835  17.519   6.433  1.00 13.31           S  \\nANISOU  518  SD  MET A  99     1572   1709   1774   -183     -3   -322       S  \\nATOM    519  CE  MET A  99      15.564  18.628   5.220  1.00 14.59           C  \\nANISOU  519  CE  MET A  99     1828   2231   1485   -212    172   -321       C  \\nATOM    520  N   CYS A 100      17.750  20.268  10.237  1.00 11.15           N  \\nANISOU  520  N   CYS A 100     1535   1111   1590    -87   -105    107       N  \\nATOM    521  CA  CYS A 100      17.708  21.723  10.354  1.00 11.84           C  \\nANISOU  521  CA  CYS A 100     1749   1142   1605   -138   -199     38       C  \\nATOM    522  C   CYS A 100      18.800  22.202  11.289  1.00 11.06           C  \\nANISOU  522  C   CYS A 100     1407   1036   1758     33   -118     -3       C  \\nATOM    523  O   CYS A 100      19.101  21.547  12.281  1.00 12.39           O  \\nANISOU  523  O   CYS A 100     1541   1204   1961    -25   -346     87       O  \\nATOM    524  CB  CYS A 100      16.372  22.102  10.905  1.00 13.43           C  \\nANISOU  524  CB  CYS A 100     1664   1321   2118    184   -340   -152       C  \\nATOM    525  SG  CYS A 100      14.948  21.413  10.078  1.00 15.57           S  \\nANISOU  525  SG  CYS A 100     1906   1645   2362     91   -311     24       S  \\nATOM    526  N   GLU A 101      19.315  23.389  11.007  1.00 11.14           N  \\nANISOU  526  N   GLU A 101     1436   1085   1709    -60   -355    -22       N  \\nATOM    527  CA  GLU A 101      20.298  23.987  11.913  1.00 12.77           C  \\nANISOU  527  CA  GLU A 101     1286   1704   1861    -50   -488      2       C  \\nATOM    528  C   GLU A 101      19.807  24.073  13.355  1.00 12.30           C  \\nANISOU  528  C   GLU A 101     1543   1301   1829    -97   -361    -52       C  \\nATOM    529  O   GLU A 101      20.575  24.014  14.352  1.00 15.64           O  \\nANISOU  529  O   GLU A 101     1819   2114   2008   -277   -550    182       O  \\nATOM    530  CB  GLU A 101      20.616  25.416  11.417  1.00 15.76           C  \\nANISOU  530  CB  GLU A 101     2065   1690   2230   -433   -615   -187       C  \\nATOM    531  CG  GLU A 101      21.426  25.445  10.162  1.00 20.52           C  \\nANISOU  531  CG  GLU A 101     2634   2658   2503   -402   -349   -114       C  \\nATOM    532  CD  GLU A 101      22.057  26.781   9.782  1.00 27.66           C  \\nANISOU  532  CD  GLU A 101     3864   3437   3207  -1125    311    156       C  \\nATOM    533  OE1 GLU A 101      21.862  27.803  10.500  1.00 46.89           O  \\nANISOU  533  OE1 GLU A 101     7477   5227   5111  -2387   -753  -2143       O  \\nATOM    534  OE2 GLU A 101      22.746  26.761   8.732  1.00 34.25           O  \\nANISOU  534  OE2 GLU A 101     4036   6069   2906  -2073    214   -432       O  \\nATOM    535  N   GLU A 102      18.497  24.279  13.464  1.00 11.51           N  \\nANISOU  535  N   GLU A 102     1472   1461   1436   -107   -236      5       N  \\nATOM    536  CA  GLU A 102      17.830  24.488  14.764  1.00 12.78           C  \\nANISOU  536  CA  GLU A 102     1737   1484   1634   -304    -71   -127       C  \\nATOM    537  C   GLU A 102      17.673  23.220  15.617  1.00 11.73           C  \\nANISOU  537  C   GLU A 102     1478   1484   1493   -147   -210   -145       C  \\nATOM    538  O   GLU A 102      17.292  23.300  16.766  1.00 14.10           O  \\nANISOU  538  O   GLU A 102     1960   1904   1492    -59   -215    -73       O  \\nATOM    539  CB  GLU A 102      16.435  25.079  14.507  1.00 13.57           C  \\nANISOU  539  CB  GLU A 102     1790   1468   1895    -80     54    -94       C  \\nATOM    540  CG  GLU A 102      16.421  26.498  13.880  1.00 14.02           C  \\nANISOU  540  CG  GLU A 102     2020   1435   1870    -36   -233   -143       C  \\nATOM    541  CD  GLU A 102      16.634  26.599  12.361  1.00 14.19           C  \\nANISOU  541  CD  GLU A 102     1894   1532   1962    -66     64   -225       C  \\nATOM    542  OE1 GLU A 102      16.714  25.545  11.670  1.00 13.05           O  \\nANISOU  542  OE1 GLU A 102     1905   1439   1613    -18   -266    -72       O  \\nATOM    543  OE2 GLU A 102      16.624  27.756  11.830  1.00 17.79           O  \\nANISOU  543  OE2 GLU A 102     3041   1785   1932   -287    171     23       O  \\nATOM    544  N   GLY A 103      17.928  22.063  15.007  1.00 11.37           N  \\nANISOU  544  N   GLY A 103     1456   1458   1403    -69   -201   -124       N  \\nATOM    545  CA  GLY A 103      17.852  20.788  15.671  1.00 12.56           C  \\nANISOU  545  CA  GLY A 103     1541   1699   1531   -148   -336     67       C  \\nATOM    546  C   GLY A 103      19.005  20.468  16.581  1.00 12.51           C  \\nANISOU  546  C   GLY A 103     1636   1576   1540    -85   -398     26       C  \\nATOM    547  O   GLY A 103      19.884  21.317  16.857  1.00 15.27           O  \\nANISOU  547  O   GLY A 103     1860   1879   2060   -327   -566    195       O  \\nATOM    548  N   GLY A 104      19.005  19.221  17.060  1.00 12.32           N  \\nANISOU  548  N   GLY A 104     1550   1596   1533   -182   -513    103       N  \\nATOM    549  CA  GLY A 104      20.057  18.792  17.975  1.00 13.54           C  \\nANISOU  549  CA  GLY A 104     1726   1714   1702    -94   -620     10       C  \\nATOM    550  C   GLY A 104      19.748  19.066  19.439  1.00 14.38           C  \\nANISOU  550  C   GLY A 104     1631   2239   1592    -60   -514    161       C  \\nATOM    551  O   GLY A 104      18.636  18.802  19.922  1.00 16.59           O  \\nANISOU  551  O   GLY A 104     1725   2764   1813     -8   -449    110       O  \\nATOM    552  N   LYS A 105      20.747  19.546  20.165  1.00 19.07           N  \\nANISOU  552  N   LYS A 105     1653   3014   2576   -159   -776   -107       N  \\nATOM    553  CA  LYS A 105      20.615  19.783  21.607  1.00 20.74           C  \\nANISOU  553  CA  LYS A 105     2184   3230   2464   -288   -423    197       C  \\nATOM    554  C   LYS A 105      19.403  20.637  21.872  1.00 19.01           C  \\nANISOU  554  C   LYS A 105     2318   2846   2057   -299   -740   -120       C  \\nATOM    555  O   LYS A 105      19.158  21.644  21.197  1.00 20.03           O  \\nANISOU  555  O   LYS A 105     2525   2536   2548   -251   -971   -359       O  \\nATOM    556  CB  LYS A 105      21.880  20.442  22.189  1.00 28.64           C  \\nANISOU  556  CB  LYS A 105     2084   5065   3733   -521   -524   -463       C  \\nATOM    557  CG  LYS A 105      23.023  19.476  22.379  1.00 38.36           C  \\nANISOU  557  CG  LYS A 105     3604   5509   5459    -12   -957   -171       C  \\nATOM    558  CD  LYS A 105      24.016  20.013  23.401  1.00 44.93           C  \\nANISOU  558  CD  LYS A 105     4957   7272   4842     10  -1807    -21       C  \\nATOM    559  CE  LYS A 105      25.173  19.042  23.620  1.00 55.13           C  \\nANISOU  559  CE  LYS A 105     6121   7216   7610    890   -821    124       C  \\nATOM    560  NZ  LYS A 105      26.180  19.555  24.595  1.00 64.22           N  \\nANISOU  560  NZ  LYS A 105     6864   9372   8163   -497   -626    -83       N  \\nATOM    561  N   GLY A 106      18.606  20.193  22.823  1.00 17.68           N  \\nANISOU  561  N   GLY A 106     1828   2805   2083   -148   -971   -167       N  \\nATOM    562  CA  GLY A 106      17.441  20.945  23.237  1.00 16.32           C  \\nANISOU  562  CA  GLY A 106     2180   2204   1814    -58   -869   -203       C  \\nATOM    563  C   GLY A 106      16.151  20.634  22.505  1.00 13.39           C  \\nANISOU  563  C   GLY A 106     1827   1699   1560      0   -535     21       C  \\nATOM    564  O   GLY A 106      15.095  21.078  22.946  1.00 15.24           O  \\nANISOU  564  O   GLY A 106     1874   1831   2085     71   -745   -325       O  \\nATOM    565  N   ALA A 107      16.226  19.855  21.430  1.00 12.51           N  \\nANISOU  565  N   ALA A 107     1396   1632   1724   -113   -412    -28       N  \\nATOM    566  CA  ALA A 107      15.023  19.606  20.688  1.00 11.60           C  \\nANISOU  566  CA  ALA A 107     1275   1534   1595    157   -314   -151       C  \\nATOM    567  C   ALA A 107      13.980  18.862  21.500  1.00 11.17           C  \\nANISOU  567  C   ALA A 107     1535   1255   1453     19   -386   -114       C  \\nATOM    568  O   ALA A 107      14.312  18.034  22.346  1.00 13.33           O  \\nANISOU  568  O   ALA A 107     1770   1553   1740      2   -445     93       O  \\nATOM    569  CB  ALA A 107      15.316  18.831  19.433  1.00 12.40           C  \\nANISOU  569  CB  ALA A 107     1674   1569   1465    106   -267   -132       C  \\nATOM    570  N   SER A 108      12.719  19.123  21.168  1.00 10.43           N  \\nANISOU  570  N   SER A 108     1499   1215   1249     24   -402    -36       N  \\nATOM    571  CA  SER A 108      11.624  18.342  21.744  1.00 11.65           C  \\nANISOU  571  CA  SER A 108     1792   1403   1229    -68   -238    -46       C  \\nATOM    572  C   SER A 108      11.615  16.947  21.152  1.00 10.93           C  \\nANISOU  572  C   SER A 108     1567   1283   1301    -11   -413     89       C  \\nATOM    573  O   SER A 108      11.651  16.813  19.927  1.00 11.52           O  \\nANISOU  573  O   SER A 108     1894   1181   1300   -116   -245     74       O  \\nATOM    574  CB  SER A 108      10.287  18.999  21.427  1.00 11.72           C  \\nANISOU  574  CB  SER A 108     1734   1364   1355    -64   -155   -175       C  \\nATOM    575  OG  SER A 108       9.144  18.335  21.961  1.00 12.22           O  \\nANISOU  575  OG  SER A 108     1711   1612   1318    -97   -186   -111       O  \\nATOM    576  N   VAL A 109      11.500  15.956  22.016  1.00 11.59           N  \\nANISOU  576  N   VAL A 109     1832   1334   1237   -190   -110     59       N  \\nATOM    577  CA  VAL A 109      11.419  14.562  21.577  1.00 12.55           C  \\nANISOU  577  CA  VAL A 109     1953   1404   1408    -63   -345    -48       C  \\nATOM    578  C   VAL A 109      10.132  13.942  22.089  1.00 12.52           C  \\nANISOU  578  C   VAL A 109     2174   1444   1139   -101   -254    169       C  \\nATOM    579  O   VAL A 109       9.781  14.114  23.264  1.00 15.42           O  \\nANISOU  579  O   VAL A 109     2740   2009   1107   -555   -243    -72       O  \\nATOM    580  CB  VAL A 109      12.631  13.766  22.017  1.00 13.71           C  \\nANISOU  580  CB  VAL A 109     2065   1404   1740     55   -392    166       C  \\nATOM    581  CG1 VAL A 109      12.519  12.320  21.585  1.00 13.98           C  \\nANISOU  581  CG1 VAL A 109     1983   1491   1838    205   -190     10       C  \\nATOM    582  CG2 VAL A 109      13.915  14.389  21.483  1.00 15.14           C  \\nANISOU  582  CG2 VAL A 109     1787   1878   2087     35   -435    -21       C  \\nATOM    583  N   ARG A 110       9.417  13.251  21.189  1.00 12.54           N  \\nANISOU  583  N   ARG A 110     1902   1395   1467   -124   -182     24       N  \\nATOM    584  CA  ARG A 110       8.157  12.611  21.576  1.00 12.56           C  \\nANISOU  584  CA  ARG A 110     2056   1335   1381   -153   -224    208       C  \\nATOM    585  C   ARG A 110       8.108  11.245  20.918  1.00 11.92           C  \\nANISOU  585  C   ARG A 110     1970   1283   1274    -68   -445    298       C  \\nATOM    586  O   ARG A 110       8.374  11.157  19.709  1.00 12.99           O  \\nANISOU  586  O   ARG A 110     2125   1443   1364   -123   -207    254       O  \\nATOM    587  CB  ARG A 110       6.950  13.449  21.178  1.00 13.28           C  \\nANISOU  587  CB  ARG A 110     2093   1426   1526     43     51    129       C  \\nATOM    588  CG  ARG A 110       5.618  12.850  21.653  1.00 14.13           C  \\nANISOU  588  CG  ARG A 110     2128   1554   1687    -35     -7     78       C  \\nATOM    589  CD  ARG A 110       4.404  13.600  21.168  1.00 14.43           C  \\nANISOU  589  CD  ARG A 110     1957   1737   1789    -18    186     29       C  \\nATOM    590  NE  ARG A 110       4.321  14.920  21.751  1.00 15.45           N  \\nANISOU  590  NE  ARG A 110     2179   1853   1838     13    120     -1       N  \\nATOM    591  CZ  ARG A 110       3.265  15.710  21.626  1.00 14.78           C  \\nANISOU  591  CZ  ARG A 110     2036   1741   1837    -86    247    315       C  \\nATOM    592  NH1 ARG A 110       2.249  15.373  20.826  1.00 15.41           N  \\nANISOU  592  NH1 ARG A 110     2258   1972   1626    122    158     42       N  \\nATOM    593  NH2 ARG A 110       3.258  16.882  22.256  1.00 16.93           N  \\nANISOU  593  NH2 ARG A 110     2286   1820   2324    219    191    122       N  \\nATOM    594  N   TYR A 111       7.615  10.227  21.647  1.00 12.42           N  \\nANISOU  594  N   TYR A 111     1947   1430   1339   -149   -337    312       N  \\nATOM    595  CA  TYR A 111       7.342   8.940  21.011  1.00 13.19           C  \\nANISOU  595  CA  TYR A 111     1983   1329   1698   -150   -348    301       C  \\nATOM    596  C   TYR A 111       6.146   9.094  20.068  1.00 12.82           C  \\nANISOU  596  C   TYR A 111     1923   1312   1635   -256   -261    316       C  \\nATOM    597  O   TYR A 111       5.082   9.562  20.510  1.00 15.26           O  \\nANISOU  597  O   TYR A 111     2080   1709   2006   -209   -312    129       O  \\nATOM    598  CB  TYR A 111       7.044   7.866  22.029  1.00 14.69           C  \\nANISOU  598  CB  TYR A 111     2222   1527   1833    -28   -179    394       C  \\nATOM    599  CG  TYR A 111       8.216   7.534  22.921  1.00 15.70           C  \\nANISOU  599  CG  TYR A 111     2539   1384   2040   -119   -577    527       C  \\nATOM    600  CD1 TYR A 111       9.375   6.952  22.385  1.00 16.75           C  \\nANISOU  600  CD1 TYR A 111     2420   1893   2051    -98   -567    793       C  \\nATOM    601  CD2 TYR A 111       8.193   7.813  24.306  1.00 19.33           C  \\nANISOU  601  CD2 TYR A 111     2781   2432   2128    193   -569    621       C  \\nATOM    602  CE1 TYR A 111      10.479   6.638  23.212  1.00 18.51           C  \\nANISOU  602  CE1 TYR A 111     2754   1779   2500     15   -845   1020       C  \\nATOM    603  CE2 TYR A 111       9.284   7.506  25.128  1.00 21.39           C  \\nANISOU  603  CE2 TYR A 111     3185   2691   2250    181   -929    519       C  \\nATOM    604  CZ  TYR A 111      10.416   6.924  24.587  1.00 20.46           C  \\nANISOU  604  CZ  TYR A 111     2684   2528   2559    -38   -817   1093       C  \\nATOM    605  OH  TYR A 111      11.499   6.630  25.402  1.00 28.21           O  \\nANISOU  605  OH  TYR A 111     3741   3827   3150     99  -1721   1303       O  \\nATOM    606  N   VAL A 112       6.328   8.702  18.806  1.00 12.29           N  \\nANISOU  606  N   VAL A 112     1623   1298   1746    -39   -332    202       N  \\nATOM    607  CA  VAL A 112       5.273   8.769  17.785  1.00 12.46           C  \\nANISOU  607  CA  VAL A 112     1613   1451   1667      2   -263     88       C  \\nATOM    608  C   VAL A 112       5.200   7.400  17.106  1.00 12.74           C  \\nANISOU  608  C   VAL A 112     1638   1331   1871     95   -414    197       C  \\nATOM    609  O   VAL A 112       6.230   6.796  16.848  1.00 12.62           O  \\nANISOU  609  O   VAL A 112     1705   1224   1863    125   -318    233       O  \\nATOM    610  CB  VAL A 112       5.630   9.852  16.729  1.00 13.12           C  \\nANISOU  610  CB  VAL A 112     1778   1316   1891    167   -285    207       C  \\nATOM    611  CG1 VAL A 112       4.683   9.796  15.525  1.00 15.41           C  \\nANISOU  611  CG1 VAL A 112     1850   1711   2293    131   -518    290       C  \\nATOM    612  CG2 VAL A 112       5.573  11.251  17.357  1.00 13.35           C  \\nANISOU  612  CG2 VAL A 112     1959   1482   1629    123   -454    133       C  \\nATOM    613  N   SER A 113       3.996   6.881  16.880  1.00 12.52           N  \\nANISOU  613  N   SER A 113     1806   1140   1808    -98   -183    185       N  \\nATOM    614  CA ASER A 113       3.838   5.595  16.196  0.50 12.63           C  \\nANISOU  614  CA ASER A 113     1849   1236   1710    -60   -314     83       C  \\nATOM    615  CA BSER A 113       3.898   5.566  16.257  0.50 13.28           C  \\nANISOU  615  CA BSER A 113     2077   1221   1748     12   -259     62       C  \\nATOM    616  C   SER A 113       4.715   5.512  14.975  1.00 13.17           C  \\nANISOU  616  C   SER A 113     2014   1142   1846    -34   -243     62       C  \\nATOM    617  O   SER A 113       4.735   6.445  14.176  1.00 14.04           O  \\nANISOU  617  O   SER A 113     2216   1132   1984    -70   -283    122       O  \\nATOM    618  CB ASER A 113       2.388   5.427  15.775  0.50 14.80           C  \\nANISOU  618  CB ASER A 113     1823   1643   2157   -111   -251    172       C  \\nATOM    619  CB BSER A 113       2.446   5.140  16.044  0.50 16.70           C  \\nANISOU  619  CB BSER A 113     2069   1902   2371     84   -251     63       C  \\nATOM    620  OG ASER A 113       1.549   5.465  16.919  0.50 16.38           O  \\nANISOU  620  OG ASER A 113     1692   1999   2530    190    -46    172       O  \\nATOM    621  OG BSER A 113       1.796   5.910  15.062  0.50 20.79           O  \\nANISOU  621  OG BSER A 113     3270   1978   2650    -55   -723    289       O  \\nATOM    622  N   SER A 114       5.388   4.376  14.779  1.00 13.42           N  \\nANISOU  622  N   SER A 114     2084   1032   1983    -53   -265     86       N  \\nATOM    623  CA  SER A 114       6.397   4.283  13.745  1.00 15.23           C  \\nANISOU  623  CA  SER A 114     2229   1192   2363    -26    -41    -36       C  \\nATOM    624  C   SER A 114       5.846   4.539  12.359  1.00 15.66           C  \\nANISOU  624  C   SER A 114     2474   1192   2284      4    -98   -213       C  \\nATOM    625  O   SER A 114       6.513   5.228  11.569  1.00 17.22           O  \\nANISOU  625  O   SER A 114     2773   1262   2508    122    -12    138       O  \\nATOM    626  CB  SER A 114       7.150   2.952  13.802  1.00 16.60           C  \\nANISOU  626  CB  SER A 114     2466   1367   2472    210    -14   -153       C  \\nATOM    627  OG  SER A 114       7.796   2.815  15.040  1.00 21.79           O  \\nANISOU  627  OG  SER A 114     3170   1856   3251     17   -572    153       O  \\nATOM    628  N   SER A 115       4.650   4.048  12.030  1.00 16.37           N  \\nANISOU  628  N   SER A 115     2809   1168   2240   -129   -223   -130       N  \\nATOM    629  CA ASER A 115       4.123   4.267  10.683  0.50 17.27           C  \\nANISOU  629  CA ASER A 115     2960   1376   2225    -28   -216   -307       C  \\nATOM    630  CA BSER A 115       4.118   4.259  10.684  0.50 17.51           C  \\nANISOU  630  CA BSER A 115     2908   1455   2289    -17   -238   -168       C  \\nATOM    631  C   SER A 115       3.849   5.742  10.414  1.00 17.90           C  \\nANISOU  631  C   SER A 115     3065   1447   2289    -99   -489   -146       C  \\nATOM    632  O   SER A 115       4.121   6.232   9.327  1.00 19.11           O  \\nANISOU  632  O   SER A 115     3080   1863   2318   -245   -489    -70       O  \\nATOM    633  CB ASER A 115       2.850   3.447  10.460  0.50 19.42           C  \\nANISOU  633  CB ASER A 115     3156   1711   2510   -305   -306   -336       C  \\nATOM    634  CB BSER A 115       2.832   3.449  10.483  0.50 22.15           C  \\nANISOU  634  CB BSER A 115     3209   2281   2924   -492   -600    -19       C  \\nATOM    635  OG ASER A 115       3.164   2.075  10.497  0.50 28.14           O  \\nANISOU  635  OG ASER A 115     4973   1859   3858    125   -536   -229       O  \\nATOM    636  OG BSER A 115       1.872   3.794  11.463  0.50 24.69           O  \\nANISOU  636  OG BSER A 115     3193   2964   3224   -413   -471    522       O  \\nATOM    637  N   ASP A 116       3.323   6.441  11.420  1.00 16.13           N  \\nANISOU  637  N   ASP A 116     2611   1372   2144    -62   -523     64       N  \\nATOM    638  CA  ASP A 116       3.056   7.876  11.302  1.00 14.28           C  \\nANISOU  638  CA  ASP A 116     2104   1414   1906     68   -474     65       C  \\nATOM    639  C   ASP A 116       4.344   8.625  11.130  1.00 13.58           C  \\nANISOU  639  C   ASP A 116     2003   1345   1810     98   -418      2       C  \\nATOM    640  O   ASP A 116       4.460   9.465  10.207  1.00 14.20           O  \\nANISOU  640  O   ASP A 116     2176   1404   1813     43   -478      8       O  \\nATOM    641  CB  ASP A 116       2.289   8.319  12.561  1.00 13.60           C  \\nANISOU  641  CB  ASP A 116     2010   1375   1780   -135   -457     98       C  \\nATOM    642  CG  ASP A 116       1.829   9.763  12.562  1.00 14.01           C  \\nANISOU  642  CG  ASP A 116     2141   1433   1747   -145   -517    119       C  \\nATOM    643  OD1 ASP A 116       2.648  10.689  12.560  1.00 13.86           O  \\nANISOU  643  OD1 ASP A 116     1891   1426   1947   -100   -412     61       O  \\nATOM    644  OD2 ASP A 116       0.600   9.992  12.763  1.00 15.25           O  \\nANISOU  644  OD2 ASP A 116     2069   1467   2256    -17   -385     -6       O  \\nATOM    645  N   ASN A 117       5.332   8.338  11.981  1.00 13.93           N  \\nANISOU  645  N   ASN A 117     2017   1278   1997    201   -333    243       N  \\nATOM    646  CA  ASN A 117       6.576   9.054  11.916  1.00 13.56           C  \\nANISOU  646  CA  ASN A 117     2123   1182   1847    139   -250    254       C  \\nATOM    647  C   ASN A 117       7.392   8.775  10.657  1.00 12.94           C  \\nANISOU  647  C   ASN A 117     2241   1064   1609    170   -426    133       C  \\nATOM    648  O   ASN A 117       8.003   9.691  10.077  1.00 13.77           O  \\nANISOU  648  O   ASN A 117     2315   1195   1718    146   -299    141       O  \\nATOM    649  CB  ASN A 117       7.414   8.815  13.144  1.00 14.47           C  \\nANISOU  649  CB  ASN A 117     1973   1647   1874    293   -278    -76       C  \\nATOM    650  CG  ASN A 117       8.365   9.951  13.408  1.00 15.92           C  \\nANISOU  650  CG  ASN A 117     2163   1696   2190    331   -239   -457       C  \\nATOM    651  OD1 ASN A 117       9.536   9.720  13.557  1.00 17.13           O  \\nANISOU  651  OD1 ASN A 117     2201   2087   2217    436   -499    -81       O  \\nATOM    652  ND2 ASN A 117       7.860  11.188  13.460  1.00 15.47           N  \\nANISOU  652  ND2 ASN A 117     2295   1785   1798    442   -276   -375       N  \\nATOM    653  N   ARG A 118       7.423   7.517  10.216  1.00 14.72           N  \\nANISOU  653  N   ARG A 118     2483   1122   1986    317   -243    -83       N  \\nATOM    654  CA  ARG A 118       8.194   7.162   9.041  1.00 14.46           C  \\nANISOU  654  CA  ARG A 118     2527   1184   1782     27   -318     16       C  \\nATOM    655  C   ARG A 118       7.550   7.731   7.774  1.00 14.85           C  \\nANISOU  655  C   ARG A 118     2521   1219   1902    182   -487   -255       C  \\nATOM    656  O   ARG A 118       8.239   8.213   6.876  1.00 17.06           O  \\nANISOU  656  O   ARG A 118     2989   1471   2020    281   -346   -119       O  \\nATOM    657  CB  ARG A 118       8.391   5.640   8.981  1.00 14.32           C  \\nANISOU  657  CB  ARG A 118     2560   1217   1662    126    -19   -305       C  \\nATOM    658  CG  ARG A 118       9.311   5.151  10.088  1.00 15.03           C  \\nANISOU  658  CG  ARG A 118     2540   1096   2074    149   -182    -92       C  \\nATOM    659  CD  ARG A 118       9.355   3.651  10.253  1.00 15.54           C  \\nANISOU  659  CD  ARG A 118     2465   1123   2314     46   -194      6       C  \\nATOM    660  NE  ARG A 118       9.969   3.290  11.508  1.00 15.78           N  \\nANISOU  660  NE  ARG A 118     2529   1118   2346    250   -238    -19       N  \\nATOM    661  CZ  ARG A 118      10.217   2.048  11.892  1.00 14.04           C  \\nANISOU  661  CZ  ARG A 118     2017   1272   2043    337    -37     64       C  \\nATOM    662  NH1 ARG A 118       9.990   1.049  11.071  1.00 15.68           N  \\nANISOU  662  NH1 ARG A 118     2419   1288   2251    155    124    -28       N  \\nATOM    663  NH2 ARG A 118      10.779   1.833  13.065  1.00 14.66           N  \\nANISOU  663  NH2 ARG A 118     2238   1170   2160     20   -131    223       N  \\nATOM    664  N   GLY A 119       6.232   7.744   7.769  1.00 15.60           N  \\nANISOU  664  N   GLY A 119     2478   1325   2122    -50   -750   -238       N  \\nATOM    665  CA  GLY A 119       5.533   8.352   6.661  1.00 14.46           C  \\nANISOU  665  CA  GLY A 119     2254   1378   1860    -84   -673   -381       C  \\nATOM    666  C   GLY A 119       5.800   9.855   6.603  1.00 14.21           C  \\nANISOU  666  C   GLY A 119     2262   1366   1772     59   -239   -319       C  \\nATOM    667  O   GLY A 119       6.071  10.390   5.546  1.00 14.70           O  \\nANISOU  667  O   GLY A 119     2471   1476   1635    -51   -404   -248       O  \\nATOM    668  N   ALA A 120       5.738  10.517   7.763  1.00 13.18           N  \\nANISOU  668  N   ALA A 120     2191   1151   1665     39   -275   -143       N  \\nATOM    669  CA  ALA A 120       5.980  11.952   7.812  1.00 11.46           C  \\nANISOU  669  CA  ALA A 120     1827   1150   1374     45   -380   -221       C  \\nATOM    670  C   ALA A 120       7.395  12.218   7.348  1.00 11.46           C  \\nANISOU  670  C   ALA A 120     1748   1015   1589    193   -368   -227       C  \\nATOM    671  O   ALA A 120       7.671  13.185   6.629  1.00 12.50           O  \\nANISOU  671  O   ALA A 120     1900   1149   1699     91   -296    -92       O  \\nATOM    672  CB  ALA A 120       5.766  12.452   9.234  1.00 12.03           C  \\nANISOU  672  CB  ALA A 120     1809   1388   1370     27   -132    -84       C  \\nATOM    673  N   GLY A 121       8.343  11.404   7.823  1.00 12.44           N  \\nANISOU  673  N   GLY A 121     1964   1049   1711    158   -613   -185       N  \\nATOM    674  CA  GLY A 121       9.743  11.624   7.438  1.00 13.23           C  \\nANISOU  674  CA  GLY A 121     1787   1310   1928    245   -742      4       C  \\nATOM    675  C   GLY A 121       9.974  11.563   5.929  1.00 14.48           C  \\nANISOU  675  C   GLY A 121     1812   1588   2101    358   -589   -157       C  \\nATOM    676  O   GLY A 121      10.723  12.373   5.362  1.00 14.88           O  \\nANISOU  676  O   GLY A 121     2093   1571   1989    292   -182   -478       O  \\nATOM    677  N   SER A 122       9.347  10.575   5.277  1.00 15.19           N  \\nANISOU  677  N   SER A 122     2300   1557   1912    103   -443   -189       N  \\nATOM    678  CA ASER A 122       9.466  10.430   3.829  0.50 16.59           C  \\nANISOU  678  CA ASER A 122     2606   1662   2033    300   -234   -449       C  \\nATOM    679  CA BSER A 122       9.474  10.417   3.837  0.50 16.64           C  \\nANISOU  679  CA BSER A 122     2608   1664   2050    329   -222   -448       C  \\nATOM    680  C   SER A 122       8.781  11.590   3.104  1.00 13.83           C  \\nANISOU  680  C   SER A 122     1515   2028   1710    100   -560   -574       C  \\nATOM    681  O   SER A 122       9.252  12.100   2.079  1.00 15.49           O  \\nANISOU  681  O   SER A 122     2157   1731   1995    329   -150   -690       O  \\nATOM    682  CB ASER A 122       8.889   9.118   3.318  0.50 20.40           C  \\nANISOU  682  CB ASER A 122     3433   1771   2546    187   -480   -639       C  \\nATOM    683  CB BSER A 122       8.879   9.080   3.400  0.50 21.01           C  \\nANISOU  683  CB BSER A 122     3514   1886   2584    100   -391   -721       C  \\nATOM    684  OG ASER A 122       9.071   9.069   1.896  0.50 23.62           O  \\nANISOU  684  OG ASER A 122     4747   1851   2374    179   -881   -282       O  \\nATOM    685  OG BSER A 122       9.579   7.991   3.983  0.50 25.92           O  \\nANISOU  685  OG BSER A 122     4482   2454   2910    558   -408   -454       O  \\nATOM    686  N   TRP A 123       7.620  11.983   3.607  1.00 12.35           N  \\nANISOU  686  N   TRP A 123     1668   1488   1533     18   -236   -290       N  \\nATOM    687  CA  TRP A 123       6.910  13.108   3.039  1.00 11.58           C  \\nANISOU  687  CA  TRP A 123     1495   1513   1389     -7   -164   -181       C  \\nATOM    688  C   TRP A 123       7.756  14.373   3.098  1.00 10.90           C  \\nANISOU  688  C   TRP A 123     1310   1550   1278    167    -58   -198       C  \\nATOM    689  O   TRP A 123       7.862  15.092   2.107  1.00 11.38           O  \\nANISOU  689  O   TRP A 123     1315   1645   1361    -18    -14   -108       O  \\nATOM    690  CB  TRP A 123       5.572  13.297   3.764  1.00 13.41           C  \\nANISOU  690  CB  TRP A 123     1558   1825   1710   -189     58     52       C  \\nATOM    691  CG  TRP A 123       4.726  14.329   3.129  1.00 14.14           C  \\nANISOU  691  CG  TRP A 123     1543   2219   1608     19     90    119       C  \\nATOM    692  CD1 TRP A 123       3.768  14.114   2.209  1.00 17.46           C  \\nANISOU  692  CD1 TRP A 123     2561   2398   1673   -107   -386      7       C  \\nATOM    693  CD2 TRP A 123       4.788  15.752   3.322  1.00 13.80           C  \\nANISOU  693  CD2 TRP A 123     1627   2112   1502    162    273    335       C  \\nATOM    694  NE1 TRP A 123       3.192  15.298   1.849  1.00 20.80           N  \\nANISOU  694  NE1 TRP A 123     2618   3113   2172    536   -397     92       N  \\nATOM    695  CE2 TRP A 123       3.847  16.329   2.459  1.00 16.83           C  \\nANISOU  695  CE2 TRP A 123     2255   2546   1591    446    285    526       C  \\nATOM    696  CE3 TRP A 123       5.561  16.595   4.116  1.00 15.01           C  \\nANISOU  696  CE3 TRP A 123     1604   2046   2050    199    177    323       C  \\nATOM    697  CZ2 TRP A 123       3.603  17.700   2.433  1.00 19.10           C  \\nANISOU  697  CZ2 TRP A 123     2411   2628   2215    452    400    396       C  \\nATOM    698  CZ3 TRP A 123       5.356  17.950   4.029  1.00 17.44           C  \\nANISOU  698  CZ3 TRP A 123     2306   2179   2141    191    269     31       C  \\nATOM    699  CH2 TRP A 123       4.414  18.488   3.176  1.00 17.16           C  \\nANISOU  699  CH2 TRP A 123     2429   2496   1593    484    776    387       C  \\nATOM    700  N   VAL A 124       8.349  14.643   4.278  1.00 11.08           N  \\nANISOU  700  N   VAL A 124     1479   1467   1263    -20   -100   -186       N  \\nATOM    701  CA  VAL A 124       9.151  15.843   4.434  1.00 12.47           C  \\nANISOU  701  CA  VAL A 124     1631   1465   1641    -86    -86   -270       C  \\nATOM    702  C   VAL A 124      10.385  15.790   3.546  1.00 12.55           C  \\nANISOU  702  C   VAL A 124     1391   1681   1694    -26   -253   -202       C  \\nATOM    703  O   VAL A 124      10.730  16.775   2.891  1.00 12.99           O  \\nANISOU  703  O   VAL A 124     1437   1756   1740   -231    -58   -342       O  \\nATOM    704  CB  VAL A 124       9.522  16.062   5.912  1.00 14.46           C  \\nANISOU  704  CB  VAL A 124     2069   1666   1758   -137   -186   -472       C  \\nATOM    705  CG1 VAL A 124      10.623  17.111   6.068  1.00 17.84           C  \\nANISOU  705  CG1 VAL A 124     2270   2460   2046   -637    -18   -411       C  \\nATOM    706  CG2 VAL A 124       8.292  16.409   6.706  1.00 14.62           C  \\nANISOU  706  CG2 VAL A 124     2231   1582   1742   -116   -109   -220       C  \\nATOM    707  N   GLY A 125      11.062  14.648   3.506  1.00 13.90           N  \\nANISOU  707  N   GLY A 125     1502   1720   2058     10   -104   -327       N  \\nATOM    708  CA  GLY A 125      12.275  14.592   2.669  1.00 15.88           C  \\nANISOU  708  CA  GLY A 125     1270   2044   2720     28    -73   -597       C  \\nATOM    709  C   GLY A 125      11.985  14.842   1.214  1.00 18.17           C  \\nANISOU  709  C   GLY A 125     1697   2688   2516    -73    194   -952       C  \\nATOM    710  O   GLY A 125      12.692  15.590   0.544  1.00 20.86           O  \\nANISOU  710  O   GLY A 125     1611   3211   3102   -137    454  -1205       O  \\nATOM    711  N   ASN A 126      10.889  14.268   0.718  1.00 14.70           N  \\nANISOU  711  N   ASN A 126     1644   1906   2034    185     94   -811       N  \\nATOM    712  CA  ASN A 126      10.505  14.529  -0.645  1.00 14.53           C  \\nANISOU  712  CA  ASN A 126     1554   2137   1827    -46    594   -718       C  \\nATOM    713  C   ASN A 126      10.055  15.943  -0.915  1.00 13.73           C  \\nANISOU  713  C   ASN A 126     1877   2123   1214   -254    294   -540       C  \\nATOM    714  O   ASN A 126      10.437  16.536  -1.911  1.00 16.96           O  \\nANISOU  714  O   ASN A 126     2210   2564   1668   -621    908   -699       O  \\nATOM    715  CB  ASN A 126       9.502  13.467  -1.095  1.00 16.67           C  \\nANISOU  715  CB  ASN A 126     1778   1939   2616    -47    330   -472       C  \\nATOM    716  CG  ASN A 126      10.237  12.311  -1.649  1.00 18.18           C  \\nANISOU  716  CG  ASN A 126     2681   1946   2277     36    590   -237       C  \\nATOM    717  OD1 ASN A 126      10.878  12.435  -2.721  1.00 23.91           O  \\nANISOU  717  OD1 ASN A 126     3747   2811   2525   -674   1233   -798       O  \\nATOM    718  ND2 ASN A 126      10.263  11.239  -0.894  1.00 20.90           N  \\nANISOU  718  ND2 ASN A 126     2694   2456   2790   -428   -158    367       N  \\nATOM    719  N   ARG A 127       9.230  16.489  -0.015  1.00 11.73           N  \\nANISOU  719  N   ARG A 127     1374   1710   1371   -320    330   -348       N  \\nATOM    720  CA  ARG A 127       8.751  17.830  -0.217  1.00 12.22           C  \\nANISOU  720  CA  ARG A 127     1799   1728   1116   -339    379   -187       C  \\nATOM    721  C   ARG A 127       9.887  18.840  -0.180  1.00 15.17           C  \\nANISOU  721  C   ARG A 127     1884   1778   2100   -320    433   -334       C  \\nATOM    722  O   ARG A 127       9.835  19.828  -0.906  1.00 22.87           O  \\nANISOU  722  O   ARG A 127     4284   2661   1742  -1504    127     24       O  \\nATOM    723  CB  ARG A 127       7.672  18.164   0.810  1.00 13.32           C  \\nANISOU  723  CB  ARG A 127     1665   1983   1410    -83    347   -362       C  \\nATOM    724  CG  ARG A 127       6.970  19.508   0.616  1.00 16.13           C  \\nANISOU  724  CG  ARG A 127     2527   1943   1658     81    235   -289       C  \\nATOM    725  CD  ARG A 127       6.023  19.442  -0.629  1.00 19.86           C  \\nANISOU  725  CD  ARG A 127     3202   2507   1837    445   -114   -119       C  \\nATOM    726  NE  ARG A 127       5.297  20.697  -0.880  1.00 25.19           N  \\nANISOU  726  NE  ARG A 127     4132   3058   2379    850   -434    -45       N  \\nATOM    727  CZ  ARG A 127       5.831  21.752  -1.483  1.00 32.19           C  \\nANISOU  727  CZ  ARG A 127     4325   3813   4092   -146    125   -269       C  \\nATOM    728  NH1 ARG A 127       7.111  21.739  -1.874  1.00 30.62           N  \\nANISOU  728  NH1 ARG A 127     3545   5059   3028   -178   -763  -1067       N  \\nATOM    729  NH2 ARG A 127       5.106  22.858  -1.667  1.00 36.83           N  \\nANISOU  729  NH2 ARG A 127     4612   4140   5241    566    254   -184       N  \\nATOM    730  N   LEU A 128      10.836  18.654   0.731  1.00 14.49           N  \\nANISOU  730  N   LEU A 128     1312   1799   2395   -260    801   -579       N  \\nATOM    731  CA  LEU A 128      11.936  19.616   0.870  1.00 15.93           C  \\nANISOU  731  CA  LEU A 128     1632   1918   2503   -473    790   -793       C  \\nATOM    732  C   LEU A 128      13.030  19.428  -0.129  1.00 16.71           C  \\nANISOU  732  C   LEU A 128     1559   1776   3012   -507    946   -849       C  \\nATOM    733  O   LEU A 128      13.902  20.269  -0.252  1.00 16.56           O  \\nANISOU  733  O   LEU A 128     1419   1833   3037   -524    647   -817       O  \\nATOM    734  CB  LEU A 128      12.477  19.602   2.303  1.00 16.46           C  \\nANISOU  734  CB  LEU A 128     1723   1945   2585   -622    314   -929       C  \\nATOM    735  CG  LEU A 128      11.522  20.105   3.415  1.00 17.79           C  \\nANISOU  735  CG  LEU A 128     2001   2036   2719   -114    372   -766       C  \\nATOM    736  CD1 LEU A 128      12.235  20.170   4.779  1.00 20.21           C  \\nANISOU  736  CD1 LEU A 128     2183   2471   3022   -223    113  -1085       C  \\nATOM    737  CD2 LEU A 128      10.906  21.443   3.023  1.00 20.05           C  \\nANISOU  737  CD2 LEU A 128     2606   2356   2654      2    383   -380       C  \\nATOM    738  N   ASN A 129      12.944  18.373  -0.922  1.00 20.86           N  \\nANISOU  738  N   ASN A 129     2133   2231   3562   -952   1598  -1356       N  \\nATOM    739  CA  ASN A 129      13.954  18.156  -1.944  1.00 18.59           C  \\nANISOU  739  CA  ASN A 129     1729   2240   3093   -555   1082  -1391       C  \\nATOM    740  C   ASN A 129      13.965  19.204  -3.045  1.00 18.29           C  \\nANISOU  740  C   ASN A 129     1737   2481   2731   -129    355  -1274       C  \\nATOM    741  O   ASN A 129      14.925  19.279  -3.836  1.00 17.87           O  \\nANISOU  741  O   ASN A 129     1681   2036   3072   -335    470   -690       O  \\nATOM    742  CB  ASN A 129      13.785  16.777  -2.584  1.00 19.64           C  \\nANISOU  742  CB  ASN A 129     1836   2287   3340   -802   1115  -1448       C  \\nATOM    743  CG  ASN A 129      14.965  16.419  -3.482  1.00 17.97           C  \\nANISOU  743  CG  ASN A 129     1787   2229   2811   -841   1136   -748       C  \\nATOM    744  OD1 ASN A 129      16.090  16.467  -3.061  1.00 19.31           O  \\nANISOU  744  OD1 ASN A 129     1904   2441   2990   -605   1117  -1155       O  \\nATOM    745  ND2 ASN A 129      14.687  16.026  -4.691  1.00 20.02           N  \\nANISOU  745  ND2 ASN A 129     2068   2491   3045   -799    763  -1145       N  \\nATOM    746  N   GLY A 130      12.942  20.058  -3.104  1.00 21.23           N  \\nANISOU  746  N   GLY A 130     1538   3161   3368   -161   -137  -1981       N  \\nATOM    747  CA  GLY A 130      12.963  21.192  -3.975  1.00 22.76           C  \\nANISOU  747  CA  GLY A 130     2451   2929   3265    448   -791  -1861       C  \\nATOM    748  C   GLY A 130      13.728  22.404  -3.455  1.00 17.72           C  \\nANISOU  748  C   GLY A 130     1641   2644   2448    460   -566  -1171       C  \\nATOM    749  O   GLY A 130      13.856  23.387  -4.143  1.00 24.02           O  \\nANISOU  749  O   GLY A 130     3364   3328   2432   1088   -795   -938       O  \\nATOM    750  N   TYR A 131      14.301  22.302  -2.263  1.00 14.58           N  \\nANISOU  750  N   TYR A 131     1416   2031   2093    305   -248   -852       N  \\nATOM    751  CA  TYR A 131      14.966  23.447  -1.649  1.00 12.97           C  \\nANISOU  751  CA  TYR A 131     1422   1805   1701    218    -78   -461       C  \\nATOM    752  C   TYR A 131      16.414  23.085  -1.303  1.00 11.82           C  \\nANISOU  752  C   TYR A 131     1415   1622   1454    155   -130   -534       C  \\nATOM    753  O   TYR A 131      16.689  22.061  -0.680  1.00 12.81           O  \\nANISOU  753  O   TYR A 131     1273   1673   1920    278    -25   -458       O  \\nATOM    754  CB  TYR A 131      14.248  23.825  -0.357  1.00 12.95           C  \\nANISOU  754  CB  TYR A 131     1541   1555   1824    278     70   -379       C  \\nATOM    755  CG  TYR A 131      12.811  24.217  -0.520  1.00 13.31           C  \\nANISOU  755  CG  TYR A 131     1594   1754   1709    288     81   -441       C  \\nATOM    756  CD1 TYR A 131      12.449  25.442  -1.081  1.00 14.15           C  \\nANISOU  756  CD1 TYR A 131     1537   1930   1908    466   -218   -320       C  \\nATOM    757  CD2 TYR A 131      11.833  23.408  -0.063  1.00 15.35           C  \\nANISOU  757  CD2 TYR A 131     1612   1914   2305    361    314    -49       C  \\nATOM    758  CE1 TYR A 131      11.109  25.811  -1.194  1.00 17.22           C  \\nANISOU  758  CE1 TYR A 131     1678   2467   2396    744   -191   -324       C  \\nATOM    759  CE2 TYR A 131      10.490  23.761  -0.143  1.00 18.54           C  \\nANISOU  759  CE2 TYR A 131     1572   2682   2788    419    430   -315       C  \\nATOM    760  CZ  TYR A 131      10.145  24.956  -0.716  1.00 16.85           C  \\nANISOU  760  CZ  TYR A 131     1521   2942   1939    591   -366   -494       C  \\nATOM    761  OH  TYR A 131       8.839  25.280  -0.770  1.00 21.97           O  \\nANISOU  761  OH  TYR A 131     1518   4119   2710    694   -300   -338       O  \\nATOM    762  N   ALA A 132      17.315  23.993  -1.657  1.00 12.61           N  \\nANISOU  762  N   ALA A 132     1427   1502   1859    170   -164   -406       N  \\nATOM    763  CA  ALA A 132      18.717  23.839  -1.338  1.00 12.72           C  \\nANISOU  763  CA  ALA A 132     1327   1536   1969    160     49   -246       C  \\nATOM    764  C   ALA A 132      18.978  23.979   0.149  1.00 11.82           C  \\nANISOU  764  C   ALA A 132     1232   1333   1925     92     12    -95       C  \\nATOM    765  O   ALA A 132      18.273  24.722   0.865  1.00 12.64           O  \\nANISOU  765  O   ALA A 132     1481   1512   1810    251   -122   -360       O  \\nATOM    766  CB  ALA A 132      19.531  24.882  -2.058  1.00 13.38           C  \\nANISOU  766  CB  ALA A 132     1746   1753   1584     47   -121    -35       C  \\nATOM    767  N   ASP A 133      20.065  23.376   0.613  1.00 12.30           N  \\nANISOU  767  N   ASP A 133     1274   1559   1841    143    -70   -154       N  \\nATOM    768  CA  ASP A 133      20.536  23.667   1.949  1.00 11.87           C  \\nANISOU  768  CA  ASP A 133     1458   1346   1705    158     70   -170       C  \\nATOM    769  C   ASP A 133      20.681  25.186   2.140  1.00 12.02           C  \\nANISOU  769  C   ASP A 133     1191   1431   1944   -152    143   -251       C  \\nATOM    770  O   ASP A 133      21.067  25.895   1.216  1.00 13.85           O  \\nANISOU  770  O   ASP A 133     1663   1566   2031    -29    -12    -33       O  \\nATOM    771  CB  ASP A 133      21.854  23.001   2.254  1.00 12.70           C  \\nANISOU  771  CB  ASP A 133     1576   1378   1869    247      9   -278       C  \\nATOM    772  CG  ASP A 133      21.742  21.485   2.417  1.00 12.76           C  \\nANISOU  772  CG  ASP A 133     1467   1371   2008     83     98   -252       C  \\nATOM    773  OD1 ASP A 133      21.290  20.827   1.454  1.00 14.08           O  \\nANISOU  773  OD1 ASP A 133     1685   1433   2231    139   -273   -208       O  \\nATOM    774  OD2 ASP A 133      22.141  21.003   3.506  1.00 15.29           O  \\nANISOU  774  OD2 ASP A 133     2049   1747   2014    -85    127     30       O  \\nATOM    775  N   GLY A 134      20.332  25.642   3.340  1.00 10.96           N  \\nANISOU  775  N   GLY A 134     1399   1003   1762   -121    161    -43       N  \\nATOM    776  CA  GLY A 134      20.339  27.057   3.637  1.00 10.82           C  \\nANISOU  776  CA  GLY A 134     1253   1040   1819   -187   -170    -53       C  \\nATOM    777  C   GLY A 134      19.004  27.738   3.519  1.00  9.88           C  \\nANISOU  777  C   GLY A 134     1415    891   1448     31    -66   -108       C  \\nATOM    778  O   GLY A 134      18.798  28.853   4.032  1.00 11.29           O  \\nANISOU  778  O   GLY A 134     1642    926   1721      0     74   -254       O  \\nATOM    779  N   THR A 135      18.082  27.125   2.795  1.00 11.09           N  \\nANISOU  779  N   THR A 135     1341   1208   1665     97   -152   -250       N  \\nATOM    780  CA  THR A 135      16.739  27.674   2.656  1.00 10.56           C  \\nANISOU  780  CA  THR A 135     1376   1187   1449    110    -65     -1       C  \\nATOM    781  C   THR A 135      16.050  27.604   3.998  1.00 10.53           C  \\nANISOU  781  C   THR A 135     1383   1205   1413    200   -145   -198       C  \\nATOM    782  O   THR A 135      16.170  26.577   4.695  1.00 11.21           O  \\nANISOU  782  O   THR A 135     1602   1099   1557     16    -31    -51       O  \\nATOM    783  CB  THR A 135      15.945  26.902   1.590  1.00 12.03           C  \\nANISOU  783  CB  THR A 135     1581   1321   1669    102   -155   -138       C  \\nATOM    784  OG1 THR A 135      16.666  26.850   0.378  1.00 14.91           O  \\nANISOU  784  OG1 THR A 135     1917   1931   1815    349    -53   -235       O  \\nATOM    785  CG2 THR A 135      14.611  27.569   1.326  1.00 13.10           C  \\nANISOU  785  CG2 THR A 135     1671   1563   1742    296    -79   -199       C  \\nATOM    786  N   ARG A 136      15.302  28.662   4.340  1.00  9.86           N  \\nANISOU  786  N   ARG A 136     1301   1033   1409    157    -63    -23       N  \\nATOM    787  CA  ARG A 136      14.577  28.745   5.593  1.00  9.69           C  \\nANISOU  787  CA  ARG A 136     1440   1029   1211    -62    -69     68       C  \\nATOM    788  C   ARG A 136      13.087  28.505   5.333  1.00 10.23           C  \\nANISOU  788  C   ARG A 136     1387   1133   1367    126     15     61       C  \\nATOM    789  O   ARG A 136      12.468  29.264   4.610  1.00 10.95           O  \\nANISOU  789  O   ARG A 136     1502   1044   1615    112   -105     52       O  \\nATOM    790  CB  ARG A 136      14.816  30.131   6.213  1.00 10.70           C  \\nANISOU  790  CB  ARG A 136     1626   1055   1384      2   -138      3       C  \\nATOM    791  CG  ARG A 136      16.298  30.375   6.506  1.00 14.43           C  \\nANISOU  791  CG  ARG A 136     1925   1552   2003    -19   -687   -282       C  \\nATOM    792  CD  ARG A 136      16.600  31.799   6.739  1.00 15.29           C  \\nANISOU  792  CD  ARG A 136     2035   1635   2139   -348   -395   -140       C  \\nATOM    793  NE  ARG A 136      16.349  32.581   5.539  1.00 14.36           N  \\nANISOU  793  NE  ARG A 136     1731   1769   1956   -340    150      8       N  \\nATOM    794  CZ  ARG A 136      16.289  33.907   5.517  1.00 18.41           C  \\nANISOU  794  CZ  ARG A 136     2513   1914   2566   -378   -230   -411       C  \\nATOM    795  NH1 ARG A 136      16.557  34.594   6.614  1.00 21.09           N  \\nANISOU  795  NH1 ARG A 136     2950   2528   2534   -415   -236   -550       N  \\nATOM    796  NH2 ARG A 136      15.990  34.510   4.377  1.00 23.25           N  \\nANISOU  796  NH2 ARG A 136     3904   2040   2887    305   -763   -486       N  \\nATOM    797  N   ILE A 137      12.556  27.411   5.875  1.00  9.33           N  \\nANISOU  797  N   ILE A 137     1097   1173   1275     15    -65    -27       N  \\nATOM    798  CA  ILE A 137      11.172  27.020   5.676  1.00  9.16           C  \\nANISOU  798  CA  ILE A 137     1173   1144   1160    -51   -142    -62       C  \\nATOM    799  C   ILE A 137      10.399  27.182   6.973  1.00  9.81           C  \\nANISOU  799  C   ILE A 137     1297   1157   1271      6    -57    -16       C  \\nATOM    800  O   ILE A 137      10.699  26.528   7.953  1.00 10.75           O  \\nANISOU  800  O   ILE A 137     1237   1510   1336     88    -76    117       O  \\nATOM    801  CB  ILE A 137      11.098  25.575   5.168  1.00 10.64           C  \\nANISOU  801  CB  ILE A 137     1317   1192   1531    -23   -124   -183       C  \\nATOM    802  CG1 ILE A 137      11.918  25.390   3.855  1.00 13.68           C  \\nANISOU  802  CG1 ILE A 137     1744   1806   1646   -193     42   -431       C  \\nATOM    803  CG2 ILE A 137       9.660  25.113   4.960  1.00 12.91           C  \\nANISOU  803  CG2 ILE A 137     1468   1576   1859   -375     76   -262       C  \\nATOM    804  CD1 ILE A 137      11.447  26.173   2.648  1.00 17.58           C  \\nANISOU  804  CD1 ILE A 137     2501   2579   1598   -359    -87   -295       C  \\nATOM    805  N   LEU A 138       9.344  28.017   6.923  1.00 10.00           N  \\nANISOU  805  N   LEU A 138     1310   1189   1298    116    -23     30       N  \\nATOM    806  CA  LEU A 138       8.411  28.131   8.041  1.00 10.19           C  \\nANISOU  806  CA  LEU A 138     1350   1217   1303     -6    -35    -65       C  \\nATOM    807  C   LEU A 138       7.403  26.987   7.967  1.00  9.61           C  \\nANISOU  807  C   LEU A 138     1316   1237   1097     21    -22    -51       C  \\nATOM    808  O   LEU A 138       6.697  26.854   6.957  1.00 12.39           O  \\nANISOU  808  O   LEU A 138     1621   1699   1387   -110   -391    218       O  \\nATOM    809  CB  LEU A 138       7.676  29.466   7.957  1.00 12.22           C  \\nANISOU  809  CB  LEU A 138     1605   1503   1535    250    -72      7       C  \\nATOM    810  CG  LEU A 138       6.581  29.648   9.000  1.00 13.38           C  \\nANISOU  810  CG  LEU A 138     1536   1619   1929    251     50   -163       C  \\nATOM    811  CD1 LEU A 138       7.101  29.722  10.386  1.00 14.98           C  \\nANISOU  811  CD1 LEU A 138     2208   1719   1764    208    197   -338       C  \\nATOM    812  CD2 LEU A 138       5.796  30.926   8.681  1.00 15.40           C  \\nANISOU  812  CD2 LEU A 138     1926   1935   1989    560   -161   -236       C  \\nATOM    813  N   PHE A 139       7.340  26.201   9.047  1.00  9.98           N  \\nANISOU  813  N   PHE A 139     1344   1262   1184    -19   -159      8       N  \\nATOM    814  CA  PHE A 139       6.334  25.151   9.148  1.00 10.43           C  \\nANISOU  814  CA  PHE A 139     1361   1118   1485      6   -178    -37       C  \\nATOM    815  C   PHE A 139       5.121  25.707   9.865  1.00 11.82           C  \\nANISOU  815  C   PHE A 139     1480   1504   1505     68   -146   -126       C  \\nATOM    816  O   PHE A 139       5.267  26.324  10.915  1.00 12.80           O  \\nANISOU  816  O   PHE A 139     1559   1836   1465    245   -102   -260       O  \\nATOM    817  CB  PHE A 139       6.859  23.953   9.938  1.00 11.48           C  \\nANISOU  817  CB  PHE A 139     1463   1380   1516     49   -188    147       C  \\nATOM    818  CG  PHE A 139       7.660  22.970   9.115  1.00 12.52           C  \\nANISOU  818  CG  PHE A 139     1409   1621   1727    156   -401    -84       C  \\nATOM    819  CD1 PHE A 139       8.838  23.329   8.475  1.00 13.68           C  \\nANISOU  819  CD1 PHE A 139     1646   1674   1875    296   -111     34       C  \\nATOM    820  CD2 PHE A 139       7.255  21.621   9.088  1.00 15.98           C  \\nANISOU  820  CD2 PHE A 139     1562   1851   2659    -68   -376   -693       C  \\nATOM    821  CE1 PHE A 139       9.552  22.396   7.715  1.00 14.85           C  \\nANISOU  821  CE1 PHE A 139     1603   1715   2323    347     60     13       C  \\nATOM    822  CE2 PHE A 139       7.977  20.671   8.379  1.00 18.52           C  \\nANISOU  822  CE2 PHE A 139     1496   2233   3305    262   -168   -642       C  \\nATOM    823  CZ  PHE A 139       9.117  21.072   7.676  1.00 16.51           C  \\nANISOU  823  CZ  PHE A 139     1623   1982   2665    225   -154   -684       C  \\nATOM    824  N   ILE A 140       3.946  25.457   9.308  1.00 11.60           N  \\nANISOU  824  N   ILE A 140     1387   1728   1289    -40     40   -270       N  \\nATOM    825  CA  ILE A 140       2.682  25.887   9.872  1.00 13.47           C  \\nANISOU  825  CA  ILE A 140     1342   2203   1571     98    -54   -283       C  \\nATOM    826  C   ILE A 140       1.814  24.660  10.106  1.00 13.09           C  \\nANISOU  826  C   ILE A 140     1177   2155   1638    111   -156   -244       C  \\nATOM    827  O   ILE A 140       1.565  23.920   9.163  1.00 13.54           O  \\nANISOU  827  O   ILE A 140     1533   2073   1538    -89    -88    -58       O  \\nATOM    828  CB  ILE A 140       1.987  26.868   8.917  1.00 16.68           C  \\nANISOU  828  CB  ILE A 140     1534   2127   2674    346   -272   -127       C  \\nATOM    829  CG1 ILE A 140       2.860  28.093   8.725  1.00 19.80           C  \\nANISOU  829  CG1 ILE A 140     1952   2020   3550    341      0     78       C  \\nATOM    830  CG2 ILE A 140       0.604  27.198   9.414  1.00 18.09           C  \\nANISOU  830  CG2 ILE A 140     1661   2251   2960    459   -191    150       C  \\nATOM    831  CD1 ILE A 140       2.270  29.181   7.878  1.00 24.51           C  \\nANISOU  831  CD1 ILE A 140     2643   2757   3912     88   -680    437       C  \\nATOM    832  N   VAL A 141       1.362  24.441  11.339  1.00 14.51           N  \\nANISOU  832  N   VAL A 141     1553   2426   1530     30   -323   -357       N  \\nATOM    833  CA  VAL A 141       0.448  23.346  11.711  1.00 16.70           C  \\nANISOU  833  CA  VAL A 141     1708   2785   1852   -248   -629   -133       C  \\nATOM    834  C   VAL A 141      -0.652  24.007  12.523  1.00 22.92           C  \\nANISOU  834  C   VAL A 141     2466   3590   2651    115    310    264       C  \\nATOM    835  O   VAL A 141      -1.796  23.784  12.263  1.00 37.89           O  \\nANISOU  835  O   VAL A 141     2763   5648   5984    178    322   -131       O  \\nATOM    836  CB  VAL A 141       1.086  22.283  12.643  1.00 23.19           C  \\nANISOU  836  CB  VAL A 141     3136   2974   2699     66   -536    546       C  \\nATOM    837  CG1 VAL A 141       0.316  20.973  12.582  1.00 25.66           C  \\nANISOU  837  CG1 VAL A 141     3028   3485   3237   -224   -896    553       C  \\nATOM    838  CG2 VAL A 141       2.551  22.087  12.399  1.00 26.05           C  \\nANISOU  838  CG2 VAL A 141     2941   3695   3260    229  -1211    -49       C  \\nATOM    839  N   GLN A 142      -0.202  24.729  13.537  1.00 30.17           N  \\nANISOU  839  N   GLN A 142     4351   4668   2442   -142    972   -386       N  \\nATOM    840  CA  GLN A 142      -0.849  25.863  14.249  1.00 43.44           C  \\nANISOU  840  CA  GLN A 142     5780   5341   5383    836    992   -581       C  \\nATOM    841  C   GLN A 142      -1.266  25.367  15.610  1.00 53.97           C  \\nANISOU  841  C   GLN A 142     6968   7519   6017    338    569   1055       C  \\nATOM    842  O   GLN A 142      -2.348  25.603  16.175  1.00 73.17           O  \\nANISOU  842  O   GLN A 142     7022  11526   9254   1465    999    695       O  \\nATOM    843  CB  GLN A 142      -1.971  26.585  13.489  1.00 51.01           C  \\nANISOU  843  CB  GLN A 142     7039   5947   6394    728    162    714       C  \\nATOM    844  CG  GLN A 142      -1.444  27.425  12.340  1.00 51.77           C  \\nANISOU  844  CG  GLN A 142     6756   6371   6540    543   1032    322       C  \\nATOM    845  CD  GLN A 142      -2.436  27.561  11.197  1.00 56.41           C  \\nANISOU  845  CD  GLN A 142     7133   7428   6871    597    522   -176       C  \\nATOM    846  OE1 GLN A 142      -3.152  26.612  10.853  1.00 68.90           O  \\nANISOU  846  OE1 GLN A 142     5619   8561  11997   1270    196  -1772       O  \\nATOM    847  NE2 GLN A 142      -2.473  28.741  10.588  1.00 44.85           N  \\nANISOU  847  NE2 GLN A 142     5119   5999   5921    714    934  -1790       N  \\nATOM    848  OXT GLN A 142      -0.406  24.689  16.162  1.00 41.66           O  \\nANISOU  848  OXT GLN A 142     6097   5135   4596   -391    602    308       O  \\nTER     849      GLN A 142                                                      \\nHETATM  850  O   HOH A 201      16.072  12.445  12.809  1.00 32.78           O  \\nANISOU  850  O   HOH A 201     3763   3016   5674    208    112   -239       O  \\nHETATM  851  O   HOH A 202      -3.461  23.570  13.871  1.00 38.54           O  \\nANISOU  851  O   HOH A 202     4713   5079   4851   -399    481   -248       O  \\nHETATM  852  O   HOH A 203       4.720  30.032  13.736  1.00 27.49           O  \\nANISOU  852  O   HOH A 203     3630   3243   3569    639    542     90       O  \\nHETATM  853  O   HOH A 204      17.540  13.609   8.791  1.00 30.95           O  \\nANISOU  853  O   HOH A 204     3823   3464   4472   1259   -221   -640       O  \\nHETATM  854  O   HOH A 205      -0.706  30.223   1.739  1.00 33.97           O  \\nANISOU  854  O   HOH A 205     4577   3928   4401   -361  -1438    768       O  \\nHETATM  855  O   HOH A 206       9.791  14.149  12.055  1.00 25.97           O  \\nANISOU  855  O   HOH A 206     3108   4134   2625   -457    -13   -514       O  \\nHETATM  856  O   HOH A 207       1.888  25.492  16.683  1.00 20.22           O  \\nANISOU  856  O   HOH A 207     2714   1804   3163    361   -280    -17       O  \\nHETATM  857  O   HOH A 208      -2.781  16.073   5.175  1.00 23.42           O  \\nANISOU  857  O   HOH A 208     2574   2223   4100    329  -1207   -107       O  \\nHETATM  858  O   HOH A 209      -0.808  22.151  19.753  1.00 33.56           O  \\nANISOU  858  O   HOH A 209     4257   4942   3552   1703   1388   1101       O  \\nHETATM  859  O   HOH A 210      23.504   9.909  15.510  1.00 34.86           O  \\nANISOU  859  O   HOH A 210     5222   2561   5463    298    -37    936       O  \\nHETATM  860  O   HOH A 211      14.469  10.626  15.427  1.00 29.66           O  \\nANISOU  860  O   HOH A 211     5111   2890   3267   -145    762   -369       O  \\nHETATM  861  O   HOH A 212       3.158   1.329  16.175  1.00 31.50           O  \\nANISOU  861  O   HOH A 212     3677   3446   4847  -1451   -989    686       O  \\nHETATM  862  O   HOH A 213      14.908  32.975  10.694  1.00 31.76           O  \\nANISOU  862  O   HOH A 213     3750   5383   2933    618   -999      4       O  \\nHETATM  863  O   HOH A 214      10.839   7.700   6.986  1.00 26.51           O  \\nANISOU  863  O   HOH A 214     3206   2844   4021    294   -732    370       O  \\nHETATM  864  O   HOH A 215      24.368  22.251   4.457  1.00 17.78           O  \\nANISOU  864  O   HOH A 215     1687   2308   2758    106   -348   -137       O  \\nHETATM  865  O   HOH A 216      16.914  14.945  -1.038  1.00 34.36           O  \\nANISOU  865  O   HOH A 216     3175   1563   8314  -1160    337   1236       O  \\nHETATM  866  O   HOH A 217      13.464  34.647   0.395  1.00 27.91           O  \\nANISOU  866  O   HOH A 217     2896   3347   4360   -325   -168   2221       O  \\nHETATM  867  O   HOH A 218       4.178  27.982  12.709  1.00 30.97           O  \\nANISOU  867  O   HOH A 218     3013   5670   3083   1866   -178  -2218       O  \\nHETATM  868  O   HOH A 219      -1.289   8.103  12.619  1.00 29.28           O  \\nANISOU  868  O   HOH A 219     3141   2584   5400   -964   -383    418       O  \\nHETATM  869  O   HOH A 220      -0.397  16.592  21.384  1.00 30.14           O  \\nANISOU  869  O   HOH A 220     3324   4517   3609   1035    515   -674       O  \\nHETATM  870  O   HOH A 221       1.135  20.720   2.443  1.00 19.47           O  \\nANISOU  870  O   HOH A 221     2827   2651   1920    120     59   -102       O  \\nHETATM  871  O   HOH A 222       3.774  26.227   0.588  1.00 38.84           O  \\nANISOU  871  O   HOH A 222     5627   6465   2665  -1404  -1173    795       O  \\nHETATM  872  O   HOH A 223       8.026  28.642  18.545  1.00 13.27           O  \\nANISOU  872  O   HOH A 223     1796   1718   1527    -23   -101   -227       O  \\nHETATM  873  O   HOH A 224       5.751  28.590  25.338  1.00 29.36           O  \\nANISOU  873  O   HOH A 224     3661   3634   3861    357    196   -396       O  \\nHETATM  874  O   HOH A 225      22.203  25.686   6.288  1.00 18.66           O  \\nANISOU  874  O   HOH A 225     2311   2068   2710   -458   -499    137       O  \\nHETATM  875  O   HOH A 226      16.926  22.028  19.155  1.00 15.42           O  \\nANISOU  875  O   HOH A 226     2170   1620   2066    -41   -397    -95       O  \\nHETATM  876  O   HOH A 227      -0.836  22.407   8.306  1.00 24.94           O  \\nANISOU  876  O   HOH A 227     2290   3537   3646   -161   -998   -141       O  \\nHETATM  877  O   HOH A 228      16.492  16.599  23.166  1.00 20.68           O  \\nANISOU  877  O   HOH A 228     2712   2417   2726    631  -1125    -20       O  \\nHETATM  878  O   HOH A 229      10.740   7.732  12.092  1.00 23.12           O  \\nANISOU  878  O   HOH A 229     3673   1908   3201   -215    366     74       O  \\nHETATM  879  O   HOH A 230      15.104  29.601  13.170  1.00 28.76           O  \\nANISOU  879  O   HOH A 230     4049   3421   3455    480   -318  -1521       O  \\nHETATM  880  O   HOH A 231       3.263  23.022  24.032  1.00 24.10           O  \\nANISOU  880  O   HOH A 231     4046   3846   1265   -256    832   -273       O  \\nHETATM  881  O   HOH A 232      18.204  13.929  24.992  1.00 30.09           O  \\nANISOU  881  O   HOH A 232     5447   3267   2718    552   -821    456       O  \\nHETATM  882  O   HOH A 233      15.048  19.640  -6.556  1.00 20.81           O  \\nANISOU  882  O   HOH A 233     2183   2233   3490    -83    563   -373       O  \\nHETATM  883  O   HOH A 234      11.699  25.820  23.720  1.00 17.71           O  \\nANISOU  883  O   HOH A 234     2473   2306   1950   -143   -366   -425       O  \\nHETATM  884  O   HOH A 235       5.091   9.419   3.166  1.00 30.28           O  \\nANISOU  884  O   HOH A 235     4821   3669   3012  -1389  -1450   -325       O  \\nHETATM  885  O   HOH A 236       1.821  13.127  19.026  1.00 15.51           O  \\nANISOU  885  O   HOH A 236     2229   2211   1450    406     17    227       O  \\nHETATM  886  O   HOH A 237       9.159  16.556  14.003  1.00 17.08           O  \\nANISOU  886  O   HOH A 237     2526   1621   2340    -39    -99    113       O  \\nHETATM  887  O   HOH A 238      10.039  21.296  25.088  1.00 33.21           O  \\nANISOU  887  O   HOH A 238     6336   3541   2739    250   1340    636       O  \\nHETATM  888  O   HOH A 239      -2.531  16.382  14.607  1.00 25.63           O  \\nANISOU  888  O   HOH A 239     2728   3401   3609   1128  -1073   -563       O  \\nHETATM  889  O   HOH A 240      23.242  25.115  -0.309  1.00 25.33           O  \\nANISOU  889  O   HOH A 240     3065   3076   3481    639    803    282       O  \\nHETATM  890  O   HOH A 241      -0.040   8.199   9.475  1.00 32.67           O  \\nANISOU  890  O   HOH A 241     4288   3271   4851    112  -1905    -58       O  \\nHETATM  891  O   HOH A 242      11.468   4.392  20.223  1.00 24.94           O  \\nANISOU  891  O   HOH A 242     2955   2079   4440   -260   -996   1032       O  \\nHETATM  892  O   HOH A 243      14.898  23.577  20.270  1.00 15.70           O  \\nANISOU  892  O   HOH A 243     2249   1757   1959    -53   -561   -231       O  \\nHETATM  893  O   HOH A 244      22.757  22.392  14.937  1.00 43.05           O  \\nANISOU  893  O   HOH A 244     3486   5789   7082    371  -2216  -3794       O  \\nHETATM  894  O   HOH A 245      18.667  30.834   9.570  1.00 27.14           O  \\nANISOU  894  O   HOH A 245     4430   2292   3588   -717   -402   -354       O  \\nHETATM  895  O   HOH A 246      17.023  25.782  18.004  1.00 27.14           O  \\nANISOU  895  O   HOH A 246     4492   2650   3170   -690    346   -805       O  \\nHETATM  896  O   HOH A 247       8.046   0.035  14.891  1.00 30.20           O  \\nANISOU  896  O   HOH A 247     5763   2139   3570    579   -144    254       O  \\nHETATM  897  O   HOH A 248       8.703  16.604  24.118  1.00 23.45           O  \\nANISOU  897  O   HOH A 248     2999   3397   2511    -68    -54    281       O  \\nHETATM  898  O   HOH A 249      16.986  17.529  16.105  1.00 14.10           O  \\nANISOU  898  O   HOH A 249     1550   1496   2311     31   -494   -116       O  \\nHETATM  899  O   HOH A 250      11.761  29.680  18.688  1.00 21.64           O  \\nANISOU  899  O   HOH A 250     3858   2302   2060   -883    140   -635       O  \\nHETATM  900  O   HOH A 251      18.781  17.698  24.105  1.00 25.53           O  \\nANISOU  900  O   HOH A 251     3199   3773   2727   -578   -713    807       O  \\nHETATM  901  O   HOH A 252       7.706  36.673  10.812  1.00 33.86           O  \\nANISOU  901  O   HOH A 252     6313   3042   3508   1431   -720   -270       O  \\nHETATM  902  O   HOH A 253      20.506  28.414   0.094  1.00 24.78           O  \\nANISOU  902  O   HOH A 253     4483   2336   2596    751    177    -37       O  \\nHETATM  903  O   HOH A 254      12.723  24.262  13.183  1.00 13.88           O  \\nANISOU  903  O   HOH A 254     2166   1493   1613    186   -208   -280       O  \\nHETATM  904  O   HOH A 255       2.598  10.572  19.644  1.00 18.30           O  \\nANISOU  904  O   HOH A 255     2518   1861   2573     49     27    576       O  \\nHETATM  905  O   HOH A 256       2.265  26.089  13.658  1.00 33.42           O  \\nANISOU  905  O   HOH A 256     2719   6246   3731   -741    405  -3000       O  \\nHETATM  906  O   HOH A 257       7.454  20.274  23.127  1.00 20.28           O  \\nANISOU  906  O   HOH A 257     3463   2510   1730    789    258    684       O  \\nHETATM  907  O   HOH A 258      11.910  11.238  13.304  1.00 26.47           O  \\nANISOU  907  O   HOH A 258     3006   3169   3881   -213    520   -534       O  \\nHETATM  908  O   HOH A 259      19.213  28.738  12.435  1.00 31.47           O  \\nANISOU  908  O   HOH A 259     3943   3328   4687   -363   -792    193       O  \\nHETATM  909  O   HOH A 260      24.402  13.298   8.952  1.00 38.57           O  \\nANISOU  909  O   HOH A 260     5443   3034   6176   1520   3487   1091       O  \\nHETATM  910  O   HOH A 261      13.079   4.068  16.144  1.00 23.20           O  \\nANISOU  910  O   HOH A 261     2634   2596   3582    160   -139   -171       O  \\nHETATM  911  O   HOH A 262       6.214  14.758  -0.208  1.00 19.80           O  \\nANISOU  911  O   HOH A 262     2431   3217   1873   -432   -250   -123       O  \\nHETATM  912  O   HOH A 263      23.271  19.734  18.796  1.00 39.14           O  \\nANISOU  912  O   HOH A 263     2750   7717   4402  -1641   -475   -697       O  \\nHETATM  913  O   HOH A 264       2.836  23.614   1.991  1.00 21.18           O  \\nANISOU  913  O   HOH A 264     1936   4404   1707     34    -49   -350       O  \\nHETATM  914  O   HOH A 265      12.482  21.621  12.225  1.00 13.37           O  \\nANISOU  914  O   HOH A 265     2166   1325   1590    355   -262     30       O  \\nHETATM  915  O   HOH A 266      13.672   6.473  17.276  1.00 20.90           O  \\nANISOU  915  O   HOH A 266     2571   2386   2984    492    397   -143       O  \\nHETATM  916  O   HOH A 267      14.689  14.637  12.770  1.00 17.31           O  \\nANISOU  916  O   HOH A 267     2330   1473   2773   -249   -359    -51       O  \\nHETATM  917  O   HOH A 268      13.051  13.763   6.369  1.00 22.40           O  \\nANISOU  917  O   HOH A 268     2331   3322   2857   -339   -239   -398       O  \\nHETATM  918  O   HOH A 269      16.346  26.483  -2.777  1.00 31.51           O  \\nANISOU  918  O   HOH A 269     4277   2709   4983    982  -1557    158       O  \\nHETATM  919  O   HOH A 270       6.127  15.898  23.802  1.00 30.32           O  \\nANISOU  919  O   HOH A 270     3865   4290   3365   -456   -413   -824       O  \\nHETATM  920  O   HOH A 271       2.996   2.102  13.555  1.00 25.82           O  \\nANISOU  920  O   HOH A 271     3884   2270   3654  -1102   -503    286       O  \\nHETATM  921  O   HOH A 272      14.438  25.854  18.825  1.00 18.08           O  \\nANISOU  921  O   HOH A 272     2326   2224   2317   -171   -624    -65       O  \\nHETATM  922  O   HOH A 273      11.956  16.571  24.846  1.00 25.64           O  \\nANISOU  922  O   HOH A 273     4536   3568   1638   -430   -436   -279       O  \\nHETATM  923  O   HOH A 274       8.709   0.860   8.438  1.00 29.62           O  \\nANISOU  923  O   HOH A 274     4409   2369   4474    486  -1324    -68       O  \\nHETATM  924  O   HOH A 275      27.033  20.327   9.682  1.00 36.23           O  \\nANISOU  924  O   HOH A 275     3657   5369   4740   -798   -508    414       O  \\nHETATM  925  O   HOH A 276      10.175  11.601  10.654  1.00 24.34           O  \\nANISOU  925  O   HOH A 276     3553   3257   2435   -284   -748    -14       O  \\nHETATM  926  O   HOH A 277       6.709  10.890  24.386  1.00 22.85           O  \\nANISOU  926  O   HOH A 277     4266   2440   1976    195    405    388       O  \\nHETATM  927  O   HOH A 278      26.310  14.622  11.205  1.00 28.28           O  \\nANISOU  927  O   HOH A 278     2395   4380   3970    832   1142   1348       O  \\nHETATM  928  O   HOH A 279      14.073  26.336  -4.385  1.00 31.42           O  \\nANISOU  928  O   HOH A 279     4903   3856   3180    491    239  -1028       O  \\nHETATM  929  O   HOH A 280      16.377  37.438   3.806  1.00 26.37           O  \\nANISOU  929  O   HOH A 280     3539   3334   3145   -962   -427    751       O  \\nHETATM  930  O   HOH A 281      12.100  14.073  13.451  1.00 28.03           O  \\nANISOU  930  O   HOH A 281     2371   5224   3053   -495    134  -1393       O  \\nHETATM  931  O   HOH A 282      17.746  33.433   9.224  1.00 26.35           O  \\nANISOU  931  O   HOH A 282     3158   4500   2352    305   -338   -781       O  \\nHETATM  932  O   HOH A 283      12.332  29.838  14.050  1.00 20.55           O  \\nANISOU  932  O   HOH A 283     3787   1940   2082   -168     76    -74       O  \\nHETATM  933  O   HOH A 284      15.492  15.212   9.923  1.00 16.33           O  \\nANISOU  933  O   HOH A 284     2209   1831   2163   -272     95   -152       O  \\nHETATM  934  O   HOH A 285      22.376  28.552   6.164  1.00 32.44           O  \\nANISOU  934  O   HOH A 285     4173   3476   4676   -414   -465   -198       O  \\nHETATM  935  O   HOH A 286      -4.524  16.181  12.875  1.00 36.39           O  \\nANISOU  935  O   HOH A 286     2979   5987   4860    -75     -7   2076       O  \\nHETATM  936  O   HOH A 287       9.518  12.846  -5.721  1.00 29.40           O  \\nANISOU  936  O   HOH A 287     3441   3644   4085    622     30   -211       O  \\nHETATM  937  O   HOH A 288      -2.563  12.458  16.875  1.00 31.88           O  \\nANISOU  937  O   HOH A 288     2880   4624   4607   -327    240    858       O  \\nHETATM  938  O   HOH A 289       6.385  30.464  17.119  1.00 22.22           O  \\nANISOU  938  O   HOH A 289     3974   2090   2378     28   -332   -146       O  \\nHETATM  939  O   HOH A 290      21.737   9.162  13.944  1.00 31.17           O  \\nANISOU  939  O   HOH A 290     5634   3125   3082   1100    399     12       O  \\nHETATM  940  O   HOH A 291      18.169  38.276   7.095  1.00 32.62           O  \\nANISOU  940  O   HOH A 291     4227   5713   2452     19     89   -168       O  \\nHETATM  941  O   HOH A 292       6.341  16.415  -2.481  1.00 32.31           O  \\nANISOU  941  O   HOH A 292     4621   3358   4297    -22  -1169   -434       O  \\nHETATM  942  O   HOH A 293       7.484  36.776   4.155  1.00 33.55           O  \\nANISOU  942  O   HOH A 293     4163   3127   5456     -4     84    859       O  \\nHETATM  943  O   HOH A 294      14.864   8.058  15.220  1.00 28.10           O  \\nANISOU  943  O   HOH A 294     3538   3461   3676   -667    -87    297       O  \\nHETATM  944  O   HOH A 295       1.707  28.226  16.735  1.00 26.12           O  \\nANISOU  944  O   HOH A 295     3503   2736   3686    142   -337   -272       O  \\nHETATM  945  O   HOH A 296       6.035  12.051  -0.584  1.00 37.96           O  \\nANISOU  945  O   HOH A 296     7336   3938   3146  -1766  -1978   -197       O  \\nHETATM  946  O   HOH A 297       3.744  30.089  16.276  1.00 23.14           O  \\nANISOU  946  O   HOH A 297     3008   2866   2917     52    -53   -145       O  \\nHETATM  947  O   HOH A 298      11.678  19.277  25.632  1.00 40.33           O  \\nANISOU  947  O   HOH A 298     6293   4875   4155    665  -2141   -778       O  \\nHETATM  948  O   HOH A 299      -0.971  12.784  19.363  1.00 25.96           O  \\nANISOU  948  O   HOH A 299     3094   4024   2744   -139    607    431       O  \\nHETATM  949  O   HOH A 300      17.256   7.664  15.084  1.00 34.89           O  \\nANISOU  949  O   HOH A 300     4123   5276   3856   -862   -126  -1003       O  \\nHETATM  950  O   HOH A 301      24.424  24.955   4.816  1.00 29.40           O  \\nANISOU  950  O   HOH A 301     4605   3102   3461  -1011    450   -446       O  \\nHETATM  951  O   HOH A 302       8.778  16.481  11.241  1.00 24.68           O  \\nANISOU  951  O   HOH A 302     3200   2868   3306    169   -216   -479       O  \\nHETATM  952  O   HOH A 303       2.308  28.141  -0.112  1.00 36.10           O  \\nANISOU  952  O   HOH A 303     2592   5656   5467   -556   -325   -568       O  \\nHETATM  953  O   HOH A 304      13.049  15.218   8.691  1.00 20.84           O  \\nANISOU  953  O   HOH A 304     2631   2990   2295    -61   -478   -306       O  \\nCONECT  254  525                                                                \\nCONECT  525  254                                                                \\nMASTER      309    0    0    3    5    0    0    6  943    1    2    9          \\nEND                                                                             \\n\");\n\tviewer_17261593772982423.setStyle({\"model\": -1, \"chain\": [\"A\", \"B\"]},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_17261593772982423.zoomTo();\nviewer_17261593772982423.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17261593772982423\"  style=\"position: relative; width: 600px; height: 400px;\">\n",
              "        <p id=\"3dmolwarning_17261593772982423\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.0/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_17261593772982423 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17261593772982423\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17261593772982423 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17261593772982423\"),{backgroundColor:\"white\"});\n",
              "viewer_17261593772982423.zoomTo();\n",
              "\tviewer_17261593772982423.addModelsAsFrames(\"HEADER    HYDROLASE                               01-AUG-17   5OMT              \\nTITLE     ENDONUCLEASE NUCB                                                     \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: NUCB;                                                      \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: BACILLUS LICHENIFORMIS;                         \\nSOURCE   3 ORGANISM_TAXID: 1402;                                                \\nSOURCE   4 GENE: NUCB;                                                          \\nSOURCE   5 EXPRESSION_SYSTEM: BACILLUS SUBTILIS;                                \\nSOURCE   6 EXPRESSION_SYSTEM_TAXID: 1423                                        \\nKEYWDS    ENDONUCLEASE, BIOFILM, BACILLUS SUBTILIS, DNA, HYDROLASE              \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    A.BASLE,R.J.LEWIS                                                     \\nREVDAT   3   24-JAN-18 5OMT    1       JRNL                                     \\nREVDAT   2   06-DEC-17 5OMT    1       JRNL                                     \\nREVDAT   1   15-NOV-17 5OMT    0                                                \\nJRNL        AUTH   A.BASLE,L.HEWITT,A.KOH,H.K.LAMB,P.THOMPSON,J.G.BURGESS,      \\nJRNL        AUTH 2 M.J.HALL,A.R.HAWKINS,H.MURRAY,R.J.LEWIS                      \\nJRNL        TITL   CRYSTAL STRUCTURE OF NUCB, A BIOFILM-DEGRADING ENDONUCLEASE. \\nJRNL        REF    NUCLEIC ACIDS RES.            V.  46   473 2018              \\nJRNL        REFN                   ESSN 1362-4962                               \\nJRNL        PMID   29165717                                                     \\nJRNL        DOI    10.1093/NAR/GKX1170                                          \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : REFMAC 5.8.0158                                      \\nREMARK   3   AUTHORS     : MURSHUDOV,VAGIN,DODSON                               \\nREMARK   3                                                                      \\nREMARK   3    REFINEMENT TARGET : NULL                                          \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 44.56                          \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : 100.0                          \\nREMARK   3   NUMBER OF REFLECTIONS             : 25934                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : THROUGHOUT                      \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : RANDOM                          \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.150                           \\nREMARK   3   R VALUE            (WORKING SET) : 0.149                           \\nREMARK   3   FREE R VALUE                     : 0.186                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 5.000                           \\nREMARK   3   FREE R VALUE TEST SET COUNT      : 1369                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : 20                           \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : 1.35                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : 1.39                         \\nREMARK   3   REFLECTION IN BIN     (WORKING SET) : 1891                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : 100.0                        \\nREMARK   3   BIN R VALUE           (WORKING SET) : 0.1840                       \\nREMARK   3   BIN FREE R VALUE SET COUNT          : 85                           \\nREMARK   3   BIN FREE R VALUE                    : 0.2330                       \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 839                                     \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 0                                       \\nREMARK   3   SOLVENT ATOMS            : 104                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : 18.44                          \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : -0.79000                                             \\nREMARK   3    B22 (A**2) : -0.79000                                             \\nREMARK   3    B33 (A**2) : 1.58000                                              \\nREMARK   3    B12 (A**2) : 0.00000                                              \\nREMARK   3    B13 (A**2) : 0.00000                                              \\nREMARK   3    B23 (A**2) : 0.00000                                              \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED OVERALL COORDINATE ERROR.                                 \\nREMARK   3   ESU BASED ON R VALUE                            (A): 0.048         \\nREMARK   3   ESU BASED ON FREE R VALUE                       (A): 0.049         \\nREMARK   3   ESU BASED ON MAXIMUM LIKELIHOOD                 (A): 0.028         \\nREMARK   3   ESU FOR B VALUES BASED ON MAXIMUM LIKELIHOOD (A**2): 1.528         \\nREMARK   3                                                                      \\nREMARK   3 CORRELATION COEFFICIENTS.                                            \\nREMARK   3   CORRELATION COEFFICIENT FO-FC      : 0.969                         \\nREMARK   3   CORRELATION COEFFICIENT FO-FC FREE : 0.958                         \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES        COUNT    RMS    WEIGHT      \\nREMARK   3   BOND LENGTHS REFINED ATOMS        (A):   870 ; 0.014 ; 0.019       \\nREMARK   3   BOND LENGTHS OTHERS               (A):   780 ; 0.002 ; 0.020       \\nREMARK   3   BOND ANGLES REFINED ATOMS   (DEGREES):  1178 ; 1.676 ; 1.943       \\nREMARK   3   BOND ANGLES OTHERS          (DEGREES):  1808 ; 0.996 ; 3.000       \\nREMARK   3   TORSION ANGLES, PERIOD 1    (DEGREES):   114 ; 5.752 ; 5.000       \\nREMARK   3   TORSION ANGLES, PERIOD 2    (DEGREES):    42 ;35.634 ;22.381       \\nREMARK   3   TORSION ANGLES, PERIOD 3    (DEGREES):   140 ;12.034 ;15.000       \\nREMARK   3   TORSION ANGLES, PERIOD 4    (DEGREES):    10 ;10.784 ;15.000       \\nREMARK   3   CHIRAL-CENTER RESTRAINTS       (A**3):   119 ; 0.109 ; 0.200       \\nREMARK   3   GENERAL PLANES REFINED ATOMS      (A):  1005 ; 0.009 ; 0.020       \\nREMARK   3   GENERAL PLANES OTHERS             (A):   197 ; 0.002 ; 0.020       \\nREMARK   3   NON-BONDED CONTACTS REFINED ATOMS (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   NON-BONDED CONTACTS OTHERS        (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   NON-BONDED TORSION REFINED ATOMS  (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   NON-BONDED TORSION OTHERS         (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   H-BOND (X...Y) REFINED ATOMS      (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   H-BOND (X...Y) OTHERS             (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   POTENTIAL METAL-ION REFINED ATOMS (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   POTENTIAL METAL-ION OTHERS        (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY VDW REFINED ATOMS        (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY VDW OTHERS               (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY H-BOND REFINED ATOMS     (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY H-BOND OTHERS            (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY METAL-ION REFINED ATOMS  (A):  NULL ;  NULL ;  NULL       \\nREMARK   3   SYMMETRY METAL-ION OTHERS         (A):  NULL ;  NULL ;  NULL       \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.     COUNT   RMS    WEIGHT      \\nREMARK   3   MAIN-CHAIN BOND REFINED ATOMS  (A**2):   441 ; 2.302 ; 1.506       \\nREMARK   3   MAIN-CHAIN BOND OTHER ATOMS    (A**2):   440 ; 2.224 ; 1.494       \\nREMARK   3   MAIN-CHAIN ANGLE REFINED ATOMS (A**2):   551 ; 2.812 ; 2.261       \\nREMARK   3   MAIN-CHAIN ANGLE OTHER ATOMS   (A**2):   552 ; 2.835 ; 2.270       \\nREMARK   3   SIDE-CHAIN BOND REFINED ATOMS  (A**2):   429 ; 3.458 ; 1.949       \\nREMARK   3   SIDE-CHAIN BOND OTHER ATOMS    (A**2):   426 ; 3.343 ; 1.925       \\nREMARK   3   SIDE-CHAIN ANGLE REFINED ATOMS (A**2):  NULL ;  NULL ;  NULL       \\nREMARK   3   SIDE-CHAIN ANGLE OTHER ATOMS   (A**2):   623 ; 3.901 ; 2.757       \\nREMARK   3   LONG RANGE B REFINED ATOMS     (A**2):  1005 ; 4.521 ;19.441       \\nREMARK   3   LONG RANGE B OTHER ATOMS       (A**2):   984 ; 4.424 ;19.011       \\nREMARK   3                                                                      \\nREMARK   3 ANISOTROPIC THERMAL FACTOR RESTRAINTS.    COUNT   RMS   WEIGHT       \\nREMARK   3   RIGID-BOND RESTRAINTS          (A**2):  1650 ; 2.176 ; 3.000       \\nREMARK   3   SPHERICITY; FREE ATOMS         (A**2):    76 ;18.023 ; 5.000       \\nREMARK   3   SPHERICITY; BONDED ATOMS       (A**2):  1656 ;10.774 ; 5.000       \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS STATISTICS                                           \\nREMARK   3   NUMBER OF DIFFERENT NCS GROUPS : NULL                              \\nREMARK   3                                                                      \\nREMARK   3  TLS DETAILS                                                         \\nREMARK   3   NUMBER OF TLS GROUPS  : NULL                                       \\nREMARK   3                                                                      \\nREMARK   3  BULK SOLVENT MODELLING.                                             \\nREMARK   3   METHOD USED : NULL                                                 \\nREMARK   3   PARAMETERS FOR MASK CALCULATION                                    \\nREMARK   3   VDW PROBE RADIUS   : 1.20                                          \\nREMARK   3   ION PROBE RADIUS   : 0.80                                          \\nREMARK   3   SHRINKAGE RADIUS   : 0.80                                          \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: HYDROGENS HAVE BEEN ADDED IN THE RIDING   \\nREMARK   3  POSITIONS                                                           \\nREMARK   4                                                                      \\nREMARK   4 5OMT COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY PDBE ON 02-AUG-17.                  \\nREMARK 100 THE DEPOSITION ID IS D_1200006052.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 14-JUL-12                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \\nREMARK 200  RADIATION SOURCE               : DIAMOND                            \\nREMARK 200  BEAMLINE                       : I04                                \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : 0.9795                             \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : CCD                                \\nREMARK 200  DETECTOR MANUFACTURER          : ADSC QUANTUM 315                   \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : XDS, AIMLESS                       \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 27361                              \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 1.350                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : 44.560                             \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : 100.0                              \\nREMARK 200  DATA REDUNDANCY                : 7.800                              \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 16.1000                            \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 1.35                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 1.37                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : 100.0                              \\nREMARK 200  DATA REDUNDANCY IN SHELL       : 7.70                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : 3.600                              \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: SAD                          \\nREMARK 200 SOFTWARE USED: SHELXCD                                               \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 50.62                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.49                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: 100 MM MES PH6.5 12% PEG 20000, VAPOR    \\nREMARK 280  DIFFUSION, SITTING DROP, TEMPERATURE 293K                           \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 41 21 2                        \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X,-Y,Z+1/2                                             \\nREMARK 290       3555   -Y+1/2,X+1/2,Z+1/4                                      \\nREMARK 290       4555   Y+1/2,-X+1/2,Z+3/4                                      \\nREMARK 290       5555   -X+1/2,Y+1/2,-Z+1/4                                     \\nREMARK 290       6555   X+1/2,-Y+1/2,-Z+3/4                                     \\nREMARK 290       7555   Y,X,-Z                                                  \\nREMARK 290       8555   -Y,-X,-Z+1/2                                            \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       33.30200            \\nREMARK 290   SMTRY1   3  0.000000 -1.000000  0.000000       29.97750            \\nREMARK 290   SMTRY2   3  1.000000  0.000000  0.000000       29.97750            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       16.65100            \\nREMARK 290   SMTRY1   4  0.000000  1.000000  0.000000       29.97750            \\nREMARK 290   SMTRY2   4 -1.000000  0.000000  0.000000       29.97750            \\nREMARK 290   SMTRY3   4  0.000000  0.000000  1.000000       49.95300            \\nREMARK 290   SMTRY1   5 -1.000000  0.000000  0.000000       29.97750            \\nREMARK 290   SMTRY2   5  0.000000  1.000000  0.000000       29.97750            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       16.65100            \\nREMARK 290   SMTRY1   6  1.000000  0.000000  0.000000       29.97750            \\nREMARK 290   SMTRY2   6  0.000000 -1.000000  0.000000       29.97750            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000       49.95300            \\nREMARK 290   SMTRY1   7  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   8  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   8 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000       33.30200            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: MONOMERIC                  \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 0 ANGSTROM**2                             \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 6030 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: 0.0 KCAL/MOL                          \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 465                                                                      \\nREMARK 465 MISSING RESIDUES                                                     \\nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \\nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \\nREMARK 465                                                                      \\nREMARK 465   M RES C SSSEQI                                                     \\nREMARK 465     ALA A    33                                                      \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: CLOSE CONTACTS IN SAME ASYMMETRIC UNIT                     \\nREMARK 500                                                                      \\nREMARK 500 THE FOLLOWING ATOMS ARE IN CLOSE CONTACT.                            \\nREMARK 500                                                                      \\nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI           DISTANCE          \\nREMARK 500   NE   ARG A    77     O    HOH A   201              2.13            \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: CLOSE CONTACTS                                             \\nREMARK 500                                                                      \\nREMARK 500 THE FOLLOWING ATOMS THAT ARE RELATED BY CRYSTALLOGRAPHIC             \\nREMARK 500 SYMMETRY ARE IN CLOSE CONTACT.  AN ATOM LOCATED WITHIN 0.15          \\nREMARK 500 ANGSTROMS OF A SYMMETRY RELATED ATOM IS ASSUMED TO BE ON A           \\nREMARK 500 SPECIAL POSITION AND IS, THEREFORE, LISTED IN REMARK 375             \\nREMARK 500 INSTEAD OF REMARK 500.  ATOMS WITH NON-BLANK ALTERNATE               \\nREMARK 500 LOCATION INDICATORS ARE NOT INCLUDED IN THE CALCULATIONS.            \\nREMARK 500                                                                      \\nREMARK 500 DISTANCE CUTOFF:                                                     \\nREMARK 500 2.2 ANGSTROMS FOR CONTACTS NOT INVOLVING HYDROGEN ATOMS              \\nREMARK 500 1.6 ANGSTROMS FOR CONTACTS INVOLVING HYDROGEN ATOMS                  \\nREMARK 500                                                                      \\nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI  SSYMOP   DISTANCE          \\nREMARK 500   O    HOH A   224     O    HOH A   227     3555     1.94            \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    VAL A 141      -53.41   -130.99                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  5OMT A   33   142  UNP    F1BV52   F1BV52_BACLI    33    142             \\nSEQRES   1 A  110  ALA ALA ARG TYR ASP ASP VAL LEU TYR PHE PRO ALA SER          \\nSEQRES   2 A  110  ARG TYR PRO GLU THR GLY ALA HIS ILE SER ASP ALA ILE          \\nSEQRES   3 A  110  LYS ALA GLY HIS ALA ASP VAL CYS THR ILE GLU ARG SER          \\nSEQRES   4 A  110  GLY ALA ASP LYS ARG ARG GLN GLU SER LEU LYS GLY ILE          \\nSEQRES   5 A  110  PRO THR LYS PRO GLY PHE ASP ARG ASP GLU TRP PRO MET          \\nSEQRES   6 A  110  ALA MET CYS GLU GLU GLY GLY LYS GLY ALA SER VAL ARG          \\nSEQRES   7 A  110  TYR VAL SER SER SER ASP ASN ARG GLY ALA GLY SER TRP          \\nSEQRES   8 A  110  VAL GLY ASN ARG LEU ASN GLY TYR ALA ASP GLY THR ARG          \\nSEQRES   9 A  110  ILE LEU PHE ILE VAL GLN                                      \\nFORMUL   2  HOH   *104(H2 O)                                                    \\nHELIX    1 AA1 TYR A   47  ALA A   60  1                                  14    \\nHELIX    2 AA2 GLY A   72  LYS A   82  1                                  11    \\nHELIX    3 AA3 SER A  114  ASN A  129  1                                  16    \\nSHEET    1 AA1 3 ASP A  38  TYR A  41  0                                        \\nSHEET    2 AA1 3 ARG A 136  ILE A 140  1  O  LEU A 138   N  LEU A  40           \\nSHEET    3 AA1 3 VAL A  65  THR A  67 -1  N  CYS A  66   O  ILE A 137           \\nSHEET    1 AA2 2 PHE A  90  TRP A  95  0                                        \\nSHEET    2 AA2 2 SER A 108  SER A 113 -1  O  VAL A 112   N  ASP A  91           \\nSSBOND   1 CYS A   66    CYS A  100                          1555   1555  2.05  \\nCISPEP   1 TRP A   95    PRO A   96          0        -0.27                     \\nCRYST1   59.955   59.955   66.604  90.00  90.00  90.00 P 41 21 2     8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.016679  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.016679  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.015014        0.00000                         \\nATOM      1  N   ALA A  34      14.458  39.288   6.017  1.00 31.14           N  \\nANISOU    1  N   ALA A  34     4143   2682   5004   -276   1064   2656       N  \\nATOM      2  CA  ALA A  34      13.318  38.299   6.191  1.00 24.44           C  \\nANISOU    2  CA  ALA A  34     3739   2283   3262   -122     42   1126       C  \\nATOM      3  C   ALA A  34      13.580  37.130   7.127  1.00 25.95           C  \\nANISOU    3  C   ALA A  34     3588   2646   3624   -142    446   1610       C  \\nATOM      4  O   ALA A  34      14.736  36.794   7.371  1.00 25.95           O  \\nANISOU    4  O   ALA A  34     4013   2603   3241    411    660    350       O  \\nATOM      5  CB  ALA A  34      12.860  37.796   4.821  1.00 28.47           C  \\nANISOU    5  CB  ALA A  34     2807   4521   3486    255   -934    858       C  \\nATOM      6  N   ARG A  35      12.506  36.565   7.716  1.00 18.00           N  \\nANISOU    6  N   ARG A  35     3214   1382   2241    119    568    -72       N  \\nATOM      7  CA  ARG A  35      12.654  35.467   8.680  1.00 16.94           C  \\nANISOU    7  CA  ARG A  35     2667   1472   2295   -111   -178   -103       C  \\nATOM      8  C   ARG A  35      12.701  34.068   8.058  1.00 15.64           C  \\nANISOU    8  C   ARG A  35     2612   1649   1678    -14    -47    -75       C  \\nATOM      9  O   ARG A  35      13.272  33.117   8.613  1.00 16.57           O  \\nANISOU    9  O   ARG A  35     2498   1768   2028    -18   -133    255       O  \\nATOM     10  CB  ARG A  35      11.497  35.573   9.689  1.00 22.02           C  \\nANISOU   10  CB  ARG A  35     3437   2420   2506    -99    276   -503       C  \\nATOM     11  CG  ARG A  35      11.719  34.920  10.993  1.00 21.39           C  \\nANISOU   11  CG  ARG A  35     3098   2242   2787    187    192   -381       C  \\nATOM     12  CD  ARG A  35      10.696  35.325  12.058  1.00 20.47           C  \\nANISOU   12  CD  ARG A  35     3275   2455   2045   -196    230   -161       C  \\nATOM     13  NE  ARG A  35       9.381  34.700  11.909  1.00 18.84           N  \\nANISOU   13  NE  ARG A  35     3109   2243   1806    145     34   -249       N  \\nATOM     14  CZ  ARG A  35       9.044  33.472  12.332  1.00 19.49           C  \\nANISOU   14  CZ  ARG A  35     3214   2398   1791   -129    396   -415       C  \\nATOM     15  NH1 ARG A  35       9.925  32.690  12.910  1.00 20.87           N  \\nANISOU   15  NH1 ARG A  35     3561   2301   2066   -497    337    166       N  \\nATOM     16  NH2 ARG A  35       7.804  33.038  12.140  1.00 24.42           N  \\nANISOU   16  NH2 ARG A  35     3499   2915   2864   -424    412   -261       N  \\nATOM     17  N   TYR A  36      12.071  33.937   6.914  1.00 12.70           N  \\nANISOU   17  N   TYR A  36     2305   1146   1372    105    175     -8       N  \\nATOM     18  CA  TYR A  36      11.982  32.660   6.183  1.00 12.66           C  \\nANISOU   18  CA  TYR A  36     2240   1234   1335    364    267   -139       C  \\nATOM     19  C   TYR A  36      11.837  32.934   4.715  1.00 13.35           C  \\nANISOU   19  C   TYR A  36     2285   1338   1446    294    123    135       C  \\nATOM     20  O   TYR A  36      11.511  34.064   4.303  1.00 15.85           O  \\nANISOU   20  O   TYR A  36     3102   1346   1574    534    132     60       O  \\nATOM     21  CB  TYR A  36      10.830  31.796   6.721  1.00 14.81           C  \\nANISOU   21  CB  TYR A  36     2841   1274   1512    225    405    138       C  \\nATOM     22  CG  TYR A  36       9.509  32.488   6.731  1.00 14.38           C  \\nANISOU   22  CG  TYR A  36     2615   1488   1360    -95    564    134       C  \\nATOM     23  CD1 TYR A  36       8.685  32.490   5.630  1.00 17.53           C  \\nANISOU   23  CD1 TYR A  36     2934   1547   2178      9     20     11       C  \\nATOM     24  CD2 TYR A  36       9.081  33.149   7.878  1.00 14.85           C  \\nANISOU   24  CD2 TYR A  36     2462   1862   1316    -52    259   -128       C  \\nATOM     25  CE1 TYR A  36       7.463  33.160   5.661  1.00 19.34           C  \\nANISOU   25  CE1 TYR A  36     2511   2279   2556   -144      0    167       C  \\nATOM     26  CE2 TYR A  36       7.896  33.820   7.917  1.00 18.93           C  \\nANISOU   26  CE2 TYR A  36     2635   1791   2764    126    431   -436       C  \\nATOM     27  CZ  TYR A  36       7.094  33.845   6.803  1.00 21.32           C  \\nANISOU   27  CZ  TYR A  36     2442   2585   3071    230    234   -167       C  \\nATOM     28  OH  TYR A  36       5.866  34.508   6.901  1.00 27.25           O  \\nANISOU   28  OH  TYR A  36     2599   3687   4067    577    276   -895       O  \\nATOM     29  N   ASP A  37      12.113  31.908   3.927  1.00 11.65           N  \\nANISOU   29  N   ASP A  37     1743   1256   1425    140    149    146       N  \\nATOM     30  CA  ASP A  37      12.075  31.991   2.490  1.00 12.55           C  \\nANISOU   30  CA  ASP A  37     1894   1434   1438    229     46    273       C  \\nATOM     31  C   ASP A  37      10.817  31.403   1.857  1.00 13.91           C  \\nANISOU   31  C   ASP A  37     1829   1803   1653    152    129    267       C  \\nATOM     32  O   ASP A  37      10.380  31.858   0.801  1.00 17.76           O  \\nANISOU   32  O   ASP A  37     2052   2752   1944   -100   -369    513       O  \\nATOM     33  CB  ASP A  37      13.320  31.285   1.905  1.00 12.52           C  \\nANISOU   33  CB  ASP A  37     1866   1498   1392     22    203     84       C  \\nATOM     34  CG  ASP A  37      14.614  31.912   2.379  1.00 13.26           C  \\nANISOU   34  CG  ASP A  37     1894   1578   1565     33    134     71       C  \\nATOM     35  OD1 ASP A  37      14.776  33.141   2.167  1.00 15.66           O  \\nANISOU   35  OD1 ASP A  37     2210   1667   2072     84    -61    520       O  \\nATOM     36  OD2 ASP A  37      15.460  31.192   2.989  1.00 14.12           O  \\nANISOU   36  OD2 ASP A  37     1989   1361   2013    -13     83    267       O  \\nATOM     37  N   ASP A  38      10.249  30.389   2.481  1.00 13.41           N  \\nANISOU   37  N   ASP A  38     1722   1647   1724    164    -60    300       N  \\nATOM     38  CA  ASP A  38       9.029  29.737   2.038  1.00 13.80           C  \\nANISOU   38  CA  ASP A  38     1913   1822   1505    127   -180    156       C  \\nATOM     39  C   ASP A  38       8.311  29.177   3.223  1.00 11.72           C  \\nANISOU   39  C   ASP A  38     1635   1325   1493     88   -301     90       C  \\nATOM     40  O   ASP A  38       8.865  29.074   4.318  1.00 12.45           O  \\nANISOU   40  O   ASP A  38     1512   1815   1400    150   -140    288       O  \\nATOM     41  CB  ASP A  38       9.235  28.708   0.912  1.00 17.86           C  \\nANISOU   41  CB  ASP A  38     2980   2531   1273   -278     55   -112       C  \\nATOM     42  CG  ASP A  38       7.978  28.564  -0.064  1.00 22.48           C  \\nANISOU   42  CG  ASP A  38     2648   3317   2574     44   -160   -190       C  \\nATOM     43  OD1 ASP A  38       6.839  29.147   0.148  1.00 22.75           O  \\nANISOU   43  OD1 ASP A  38     3108   3830   1702    484     17    222       O  \\nATOM     44  OD2 ASP A  38       8.121  27.808  -1.077  1.00 26.00           O  \\nANISOU   44  OD2 ASP A  38     3351   3723   2803    742   -434   -438       O  \\nATOM     45  N   VAL A  39       7.064  28.776   2.988  1.00 12.94           N  \\nANISOU   45  N   VAL A  39     1656   1832   1429   -133   -381    257       N  \\nATOM     46  CA  VAL A  39       6.205  28.170   4.001  1.00 13.18           C  \\nANISOU   46  CA  VAL A  39     1706   1513   1788    -55   -198    328       C  \\nATOM     47  C   VAL A  39       5.850  26.760   3.579  1.00 12.90           C  \\nANISOU   47  C   VAL A  39     1620   1706   1575   -155   -257    183       C  \\nATOM     48  O   VAL A  39       5.751  26.465   2.388  1.00 17.73           O  \\nANISOU   48  O   VAL A  39     2728   2291   1717   -405   -210     11       O  \\nATOM     49  CB  VAL A  39       4.942  29.010   4.256  1.00 17.00           C  \\nANISOU   49  CB  VAL A  39     2008   2261   2191    197    115    258       C  \\nATOM     50  CG1 VAL A  39       5.284  30.440   4.537  1.00 18.59           C  \\nANISOU   50  CG1 VAL A  39     2311   2247   2503    291    181     71       C  \\nATOM     51  CG2 VAL A  39       3.899  28.876   3.171  1.00 20.54           C  \\nANISOU   51  CG2 VAL A  39     2543   2821   2438    135   -128    354       C  \\nATOM     52  N   LEU A  40       5.644  25.906   4.583  1.00 11.67           N  \\nANISOU   52  N   LEU A  40     1286   1643   1504   -114     37     41       N  \\nATOM     53  CA  LEU A  40       5.170  24.533   4.404  1.00 12.74           C  \\nANISOU   53  CA  LEU A  40     1337   1811   1690   -221     95   -153       C  \\nATOM     54  C   LEU A  40       4.005  24.329   5.383  1.00 10.91           C  \\nANISOU   54  C   LEU A  40     1221   1587   1335      8    -69   -155       C  \\nATOM     55  O   LEU A  40       4.148  24.504   6.585  1.00 12.22           O  \\nANISOU   55  O   LEU A  40     1347   1842   1454    -35   -148   -388       O  \\nATOM     56  CB  LEU A  40       6.299  23.578   4.709  1.00 15.21           C  \\nANISOU   56  CB  LEU A  40     1626   1998   2153     74    247   -227       C  \\nATOM     57  CG  LEU A  40       6.042  22.094   4.494  1.00 17.02           C  \\nANISOU   57  CG  LEU A  40     1947   2114   2405   -178    -18   -309       C  \\nATOM     58  CD1 LEU A  40       7.328  21.386   4.125  1.00 19.92           C  \\nANISOU   58  CD1 LEU A  40     2220   2396   2953    182    -73   -489       C  \\nATOM     59  CD2 LEU A  40       5.406  21.403   5.700  1.00 19.32           C  \\nANISOU   59  CD2 LEU A  40     2320   2470   2550    234    244   -228       C  \\nATOM     60  N   TYR A  41       2.854  23.930   4.852  1.00 11.26           N  \\nANISOU   60  N   TYR A  41     1212   1693   1373    -52   -195      4       N  \\nATOM     61  CA  TYR A  41       1.697  23.554   5.682  1.00 11.67           C  \\nANISOU   61  CA  TYR A  41     1217   1711   1504    -32   -168    191       C  \\nATOM     62  C   TYR A  41       1.803  22.046   5.928  1.00 12.04           C  \\nANISOU   62  C   TYR A  41     1457   1774   1342    134    -38    126       C  \\nATOM     63  O   TYR A  41       1.675  21.213   5.022  1.00 13.27           O  \\nANISOU   63  O   TYR A  41     1686   1788   1566     83   -321    120       O  \\nATOM     64  CB  TYR A  41       0.412  23.917   4.963  1.00 13.18           C  \\nANISOU   64  CB  TYR A  41     1276   1789   1941   -122   -395     98       C  \\nATOM     65  CG  TYR A  41       0.282  25.391   4.678  1.00 12.81           C  \\nANISOU   65  CG  TYR A  41     1115   1966   1784   -136   -342    277       C  \\nATOM     66  CD1 TYR A  41       0.026  26.279   5.709  1.00 15.19           C  \\nANISOU   66  CD1 TYR A  41     1624   2032   2113    269   -323    127       C  \\nATOM     67  CD2 TYR A  41       0.412  25.894   3.404  1.00 14.60           C  \\nANISOU   67  CD2 TYR A  41     1876   1825   1845    279    -69    307       C  \\nATOM     68  CE1 TYR A  41      -0.075  27.641   5.488  1.00 16.92           C  \\nANISOU   68  CE1 TYR A  41     1997   1998   2433    335   -390     61       C  \\nATOM     69  CE2 TYR A  41       0.330  27.246   3.205  1.00 16.10           C  \\nANISOU   69  CE2 TYR A  41     2459   1864   1793     68   -240    723       C  \\nATOM     70  CZ  TYR A  41       0.082  28.108   4.249  1.00 16.84           C  \\nANISOU   70  CZ  TYR A  41     2099   1866   2430    232   -692    302       C  \\nATOM     71  OH  TYR A  41      -0.028  29.475   3.995  1.00 21.92           O  \\nANISOU   71  OH  TYR A  41     2933   1727   3665    -66   -994    149       O  \\nATOM     72  N   PHE A  42       2.081  21.697   7.179  1.00 12.09           N  \\nANISOU   72  N   PHE A  42     1616   1561   1415    -70   -393     70       N  \\nATOM     73  CA  PHE A  42       2.342  20.304   7.482  1.00 11.25           C  \\nANISOU   73  CA  PHE A  42     1315   1587   1372    -54   -351    159       C  \\nATOM     74  C   PHE A  42       1.065  19.456   7.281  1.00 12.19           C  \\nANISOU   74  C   PHE A  42     1407   1632   1593   -131   -348    123       C  \\nATOM     75  O   PHE A  42      -0.004  19.879   7.679  1.00 12.37           O  \\nANISOU   75  O   PHE A  42     1512   1689   1500    -80   -317    169       O  \\nATOM     76  CB  PHE A  42       2.828  20.171   8.916  1.00 11.73           C  \\nANISOU   76  CB  PHE A  42     1550   1549   1356   -183   -326    213       C  \\nATOM     77  CG  PHE A  42       3.246  18.755   9.251  1.00 11.75           C  \\nANISOU   77  CG  PHE A  42     1659   1541   1265     16   -444    -14       C  \\nATOM     78  CD1 PHE A  42       4.510  18.308   8.930  1.00 13.02           C  \\nANISOU   78  CD1 PHE A  42     1548   1911   1486    -36   -466     96       C  \\nATOM     79  CD2 PHE A  42       2.341  17.865   9.767  1.00 11.29           C  \\nANISOU   79  CD2 PHE A  42     1353   1644   1291    180   -285     20       C  \\nATOM     80  CE1 PHE A  42       4.870  16.994   9.181  1.00 14.43           C  \\nANISOU   80  CE1 PHE A  42     1721   1969   1792    443    -77   -308       C  \\nATOM     81  CE2 PHE A  42       2.697  16.555   9.991  1.00 12.96           C  \\nANISOU   81  CE2 PHE A  42     1826   1711   1385    104   -484    202       C  \\nATOM     82  CZ  PHE A  42       3.971  16.138   9.700  1.00 14.58           C  \\nANISOU   82  CZ  PHE A  42     1970   1870   1699    299   -615     89       C  \\nATOM     83  N   PRO A  43       1.175  18.237   6.699  1.00 11.92           N  \\nANISOU   83  N   PRO A  43     1360   1620   1547    -11   -168    201       N  \\nATOM     84  CA  PRO A  43       0.035  17.374   6.405  1.00 12.93           C  \\nANISOU   84  CA  PRO A  43     1433   1727   1753      6   -248     52       C  \\nATOM     85  C   PRO A  43      -0.477  16.654   7.633  1.00 12.74           C  \\nANISOU   85  C   PRO A  43     1499   1650   1690     77    -43    -41       C  \\nATOM     86  O   PRO A  43      -0.318  15.424   7.791  1.00 12.88           O  \\nANISOU   86  O   PRO A  43     1518   1674   1701     76   -225     91       O  \\nATOM     87  CB  PRO A  43       0.595  16.419   5.336  1.00 14.05           C  \\nANISOU   87  CB  PRO A  43     1976   1788   1572     82   -496   -131       C  \\nATOM     88  CG  PRO A  43       2.033  16.292   5.735  1.00 14.13           C  \\nANISOU   88  CG  PRO A  43     1692   1984   1690    107     14   -115       C  \\nATOM     89  CD  PRO A  43       2.404  17.685   6.116  1.00 13.89           C  \\nANISOU   89  CD  PRO A  43     1595   1971   1711    117     -6    -26       C  \\nATOM     90  N   ALA A  44      -1.058  17.428   8.548  1.00 12.12           N  \\nANISOU   90  N   ALA A  44     1414   1556   1633    125    -76     83       N  \\nATOM     91  CA  ALA A  44      -1.484  16.884   9.813  1.00 14.63           C  \\nANISOU   91  CA  ALA A  44     1680   1971   1906    -72      7    325       C  \\nATOM     92  C   ALA A  44      -2.581  15.833   9.675  1.00 15.11           C  \\nANISOU   92  C   ALA A  44     1709   2059   1973   -143   -136    395       C  \\nATOM     93  O   ALA A  44      -2.648  14.919  10.489  1.00 18.33           O  \\nANISOU   93  O   ALA A  44     2358   2345   2258    -80   -530    673       O  \\nATOM     94  CB  ALA A  44      -1.945  18.015  10.712  1.00 18.87           C  \\nANISOU   94  CB  ALA A  44     2274   2422   2473    -56    625    -26       C  \\nATOM     95  N   SER A  45      -3.401  15.902   8.632  1.00 15.52           N  \\nANISOU   95  N   SER A  45     1582   2199   2114   -182   -142    676       N  \\nATOM     96  CA  SER A  45      -4.445  14.866   8.500  1.00 18.43           C  \\nANISOU   96  CA  SER A  45     1527   2481   2995   -227   -563    302       C  \\nATOM     97  C   SER A  45      -3.841  13.498   8.191  1.00 18.39           C  \\nANISOU   97  C   SER A  45     1875   2446   2663   -300   -743    242       C  \\nATOM     98  O   SER A  45      -4.376  12.456   8.589  1.00 23.63           O  \\nANISOU   98  O   SER A  45     2265   3028   3684   -664   -835    815       O  \\nATOM     99  CB  SER A  45      -5.476  15.237   7.423  1.00 20.57           C  \\nANISOU   99  CB  SER A  45     1740   3230   2845    197   -438    546       C  \\nATOM    100  OG  SER A  45      -4.934  15.192   6.083  1.00 27.19           O  \\nANISOU  100  OG  SER A  45     2773   4833   2725    113   -420   1297       O  \\nATOM    101  N   ARG A  46      -2.744  13.504   7.448  1.00 16.63           N  \\nANISOU  101  N   ARG A  46     1710   2283   2326   -115  -1051    164       N  \\nATOM    102  CA  ARG A  46      -2.094  12.269   7.022  1.00 16.41           C  \\nANISOU  102  CA  ARG A  46     2245   1826   2161   -328  -1192   -358       C  \\nATOM    103  C   ARG A  46      -1.213  11.696   8.159  1.00 17.24           C  \\nANISOU  103  C   ARG A  46     2081   2004   2464   -243  -1206   -268       C  \\nATOM    104  O   ARG A  46      -1.156  10.470   8.345  1.00 18.21           O  \\nANISOU  104  O   ARG A  46     2626   1988   2305   -211   -916   -231       O  \\nATOM    105  CB  ARG A  46      -1.219  12.477   5.777  1.00 23.50           C  \\nANISOU  105  CB  ARG A  46     2706   3579   2643   -488   -957    207       C  \\nATOM    106  CG  ARG A  46      -1.929  12.619   4.434  1.00 28.14           C  \\nANISOU  106  CG  ARG A  46     3804   3946   2940   -360  -1334    367       C  \\nATOM    107  CD  ARG A  46      -0.971  13.013   3.313  1.00 38.97           C  \\nANISOU  107  CD  ARG A  46     5066   5967   3771   -658     92   -647       C  \\nATOM    108  NE  ARG A  46       0.052  12.007   2.967  1.00 49.36           N  \\nANISOU  108  NE  ARG A  46     6649   6411   5692    571  -1323   -718       N  \\nATOM    109  CZ  ARG A  46      -0.164  10.902   2.246  1.00 51.28           C  \\nANISOU  109  CZ  ARG A  46     6713   6406   6364   1688   -425  -1407       C  \\nATOM    110  NH1 ARG A  46      -1.381  10.591   1.798  1.00 62.38           N  \\nANISOU  110  NH1 ARG A  46     9037   7342   7321   -391  -2428  -3363       N  \\nATOM    111  NH2 ARG A  46       0.840  10.085   1.964  1.00 50.85           N  \\nANISOU  111  NH2 ARG A  46     6176   7779   5364   2380   -751    -76       N  \\nATOM    112  N   TYR A  47      -0.545  12.578   8.912  1.00 12.74           N  \\nANISOU  112  N   TYR A  47     1833   1554   1452   -117   -621    -52       N  \\nATOM    113  CA  TYR A  47       0.444  12.162   9.919  1.00 12.44           C  \\nANISOU  113  CA  TYR A  47     1696   1485   1544    -79   -536     72       C  \\nATOM    114  C   TYR A  47       0.107  12.923  11.186  1.00 11.86           C  \\nANISOU  114  C   TYR A  47     1426   1447   1630   -236   -484     11       C  \\nATOM    115  O   TYR A  47       0.852  13.810  11.616  1.00 11.57           O  \\nANISOU  115  O   TYR A  47     1464   1435   1495   -285   -402    -20       O  \\nATOM    116  CB  TYR A  47       1.880  12.407   9.459  1.00 13.30           C  \\nANISOU  116  CB  TYR A  47     1694   1754   1604    -62   -556    -87       C  \\nATOM    117  CG  TYR A  47       2.138  11.885   8.087  1.00 14.23           C  \\nANISOU  117  CG  TYR A  47     1753   1880   1772    -41   -562   -238       C  \\nATOM    118  CD1 TYR A  47       2.112  10.535   7.841  1.00 16.56           C  \\nANISOU  118  CD1 TYR A  47     2532   2016   1741    300   -768   -374       C  \\nATOM    119  CD2 TYR A  47       2.401  12.743   7.032  1.00 15.43           C  \\nANISOU  119  CD2 TYR A  47     1834   2414   1615     52   -195   -232       C  \\nATOM    120  CE1 TYR A  47       2.257  10.081   6.552  1.00 19.33           C  \\nANISOU  120  CE1 TYR A  47     2485   2578   2280    486   -475  -1216       C  \\nATOM    121  CE2 TYR A  47       2.550  12.286   5.747  1.00 20.35           C  \\nANISOU  121  CE2 TYR A  47     2279   3618   1833    -66    -83   -629       C  \\nATOM    122  CZ  TYR A  47       2.512  10.955   5.504  1.00 22.07           C  \\nANISOU  122  CZ  TYR A  47     2654   3570   2162    262   -228   -963       C  \\nATOM    123  OH  TYR A  47       2.689  10.482   4.187  1.00 30.14           O  \\nANISOU  123  OH  TYR A  47     2786   5878   2786   -137    272  -2029       O  \\nATOM    124  N   PRO A  48      -1.035  12.594  11.806  1.00 11.98           N  \\nANISOU  124  N   PRO A  48     1526   1601   1423   -249   -405     27       N  \\nATOM    125  CA  PRO A  48      -1.496  13.404  12.900  1.00 13.00           C  \\nANISOU  125  CA  PRO A  48     1515   1742   1681     28   -394     58       C  \\nATOM    126  C   PRO A  48      -0.659  13.389  14.143  1.00 11.52           C  \\nANISOU  126  C   PRO A  48     1407   1406   1562   -107   -290    226       C  \\nATOM    127  O   PRO A  48      -0.632  14.389  14.868  1.00 13.19           O  \\nANISOU  127  O   PRO A  48     1703   1551   1755   -105   -207    100       O  \\nATOM    128  CB  PRO A  48      -2.908  12.851  13.184  1.00 15.18           C  \\nANISOU  128  CB  PRO A  48     1595   2378   1791   -216   -235    -11       C  \\nATOM    129  CG  PRO A  48      -2.880  11.477  12.676  1.00 15.97           C  \\nANISOU  129  CG  PRO A  48     1601   2199   2268   -578   -505     69       C  \\nATOM    130  CD  PRO A  48      -2.006  11.539  11.460  1.00 14.77           C  \\nANISOU  130  CD  PRO A  48     1835   1697   2079   -370   -548    -93       C  \\nATOM    131  N   GLU A  49      -0.012  12.281  14.444  1.00 12.58           N  \\nANISOU  131  N   GLU A  49     1671   1270   1836    -54   -276     62       N  \\nATOM    132  CA  GLU A  49       0.803  12.238  15.665  1.00 12.03           C  \\nANISOU  132  CA  GLU A  49     1568   1307   1694    -30   -141    -22       C  \\nATOM    133  C   GLU A  49       2.056  13.078  15.507  1.00 11.40           C  \\nANISOU  133  C   GLU A  49     1577   1270   1483     10   -169    102       C  \\nATOM    134  O   GLU A  49       2.442  13.818  16.436  1.00 12.12           O  \\nANISOU  134  O   GLU A  49     1653   1596   1355      2   -259      0       O  \\nATOM    135  CB  GLU A  49       1.196  10.806  16.029  1.00 14.74           C  \\nANISOU  135  CB  GLU A  49     2153   1339   2105    -52   -199    213       C  \\nATOM    136  CG  GLU A  49       0.115   9.950  16.550  1.00 17.40           C  \\nANISOU  136  CG  GLU A  49     2212   1793   2604    -26     26    203       C  \\nATOM    137  CD  GLU A  49       0.706   8.607  17.015  1.00 20.70           C  \\nANISOU  137  CD  GLU A  49     2275   1917   3670    -24    265    693       C  \\nATOM    138  OE1 GLU A  49       1.732   8.490  17.785  1.00 20.18           O  \\nANISOU  138  OE1 GLU A  49     2373   1852   3442    217    533    251       O  \\nATOM    139  OE2 GLU A  49       0.149   7.640  16.556  1.00 25.98           O  \\nANISOU  139  OE2 GLU A  49     3040   2366   4464    109   -626     72       O  \\nATOM    140  N   THR A  50       2.709  12.987  14.352  1.00 10.98           N  \\nANISOU  140  N   THR A  50     1433   1359   1379    -33   -261    -13       N  \\nATOM    141  CA  THR A  50       3.863  13.862  14.078  1.00 10.30           C  \\nANISOU  141  CA  THR A  50     1283   1314   1314     70   -229      2       C  \\nATOM    142  C   THR A  50       3.436  15.324  14.094  1.00  9.95           C  \\nANISOU  142  C   THR A  50     1309   1253   1216    -52   -384      0       C  \\nATOM    143  O   THR A  50       4.073  16.177  14.693  1.00 10.96           O  \\nANISOU  143  O   THR A  50     1420   1539   1204    -96   -333   -154       O  \\nATOM    144  CB  THR A  50       4.548  13.441  12.792  1.00 10.51           C  \\nANISOU  144  CB  THR A  50     1493   1088   1411     69   -136    -84       C  \\nATOM    145  OG1 THR A  50       4.958  12.066  12.896  1.00 11.72           O  \\nANISOU  145  OG1 THR A  50     1622   1183   1646    219   -240     -6       O  \\nATOM    146  CG2 THR A  50       5.755  14.255  12.518  1.00 11.56           C  \\nANISOU  146  CG2 THR A  50     1569   1426   1393      0    -52   -103       C  \\nATOM    147  N   GLY A  51       2.310  15.611  13.431  1.00 10.02           N  \\nANISOU  147  N   GLY A  51     1402   1174   1229    -56   -427    -53       N  \\nATOM    148  CA  GLY A  51       1.837  16.973  13.396  1.00  9.90           C  \\nANISOU  148  CA  GLY A  51     1394   1167   1199     -7   -356    -54       C  \\nATOM    149  C   GLY A  51       1.524  17.539  14.752  1.00  9.93           C  \\nANISOU  149  C   GLY A  51     1343   1144   1284     54   -104     31       C  \\nATOM    150  O   GLY A  51       1.802  18.708  15.023  1.00 10.97           O  \\nANISOU  150  O   GLY A  51     1557   1070   1540     23   -192    -72       O  \\nATOM    151  N   ALA A  52       0.898  16.735  15.617  1.00 10.43           N  \\nANISOU  151  N   ALA A  52     1358   1256   1349   -131    -66     -1       N  \\nATOM    152  CA  ALA A  52       0.595  17.195  16.961  1.00 11.37           C  \\nANISOU  152  CA  ALA A  52     1258   1537   1525    165    -16   -155       C  \\nATOM    153  C   ALA A  52       1.864  17.533  17.738  1.00 10.84           C  \\nANISOU  153  C   ALA A  52     1359   1475   1283    -22    -12    -76       C  \\nATOM    154  O   ALA A  52       1.898  18.497  18.516  1.00 12.56           O  \\nANISOU  154  O   ALA A  52     1823   1502   1448    -33    -23   -155       O  \\nATOM    155  CB  ALA A  52      -0.227  16.163  17.724  1.00 12.94           C  \\nANISOU  155  CB  ALA A  52     1530   1887   1498    -71     99   -108       C  \\nATOM    156  N   HIS A  53       2.903  16.721  17.566  1.00 10.29           N  \\nANISOU  156  N   HIS A  53     1330   1126   1454   -138    -92   -207       N  \\nATOM    157  CA  HIS A  53       4.177  17.033  18.189  1.00  9.80           C  \\nANISOU  157  CA  HIS A  53     1349   1296   1077   -104    -46    -95       C  \\nATOM    158  C   HIS A  53       4.722  18.368  17.714  1.00 10.19           C  \\nANISOU  158  C   HIS A  53     1524   1256   1089    -83   -177   -118       C  \\nATOM    159  O   HIS A  53       5.118  19.215  18.538  1.00 10.33           O  \\nANISOU  159  O   HIS A  53     1702   1279    943     17   -184   -178       O  \\nATOM    160  CB  HIS A  53       5.176  15.915  17.990  1.00 10.64           C  \\nANISOU  160  CB  HIS A  53     1571   1175   1296    -45   -190     21       C  \\nATOM    161  CG  HIS A  53       6.469  16.165  18.686  1.00 10.42           C  \\nANISOU  161  CG  HIS A  53     1489   1089   1381     -9   -143     26       C  \\nATOM    162  ND1 HIS A  53       7.681  15.705  18.204  1.00 11.81           N  \\nANISOU  162  ND1 HIS A  53     1478   1421   1586    -24   -126     55       N  \\nATOM    163  CD2 HIS A  53       6.742  16.842  19.832  1.00 11.10           C  \\nANISOU  163  CD2 HIS A  53     1469   1363   1382    125   -148    -62       C  \\nATOM    164  CE1 HIS A  53       8.642  16.113  19.022  1.00 11.59           C  \\nANISOU  164  CE1 HIS A  53     1451   1342   1607     63   -135    -69       C  \\nATOM    165  NE2 HIS A  53       8.096  16.797  20.011  1.00 11.06           N  \\nANISOU  165  NE2 HIS A  53     1498   1258   1444     31   -284    -15       N  \\nATOM    166  N   ILE A  54       4.769  18.555  16.393  1.00  9.71           N  \\nANISOU  166  N   ILE A  54     1475   1152   1062   -103   -140   -126       N  \\nATOM    167  CA  ILE A  54       5.320  19.790  15.859  1.00  9.20           C  \\nANISOU  167  CA  ILE A  54     1268   1265    963     14   -164     68       C  \\nATOM    168  C   ILE A  54       4.518  20.989  16.404  1.00  9.88           C  \\nANISOU  168  C   ILE A  54     1414   1185   1153    -14   -309    -92       C  \\nATOM    169  O   ILE A  54       5.096  21.987  16.840  1.00 10.12           O  \\nANISOU  169  O   ILE A  54     1541   1254   1050    -67   -206   -205       O  \\nATOM    170  CB  ILE A  54       5.320  19.775  14.331  1.00  9.72           C  \\nANISOU  170  CB  ILE A  54     1419   1277    997    -66   -243    -41       C  \\nATOM    171  CG1 ILE A  54       6.195  18.630  13.821  1.00 11.25           C  \\nANISOU  171  CG1 ILE A  54     1441   1289   1544   -150     -6    -92       C  \\nATOM    172  CG2 ILE A  54       5.745  21.116  13.832  1.00 11.39           C  \\nANISOU  172  CG2 ILE A  54     1623   1330   1374   -116   -169     37       C  \\nATOM    173  CD1 ILE A  54       6.080  18.414  12.336  1.00 13.38           C  \\nANISOU  173  CD1 ILE A  54     1765   1664   1654   -170    112   -356       C  \\nATOM    174  N   SER A  55       3.200  20.890  16.375  1.00 10.18           N  \\nANISOU  174  N   SER A  55     1418   1191   1256     96   -174   -145       N  \\nATOM    175  CA  SER A  55       2.353  21.984  16.849  1.00 10.89           C  \\nANISOU  175  CA  SER A  55     1457   1425   1255    244   -287   -187       C  \\nATOM    176  C   SER A  55       2.662  22.331  18.311  1.00 10.70           C  \\nANISOU  176  C   SER A  55     1555   1336   1174     25   -200    -66       C  \\nATOM    177  O   SER A  55       2.850  23.491  18.650  1.00 12.40           O  \\nANISOU  177  O   SER A  55     1673   1401   1636      1    157   -322       O  \\nATOM    178  CB  SER A  55       0.881  21.600  16.715  1.00 12.88           C  \\nANISOU  178  CB  SER A  55     1337   2021   1533    296   -266   -205       C  \\nATOM    179  OG  SER A  55       0.025  22.603  17.231  1.00 17.85           O  \\nANISOU  179  OG  SER A  55     2010   2208   2564    584   -264   -444       O  \\nATOM    180  N   ASP A  56       2.733  21.316  19.166  1.00 10.26           N  \\nANISOU  180  N   ASP A  56     1518   1360   1019    -70    -25    -56       N  \\nATOM    181  CA  ASP A  56       2.988  21.516  20.587  1.00 11.35           C  \\nANISOU  181  CA  ASP A  56     1673   1598   1040     19     31   -112       C  \\nATOM    182  C   ASP A  56       4.407  22.067  20.802  1.00 11.77           C  \\nANISOU  182  C   ASP A  56     1804   1570   1097   -138     -6   -186       C  \\nATOM    183  O   ASP A  56       4.624  22.914  21.654  1.00 12.21           O  \\nANISOU  183  O   ASP A  56     1783   1629   1225    -96    -83   -322       O  \\nATOM    184  CB  ASP A  56       2.805  20.208  21.368  1.00 13.08           C  \\nANISOU  184  CB  ASP A  56     2095   1677   1198    -50     21    -37       C  \\nATOM    185  CG  ASP A  56       1.362  19.807  21.569  1.00 17.02           C  \\nANISOU  185  CG  ASP A  56     2170   2162   2133     31    183     86       C  \\nATOM    186  OD1 ASP A  56       0.462  20.617  21.320  1.00 21.40           O  \\nANISOU  186  OD1 ASP A  56     2152   2819   3157    120    340    675       O  \\nATOM    187  OD2 ASP A  56       1.167  18.673  22.018  1.00 19.66           O  \\nANISOU  187  OD2 ASP A  56     2365   2452   2653    -63    187    366       O  \\nATOM    188  N   ALA A  57       5.371  21.538  20.059  1.00 10.48           N  \\nANISOU  188  N   ALA A  57     1581   1307   1091   -104    -73    -15       N  \\nATOM    189  CA  ALA A  57       6.753  21.961  20.233  1.00 11.12           C  \\nANISOU  189  CA  ALA A  57     1552   1384   1288    -28   -148   -178       C  \\nATOM    190  C   ALA A  57       6.873  23.441  19.868  1.00 10.47           C  \\nANISOU  190  C   ALA A  57     1473   1437   1068     -2   -274   -138       C  \\nATOM    191  O   ALA A  57       7.565  24.187  20.570  1.00 10.98           O  \\nANISOU  191  O   ALA A  57     1689   1376   1107    -22   -251   -326       O  \\nATOM    192  CB  ALA A  57       7.700  21.102  19.403  1.00 10.34           C  \\nANISOU  192  CB  ALA A  57     1485   1235   1209    -20   -204    -71       C  \\nATOM    193  N   ILE A  58       6.241  23.859  18.780  1.00  9.42           N  \\nANISOU  193  N   ILE A  58     1552   1117    909      9   -153   -142       N  \\nATOM    194  CA  ILE A  58       6.291  25.295  18.426  1.00  9.54           C  \\nANISOU  194  CA  ILE A  58     1477   1127   1017      7    -21   -148       C  \\nATOM    195  C   ILE A  58       5.635  26.136  19.540  1.00  9.46           C  \\nANISOU  195  C   ILE A  58     1533   1278    784    -67   -175   -276       C  \\nATOM    196  O   ILE A  58       6.162  27.215  19.880  1.00 11.33           O  \\nANISOU  196  O   ILE A  58     1816   1164   1325    -45      7   -419       O  \\nATOM    197  CB  ILE A  58       5.634  25.485  17.025  1.00  9.92           C  \\nANISOU  197  CB  ILE A  58     1510   1156   1101    -76   -136   -101       C  \\nATOM    198  CG1 ILE A  58       6.548  24.896  15.929  1.00 10.99           C  \\nANISOU  198  CG1 ILE A  58     1718   1238   1218    -57     28     -8       C  \\nATOM    199  CG2 ILE A  58       5.302  26.954  16.779  1.00 10.16           C  \\nANISOU  199  CG2 ILE A  58     1474   1218   1167     26    -98   -103       C  \\nATOM    200  CD1 ILE A  58       5.874  24.802  14.571  1.00 11.68           C  \\nANISOU  200  CD1 ILE A  58     2116   1126   1194     25     51    -33       C  \\nATOM    201  N   LYS A  59       4.538  25.672  20.117  1.00 10.95           N  \\nANISOU  201  N   LYS A  59     1660   1211   1286     -3    -28   -294       N  \\nATOM    202  CA  LYS A  59       3.902  26.422  21.224  1.00 11.52           C  \\nANISOU  202  CA  LYS A  59     1690   1290   1397      9     87   -310       C  \\nATOM    203  C   LYS A  59       4.717  26.379  22.540  1.00 11.95           C  \\nANISOU  203  C   LYS A  59     1723   1563   1252    188    248   -141       C  \\nATOM    204  O   LYS A  59       4.455  27.166  23.416  1.00 17.26           O  \\nANISOU  204  O   LYS A  59     2735   1898   1922    452    155   -687       O  \\nATOM    205  CB  LYS A  59       2.476  25.925  21.442  1.00 13.42           C  \\nANISOU  205  CB  LYS A  59     1858   1693   1547    -68    353   -494       C  \\nATOM    206  CG  LYS A  59       1.540  26.275  20.292  1.00 16.54           C  \\nANISOU  206  CG  LYS A  59     2046   1884   2351    291   -191   -658       C  \\nATOM    207  CD  LYS A  59       0.136  25.849  20.670  1.00 25.52           C  \\nANISOU  207  CD  LYS A  59     2435   3526   3732   -202    157   -407       C  \\nATOM    208  CE  LYS A  59      -0.884  26.259  19.640  1.00 35.12           C  \\nANISOU  208  CE  LYS A  59     4348   4676   4317   -161   -553    -27       C  \\nATOM    209  NZ  LYS A  59      -0.896  25.251  18.579  1.00 39.77           N  \\nANISOU  209  NZ  LYS A  59     5396   4901   4814   -150    376   -192       N  \\nATOM    210  N   ALA A  60       5.728  25.515  22.605  1.00 11.33           N  \\nANISOU  210  N   ALA A  60     1791   1234   1277    148    -17   -344       N  \\nATOM    211  CA  ALA A  60       6.710  25.459  23.672  1.00 11.93           C  \\nANISOU  211  CA  ALA A  60     1809   1512   1210    212     80   -388       C  \\nATOM    212  C   ALA A  60       7.990  26.240  23.375  1.00 11.71           C  \\nANISOU  212  C   ALA A  60     1883   1629    935     83    -57   -299       C  \\nATOM    213  O   ALA A  60       9.001  26.178  24.112  1.00 14.41           O  \\nANISOU  213  O   ALA A  60     1921   1861   1692    -73   -357   -206       O  \\nATOM    214  CB  ALA A  60       6.981  24.022  24.088  1.00 13.04           C  \\nANISOU  214  CB  ALA A  60     1968   1736   1251     11     30    -23       C  \\nATOM    215  N   GLY A  61       7.986  26.982  22.258  1.00 11.50           N  \\nANISOU  215  N   GLY A  61     1894   1551    922    -63     31   -294       N  \\nATOM    216  CA  GLY A  61       9.079  27.865  21.931  1.00 12.16           C  \\nANISOU  216  CA  GLY A  61     1751   1531   1336    -78   -135   -272       C  \\nATOM    217  C   GLY A  61      10.151  27.293  21.061  1.00 11.43           C  \\nANISOU  217  C   GLY A  61     1787   1209   1345    -30   -200   -229       C  \\nATOM    218  O   GLY A  61      11.162  27.951  20.816  1.00 13.18           O  \\nANISOU  218  O   GLY A  61     1964   1397   1645   -248   -124   -477       O  \\nATOM    219  N   HIS A  62       9.969  26.070  20.571  1.00 10.94           N  \\nANISOU  219  N   HIS A  62     1875   1132   1148    -79   -110   -124       N  \\nATOM    220  CA  HIS A  62      10.929  25.525  19.610  1.00 11.03           C  \\nANISOU  220  CA  HIS A  62     1643   1332   1213    -89   -181   -161       C  \\nATOM    221  C   HIS A  62      10.756  26.205  18.262  1.00 10.70           C  \\nANISOU  221  C   HIS A  62     1620   1389   1056    123    -36   -312       C  \\nATOM    222  O   HIS A  62       9.640  26.566  17.858  1.00 12.02           O  \\nANISOU  222  O   HIS A  62     1662   1770   1133    188   -144   -188       O  \\nATOM    223  CB  HIS A  62      10.759  24.010  19.506  1.00 11.18           C  \\nANISOU  223  CB  HIS A  62     1707   1326   1213     55   -269   -302       C  \\nATOM    224  CG  HIS A  62      11.055  23.313  20.781  1.00 11.13           C  \\nANISOU  224  CG  HIS A  62     1846   1185   1194     53   -236   -353       C  \\nATOM    225  ND1 HIS A  62      12.334  22.928  21.111  1.00 13.27           N  \\nANISOU  225  ND1 HIS A  62     2164   1389   1490    309   -567   -353       N  \\nATOM    226  CD2 HIS A  62      10.265  22.982  21.834  1.00 12.34           C  \\nANISOU  226  CD2 HIS A  62     1877   1621   1187     38   -297   -295       C  \\nATOM    227  CE1 HIS A  62      12.312  22.375  22.319  1.00 12.93           C  \\nANISOU  227  CE1 HIS A  62     1880   1693   1340    151   -496   -381       C  \\nATOM    228  NE2 HIS A  62      11.068  22.392  22.774  1.00 13.17           N  \\nANISOU  228  NE2 HIS A  62     2032   1605   1366    -29   -395   -150       N  \\nATOM    229  N   ALA A  63      11.858  26.320  17.534  1.00 11.39           N  \\nANISOU  229  N   ALA A  63     1698   1365   1264    102     26   -192       N  \\nATOM    230  CA  ALA A  63      11.856  27.070  16.256  1.00 10.78           C  \\nANISOU  230  CA  ALA A  63     1684   1193   1217     74     18   -268       C  \\nATOM    231  C   ALA A  63      10.781  26.564  15.309  1.00 10.72           C  \\nANISOU  231  C   ALA A  63     1882   1162   1028      0    -49    -53       C  \\nATOM    232  O   ALA A  63      10.693  25.358  15.054  1.00 11.29           O  \\nANISOU  232  O   ALA A  63     1898   1073   1319    -50   -217    -37       O  \\nATOM    233  CB  ALA A  63      13.216  26.984  15.623  1.00 12.62           C  \\nANISOU  233  CB  ALA A  63     1759   1543   1491    -31    129   -143       C  \\nATOM    234  N   ASP A  64      10.034  27.471  14.725  1.00 11.28           N  \\nANISOU  234  N   ASP A  64     1812   1140   1331    -76   -206    -53       N  \\nATOM    235  CA  ASP A  64       9.091  27.162  13.677  1.00 10.83           C  \\nANISOU  235  CA  ASP A  64     1682   1410   1023    -32    -99    168       C  \\nATOM    236  C   ASP A  64       9.692  27.226  12.279  1.00 10.45           C  \\nANISOU  236  C   ASP A  64     1484   1336   1148    -22    -48     72       C  \\nATOM    237  O   ASP A  64       9.081  26.710  11.328  1.00 12.17           O  \\nANISOU  237  O   ASP A  64     1599   1844   1178   -238    -96     -2       O  \\nATOM    238  CB  ASP A  64       7.817  28.036  13.756  1.00 13.95           C  \\nANISOU  238  CB  ASP A  64     1614   2239   1445    205    181    454       C  \\nATOM    239  CG  ASP A  64       8.071  29.586  13.722  1.00 17.09           C  \\nANISOU  239  CG  ASP A  64     2323   2111   2057    695    306    197       C  \\nATOM    240  OD1 ASP A  64       9.236  30.027  13.750  1.00 19.44           O  \\nANISOU  240  OD1 ASP A  64     2971   1514   2898    551    521    -65       O  \\nATOM    241  OD2 ASP A  64       7.086  30.352  13.671  1.00 23.24           O  \\nANISOU  241  OD2 ASP A  64     3044   2991   2794   1375    634    597       O  \\nATOM    242  N   VAL A  65      10.881  27.809  12.155  1.00 10.43           N  \\nANISOU  242  N   VAL A  65     1498   1333   1131    -83   -169    -29       N  \\nATOM    243  CA  VAL A  65      11.608  27.824  10.889  1.00 10.68           C  \\nANISOU  243  CA  VAL A  65     1488   1286   1284    -70    -81   -137       C  \\nATOM    244  C   VAL A  65      12.696  26.734  10.896  1.00 11.28           C  \\nANISOU  244  C   VAL A  65     1413   1543   1330    -30    -16    -14       C  \\nATOM    245  O   VAL A  65      13.458  26.641  11.865  1.00 13.03           O  \\nANISOU  245  O   VAL A  65     1590   1788   1572     24   -202   -148       O  \\nATOM    246  CB  VAL A  65      12.224  29.210  10.637  1.00 12.85           C  \\nANISOU  246  CB  VAL A  65     1843   1339   1700   -163     74   -136       C  \\nATOM    247  CG1 VAL A  65      13.120  29.179   9.404  1.00 14.99           C  \\nANISOU  247  CG1 VAL A  65     1899   1776   2018   -189    342     54       C  \\nATOM    248  CG2 VAL A  65      11.121  30.236  10.490  1.00 13.64           C  \\nANISOU  248  CG2 VAL A  65     2164   1418   1598     36    128     18       C  \\nATOM    249  N   CYS A  66      12.759  25.969   9.802  1.00 11.30           N  \\nANISOU  249  N   CYS A  66     1524   1491   1277    176   -146     48       N  \\nATOM    250  CA  CYS A  66      13.823  25.027   9.566  1.00 11.27           C  \\nANISOU  250  CA  CYS A  66     1457   1502   1320    190   -219    132       C  \\nATOM    251  C   CYS A  66      14.787  25.666   8.556  1.00 10.91           C  \\nANISOU  251  C   CYS A  66     1394   1205   1545    113   -103     25       C  \\nATOM    252  O   CYS A  66      14.408  25.837   7.391  1.00 11.68           O  \\nANISOU  252  O   CYS A  66     1339   1493   1606    130   -133     52       O  \\nATOM    253  CB  CYS A  66      13.284  23.728   8.966  1.00 13.28           C  \\nANISOU  253  CB  CYS A  66     1966   1382   1695    112   -146    150       C  \\nATOM    254  SG  CYS A  66      14.616  22.561   8.414  1.00 16.12           S  \\nANISOU  254  SG  CYS A  66     2350   1542   2230    -48    128   -244       S  \\nATOM    255  N   THR A  67      16.022  25.920   8.999  1.00 10.44           N  \\nANISOU  255  N   THR A  67     1422   1205   1337    125    -98     32       N  \\nATOM    256  CA  THR A  67      17.094  26.336   8.109  1.00 10.70           C  \\nANISOU  256  CA  THR A  67     1382   1142   1541     88    -57    -91       C  \\nATOM    257  C   THR A  67      17.772  25.025   7.686  1.00 10.34           C  \\nANISOU  257  C   THR A  67     1362   1119   1444    -33    -26   -125       C  \\nATOM    258  O   THR A  67      18.448  24.385   8.485  1.00 10.56           O  \\nANISOU  258  O   THR A  67     1347   1079   1585     26   -177   -213       O  \\nATOM    259  CB  THR A  67      18.089  27.286   8.785  1.00 12.01           C  \\nANISOU  259  CB  THR A  67     1565   1224   1774     23   -243    -34       C  \\nATOM    260  OG1 THR A  67      17.355  28.389   9.330  1.00 13.39           O  \\nANISOU  260  OG1 THR A  67     2089   1166   1830     15    -50    -89       O  \\nATOM    261  CG2 THR A  67      19.158  27.766   7.816  1.00 14.19           C  \\nANISOU  261  CG2 THR A  67     1782   1408   2199   -199    -58   -157       C  \\nATOM    262  N   ILE A  68      17.525  24.614   6.455  1.00  9.48           N  \\nANISOU  262  N   ILE A  68     1302    825   1474     13    -26   -161       N  \\nATOM    263  CA  ILE A  68      17.918  23.275   6.034  1.00  9.61           C  \\nANISOU  263  CA  ILE A  68     1367    867   1414    121    -51   -130       C  \\nATOM    264  C   ILE A  68      19.439  23.133   6.090  1.00 10.82           C  \\nANISOU  264  C   ILE A  68     1370   1104   1637     87     99     64       C  \\nATOM    265  O   ILE A  68      20.166  23.902   5.482  1.00 11.97           O  \\nANISOU  265  O   ILE A  68     1392   1252   1901    -63    -11    145       O  \\nATOM    266  CB  ILE A  68      17.399  22.962   4.626  1.00  9.85           C  \\nANISOU  266  CB  ILE A  68     1357   1094   1290     84    139   -163       C  \\nATOM    267  CG1 ILE A  68      15.867  23.009   4.575  1.00 10.87           C  \\nANISOU  267  CG1 ILE A  68     1349   1374   1407    144    -52   -224       C  \\nATOM    268  CG2 ILE A  68      17.881  21.590   4.174  1.00 10.80           C  \\nANISOU  268  CG2 ILE A  68     1397   1184   1521    140     82   -277       C  \\nATOM    269  CD1 ILE A  68      15.309  22.891   3.195  1.00 11.84           C  \\nANISOU  269  CD1 ILE A  68     1585   1420   1491     11   -280    -18       C  \\nATOM    270  N   GLU A  69      19.876  22.067   6.769  1.00 10.89           N  \\nANISOU  270  N   GLU A  69     1236   1150   1751    147     -2     26       N  \\nATOM    271  CA  GLU A  69      21.298  21.754   6.832  1.00 11.62           C  \\nANISOU  271  CA  GLU A  69     1200   1374   1840    121    109    -78       C  \\nATOM    272  C   GLU A  69      21.379  20.225   6.948  1.00 12.16           C  \\nANISOU  272  C   GLU A  69     1321   1400   1897    123    -56     48       C  \\nATOM    273  O   GLU A  69      21.375  19.675   8.032  1.00 13.82           O  \\nANISOU  273  O   GLU A  69     1856   1333   2060    204    127    137       O  \\nATOM    274  CB  GLU A  69      21.990  22.458   7.991  1.00 12.76           C  \\nANISOU  274  CB  GLU A  69     1366   1459   2020    -35   -137     99       C  \\nATOM    275  CG  GLU A  69      23.522  22.525   7.867  1.00 14.60           C  \\nANISOU  275  CG  GLU A  69     1354   1932   2260   -252   -348     64       C  \\nATOM    276  CD  GLU A  69      24.149  21.136   7.750  1.00 16.46           C  \\nANISOU  276  CD  GLU A  69     1326   2212   2716    104   -215    341       C  \\nATOM    277  OE1 GLU A  69      24.231  20.492   8.809  1.00 20.20           O  \\nANISOU  277  OE1 GLU A  69     1929   2742   3003    339     55    725       O  \\nATOM    278  OE2 GLU A  69      24.563  20.714   6.621  1.00 17.90           O  \\nANISOU  278  OE2 GLU A  69     1732   2191   2877    -90     75    283       O  \\nATOM    279  N   ARG A  70      21.459  19.572   5.829  1.00 12.05           N  \\nANISOU  279  N   ARG A  70     1339   1234   2002    162    166    158       N  \\nATOM    280  CA  ARG A  70      21.260  18.139   5.780  1.00 12.03           C  \\nANISOU  280  CA  ARG A  70     1363   1247   1960     71    144    -24       C  \\nATOM    281  C   ARG A  70      22.436  17.335   6.298  1.00 12.75           C  \\nANISOU  281  C   ARG A  70     1522   1299   2021    172     94     55       C  \\nATOM    282  O   ARG A  70      22.257  16.230   6.813  1.00 15.49           O  \\nANISOU  282  O   ARG A  70     1638   1377   2869    271    364    373       O  \\nATOM    283  CB  ARG A  70      20.972  17.738   4.339  1.00 11.97           C  \\nANISOU  283  CB  ARG A  70     1384   1280   1884     47    298     -8       C  \\nATOM    284  CG  ARG A  70      19.609  18.189   3.813  1.00 11.77           C  \\nANISOU  284  CG  ARG A  70     1344   1266   1860     17    271    -50       C  \\nATOM    285  CD  ARG A  70      19.436  17.996   2.317  1.00 12.61           C  \\nANISOU  285  CD  ARG A  70     1467   1325   1996     49    294   -232       C  \\nATOM    286  NE  ARG A  70      18.079  18.271   1.909  1.00 12.33           N  \\nANISOU  286  NE  ARG A  70     1475   1561   1647    -50    328   -531       N  \\nATOM    287  CZ  ARG A  70      17.645  19.364   1.289  1.00 12.42           C  \\nANISOU  287  CZ  ARG A  70     1288   1556   1873      5    257   -479       C  \\nATOM    288  NH1 ARG A  70      18.457  20.357   0.948  1.00 13.23           N  \\nANISOU  288  NH1 ARG A  70     1378   1335   2311    -36    268   -637       N  \\nATOM    289  NH2 ARG A  70      16.370  19.471   0.940  1.00 13.99           N  \\nANISOU  289  NH2 ARG A  70     1199   2132   1984   -186    227   -604       N  \\nATOM    290  N   SER A  71      23.639  17.868   6.169  1.00 14.67           N  \\nANISOU  290  N   SER A  71     1303   1595   2674    389    293    120       N  \\nATOM    291  CA  SER A  71      24.867  17.095   6.447  1.00 17.25           C  \\nANISOU  291  CA  SER A  71     1310   2263   2979    550    519    413       C  \\nATOM    292  C   SER A  71      25.043  16.782   7.938  1.00 17.94           C  \\nANISOU  292  C   SER A  71     1416   2210   3190    274    354    954       C  \\nATOM    293  O   SER A  71      25.630  15.782   8.332  1.00 24.25           O  \\nANISOU  293  O   SER A  71     1958   2678   4575    744    594   1573       O  \\nATOM    294  CB  SER A  71      26.050  17.836   5.847  1.00 19.82           C  \\nANISOU  294  CB  SER A  71      839   3352   3339    929    706    937       C  \\nATOM    295  OG  SER A  71      26.422  18.898   6.675  1.00 28.70           O  \\nANISOU  295  OG  SER A  71     2458   4350   4095   -136    360    560       O  \\nATOM    296  N   GLY A  72      24.525  17.649   8.769  1.00 17.08           N  \\nANISOU  296  N   GLY A  72     1159   2295   3034    231    145    715       N  \\nATOM    297  CA  GLY A  72      24.703  17.555  10.214  1.00 17.60           C  \\nANISOU  297  CA  GLY A  72     1644   2152   2892    103    232    711       C  \\nATOM    298  C   GLY A  72      23.704  16.677  10.947  1.00 16.43           C  \\nANISOU  298  C   GLY A  72     1478   2160   2604    188    338    463       C  \\nATOM    299  O   GLY A  72      23.699  16.659  12.169  1.00 17.97           O  \\nANISOU  299  O   GLY A  72     1600   2423   2804    339    339    955       O  \\nATOM    300  N   ALA A  73      22.801  16.020  10.213  1.00 15.51           N  \\nANISOU  300  N   ALA A  73     1644   1745   2504    244    609    432       N  \\nATOM    301  CA  ALA A  73      21.659  15.387  10.839  1.00 16.62           C  \\nANISOU  301  CA  ALA A  73     1665   1837   2811     55    557    457       C  \\nATOM    302  C   ALA A  73      22.030  14.254  11.795  1.00 17.19           C  \\nANISOU  302  C   ALA A  73     1915   1654   2959    270    801    387       C  \\nATOM    303  O   ALA A  73      21.434  14.120  12.859  1.00 19.38           O  \\nANISOU  303  O   ALA A  73     2165   1989   3210    202   1039    825       O  \\nATOM    304  CB  ALA A  73      20.683  14.932   9.790  1.00 17.21           C  \\nANISOU  304  CB  ALA A  73     2097   1584   2858    123    518     -9       C  \\nATOM    305  N   ASP A  74      23.043  13.452  11.458  1.00 22.28           N  \\nANISOU  305  N   ASP A  74     2504   2236   3724    793   1272    822       N  \\nATOM    306  CA  ASP A  74      23.475  12.400  12.402  1.00 25.73           C  \\nANISOU  306  CA  ASP A  74     3275   2258   4241   1182   1334    903       C  \\nATOM    307  C   ASP A  74      23.965  12.923  13.713  1.00 24.12           C  \\nANISOU  307  C   ASP A  74     2684   2619   3860   1259   1264   1440       C  \\nATOM    308  O   ASP A  74      23.589  12.368  14.760  1.00 30.09           O  \\nANISOU  308  O   ASP A  74     3643   3225   4562   1928   1780   2252       O  \\nATOM    309  CB  ASP A  74      24.572  11.553  11.796  1.00 29.13           C  \\nANISOU  309  CB  ASP A  74     3251   2596   5217   1345   1333    767       C  \\nATOM    310  CG  ASP A  74      24.055  10.665  10.711  1.00 34.40           C  \\nANISOU  310  CG  ASP A  74     5386   2289   5396   1778   1624    115       C  \\nATOM    311  OD1 ASP A  74      22.837  10.404  10.641  1.00 41.43           O  \\nANISOU  311  OD1 ASP A  74     6030   2977   6732    989   3329    -80       O  \\nATOM    312  OD2 ASP A  74      24.874  10.179   9.929  1.00 46.88           O  \\nANISOU  312  OD2 ASP A  74     6466   4307   7037   1646   3405    402       O  \\nATOM    313  N   LYS A  75      24.827  13.943  13.680  1.00 23.81           N  \\nANISOU  313  N   LYS A  75     2305   3248   3491   1147    852    551       N  \\nATOM    314  CA  LYS A  75      25.326  14.552  14.916  1.00 22.46           C  \\nANISOU  314  CA  LYS A  75     3866   2264   2404   1998    158   1459       C  \\nATOM    315  C   LYS A  75      24.122  15.105  15.679  1.00 24.80           C  \\nANISOU  315  C   LYS A  75     2621   3309   3491   1564   -236   1038       C  \\nATOM    316  O   LYS A  75      24.027  14.902  16.889  1.00 29.50           O  \\nANISOU  316  O   LYS A  75     2391   5104   3713   1405    139   2236       O  \\nATOM    317  CB  LYS A  75      26.287  15.737  14.646  1.00 30.01           C  \\nANISOU  317  CB  LYS A  75     3923   3542   3937    810    324    821       C  \\nATOM    318  CG  LYS A  75      27.731  15.371  14.317  1.00 40.91           C  \\nANISOU  318  CG  LYS A  75     4167   5912   5463    847    582    333       C  \\nATOM    319  CD  LYS A  75      28.624  16.616  14.208  1.00 47.87           C  \\nANISOU  319  CD  LYS A  75     6232   6097   5859   -139    247    628       C  \\nATOM    320  CE  LYS A  75      28.179  17.642  13.140  1.00 45.85           C  \\nANISOU  320  CE  LYS A  75     5743   6462   5214     29    241    389       C  \\nATOM    321  NZ  LYS A  75      28.111  17.176  11.704  1.00 34.24           N  \\nANISOU  321  NZ  LYS A  75     2393   6045   4572   -991   -473   1491       N  \\nATOM    322  N   ARG A  76      23.179  15.757  14.988  1.00 17.99           N  \\nANISOU  322  N   ARG A  76     1451   2638   2742    432    163   1261       N  \\nATOM    323  CA  ARG A  76      22.072  16.329  15.695  1.00 15.56           C  \\nANISOU  323  CA  ARG A  76     1824   1739   2348    206    -43    342       C  \\nATOM    324  C   ARG A  76      21.136  15.295  16.333  1.00 15.13           C  \\nANISOU  324  C   ARG A  76     1681   1745   2319    423     68    393       C  \\nATOM    325  O   ARG A  76      20.656  15.517  17.436  1.00 18.16           O  \\nANISOU  325  O   ARG A  76     2500   1889   2509    270    480    137       O  \\nATOM    326  CB  ARG A  76      21.266  17.267  14.790  1.00 16.28           C  \\nANISOU  326  CB  ARG A  76     1699   1751   2736     50   -431    205       C  \\nATOM    327  CG  ARG A  76      22.025  18.527  14.412  1.00 15.16           C  \\nANISOU  327  CG  ARG A  76     1628   1930   2200    -55   -444    297       C  \\nATOM    328  CD  ARG A  76      21.168  19.564  13.760  1.00 14.67           C  \\nANISOU  328  CD  ARG A  76     1730   1689   2154    -54   -414    211       C  \\nATOM    329  NE  ARG A  76      22.026  20.488  13.035  1.00 15.24           N  \\nANISOU  329  NE  ARG A  76     1332   2170   2287   -199   -484    258       N  \\nATOM    330  CZ  ARG A  76      22.323  20.448  11.753  1.00 14.54           C  \\nANISOU  330  CZ  ARG A  76     1385   1920   2218   -194   -412    356       C  \\nATOM    331  NH1 ARG A  76      21.763  19.576  10.912  1.00 14.55           N  \\nANISOU  331  NH1 ARG A  76     1588   1747   2191   -199   -118    170       N  \\nATOM    332  NH2 ARG A  76      23.199  21.338  11.272  1.00 17.41           N  \\nANISOU  332  NH2 ARG A  76     1617   2294   2701   -447   -221    356       N  \\nATOM    333  N   ARG A  77      20.884  14.172  15.676  1.00 18.60           N  \\nANISOU  333  N   ARG A  77     2562   1660   2843    298    934    177       N  \\nATOM    334  CA  ARG A  77      20.182  13.077  16.339  1.00 22.77           C  \\nANISOU  334  CA  ARG A  77     2956   2398   3296   -312   1256    110       C  \\nATOM    335  C   ARG A  77      20.908  12.549  17.550  1.00 25.58           C  \\nANISOU  335  C   ARG A  77     3621   2267   3830     -8   1244    807       C  \\nATOM    336  O   ARG A  77      20.246  12.195  18.552  1.00 27.36           O  \\nANISOU  336  O   ARG A  77     4617   2142   3637   -242   1726    187       O  \\nATOM    337  CB  ARG A  77      20.015  11.913  15.391  1.00 25.93           C  \\nANISOU  337  CB  ARG A  77     3712   2687   3454   -160    642   -123       C  \\nATOM    338  CG  ARG A  77      18.879  12.150  14.484  1.00 24.45           C  \\nANISOU  338  CG  ARG A  77     3589   2396   3303    705    476   -842       C  \\nATOM    339  CD  ARG A  77      18.265  10.889  13.850  1.00 30.79           C  \\nANISOU  339  CD  ARG A  77     4938   3620   3138   -144    495  -1652       C  \\nATOM    340  NE  ARG A  77      17.816  11.266  12.504  1.00 33.79           N  \\nANISOU  340  NE  ARG A  77     4150   4623   4065    146    349  -1360       N  \\nATOM    341  CZ  ARG A  77      18.654  11.407  11.479  1.00 30.20           C  \\nANISOU  341  CZ  ARG A  77     3270   5423   2781    765   -301  -1792       C  \\nATOM    342  NH1 ARG A  77      19.939  11.132  11.620  1.00 34.91           N  \\nANISOU  342  NH1 ARG A  77     3372   2732   7158    744  -1793  -1028       N  \\nATOM    343  NH2 ARG A  77      18.212  11.803  10.324  1.00 36.26           N  \\nANISOU  343  NH2 ARG A  77     4471   4747   4556    201   -255    750       N  \\nATOM    344  N   GLN A  78      22.242  12.435  17.465  1.00 28.13           N  \\nANISOU  344  N   GLN A  78     3867   2397   4421   1361   1259   1022       N  \\nATOM    345  CA  GLN A  78      23.023  11.925  18.636  1.00 33.97           C  \\nANISOU  345  CA  GLN A  78     4602   3519   4782   1564   1114   2410       C  \\nATOM    346  C   GLN A  78      22.686  12.800  19.822  1.00 33.47           C  \\nANISOU  346  C   GLN A  78     4271   4028   4416   1335    522   2311       C  \\nATOM    347  O   GLN A  78      22.396  12.314  20.901  1.00 42.10           O  \\nANISOU  347  O   GLN A  78     5175   6128   4691   2076   1075   3104       O  \\nATOM    348  CB  GLN A  78      24.557  12.020  18.451  1.00 39.30           C  \\nANISOU  348  CB  GLN A  78     4377   5545   5009   1420   -476   1819       C  \\nATOM    349  CG  GLN A  78      25.307  10.885  17.763  1.00 49.80           C  \\nANISOU  349  CG  GLN A  78     7001   5415   6502    800   -188    598       C  \\nATOM    350  CD  GLN A  78      25.063   9.518  18.374  1.00 50.32           C  \\nANISOU  350  CD  GLN A  78     7770   7188   4159     83  -1373   1923       C  \\nATOM    351  OE1 GLN A  78      24.083   8.837  18.049  1.00 62.39           O  \\nANISOU  351  OE1 GLN A  78     8342   9088   6275   -659   -825   -757       O  \\nATOM    352  NE2 GLN A  78      25.966   9.098  19.249  1.00 58.57           N  \\nANISOU  352  NE2 GLN A  78     6073   8565   7613   1102  -2420    222       N  \\nATOM    353  N   GLU A  79      22.690  14.109  19.587  1.00 27.21           N  \\nANISOU  353  N   GLU A  79     2382   3963   3992    707    503   2307       N  \\nATOM    354  CA  GLU A  79      22.455  15.081  20.621  1.00 30.44           C  \\nANISOU  354  CA  GLU A  79     2610   4599   4356    707   -671   1356       C  \\nATOM    355  C   GLU A  79      21.007  15.037  21.105  1.00 23.43           C  \\nANISOU  355  C   GLU A  79     2770   3280   2852    360   -658    784       C  \\nATOM    356  O   GLU A  79      20.763  14.951  22.296  1.00 25.22           O  \\nANISOU  356  O   GLU A  79     2767   4028   2785     -4   -709    971       O  \\nATOM    357  CB  GLU A  79      22.806  16.487  20.121  1.00 33.74           C  \\nANISOU  357  CB  GLU A  79     3863   5595   3360   -691  -2148   1776       C  \\nATOM    358  CG  GLU A  79      24.287  16.687  19.740  1.00 43.00           C  \\nANISOU  358  CG  GLU A  79     4356   6645   5336     61   -444    671       C  \\nATOM    359  CD  GLU A  79      25.284  16.603  20.899  1.00 56.18           C  \\nANISOU  359  CD  GLU A  79     5539   9068   6738    -99  -1520    583       C  \\nATOM    360  OE1 GLU A  79      24.888  16.357  22.063  1.00 71.67           O  \\nANISOU  360  OE1 GLU A  79     9744   9972   7512  -1021    -22    686       O  \\nATOM    361  OE2 GLU A  79      26.497  16.788  20.641  1.00 74.16           O  \\nANISOU  361  OE2 GLU A  79     6265  11159  10752   -577    600    318       O  \\nATOM    362  N   SER A  80      20.040  15.059  20.205  1.00 18.47           N  \\nANISOU  362  N   SER A  80     2492   2035   2490    145   -319    394       N  \\nATOM    363  CA  SER A  80      18.649  15.213  20.665  1.00 18.60           C  \\nANISOU  363  CA  SER A  80     2206   2292   2567   -109   -386    243       C  \\nATOM    364  C   SER A  80      18.162  13.992  21.383  1.00 16.37           C  \\nANISOU  364  C   SER A  80     2099   2128   1994     72   -523    106       C  \\nATOM    365  O   SER A  80      17.353  14.107  22.307  1.00 17.66           O  \\nANISOU  365  O   SER A  80     2551   2255   1902    -38   -456     86       O  \\nATOM    366  CB  SER A  80      17.705  15.476  19.522  1.00 19.06           C  \\nANISOU  366  CB  SER A  80     2375   2435   2430     79   -241    605       C  \\nATOM    367  OG  SER A  80      17.866  14.451  18.633  1.00 21.31           O  \\nANISOU  367  OG  SER A  80     2590   2902   2603    404   -306    507       O  \\nATOM    368  N   LEU A  81      18.668  12.817  20.971  1.00 16.04           N  \\nANISOU  368  N   LEU A  81     1919   2008   2166     64   -211    477       N  \\nATOM    369  CA  LEU A  81      18.158  11.532  21.481  1.00 17.24           C  \\nANISOU  369  CA  LEU A  81     2120   2182   2247    -13   -411    565       C  \\nATOM    370  C   LEU A  81      18.954  10.962  22.641  1.00 19.79           C  \\nANISOU  370  C   LEU A  81     2768   2207   2541    220   -600    755       C  \\nATOM    371  O   LEU A  81      18.628   9.881  23.118  1.00 20.53           O  \\nANISOU  371  O   LEU A  81     2878   2559   2362   -274   -618    959       O  \\nATOM    372  CB  LEU A  81      18.056  10.522  20.341  1.00 19.18           C  \\nANISOU  372  CB  LEU A  81     2538   2249   2500    460   -269    317       C  \\nATOM    373  CG  LEU A  81      17.161  10.944  19.178  1.00 20.40           C  \\nANISOU  373  CG  LEU A  81     2893   2317   2540    569   -508    121       C  \\nATOM    374  CD1 LEU A  81      17.027   9.781  18.206  1.00 25.01           C  \\nANISOU  374  CD1 LEU A  81     4191   2301   3010    729   -309   -137       C  \\nATOM    375  CD2 LEU A  81      15.775  11.364  19.615  1.00 21.17           C  \\nANISOU  375  CD2 LEU A  81     2321   2710   3013   -140   -677    170       C  \\nATOM    376  N   LYS A  82      20.006  11.647  23.061  1.00 21.79           N  \\nANISOU  376  N   LYS A  82     2933   2721   2622   -152   -930   1122       N  \\nATOM    377  CA  LYS A  82      20.885  11.130  24.114  1.00 26.48           C  \\nANISOU  377  CA  LYS A  82     3382   3396   3283    225  -1441   1130       C  \\nATOM    378  C   LYS A  82      20.052  10.746  25.331  1.00 24.96           C  \\nANISOU  378  C   LYS A  82     3499   3459   2525   -165  -1819    719       C  \\nATOM    379  O   LYS A  82      19.281  11.545  25.825  1.00 26.26           O  \\nANISOU  379  O   LYS A  82     3539   3430   3006   -558  -1381    862       O  \\nATOM    380  CB  LYS A  82      21.930  12.200  24.505  1.00 29.81           C  \\nANISOU  380  CB  LYS A  82     3555   3910   3859   -191  -1328   1162       C  \\nATOM    381  CG  LYS A  82      23.039  11.712  25.439  1.00 32.71           C  \\nANISOU  381  CG  LYS A  82     3945   5067   3414    397  -1173    999       C  \\nATOM    382  CD  LYS A  82      23.763  12.886  26.086  1.00 41.51           C  \\nANISOU  382  CD  LYS A  82     4784   5784   5203   -509   -677    264       C  \\nATOM    383  CE  LYS A  82      24.834  12.397  27.051  1.00 44.72           C  \\nANISOU  383  CE  LYS A  82     5308   6213   5469    348   -489    370       C  \\nATOM    384  NZ  LYS A  82      25.988  11.788  26.340  1.00 48.96           N  \\nANISOU  384  NZ  LYS A  82     5779   6414   6409    437    125     63       N  \\nATOM    385  N   GLY A  83      20.213   9.518  25.818  1.00 25.97           N  \\nANISOU  385  N   GLY A  83     3100   3734   3033    238  -1300   1012       N  \\nATOM    386  CA  GLY A  83      19.551   9.094  27.050  1.00 26.22           C  \\nANISOU  386  CA  GLY A  83     3176   3852   2931    -35  -1172    847       C  \\nATOM    387  C   GLY A  83      18.055   8.791  26.967  1.00 24.32           C  \\nANISOU  387  C   GLY A  83     2997   3842   2402    283   -721   1355       C  \\nATOM    388  O   GLY A  83      17.444   8.441  27.968  1.00 34.77           O  \\nANISOU  388  O   GLY A  83     5190   5425   2595  -1296   -205   1204       O  \\nATOM    389  N   ILE A  84      17.470   8.888  25.771  1.00 23.61           N  \\nANISOU  389  N   ILE A  84     3180   3221   2569    -45   -975    963       N  \\nATOM    390  CA  ILE A  84      16.050   8.622  25.575  1.00 21.65           C  \\nANISOU  390  CA  ILE A  84     3074   2619   2531     23   -409    445       C  \\nATOM    391  C   ILE A  84      15.903   7.170  25.095  1.00 21.50           C  \\nANISOU  391  C   ILE A  84     2850   2613   2706    182   -612    414       C  \\nATOM    392  O   ILE A  84      16.314   6.833  23.990  1.00 22.78           O  \\nANISOU  392  O   ILE A  84     2809   2998   2847    175   -609    -69       O  \\nATOM    393  CB  ILE A  84      15.422   9.595  24.579  1.00 21.87           C  \\nANISOU  393  CB  ILE A  84     2773   2797   2738    159   -298    600       C  \\nATOM    394  CG1 ILE A  84      15.500  11.008  25.147  1.00 23.17           C  \\nANISOU  394  CG1 ILE A  84     3035   2833   2934   -237   -533    710       C  \\nATOM    395  CG2 ILE A  84      13.962   9.221  24.328  1.00 20.45           C  \\nANISOU  395  CG2 ILE A  84     2937   2456   2375    -84   -431    491       C  \\nATOM    396  CD1 ILE A  84      14.953  12.075  24.247  1.00 22.91           C  \\nANISOU  396  CD1 ILE A  84     3418   2795   2492    148   -521    268       C  \\nATOM    397  N   PRO A  85      15.299   6.311  25.930  1.00 24.84           N  \\nANISOU  397  N   PRO A  85     3210   3419   2807     81   -529    895       N  \\nATOM    398  CA  PRO A  85      15.221   4.913  25.530  1.00 27.60           C  \\nANISOU  398  CA  PRO A  85     3247   3760   3477     15  -1292    814       C  \\nATOM    399  C   PRO A  85      14.257   4.697  24.383  1.00 25.12           C  \\nANISOU  399  C   PRO A  85     3090   2907   3546   -271  -1375   1383       C  \\nATOM    400  O   PRO A  85      13.374   5.516  24.139  1.00 27.39           O  \\nANISOU  400  O   PRO A  85     3102   3067   4238   -322  -1274   1861       O  \\nATOM    401  CB  PRO A  85      14.690   4.214  26.775  1.00 29.13           C  \\nANISOU  401  CB  PRO A  85     3924   3020   4124    107  -1037   1494       C  \\nATOM    402  CG  PRO A  85      13.952   5.267  27.529  1.00 30.25           C  \\nANISOU  402  CG  PRO A  85     4166   3556   3770   -147   -610   1308       C  \\nATOM    403  CD  PRO A  85      14.646   6.563  27.230  1.00 27.51           C  \\nANISOU  403  CD  PRO A  85     4106   3497   2850     23   -324   1280       C  \\nATOM    404  N   THR A  86      14.452   3.596  23.683  1.00 29.56           N  \\nANISOU  404  N   THR A  86     3575   3330   4324    782  -1475   1104       N  \\nATOM    405  CA  THR A  86      13.554   3.221  22.611  1.00 33.08           C  \\nANISOU  405  CA  THR A  86     5479   2083   5004    282  -2569   1573       C  \\nATOM    406  C   THR A  86      12.330   2.602  23.210  1.00 33.21           C  \\nANISOU  406  C   THR A  86     5243   3012   4360   -175  -2806   1246       C  \\nATOM    407  O   THR A  86      12.324   2.194  24.373  1.00 36.34           O  \\nANISOU  407  O   THR A  86     5467   3454   4885   -934  -3028   1708       O  \\nATOM    408  CB  THR A  86      14.197   2.295  21.592  1.00 34.29           C  \\nANISOU  408  CB  THR A  86     4892   2625   5509   -108  -1941    712       C  \\nATOM    409  OG1 THR A  86      14.518   1.079  22.229  1.00 42.69           O  \\nANISOU  409  OG1 THR A  86     6307   2134   7778    891  -2351    299       O  \\nATOM    410  CG2 THR A  86      15.476   2.948  21.018  1.00 35.37           C  \\nANISOU  410  CG2 THR A  86     4793   4111   4533    676   -765    272       C  \\nATOM    411  N   LYS A  87      11.254   2.632  22.432  1.00 29.53           N  \\nANISOU  411  N   LYS A  87     4295   2239   4684   -470  -2390   1546       N  \\nATOM    412  CA  LYS A  87       9.984   2.043  22.881  1.00 32.34           C  \\nANISOU  412  CA  LYS A  87     5470   2589   4228  -1024  -1989   1933       C  \\nATOM    413  C   LYS A  87       9.364   1.296  21.710  1.00 34.07           C  \\nANISOU  413  C   LYS A  87     5482   2359   5103   -757  -2240   1272       C  \\nATOM    414  O   LYS A  87       9.349   1.793  20.575  1.00 32.47           O  \\nANISOU  414  O   LYS A  87     4980   1678   5677   -833  -2704   1685       O  \\nATOM    415  CB  LYS A  87       9.106   3.168  23.395  1.00 33.39           C  \\nANISOU  415  CB  LYS A  87     6308   3135   3242   -651  -1542   2628       C  \\nATOM    416  CG  LYS A  87       7.703   2.860  23.883  1.00 43.91           C  \\nANISOU  416  CG  LYS A  87     6353   4579   5752  -1013   -656   1342       C  \\nATOM    417  CD  LYS A  87       7.144   4.140  24.515  1.00 50.08           C  \\nANISOU  417  CD  LYS A  87     6730   5554   6742   -613   -488    601       C  \\nATOM    418  CE  LYS A  87       5.636   4.251  24.439  1.00 52.22           C  \\nANISOU  418  CE  LYS A  87     6644   6224   6971   -515   -694    172       C  \\nATOM    419  NZ  LYS A  87       4.988   3.311  25.387  1.00 57.09           N  \\nANISOU  419  NZ  LYS A  87     7105   7115   7468   -999   -730    317       N  \\nATOM    420  N   PRO A  88       8.880   0.068  21.960  1.00 32.37           N  \\nANISOU  420  N   PRO A  88     4788   2594   4916   -757  -2425   1645       N  \\nATOM    421  CA  PRO A  88       8.492  -0.727  20.801  1.00 26.65           C  \\nANISOU  421  CA  PRO A  88     3716   1820   4589   -250  -1604   1316       C  \\nATOM    422  C   PRO A  88       7.238  -0.181  20.137  1.00 26.81           C  \\nANISOU  422  C   PRO A  88     3463   2409   4315   -631  -1661   1294       C  \\nATOM    423  O   PRO A  88       6.312   0.240  20.821  1.00 27.34           O  \\nANISOU  423  O   PRO A  88     4107   2181   4098   -708  -1446   1470       O  \\nATOM    424  CB  PRO A  88       8.295  -2.145  21.398  1.00 30.26           C  \\nANISOU  424  CB  PRO A  88     4383   1971   5144   -578   -784   1314       C  \\nATOM    425  CG  PRO A  88       8.031  -1.896  22.860  1.00 30.89           C  \\nANISOU  425  CG  PRO A  88     4296   2201   5239   -403    -10   1262       C  \\nATOM    426  CD  PRO A  88       8.934  -0.763  23.185  1.00 30.92           C  \\nANISOU  426  CD  PRO A  88     4303   2513   4931    -55  -2112   1617       C  \\nATOM    427  N   GLY A  89       7.235  -0.162  18.810  1.00 23.11           N  \\nANISOU  427  N   GLY A  89     3051   1294   4435   -476  -1012    878       N  \\nATOM    428  CA  GLY A  89       6.103   0.383  18.050  1.00 19.81           C  \\nANISOU  428  CA  GLY A  89     2875   1562   3087   -678   -843    771       C  \\nATOM    429  C   GLY A  89       6.183   1.898  17.797  1.00 17.43           C  \\nANISOU  429  C   GLY A  89     2147   1413   3060   -201   -448    451       C  \\nATOM    430  O   GLY A  89       5.324   2.442  17.097  1.00 17.38           O  \\nANISOU  430  O   GLY A  89     2149   1520   2933   -113   -353    591       O  \\nATOM    431  N   PHE A  90       7.162   2.575  18.429  1.00 17.35           N  \\nANISOU  431  N   PHE A  90     2202   1294   3096   -272   -670    857       N  \\nATOM    432  CA  PHE A  90       7.278   4.025  18.383  1.00 15.80           C  \\nANISOU  432  CA  PHE A  90     1956   1253   2795    -50   -235    720       C  \\nATOM    433  C   PHE A  90       8.653   4.417  17.929  1.00 16.98           C  \\nANISOU  433  C   PHE A  90     1978   1431   3042   -116   -260    872       C  \\nATOM    434  O   PHE A  90       9.650   3.746  18.233  1.00 22.11           O  \\nANISOU  434  O   PHE A  90     2071   1960   4367     58   -301   1314       O  \\nATOM    435  CB  PHE A  90       7.014   4.639  19.757  1.00 18.53           C  \\nANISOU  435  CB  PHE A  90     2334   2055   2651    184   -373    691       C  \\nATOM    436  CG  PHE A  90       5.623   4.480  20.219  1.00 19.08           C  \\nANISOU  436  CG  PHE A  90     2720   2147   2383   -224     80    763       C  \\nATOM    437  CD1 PHE A  90       5.213   3.306  20.792  1.00 27.96           C  \\nANISOU  437  CD1 PHE A  90     4095   2739   3788     99    860   1802       C  \\nATOM    438  CD2 PHE A  90       4.706   5.475  20.030  1.00 19.74           C  \\nANISOU  438  CD2 PHE A  90     2278   2393   2828   -354    -70   1140       C  \\nATOM    439  CE1 PHE A  90       3.898   3.122  21.192  1.00 37.34           C  \\nANISOU  439  CE1 PHE A  90     5002   2817   6366    270   2206   3101       C  \\nATOM    440  CE2 PHE A  90       3.399   5.330  20.413  1.00 22.15           C  \\nANISOU  440  CE2 PHE A  90     2406   2482   3526    -46    318    844       C  \\nATOM    441  CZ  PHE A  90       2.992   4.150  20.998  1.00 31.01           C  \\nANISOU  441  CZ  PHE A  90     3457   2850   5474   -431    788   1372       C  \\nATOM    442  N   ASP A  91       8.702   5.520  17.184  1.00 13.09           N  \\nANISOU  442  N   ASP A  91     1851   1202   1919   -108   -393    347       N  \\nATOM    443  CA  ASP A  91       9.954   6.185  16.916  1.00 13.60           C  \\nANISOU  443  CA  ASP A  91     1540   1473   2154    -16   -237   -141       C  \\nATOM    444  C   ASP A  91      10.054   7.442  17.785  1.00 12.49           C  \\nANISOU  444  C   ASP A  91     1716   1163   1865     63   -247    152       C  \\nATOM    445  O   ASP A  91       9.068   7.932  18.265  1.00 14.49           O  \\nANISOU  445  O   ASP A  91     1578   1633   2294    -56   -195   -102       O  \\nATOM    446  CB  ASP A  91      10.059   6.532  15.435  1.00 16.51           C  \\nANISOU  446  CB  ASP A  91     2147   1807   2317   -152   -115     -3       C  \\nATOM    447  CG  ASP A  91      10.183   5.284  14.537  1.00 17.56           C  \\nANISOU  447  CG  ASP A  91     2517   1558   2595    104   -163    103       C  \\nATOM    448  OD1 ASP A  91      10.466   4.151  15.022  1.00 19.63           O  \\nANISOU  448  OD1 ASP A  91     2849   1416   3191     89    -87    152       O  \\nATOM    449  OD2 ASP A  91       9.968   5.428  13.349  1.00 18.72           O  \\nANISOU  449  OD2 ASP A  91     2926   1348   2837     89   -264    205       O  \\nATOM    450  N   ARG A  92      11.274   7.920  17.992  1.00 13.92           N  \\nANISOU  450  N   ARG A  92     1679   1362   2245      2   -266    -38       N  \\nATOM    451  CA  ARG A  92      11.517   9.134  18.796  1.00 13.52           C  \\nANISOU  451  CA  ARG A  92     1749   1388   2000    -11    -28     40       C  \\nATOM    452  C   ARG A  92      11.585  10.319  17.864  1.00 14.72           C  \\nANISOU  452  C   ARG A  92     2037   1421   2136     49    -33    166       C  \\nATOM    453  O   ARG A  92      12.614  10.550  17.239  1.00 20.23           O  \\nANISOU  453  O   ARG A  92     2205   2067   3415     27    289    875       O  \\nATOM    454  CB  ARG A  92      12.823   8.985  19.568  1.00 14.16           C  \\nANISOU  454  CB  ARG A  92     1813   1262   2304      0   -198   -124       C  \\nATOM    455  CG  ARG A  92      12.767   7.879  20.618  1.00 16.67           C  \\nANISOU  455  CG  ARG A  92     2040   1529   2762   -101   -207    267       C  \\nATOM    456  CD  ARG A  92      14.092   7.444  21.159  1.00 18.43           C  \\nANISOU  456  CD  ARG A  92     2451   2052   2499     58   -391    252       C  \\nATOM    457  NE  ARG A  92      14.848   6.812  20.092  1.00 19.44           N  \\nANISOU  457  NE  ARG A  92     2482   1942   2959    251   -157    309       N  \\nATOM    458  CZ  ARG A  92      16.137   6.533  20.163  1.00 22.11           C  \\nANISOU  458  CZ  ARG A  92     2632   2478   3290    631   -135      5       C  \\nATOM    459  NH1 ARG A  92      16.834   6.788  21.269  1.00 25.51           N  \\nANISOU  459  NH1 ARG A  92     3060   2955   3676    777   -604    291       N  \\nATOM    460  NH2 ARG A  92      16.741   6.035  19.122  1.00 21.57           N  \\nANISOU  460  NH2 ARG A  92     2671   2681   2842    675   -660   -291       N  \\nATOM    461  N   ASP A  93      10.470  11.029  17.732  1.00 11.71           N  \\nANISOU  461  N   ASP A  93     1687   1173   1587   -287    -12    -32       N  \\nATOM    462  CA  ASP A  93      10.360  12.157  16.827  1.00 12.53           C  \\nANISOU  462  CA  ASP A  93     1996   1147   1615   -165   -117    -49       C  \\nATOM    463  C   ASP A  93      10.990  13.377  17.446  1.00 10.96           C  \\nANISOU  463  C   ASP A  93     1678   1108   1377     24   -229    -42       C  \\nATOM    464  O   ASP A  93      10.770  13.661  18.635  1.00 13.28           O  \\nANISOU  464  O   ASP A  93     2152   1480   1413   -269   -251   -209       O  \\nATOM    465  CB  ASP A  93       8.883  12.492  16.564  1.00 12.78           C  \\nANISOU  465  CB  ASP A  93     1841   1391   1624   -312   -212     17       C  \\nATOM    466  CG  ASP A  93       8.702  13.577  15.474  1.00 14.23           C  \\nANISOU  466  CG  ASP A  93     2152   1560   1693   -300   -277     88       C  \\nATOM    467  OD1 ASP A  93       9.212  13.366  14.329  1.00 16.83           O  \\nANISOU  467  OD1 ASP A  93     2631   1842   1921     59    -52   -189       O  \\nATOM    468  OD2 ASP A  93       8.098  14.632  15.788  1.00 15.24           O  \\nANISOU  468  OD2 ASP A  93     2283   2007   1500    -25   -327   -201       O  \\nATOM    469  N   GLU A  94      11.690  14.148  16.615  1.00 11.49           N  \\nANISOU  469  N   GLU A  94     1876   1193   1295    -55   -263   -157       N  \\nATOM    470  CA  GLU A  94      12.389  15.342  17.062  1.00 11.18           C  \\nANISOU  470  CA  GLU A  94     1752   1102   1394     46   -367   -261       C  \\nATOM    471  C   GLU A  94      11.832  16.580  16.369  1.00 10.09           C  \\nANISOU  471  C   GLU A  94     1416   1179   1236    -18   -102   -140       C  \\nATOM    472  O   GLU A  94      11.668  16.591  15.141  1.00 11.69           O  \\nANISOU  472  O   GLU A  94     1940   1233   1269    -19   -165   -127       O  \\nATOM    473  CB  GLU A  94      13.862  15.270  16.627  1.00 14.85           C  \\nANISOU  473  CB  GLU A  94     1858   1391   2392   -102   -286   -275       C  \\nATOM    474  CG  GLU A  94      14.665  14.085  17.018  1.00 16.79           C  \\nANISOU  474  CG  GLU A  94     2207   1821   2352    -37   -361    258       C  \\nATOM    475  CD  GLU A  94      15.868  13.874  16.102  1.00 21.15           C  \\nANISOU  475  CD  GLU A  94     2309   2580   3145    125   -183   -252       C  \\nATOM    476  OE1 GLU A  94      16.840  14.690  16.143  1.00 20.28           O  \\nANISOU  476  OE1 GLU A  94     2703   2039   2962     72   -604    206       O  \\nATOM    477  OE2 GLU A  94      15.787  12.952  15.261  1.00 28.95           O  \\nANISOU  477  OE2 GLU A  94     3623   3423   3954    140    405  -1159       O  \\nATOM    478  N   TRP A  95      11.629  17.644  17.143  1.00  9.84           N  \\nANISOU  478  N   TRP A  95     1612   1111   1015     11   -321    -32       N  \\nATOM    479  CA  TRP A  95      11.333  18.957  16.588  1.00  9.54           C  \\nANISOU  479  CA  TRP A  95     1523   1000   1101   -123   -160    -27       C  \\nATOM    480  C   TRP A  95      12.117  19.968  17.428  1.00  9.80           C  \\nANISOU  480  C   TRP A  95     1401   1077   1243      1   -120   -201       C  \\nATOM    481  O   TRP A  95      11.882  20.028  18.647  1.00 10.80           O  \\nANISOU  481  O   TRP A  95     1604   1313   1184    -18   -179   -107       O  \\nATOM    482  CB  TRP A  95       9.857  19.299  16.555  1.00  9.22           C  \\nANISOU  482  CB  TRP A  95     1559   1034    907     31   -168   -122       C  \\nATOM    483  CG  TRP A  95       9.661  20.600  15.879  1.00  9.81           C  \\nANISOU  483  CG  TRP A  95     1466   1042   1220     19    -52    -36       C  \\nATOM    484  CD1 TRP A  95       9.702  21.861  16.448  1.00 10.50           C  \\nANISOU  484  CD1 TRP A  95     1542   1088   1358     91     14    -71       C  \\nATOM    485  CD2 TRP A  95       9.546  20.792  14.476  1.00  9.53           C  \\nANISOU  485  CD2 TRP A  95     1268   1123   1228    155   -144   -202       C  \\nATOM    486  NE1 TRP A  95       9.585  22.813  15.461  1.00 10.90           N  \\nANISOU  486  NE1 TRP A  95     1701   1146   1292    126    113    -25       N  \\nATOM    487  CE2 TRP A  95       9.526  22.187  14.239  1.00  9.99           C  \\nANISOU  487  CE2 TRP A  95     1462   1226   1105    318    -59    113       C  \\nATOM    488  CE3 TRP A  95       9.511  19.910  13.377  1.00 10.03           C  \\nANISOU  488  CE3 TRP A  95     1418   1375   1017    191    -33   -205       C  \\nATOM    489  CZ2 TRP A  95       9.403  22.711  12.962  1.00 11.57           C  \\nANISOU  489  CZ2 TRP A  95     1644   1553   1198    218   -249    229       C  \\nATOM    490  CZ3 TRP A  95       9.409  20.444  12.113  1.00 12.14           C  \\nANISOU  490  CZ3 TRP A  95     1649   1830   1133    250   -236   -106       C  \\nATOM    491  CH2 TRP A  95       9.324  21.828  11.920  1.00 13.19           C  \\nANISOU  491  CH2 TRP A  95     1965   1853   1192    285   -287    125       C  \\nATOM    492  N   PRO A  96      12.965  20.806  16.840  1.00  9.33           N  \\nANISOU  492  N   PRO A  96     1500    995   1050     10   -163   -195       N  \\nATOM    493  CA  PRO A  96      13.261  20.882  15.415  1.00  9.35           C  \\nANISOU  493  CA  PRO A  96     1385   1065   1101    -26    -62   -183       C  \\nATOM    494  C   PRO A  96      13.885  19.588  14.891  1.00  9.49           C  \\nANISOU  494  C   PRO A  96     1296   1069   1241     31   -248   -150       C  \\nATOM    495  O   PRO A  96      14.501  18.835  15.647  1.00 10.41           O  \\nANISOU  495  O   PRO A  96     1370   1318   1268    107   -413    -95       O  \\nATOM    496  CB  PRO A  96      14.182  22.101  15.297  1.00  9.90           C  \\nANISOU  496  CB  PRO A  96     1364   1124   1273    -91   -213    -46       C  \\nATOM    497  CG  PRO A  96      13.901  22.890  16.566  1.00  9.96           C  \\nANISOU  497  CG  PRO A  96     1440    892   1451    -45   -230   -102       C  \\nATOM    498  CD  PRO A  96      13.648  21.869  17.602  1.00  9.77           C  \\nANISOU  498  CD  PRO A  96     1446   1024   1240   -140    -47   -217       C  \\nATOM    499  N   MET A  97      13.740  19.413  13.598  1.00  9.08           N  \\nANISOU  499  N   MET A  97     1331    977   1142    156    -52     -6       N  \\nATOM    500  CA  MET A  97      14.249  18.213  12.948  1.00  9.36           C  \\nANISOU  500  CA  MET A  97     1272    940   1343    126   -176    -97       C  \\nATOM    501  C   MET A  97      15.768  18.220  12.860  1.00  9.59           C  \\nANISOU  501  C   MET A  97     1225   1098   1319    -97   -153     26       C  \\nATOM    502  O   MET A  97      16.431  19.237  12.795  1.00 10.61           O  \\nANISOU  502  O   MET A  97     1446    989   1596    -83   -102   -162       O  \\nATOM    503  CB  MET A  97      13.706  18.115  11.511  1.00 10.06           C  \\nANISOU  503  CB  MET A  97     1372   1187   1261    113    -45   -320       C  \\nATOM    504  CG  MET A  97      12.191  18.017  11.480  1.00 11.12           C  \\nANISOU  504  CG  MET A  97     1449   1355   1419    -76    -28   -194       C  \\nATOM    505  SD  MET A  97      11.524  17.886   9.820  1.00 13.65           S  \\nANISOU  505  SD  MET A  97     1781   1673   1731     74   -393   -428       S  \\nATOM    506  CE  MET A  97      12.018  19.454   9.127  1.00 15.04           C  \\nANISOU  506  CE  MET A  97     2002   1868   1843    170   -363   -193       C  \\nATOM    507  N   ALA A  98      16.330  17.018  12.784  1.00 10.05           N  \\nANISOU  507  N   ALA A  98     1101   1170   1545    -32     95     38       N  \\nATOM    508  CA  ALA A  98      17.770  16.857  12.714  1.00 10.21           C  \\nANISOU  508  CA  ALA A  98     1170   1252   1454    119    -40   -164       C  \\nATOM    509  C   ALA A  98      18.353  17.422  11.410  1.00  9.81           C  \\nANISOU  509  C   ALA A  98     1124   1042   1561    107    -43      8       C  \\nATOM    510  O   ALA A  98      19.523  17.783  11.398  1.00 11.53           O  \\nANISOU  510  O   ALA A  98     1115   1317   1946     29    -60     52       O  \\nATOM    511  CB  ALA A  98      18.128  15.397  12.839  1.00 11.62           C  \\nANISOU  511  CB  ALA A  98     1485   1247   1682     88    -69    -19       C  \\nATOM    512  N   MET A  99      17.557  17.538  10.330  1.00 10.35           N  \\nANISOU  512  N   MET A  99     1237   1152   1541     62      2     96       N  \\nATOM    513  CA  MET A  99      17.998  18.125   9.072  1.00 10.92           C  \\nANISOU  513  CA  MET A  99     1408   1217   1522     25    146   -130       C  \\nATOM    514  C   MET A  99      17.973  19.635   9.087  1.00 10.93           C  \\nANISOU  514  C   MET A  99     1360   1251   1539    -16   -113     68       C  \\nATOM    515  O   MET A  99      18.169  20.230   8.023  1.00 12.02           O  \\nANISOU  515  O   MET A  99     1654   1318   1594   -187    -73    136       O  \\nATOM    516  CB  MET A  99      17.193  17.579   7.868  1.00 10.94           C  \\nANISOU  516  CB  MET A  99     1356   1320   1481    184    288   -255       C  \\nATOM    517  CG  MET A  99      15.744  17.952   7.903  1.00 11.76           C  \\nANISOU  517  CG  MET A  99     1244   1470   1752     11     -5   -299       C  \\nATOM    518  SD  MET A  99      14.835  17.519   6.433  1.00 13.31           S  \\nANISOU  518  SD  MET A  99     1572   1709   1774   -183     -3   -322       S  \\nATOM    519  CE  MET A  99      15.564  18.628   5.220  1.00 14.59           C  \\nANISOU  519  CE  MET A  99     1828   2231   1485   -212    172   -321       C  \\nATOM    520  N   CYS A 100      17.750  20.268  10.237  1.00 11.15           N  \\nANISOU  520  N   CYS A 100     1535   1111   1590    -87   -105    107       N  \\nATOM    521  CA  CYS A 100      17.708  21.723  10.354  1.00 11.84           C  \\nANISOU  521  CA  CYS A 100     1749   1142   1605   -138   -199     38       C  \\nATOM    522  C   CYS A 100      18.800  22.202  11.289  1.00 11.06           C  \\nANISOU  522  C   CYS A 100     1407   1036   1758     33   -118     -3       C  \\nATOM    523  O   CYS A 100      19.101  21.547  12.281  1.00 12.39           O  \\nANISOU  523  O   CYS A 100     1541   1204   1961    -25   -346     87       O  \\nATOM    524  CB  CYS A 100      16.372  22.102  10.905  1.00 13.43           C  \\nANISOU  524  CB  CYS A 100     1664   1321   2118    184   -340   -152       C  \\nATOM    525  SG  CYS A 100      14.948  21.413  10.078  1.00 15.57           S  \\nANISOU  525  SG  CYS A 100     1906   1645   2362     91   -311     24       S  \\nATOM    526  N   GLU A 101      19.315  23.389  11.007  1.00 11.14           N  \\nANISOU  526  N   GLU A 101     1436   1085   1709    -60   -355    -22       N  \\nATOM    527  CA  GLU A 101      20.298  23.987  11.913  1.00 12.77           C  \\nANISOU  527  CA  GLU A 101     1286   1704   1861    -50   -488      2       C  \\nATOM    528  C   GLU A 101      19.807  24.073  13.355  1.00 12.30           C  \\nANISOU  528  C   GLU A 101     1543   1301   1829    -97   -361    -52       C  \\nATOM    529  O   GLU A 101      20.575  24.014  14.352  1.00 15.64           O  \\nANISOU  529  O   GLU A 101     1819   2114   2008   -277   -550    182       O  \\nATOM    530  CB  GLU A 101      20.616  25.416  11.417  1.00 15.76           C  \\nANISOU  530  CB  GLU A 101     2065   1690   2230   -433   -615   -187       C  \\nATOM    531  CG  GLU A 101      21.426  25.445  10.162  1.00 20.52           C  \\nANISOU  531  CG  GLU A 101     2634   2658   2503   -402   -349   -114       C  \\nATOM    532  CD  GLU A 101      22.057  26.781   9.782  1.00 27.66           C  \\nANISOU  532  CD  GLU A 101     3864   3437   3207  -1125    311    156       C  \\nATOM    533  OE1 GLU A 101      21.862  27.803  10.500  1.00 46.89           O  \\nANISOU  533  OE1 GLU A 101     7477   5227   5111  -2387   -753  -2143       O  \\nATOM    534  OE2 GLU A 101      22.746  26.761   8.732  1.00 34.25           O  \\nANISOU  534  OE2 GLU A 101     4036   6069   2906  -2073    214   -432       O  \\nATOM    535  N   GLU A 102      18.497  24.279  13.464  1.00 11.51           N  \\nANISOU  535  N   GLU A 102     1472   1461   1436   -107   -236      5       N  \\nATOM    536  CA  GLU A 102      17.830  24.488  14.764  1.00 12.78           C  \\nANISOU  536  CA  GLU A 102     1737   1484   1634   -304    -71   -127       C  \\nATOM    537  C   GLU A 102      17.673  23.220  15.617  1.00 11.73           C  \\nANISOU  537  C   GLU A 102     1478   1484   1493   -147   -210   -145       C  \\nATOM    538  O   GLU A 102      17.292  23.300  16.766  1.00 14.10           O  \\nANISOU  538  O   GLU A 102     1960   1904   1492    -59   -215    -73       O  \\nATOM    539  CB  GLU A 102      16.435  25.079  14.507  1.00 13.57           C  \\nANISOU  539  CB  GLU A 102     1790   1468   1895    -80     54    -94       C  \\nATOM    540  CG  GLU A 102      16.421  26.498  13.880  1.00 14.02           C  \\nANISOU  540  CG  GLU A 102     2020   1435   1870    -36   -233   -143       C  \\nATOM    541  CD  GLU A 102      16.634  26.599  12.361  1.00 14.19           C  \\nANISOU  541  CD  GLU A 102     1894   1532   1962    -66     64   -225       C  \\nATOM    542  OE1 GLU A 102      16.714  25.545  11.670  1.00 13.05           O  \\nANISOU  542  OE1 GLU A 102     1905   1439   1613    -18   -266    -72       O  \\nATOM    543  OE2 GLU A 102      16.624  27.756  11.830  1.00 17.79           O  \\nANISOU  543  OE2 GLU A 102     3041   1785   1932   -287    171     23       O  \\nATOM    544  N   GLY A 103      17.928  22.063  15.007  1.00 11.37           N  \\nANISOU  544  N   GLY A 103     1456   1458   1403    -69   -201   -124       N  \\nATOM    545  CA  GLY A 103      17.852  20.788  15.671  1.00 12.56           C  \\nANISOU  545  CA  GLY A 103     1541   1699   1531   -148   -336     67       C  \\nATOM    546  C   GLY A 103      19.005  20.468  16.581  1.00 12.51           C  \\nANISOU  546  C   GLY A 103     1636   1576   1540    -85   -398     26       C  \\nATOM    547  O   GLY A 103      19.884  21.317  16.857  1.00 15.27           O  \\nANISOU  547  O   GLY A 103     1860   1879   2060   -327   -566    195       O  \\nATOM    548  N   GLY A 104      19.005  19.221  17.060  1.00 12.32           N  \\nANISOU  548  N   GLY A 104     1550   1596   1533   -182   -513    103       N  \\nATOM    549  CA  GLY A 104      20.057  18.792  17.975  1.00 13.54           C  \\nANISOU  549  CA  GLY A 104     1726   1714   1702    -94   -620     10       C  \\nATOM    550  C   GLY A 104      19.748  19.066  19.439  1.00 14.38           C  \\nANISOU  550  C   GLY A 104     1631   2239   1592    -60   -514    161       C  \\nATOM    551  O   GLY A 104      18.636  18.802  19.922  1.00 16.59           O  \\nANISOU  551  O   GLY A 104     1725   2764   1813     -8   -449    110       O  \\nATOM    552  N   LYS A 105      20.747  19.546  20.165  1.00 19.07           N  \\nANISOU  552  N   LYS A 105     1653   3014   2576   -159   -776   -107       N  \\nATOM    553  CA  LYS A 105      20.615  19.783  21.607  1.00 20.74           C  \\nANISOU  553  CA  LYS A 105     2184   3230   2464   -288   -423    197       C  \\nATOM    554  C   LYS A 105      19.403  20.637  21.872  1.00 19.01           C  \\nANISOU  554  C   LYS A 105     2318   2846   2057   -299   -740   -120       C  \\nATOM    555  O   LYS A 105      19.158  21.644  21.197  1.00 20.03           O  \\nANISOU  555  O   LYS A 105     2525   2536   2548   -251   -971   -359       O  \\nATOM    556  CB  LYS A 105      21.880  20.442  22.189  1.00 28.64           C  \\nANISOU  556  CB  LYS A 105     2084   5065   3733   -521   -524   -463       C  \\nATOM    557  CG  LYS A 105      23.023  19.476  22.379  1.00 38.36           C  \\nANISOU  557  CG  LYS A 105     3604   5509   5459    -12   -957   -171       C  \\nATOM    558  CD  LYS A 105      24.016  20.013  23.401  1.00 44.93           C  \\nANISOU  558  CD  LYS A 105     4957   7272   4842     10  -1807    -21       C  \\nATOM    559  CE  LYS A 105      25.173  19.042  23.620  1.00 55.13           C  \\nANISOU  559  CE  LYS A 105     6121   7216   7610    890   -821    124       C  \\nATOM    560  NZ  LYS A 105      26.180  19.555  24.595  1.00 64.22           N  \\nANISOU  560  NZ  LYS A 105     6864   9372   8163   -497   -626    -83       N  \\nATOM    561  N   GLY A 106      18.606  20.193  22.823  1.00 17.68           N  \\nANISOU  561  N   GLY A 106     1828   2805   2083   -148   -971   -167       N  \\nATOM    562  CA  GLY A 106      17.441  20.945  23.237  1.00 16.32           C  \\nANISOU  562  CA  GLY A 106     2180   2204   1814    -58   -869   -203       C  \\nATOM    563  C   GLY A 106      16.151  20.634  22.505  1.00 13.39           C  \\nANISOU  563  C   GLY A 106     1827   1699   1560      0   -535     21       C  \\nATOM    564  O   GLY A 106      15.095  21.078  22.946  1.00 15.24           O  \\nANISOU  564  O   GLY A 106     1874   1831   2085     71   -745   -325       O  \\nATOM    565  N   ALA A 107      16.226  19.855  21.430  1.00 12.51           N  \\nANISOU  565  N   ALA A 107     1396   1632   1724   -113   -412    -28       N  \\nATOM    566  CA  ALA A 107      15.023  19.606  20.688  1.00 11.60           C  \\nANISOU  566  CA  ALA A 107     1275   1534   1595    157   -314   -151       C  \\nATOM    567  C   ALA A 107      13.980  18.862  21.500  1.00 11.17           C  \\nANISOU  567  C   ALA A 107     1535   1255   1453     19   -386   -114       C  \\nATOM    568  O   ALA A 107      14.312  18.034  22.346  1.00 13.33           O  \\nANISOU  568  O   ALA A 107     1770   1553   1740      2   -445     93       O  \\nATOM    569  CB  ALA A 107      15.316  18.831  19.433  1.00 12.40           C  \\nANISOU  569  CB  ALA A 107     1674   1569   1465    106   -267   -132       C  \\nATOM    570  N   SER A 108      12.719  19.123  21.168  1.00 10.43           N  \\nANISOU  570  N   SER A 108     1499   1215   1249     24   -402    -36       N  \\nATOM    571  CA  SER A 108      11.624  18.342  21.744  1.00 11.65           C  \\nANISOU  571  CA  SER A 108     1792   1403   1229    -68   -238    -46       C  \\nATOM    572  C   SER A 108      11.615  16.947  21.152  1.00 10.93           C  \\nANISOU  572  C   SER A 108     1567   1283   1301    -11   -413     89       C  \\nATOM    573  O   SER A 108      11.651  16.813  19.927  1.00 11.52           O  \\nANISOU  573  O   SER A 108     1894   1181   1300   -116   -245     74       O  \\nATOM    574  CB  SER A 108      10.287  18.999  21.427  1.00 11.72           C  \\nANISOU  574  CB  SER A 108     1734   1364   1355    -64   -155   -175       C  \\nATOM    575  OG  SER A 108       9.144  18.335  21.961  1.00 12.22           O  \\nANISOU  575  OG  SER A 108     1711   1612   1318    -97   -186   -111       O  \\nATOM    576  N   VAL A 109      11.500  15.956  22.016  1.00 11.59           N  \\nANISOU  576  N   VAL A 109     1832   1334   1237   -190   -110     59       N  \\nATOM    577  CA  VAL A 109      11.419  14.562  21.577  1.00 12.55           C  \\nANISOU  577  CA  VAL A 109     1953   1404   1408    -63   -345    -48       C  \\nATOM    578  C   VAL A 109      10.132  13.942  22.089  1.00 12.52           C  \\nANISOU  578  C   VAL A 109     2174   1444   1139   -101   -254    169       C  \\nATOM    579  O   VAL A 109       9.781  14.114  23.264  1.00 15.42           O  \\nANISOU  579  O   VAL A 109     2740   2009   1107   -555   -243    -72       O  \\nATOM    580  CB  VAL A 109      12.631  13.766  22.017  1.00 13.71           C  \\nANISOU  580  CB  VAL A 109     2065   1404   1740     55   -392    166       C  \\nATOM    581  CG1 VAL A 109      12.519  12.320  21.585  1.00 13.98           C  \\nANISOU  581  CG1 VAL A 109     1983   1491   1838    205   -190     10       C  \\nATOM    582  CG2 VAL A 109      13.915  14.389  21.483  1.00 15.14           C  \\nANISOU  582  CG2 VAL A 109     1787   1878   2087     35   -435    -21       C  \\nATOM    583  N   ARG A 110       9.417  13.251  21.189  1.00 12.54           N  \\nANISOU  583  N   ARG A 110     1902   1395   1467   -124   -182     24       N  \\nATOM    584  CA  ARG A 110       8.157  12.611  21.576  1.00 12.56           C  \\nANISOU  584  CA  ARG A 110     2056   1335   1381   -153   -224    208       C  \\nATOM    585  C   ARG A 110       8.108  11.245  20.918  1.00 11.92           C  \\nANISOU  585  C   ARG A 110     1970   1283   1274    -68   -445    298       C  \\nATOM    586  O   ARG A 110       8.374  11.157  19.709  1.00 12.99           O  \\nANISOU  586  O   ARG A 110     2125   1443   1364   -123   -207    254       O  \\nATOM    587  CB  ARG A 110       6.950  13.449  21.178  1.00 13.28           C  \\nANISOU  587  CB  ARG A 110     2093   1426   1526     43     51    129       C  \\nATOM    588  CG  ARG A 110       5.618  12.850  21.653  1.00 14.13           C  \\nANISOU  588  CG  ARG A 110     2128   1554   1687    -35     -7     78       C  \\nATOM    589  CD  ARG A 110       4.404  13.600  21.168  1.00 14.43           C  \\nANISOU  589  CD  ARG A 110     1957   1737   1789    -18    186     29       C  \\nATOM    590  NE  ARG A 110       4.321  14.920  21.751  1.00 15.45           N  \\nANISOU  590  NE  ARG A 110     2179   1853   1838     13    120     -1       N  \\nATOM    591  CZ  ARG A 110       3.265  15.710  21.626  1.00 14.78           C  \\nANISOU  591  CZ  ARG A 110     2036   1741   1837    -86    247    315       C  \\nATOM    592  NH1 ARG A 110       2.249  15.373  20.826  1.00 15.41           N  \\nANISOU  592  NH1 ARG A 110     2258   1972   1626    122    158     42       N  \\nATOM    593  NH2 ARG A 110       3.258  16.882  22.256  1.00 16.93           N  \\nANISOU  593  NH2 ARG A 110     2286   1820   2324    219    191    122       N  \\nATOM    594  N   TYR A 111       7.615  10.227  21.647  1.00 12.42           N  \\nANISOU  594  N   TYR A 111     1947   1430   1339   -149   -337    312       N  \\nATOM    595  CA  TYR A 111       7.342   8.940  21.011  1.00 13.19           C  \\nANISOU  595  CA  TYR A 111     1983   1329   1698   -150   -348    301       C  \\nATOM    596  C   TYR A 111       6.146   9.094  20.068  1.00 12.82           C  \\nANISOU  596  C   TYR A 111     1923   1312   1635   -256   -261    316       C  \\nATOM    597  O   TYR A 111       5.082   9.562  20.510  1.00 15.26           O  \\nANISOU  597  O   TYR A 111     2080   1709   2006   -209   -312    129       O  \\nATOM    598  CB  TYR A 111       7.044   7.866  22.029  1.00 14.69           C  \\nANISOU  598  CB  TYR A 111     2222   1527   1833    -28   -179    394       C  \\nATOM    599  CG  TYR A 111       8.216   7.534  22.921  1.00 15.70           C  \\nANISOU  599  CG  TYR A 111     2539   1384   2040   -119   -577    527       C  \\nATOM    600  CD1 TYR A 111       9.375   6.952  22.385  1.00 16.75           C  \\nANISOU  600  CD1 TYR A 111     2420   1893   2051    -98   -567    793       C  \\nATOM    601  CD2 TYR A 111       8.193   7.813  24.306  1.00 19.33           C  \\nANISOU  601  CD2 TYR A 111     2781   2432   2128    193   -569    621       C  \\nATOM    602  CE1 TYR A 111      10.479   6.638  23.212  1.00 18.51           C  \\nANISOU  602  CE1 TYR A 111     2754   1779   2500     15   -845   1020       C  \\nATOM    603  CE2 TYR A 111       9.284   7.506  25.128  1.00 21.39           C  \\nANISOU  603  CE2 TYR A 111     3185   2691   2250    181   -929    519       C  \\nATOM    604  CZ  TYR A 111      10.416   6.924  24.587  1.00 20.46           C  \\nANISOU  604  CZ  TYR A 111     2684   2528   2559    -38   -817   1093       C  \\nATOM    605  OH  TYR A 111      11.499   6.630  25.402  1.00 28.21           O  \\nANISOU  605  OH  TYR A 111     3741   3827   3150     99  -1721   1303       O  \\nATOM    606  N   VAL A 112       6.328   8.702  18.806  1.00 12.29           N  \\nANISOU  606  N   VAL A 112     1623   1298   1746    -39   -332    202       N  \\nATOM    607  CA  VAL A 112       5.273   8.769  17.785  1.00 12.46           C  \\nANISOU  607  CA  VAL A 112     1613   1451   1667      2   -263     88       C  \\nATOM    608  C   VAL A 112       5.200   7.400  17.106  1.00 12.74           C  \\nANISOU  608  C   VAL A 112     1638   1331   1871     95   -414    197       C  \\nATOM    609  O   VAL A 112       6.230   6.796  16.848  1.00 12.62           O  \\nANISOU  609  O   VAL A 112     1705   1224   1863    125   -318    233       O  \\nATOM    610  CB  VAL A 112       5.630   9.852  16.729  1.00 13.12           C  \\nANISOU  610  CB  VAL A 112     1778   1316   1891    167   -285    207       C  \\nATOM    611  CG1 VAL A 112       4.683   9.796  15.525  1.00 15.41           C  \\nANISOU  611  CG1 VAL A 112     1850   1711   2293    131   -518    290       C  \\nATOM    612  CG2 VAL A 112       5.573  11.251  17.357  1.00 13.35           C  \\nANISOU  612  CG2 VAL A 112     1959   1482   1629    123   -454    133       C  \\nATOM    613  N   SER A 113       3.996   6.881  16.880  1.00 12.52           N  \\nANISOU  613  N   SER A 113     1806   1140   1808    -98   -183    185       N  \\nATOM    614  CA ASER A 113       3.838   5.595  16.196  0.50 12.63           C  \\nANISOU  614  CA ASER A 113     1849   1236   1710    -60   -314     83       C  \\nATOM    615  CA BSER A 113       3.898   5.566  16.257  0.50 13.28           C  \\nANISOU  615  CA BSER A 113     2077   1221   1748     12   -259     62       C  \\nATOM    616  C   SER A 113       4.715   5.512  14.975  1.00 13.17           C  \\nANISOU  616  C   SER A 113     2014   1142   1846    -34   -243     62       C  \\nATOM    617  O   SER A 113       4.735   6.445  14.176  1.00 14.04           O  \\nANISOU  617  O   SER A 113     2216   1132   1984    -70   -283    122       O  \\nATOM    618  CB ASER A 113       2.388   5.427  15.775  0.50 14.80           C  \\nANISOU  618  CB ASER A 113     1823   1643   2157   -111   -251    172       C  \\nATOM    619  CB BSER A 113       2.446   5.140  16.044  0.50 16.70           C  \\nANISOU  619  CB BSER A 113     2069   1902   2371     84   -251     63       C  \\nATOM    620  OG ASER A 113       1.549   5.465  16.919  0.50 16.38           O  \\nANISOU  620  OG ASER A 113     1692   1999   2530    190    -46    172       O  \\nATOM    621  OG BSER A 113       1.796   5.910  15.062  0.50 20.79           O  \\nANISOU  621  OG BSER A 113     3270   1978   2650    -55   -723    289       O  \\nATOM    622  N   SER A 114       5.388   4.376  14.779  1.00 13.42           N  \\nANISOU  622  N   SER A 114     2084   1032   1983    -53   -265     86       N  \\nATOM    623  CA  SER A 114       6.397   4.283  13.745  1.00 15.23           C  \\nANISOU  623  CA  SER A 114     2229   1192   2363    -26    -41    -36       C  \\nATOM    624  C   SER A 114       5.846   4.539  12.359  1.00 15.66           C  \\nANISOU  624  C   SER A 114     2474   1192   2284      4    -98   -213       C  \\nATOM    625  O   SER A 114       6.513   5.228  11.569  1.00 17.22           O  \\nANISOU  625  O   SER A 114     2773   1262   2508    122    -12    138       O  \\nATOM    626  CB  SER A 114       7.150   2.952  13.802  1.00 16.60           C  \\nANISOU  626  CB  SER A 114     2466   1367   2472    210    -14   -153       C  \\nATOM    627  OG  SER A 114       7.796   2.815  15.040  1.00 21.79           O  \\nANISOU  627  OG  SER A 114     3170   1856   3251     17   -572    153       O  \\nATOM    628  N   SER A 115       4.650   4.048  12.030  1.00 16.37           N  \\nANISOU  628  N   SER A 115     2809   1168   2240   -129   -223   -130       N  \\nATOM    629  CA ASER A 115       4.123   4.267  10.683  0.50 17.27           C  \\nANISOU  629  CA ASER A 115     2960   1376   2225    -28   -216   -307       C  \\nATOM    630  CA BSER A 115       4.118   4.259  10.684  0.50 17.51           C  \\nANISOU  630  CA BSER A 115     2908   1455   2289    -17   -238   -168       C  \\nATOM    631  C   SER A 115       3.849   5.742  10.414  1.00 17.90           C  \\nANISOU  631  C   SER A 115     3065   1447   2289    -99   -489   -146       C  \\nATOM    632  O   SER A 115       4.121   6.232   9.327  1.00 19.11           O  \\nANISOU  632  O   SER A 115     3080   1863   2318   -245   -489    -70       O  \\nATOM    633  CB ASER A 115       2.850   3.447  10.460  0.50 19.42           C  \\nANISOU  633  CB ASER A 115     3156   1711   2510   -305   -306   -336       C  \\nATOM    634  CB BSER A 115       2.832   3.449  10.483  0.50 22.15           C  \\nANISOU  634  CB BSER A 115     3209   2281   2924   -492   -600    -19       C  \\nATOM    635  OG ASER A 115       3.164   2.075  10.497  0.50 28.14           O  \\nANISOU  635  OG ASER A 115     4973   1859   3858    125   -536   -229       O  \\nATOM    636  OG BSER A 115       1.872   3.794  11.463  0.50 24.69           O  \\nANISOU  636  OG BSER A 115     3193   2964   3224   -413   -471    522       O  \\nATOM    637  N   ASP A 116       3.323   6.441  11.420  1.00 16.13           N  \\nANISOU  637  N   ASP A 116     2611   1372   2144    -62   -523     64       N  \\nATOM    638  CA  ASP A 116       3.056   7.876  11.302  1.00 14.28           C  \\nANISOU  638  CA  ASP A 116     2104   1414   1906     68   -474     65       C  \\nATOM    639  C   ASP A 116       4.344   8.625  11.130  1.00 13.58           C  \\nANISOU  639  C   ASP A 116     2003   1345   1810     98   -418      2       C  \\nATOM    640  O   ASP A 116       4.460   9.465  10.207  1.00 14.20           O  \\nANISOU  640  O   ASP A 116     2176   1404   1813     43   -478      8       O  \\nATOM    641  CB  ASP A 116       2.289   8.319  12.561  1.00 13.60           C  \\nANISOU  641  CB  ASP A 116     2010   1375   1780   -135   -457     98       C  \\nATOM    642  CG  ASP A 116       1.829   9.763  12.562  1.00 14.01           C  \\nANISOU  642  CG  ASP A 116     2141   1433   1747   -145   -517    119       C  \\nATOM    643  OD1 ASP A 116       2.648  10.689  12.560  1.00 13.86           O  \\nANISOU  643  OD1 ASP A 116     1891   1426   1947   -100   -412     61       O  \\nATOM    644  OD2 ASP A 116       0.600   9.992  12.763  1.00 15.25           O  \\nANISOU  644  OD2 ASP A 116     2069   1467   2256    -17   -385     -6       O  \\nATOM    645  N   ASN A 117       5.332   8.338  11.981  1.00 13.93           N  \\nANISOU  645  N   ASN A 117     2017   1278   1997    201   -333    243       N  \\nATOM    646  CA  ASN A 117       6.576   9.054  11.916  1.00 13.56           C  \\nANISOU  646  CA  ASN A 117     2123   1182   1847    139   -250    254       C  \\nATOM    647  C   ASN A 117       7.392   8.775  10.657  1.00 12.94           C  \\nANISOU  647  C   ASN A 117     2241   1064   1609    170   -426    133       C  \\nATOM    648  O   ASN A 117       8.003   9.691  10.077  1.00 13.77           O  \\nANISOU  648  O   ASN A 117     2315   1195   1718    146   -299    141       O  \\nATOM    649  CB  ASN A 117       7.414   8.815  13.144  1.00 14.47           C  \\nANISOU  649  CB  ASN A 117     1973   1647   1874    293   -278    -76       C  \\nATOM    650  CG  ASN A 117       8.365   9.951  13.408  1.00 15.92           C  \\nANISOU  650  CG  ASN A 117     2163   1696   2190    331   -239   -457       C  \\nATOM    651  OD1 ASN A 117       9.536   9.720  13.557  1.00 17.13           O  \\nANISOU  651  OD1 ASN A 117     2201   2087   2217    436   -499    -81       O  \\nATOM    652  ND2 ASN A 117       7.860  11.188  13.460  1.00 15.47           N  \\nANISOU  652  ND2 ASN A 117     2295   1785   1798    442   -276   -375       N  \\nATOM    653  N   ARG A 118       7.423   7.517  10.216  1.00 14.72           N  \\nANISOU  653  N   ARG A 118     2483   1122   1986    317   -243    -83       N  \\nATOM    654  CA  ARG A 118       8.194   7.162   9.041  1.00 14.46           C  \\nANISOU  654  CA  ARG A 118     2527   1184   1782     27   -318     16       C  \\nATOM    655  C   ARG A 118       7.550   7.731   7.774  1.00 14.85           C  \\nANISOU  655  C   ARG A 118     2521   1219   1902    182   -487   -255       C  \\nATOM    656  O   ARG A 118       8.239   8.213   6.876  1.00 17.06           O  \\nANISOU  656  O   ARG A 118     2989   1471   2020    281   -346   -119       O  \\nATOM    657  CB  ARG A 118       8.391   5.640   8.981  1.00 14.32           C  \\nANISOU  657  CB  ARG A 118     2560   1217   1662    126    -19   -305       C  \\nATOM    658  CG  ARG A 118       9.311   5.151  10.088  1.00 15.03           C  \\nANISOU  658  CG  ARG A 118     2540   1096   2074    149   -182    -92       C  \\nATOM    659  CD  ARG A 118       9.355   3.651  10.253  1.00 15.54           C  \\nANISOU  659  CD  ARG A 118     2465   1123   2314     46   -194      6       C  \\nATOM    660  NE  ARG A 118       9.969   3.290  11.508  1.00 15.78           N  \\nANISOU  660  NE  ARG A 118     2529   1118   2346    250   -238    -19       N  \\nATOM    661  CZ  ARG A 118      10.217   2.048  11.892  1.00 14.04           C  \\nANISOU  661  CZ  ARG A 118     2017   1272   2043    337    -37     64       C  \\nATOM    662  NH1 ARG A 118       9.990   1.049  11.071  1.00 15.68           N  \\nANISOU  662  NH1 ARG A 118     2419   1288   2251    155    124    -28       N  \\nATOM    663  NH2 ARG A 118      10.779   1.833  13.065  1.00 14.66           N  \\nANISOU  663  NH2 ARG A 118     2238   1170   2160     20   -131    223       N  \\nATOM    664  N   GLY A 119       6.232   7.744   7.769  1.00 15.60           N  \\nANISOU  664  N   GLY A 119     2478   1325   2122    -50   -750   -238       N  \\nATOM    665  CA  GLY A 119       5.533   8.352   6.661  1.00 14.46           C  \\nANISOU  665  CA  GLY A 119     2254   1378   1860    -84   -673   -381       C  \\nATOM    666  C   GLY A 119       5.800   9.855   6.603  1.00 14.21           C  \\nANISOU  666  C   GLY A 119     2262   1366   1772     59   -239   -319       C  \\nATOM    667  O   GLY A 119       6.071  10.390   5.546  1.00 14.70           O  \\nANISOU  667  O   GLY A 119     2471   1476   1635    -51   -404   -248       O  \\nATOM    668  N   ALA A 120       5.738  10.517   7.763  1.00 13.18           N  \\nANISOU  668  N   ALA A 120     2191   1151   1665     39   -275   -143       N  \\nATOM    669  CA  ALA A 120       5.980  11.952   7.812  1.00 11.46           C  \\nANISOU  669  CA  ALA A 120     1827   1150   1374     45   -380   -221       C  \\nATOM    670  C   ALA A 120       7.395  12.218   7.348  1.00 11.46           C  \\nANISOU  670  C   ALA A 120     1748   1015   1589    193   -368   -227       C  \\nATOM    671  O   ALA A 120       7.671  13.185   6.629  1.00 12.50           O  \\nANISOU  671  O   ALA A 120     1900   1149   1699     91   -296    -92       O  \\nATOM    672  CB  ALA A 120       5.766  12.452   9.234  1.00 12.03           C  \\nANISOU  672  CB  ALA A 120     1809   1388   1370     27   -132    -84       C  \\nATOM    673  N   GLY A 121       8.343  11.404   7.823  1.00 12.44           N  \\nANISOU  673  N   GLY A 121     1964   1049   1711    158   -613   -185       N  \\nATOM    674  CA  GLY A 121       9.743  11.624   7.438  1.00 13.23           C  \\nANISOU  674  CA  GLY A 121     1787   1310   1928    245   -742      4       C  \\nATOM    675  C   GLY A 121       9.974  11.563   5.929  1.00 14.48           C  \\nANISOU  675  C   GLY A 121     1812   1588   2101    358   -589   -157       C  \\nATOM    676  O   GLY A 121      10.723  12.373   5.362  1.00 14.88           O  \\nANISOU  676  O   GLY A 121     2093   1571   1989    292   -182   -478       O  \\nATOM    677  N   SER A 122       9.347  10.575   5.277  1.00 15.19           N  \\nANISOU  677  N   SER A 122     2300   1557   1912    103   -443   -189       N  \\nATOM    678  CA ASER A 122       9.466  10.430   3.829  0.50 16.59           C  \\nANISOU  678  CA ASER A 122     2606   1662   2033    300   -234   -449       C  \\nATOM    679  CA BSER A 122       9.474  10.417   3.837  0.50 16.64           C  \\nANISOU  679  CA BSER A 122     2608   1664   2050    329   -222   -448       C  \\nATOM    680  C   SER A 122       8.781  11.590   3.104  1.00 13.83           C  \\nANISOU  680  C   SER A 122     1515   2028   1710    100   -560   -574       C  \\nATOM    681  O   SER A 122       9.252  12.100   2.079  1.00 15.49           O  \\nANISOU  681  O   SER A 122     2157   1731   1995    329   -150   -690       O  \\nATOM    682  CB ASER A 122       8.889   9.118   3.318  0.50 20.40           C  \\nANISOU  682  CB ASER A 122     3433   1771   2546    187   -480   -639       C  \\nATOM    683  CB BSER A 122       8.879   9.080   3.400  0.50 21.01           C  \\nANISOU  683  CB BSER A 122     3514   1886   2584    100   -391   -721       C  \\nATOM    684  OG ASER A 122       9.071   9.069   1.896  0.50 23.62           O  \\nANISOU  684  OG ASER A 122     4747   1851   2374    179   -881   -282       O  \\nATOM    685  OG BSER A 122       9.579   7.991   3.983  0.50 25.92           O  \\nANISOU  685  OG BSER A 122     4482   2454   2910    558   -408   -454       O  \\nATOM    686  N   TRP A 123       7.620  11.983   3.607  1.00 12.35           N  \\nANISOU  686  N   TRP A 123     1668   1488   1533     18   -236   -290       N  \\nATOM    687  CA  TRP A 123       6.910  13.108   3.039  1.00 11.58           C  \\nANISOU  687  CA  TRP A 123     1495   1513   1389     -7   -164   -181       C  \\nATOM    688  C   TRP A 123       7.756  14.373   3.098  1.00 10.90           C  \\nANISOU  688  C   TRP A 123     1310   1550   1278    167    -58   -198       C  \\nATOM    689  O   TRP A 123       7.862  15.092   2.107  1.00 11.38           O  \\nANISOU  689  O   TRP A 123     1315   1645   1361    -18    -14   -108       O  \\nATOM    690  CB  TRP A 123       5.572  13.297   3.764  1.00 13.41           C  \\nANISOU  690  CB  TRP A 123     1558   1825   1710   -189     58     52       C  \\nATOM    691  CG  TRP A 123       4.726  14.329   3.129  1.00 14.14           C  \\nANISOU  691  CG  TRP A 123     1543   2219   1608     19     90    119       C  \\nATOM    692  CD1 TRP A 123       3.768  14.114   2.209  1.00 17.46           C  \\nANISOU  692  CD1 TRP A 123     2561   2398   1673   -107   -386      7       C  \\nATOM    693  CD2 TRP A 123       4.788  15.752   3.322  1.00 13.80           C  \\nANISOU  693  CD2 TRP A 123     1627   2112   1502    162    273    335       C  \\nATOM    694  NE1 TRP A 123       3.192  15.298   1.849  1.00 20.80           N  \\nANISOU  694  NE1 TRP A 123     2618   3113   2172    536   -397     92       N  \\nATOM    695  CE2 TRP A 123       3.847  16.329   2.459  1.00 16.83           C  \\nANISOU  695  CE2 TRP A 123     2255   2546   1591    446    285    526       C  \\nATOM    696  CE3 TRP A 123       5.561  16.595   4.116  1.00 15.01           C  \\nANISOU  696  CE3 TRP A 123     1604   2046   2050    199    177    323       C  \\nATOM    697  CZ2 TRP A 123       3.603  17.700   2.433  1.00 19.10           C  \\nANISOU  697  CZ2 TRP A 123     2411   2628   2215    452    400    396       C  \\nATOM    698  CZ3 TRP A 123       5.356  17.950   4.029  1.00 17.44           C  \\nANISOU  698  CZ3 TRP A 123     2306   2179   2141    191    269     31       C  \\nATOM    699  CH2 TRP A 123       4.414  18.488   3.176  1.00 17.16           C  \\nANISOU  699  CH2 TRP A 123     2429   2496   1593    484    776    387       C  \\nATOM    700  N   VAL A 124       8.349  14.643   4.278  1.00 11.08           N  \\nANISOU  700  N   VAL A 124     1479   1467   1263    -20   -100   -186       N  \\nATOM    701  CA  VAL A 124       9.151  15.843   4.434  1.00 12.47           C  \\nANISOU  701  CA  VAL A 124     1631   1465   1641    -86    -86   -270       C  \\nATOM    702  C   VAL A 124      10.385  15.790   3.546  1.00 12.55           C  \\nANISOU  702  C   VAL A 124     1391   1681   1694    -26   -253   -202       C  \\nATOM    703  O   VAL A 124      10.730  16.775   2.891  1.00 12.99           O  \\nANISOU  703  O   VAL A 124     1437   1756   1740   -231    -58   -342       O  \\nATOM    704  CB  VAL A 124       9.522  16.062   5.912  1.00 14.46           C  \\nANISOU  704  CB  VAL A 124     2069   1666   1758   -137   -186   -472       C  \\nATOM    705  CG1 VAL A 124      10.623  17.111   6.068  1.00 17.84           C  \\nANISOU  705  CG1 VAL A 124     2270   2460   2046   -637    -18   -411       C  \\nATOM    706  CG2 VAL A 124       8.292  16.409   6.706  1.00 14.62           C  \\nANISOU  706  CG2 VAL A 124     2231   1582   1742   -116   -109   -220       C  \\nATOM    707  N   GLY A 125      11.062  14.648   3.506  1.00 13.90           N  \\nANISOU  707  N   GLY A 125     1502   1720   2058     10   -104   -327       N  \\nATOM    708  CA  GLY A 125      12.275  14.592   2.669  1.00 15.88           C  \\nANISOU  708  CA  GLY A 125     1270   2044   2720     28    -73   -597       C  \\nATOM    709  C   GLY A 125      11.985  14.842   1.214  1.00 18.17           C  \\nANISOU  709  C   GLY A 125     1697   2688   2516    -73    194   -952       C  \\nATOM    710  O   GLY A 125      12.692  15.590   0.544  1.00 20.86           O  \\nANISOU  710  O   GLY A 125     1611   3211   3102   -137    454  -1205       O  \\nATOM    711  N   ASN A 126      10.889  14.268   0.718  1.00 14.70           N  \\nANISOU  711  N   ASN A 126     1644   1906   2034    185     94   -811       N  \\nATOM    712  CA  ASN A 126      10.505  14.529  -0.645  1.00 14.53           C  \\nANISOU  712  CA  ASN A 126     1554   2137   1827    -46    594   -718       C  \\nATOM    713  C   ASN A 126      10.055  15.943  -0.915  1.00 13.73           C  \\nANISOU  713  C   ASN A 126     1877   2123   1214   -254    294   -540       C  \\nATOM    714  O   ASN A 126      10.437  16.536  -1.911  1.00 16.96           O  \\nANISOU  714  O   ASN A 126     2210   2564   1668   -621    908   -699       O  \\nATOM    715  CB  ASN A 126       9.502  13.467  -1.095  1.00 16.67           C  \\nANISOU  715  CB  ASN A 126     1778   1939   2616    -47    330   -472       C  \\nATOM    716  CG  ASN A 126      10.237  12.311  -1.649  1.00 18.18           C  \\nANISOU  716  CG  ASN A 126     2681   1946   2277     36    590   -237       C  \\nATOM    717  OD1 ASN A 126      10.878  12.435  -2.721  1.00 23.91           O  \\nANISOU  717  OD1 ASN A 126     3747   2811   2525   -674   1233   -798       O  \\nATOM    718  ND2 ASN A 126      10.263  11.239  -0.894  1.00 20.90           N  \\nANISOU  718  ND2 ASN A 126     2694   2456   2790   -428   -158    367       N  \\nATOM    719  N   ARG A 127       9.230  16.489  -0.015  1.00 11.73           N  \\nANISOU  719  N   ARG A 127     1374   1710   1371   -320    330   -348       N  \\nATOM    720  CA  ARG A 127       8.751  17.830  -0.217  1.00 12.22           C  \\nANISOU  720  CA  ARG A 127     1799   1728   1116   -339    379   -187       C  \\nATOM    721  C   ARG A 127       9.887  18.840  -0.180  1.00 15.17           C  \\nANISOU  721  C   ARG A 127     1884   1778   2100   -320    433   -334       C  \\nATOM    722  O   ARG A 127       9.835  19.828  -0.906  1.00 22.87           O  \\nANISOU  722  O   ARG A 127     4284   2661   1742  -1504    127     24       O  \\nATOM    723  CB  ARG A 127       7.672  18.164   0.810  1.00 13.32           C  \\nANISOU  723  CB  ARG A 127     1665   1983   1410    -83    347   -362       C  \\nATOM    724  CG  ARG A 127       6.970  19.508   0.616  1.00 16.13           C  \\nANISOU  724  CG  ARG A 127     2527   1943   1658     81    235   -289       C  \\nATOM    725  CD  ARG A 127       6.023  19.442  -0.629  1.00 19.86           C  \\nANISOU  725  CD  ARG A 127     3202   2507   1837    445   -114   -119       C  \\nATOM    726  NE  ARG A 127       5.297  20.697  -0.880  1.00 25.19           N  \\nANISOU  726  NE  ARG A 127     4132   3058   2379    850   -434    -45       N  \\nATOM    727  CZ  ARG A 127       5.831  21.752  -1.483  1.00 32.19           C  \\nANISOU  727  CZ  ARG A 127     4325   3813   4092   -146    125   -269       C  \\nATOM    728  NH1 ARG A 127       7.111  21.739  -1.874  1.00 30.62           N  \\nANISOU  728  NH1 ARG A 127     3545   5059   3028   -178   -763  -1067       N  \\nATOM    729  NH2 ARG A 127       5.106  22.858  -1.667  1.00 36.83           N  \\nANISOU  729  NH2 ARG A 127     4612   4140   5241    566    254   -184       N  \\nATOM    730  N   LEU A 128      10.836  18.654   0.731  1.00 14.49           N  \\nANISOU  730  N   LEU A 128     1312   1799   2395   -260    801   -579       N  \\nATOM    731  CA  LEU A 128      11.936  19.616   0.870  1.00 15.93           C  \\nANISOU  731  CA  LEU A 128     1632   1918   2503   -473    790   -793       C  \\nATOM    732  C   LEU A 128      13.030  19.428  -0.129  1.00 16.71           C  \\nANISOU  732  C   LEU A 128     1559   1776   3012   -507    946   -849       C  \\nATOM    733  O   LEU A 128      13.902  20.269  -0.252  1.00 16.56           O  \\nANISOU  733  O   LEU A 128     1419   1833   3037   -524    647   -817       O  \\nATOM    734  CB  LEU A 128      12.477  19.602   2.303  1.00 16.46           C  \\nANISOU  734  CB  LEU A 128     1723   1945   2585   -622    314   -929       C  \\nATOM    735  CG  LEU A 128      11.522  20.105   3.415  1.00 17.79           C  \\nANISOU  735  CG  LEU A 128     2001   2036   2719   -114    372   -766       C  \\nATOM    736  CD1 LEU A 128      12.235  20.170   4.779  1.00 20.21           C  \\nANISOU  736  CD1 LEU A 128     2183   2471   3022   -223    113  -1085       C  \\nATOM    737  CD2 LEU A 128      10.906  21.443   3.023  1.00 20.05           C  \\nANISOU  737  CD2 LEU A 128     2606   2356   2654      2    383   -380       C  \\nATOM    738  N   ASN A 129      12.944  18.373  -0.922  1.00 20.86           N  \\nANISOU  738  N   ASN A 129     2133   2231   3562   -952   1598  -1356       N  \\nATOM    739  CA  ASN A 129      13.954  18.156  -1.944  1.00 18.59           C  \\nANISOU  739  CA  ASN A 129     1729   2240   3093   -555   1082  -1391       C  \\nATOM    740  C   ASN A 129      13.965  19.204  -3.045  1.00 18.29           C  \\nANISOU  740  C   ASN A 129     1737   2481   2731   -129    355  -1274       C  \\nATOM    741  O   ASN A 129      14.925  19.279  -3.836  1.00 17.87           O  \\nANISOU  741  O   ASN A 129     1681   2036   3072   -335    470   -690       O  \\nATOM    742  CB  ASN A 129      13.785  16.777  -2.584  1.00 19.64           C  \\nANISOU  742  CB  ASN A 129     1836   2287   3340   -802   1115  -1448       C  \\nATOM    743  CG  ASN A 129      14.965  16.419  -3.482  1.00 17.97           C  \\nANISOU  743  CG  ASN A 129     1787   2229   2811   -841   1136   -748       C  \\nATOM    744  OD1 ASN A 129      16.090  16.467  -3.061  1.00 19.31           O  \\nANISOU  744  OD1 ASN A 129     1904   2441   2990   -605   1117  -1155       O  \\nATOM    745  ND2 ASN A 129      14.687  16.026  -4.691  1.00 20.02           N  \\nANISOU  745  ND2 ASN A 129     2068   2491   3045   -799    763  -1145       N  \\nATOM    746  N   GLY A 130      12.942  20.058  -3.104  1.00 21.23           N  \\nANISOU  746  N   GLY A 130     1538   3161   3368   -161   -137  -1981       N  \\nATOM    747  CA  GLY A 130      12.963  21.192  -3.975  1.00 22.76           C  \\nANISOU  747  CA  GLY A 130     2451   2929   3265    448   -791  -1861       C  \\nATOM    748  C   GLY A 130      13.728  22.404  -3.455  1.00 17.72           C  \\nANISOU  748  C   GLY A 130     1641   2644   2448    460   -566  -1171       C  \\nATOM    749  O   GLY A 130      13.856  23.387  -4.143  1.00 24.02           O  \\nANISOU  749  O   GLY A 130     3364   3328   2432   1088   -795   -938       O  \\nATOM    750  N   TYR A 131      14.301  22.302  -2.263  1.00 14.58           N  \\nANISOU  750  N   TYR A 131     1416   2031   2093    305   -248   -852       N  \\nATOM    751  CA  TYR A 131      14.966  23.447  -1.649  1.00 12.97           C  \\nANISOU  751  CA  TYR A 131     1422   1805   1701    218    -78   -461       C  \\nATOM    752  C   TYR A 131      16.414  23.085  -1.303  1.00 11.82           C  \\nANISOU  752  C   TYR A 131     1415   1622   1454    155   -130   -534       C  \\nATOM    753  O   TYR A 131      16.689  22.061  -0.680  1.00 12.81           O  \\nANISOU  753  O   TYR A 131     1273   1673   1920    278    -25   -458       O  \\nATOM    754  CB  TYR A 131      14.248  23.825  -0.357  1.00 12.95           C  \\nANISOU  754  CB  TYR A 131     1541   1555   1824    278     70   -379       C  \\nATOM    755  CG  TYR A 131      12.811  24.217  -0.520  1.00 13.31           C  \\nANISOU  755  CG  TYR A 131     1594   1754   1709    288     81   -441       C  \\nATOM    756  CD1 TYR A 131      12.449  25.442  -1.081  1.00 14.15           C  \\nANISOU  756  CD1 TYR A 131     1537   1930   1908    466   -218   -320       C  \\nATOM    757  CD2 TYR A 131      11.833  23.408  -0.063  1.00 15.35           C  \\nANISOU  757  CD2 TYR A 131     1612   1914   2305    361    314    -49       C  \\nATOM    758  CE1 TYR A 131      11.109  25.811  -1.194  1.00 17.22           C  \\nANISOU  758  CE1 TYR A 131     1678   2467   2396    744   -191   -324       C  \\nATOM    759  CE2 TYR A 131      10.490  23.761  -0.143  1.00 18.54           C  \\nANISOU  759  CE2 TYR A 131     1572   2682   2788    419    430   -315       C  \\nATOM    760  CZ  TYR A 131      10.145  24.956  -0.716  1.00 16.85           C  \\nANISOU  760  CZ  TYR A 131     1521   2942   1939    591   -366   -494       C  \\nATOM    761  OH  TYR A 131       8.839  25.280  -0.770  1.00 21.97           O  \\nANISOU  761  OH  TYR A 131     1518   4119   2710    694   -300   -338       O  \\nATOM    762  N   ALA A 132      17.315  23.993  -1.657  1.00 12.61           N  \\nANISOU  762  N   ALA A 132     1427   1502   1859    170   -164   -406       N  \\nATOM    763  CA  ALA A 132      18.717  23.839  -1.338  1.00 12.72           C  \\nANISOU  763  CA  ALA A 132     1327   1536   1969    160     49   -246       C  \\nATOM    764  C   ALA A 132      18.978  23.979   0.149  1.00 11.82           C  \\nANISOU  764  C   ALA A 132     1232   1333   1925     92     12    -95       C  \\nATOM    765  O   ALA A 132      18.273  24.722   0.865  1.00 12.64           O  \\nANISOU  765  O   ALA A 132     1481   1512   1810    251   -122   -360       O  \\nATOM    766  CB  ALA A 132      19.531  24.882  -2.058  1.00 13.38           C  \\nANISOU  766  CB  ALA A 132     1746   1753   1584     47   -121    -35       C  \\nATOM    767  N   ASP A 133      20.065  23.376   0.613  1.00 12.30           N  \\nANISOU  767  N   ASP A 133     1274   1559   1841    143    -70   -154       N  \\nATOM    768  CA  ASP A 133      20.536  23.667   1.949  1.00 11.87           C  \\nANISOU  768  CA  ASP A 133     1458   1346   1705    158     70   -170       C  \\nATOM    769  C   ASP A 133      20.681  25.186   2.140  1.00 12.02           C  \\nANISOU  769  C   ASP A 133     1191   1431   1944   -152    143   -251       C  \\nATOM    770  O   ASP A 133      21.067  25.895   1.216  1.00 13.85           O  \\nANISOU  770  O   ASP A 133     1663   1566   2031    -29    -12    -33       O  \\nATOM    771  CB  ASP A 133      21.854  23.001   2.254  1.00 12.70           C  \\nANISOU  771  CB  ASP A 133     1576   1378   1869    247      9   -278       C  \\nATOM    772  CG  ASP A 133      21.742  21.485   2.417  1.00 12.76           C  \\nANISOU  772  CG  ASP A 133     1467   1371   2008     83     98   -252       C  \\nATOM    773  OD1 ASP A 133      21.290  20.827   1.454  1.00 14.08           O  \\nANISOU  773  OD1 ASP A 133     1685   1433   2231    139   -273   -208       O  \\nATOM    774  OD2 ASP A 133      22.141  21.003   3.506  1.00 15.29           O  \\nANISOU  774  OD2 ASP A 133     2049   1747   2014    -85    127     30       O  \\nATOM    775  N   GLY A 134      20.332  25.642   3.340  1.00 10.96           N  \\nANISOU  775  N   GLY A 134     1399   1003   1762   -121    161    -43       N  \\nATOM    776  CA  GLY A 134      20.339  27.057   3.637  1.00 10.82           C  \\nANISOU  776  CA  GLY A 134     1253   1040   1819   -187   -170    -53       C  \\nATOM    777  C   GLY A 134      19.004  27.738   3.519  1.00  9.88           C  \\nANISOU  777  C   GLY A 134     1415    891   1448     31    -66   -108       C  \\nATOM    778  O   GLY A 134      18.798  28.853   4.032  1.00 11.29           O  \\nANISOU  778  O   GLY A 134     1642    926   1721      0     74   -254       O  \\nATOM    779  N   THR A 135      18.082  27.125   2.795  1.00 11.09           N  \\nANISOU  779  N   THR A 135     1341   1208   1665     97   -152   -250       N  \\nATOM    780  CA  THR A 135      16.739  27.674   2.656  1.00 10.56           C  \\nANISOU  780  CA  THR A 135     1376   1187   1449    110    -65     -1       C  \\nATOM    781  C   THR A 135      16.050  27.604   3.998  1.00 10.53           C  \\nANISOU  781  C   THR A 135     1383   1205   1413    200   -145   -198       C  \\nATOM    782  O   THR A 135      16.170  26.577   4.695  1.00 11.21           O  \\nANISOU  782  O   THR A 135     1602   1099   1557     16    -31    -51       O  \\nATOM    783  CB  THR A 135      15.945  26.902   1.590  1.00 12.03           C  \\nANISOU  783  CB  THR A 135     1581   1321   1669    102   -155   -138       C  \\nATOM    784  OG1 THR A 135      16.666  26.850   0.378  1.00 14.91           O  \\nANISOU  784  OG1 THR A 135     1917   1931   1815    349    -53   -235       O  \\nATOM    785  CG2 THR A 135      14.611  27.569   1.326  1.00 13.10           C  \\nANISOU  785  CG2 THR A 135     1671   1563   1742    296    -79   -199       C  \\nATOM    786  N   ARG A 136      15.302  28.662   4.340  1.00  9.86           N  \\nANISOU  786  N   ARG A 136     1301   1033   1409    157    -63    -23       N  \\nATOM    787  CA  ARG A 136      14.577  28.745   5.593  1.00  9.69           C  \\nANISOU  787  CA  ARG A 136     1440   1029   1211    -62    -69     68       C  \\nATOM    788  C   ARG A 136      13.087  28.505   5.333  1.00 10.23           C  \\nANISOU  788  C   ARG A 136     1387   1133   1367    126     15     61       C  \\nATOM    789  O   ARG A 136      12.468  29.264   4.610  1.00 10.95           O  \\nANISOU  789  O   ARG A 136     1502   1044   1615    112   -105     52       O  \\nATOM    790  CB  ARG A 136      14.816  30.131   6.213  1.00 10.70           C  \\nANISOU  790  CB  ARG A 136     1626   1055   1384      2   -138      3       C  \\nATOM    791  CG  ARG A 136      16.298  30.375   6.506  1.00 14.43           C  \\nANISOU  791  CG  ARG A 136     1925   1552   2003    -19   -687   -282       C  \\nATOM    792  CD  ARG A 136      16.600  31.799   6.739  1.00 15.29           C  \\nANISOU  792  CD  ARG A 136     2035   1635   2139   -348   -395   -140       C  \\nATOM    793  NE  ARG A 136      16.349  32.581   5.539  1.00 14.36           N  \\nANISOU  793  NE  ARG A 136     1731   1769   1956   -340    150      8       N  \\nATOM    794  CZ  ARG A 136      16.289  33.907   5.517  1.00 18.41           C  \\nANISOU  794  CZ  ARG A 136     2513   1914   2566   -378   -230   -411       C  \\nATOM    795  NH1 ARG A 136      16.557  34.594   6.614  1.00 21.09           N  \\nANISOU  795  NH1 ARG A 136     2950   2528   2534   -415   -236   -550       N  \\nATOM    796  NH2 ARG A 136      15.990  34.510   4.377  1.00 23.25           N  \\nANISOU  796  NH2 ARG A 136     3904   2040   2887    305   -763   -486       N  \\nATOM    797  N   ILE A 137      12.556  27.411   5.875  1.00  9.33           N  \\nANISOU  797  N   ILE A 137     1097   1173   1275     15    -65    -27       N  \\nATOM    798  CA  ILE A 137      11.172  27.020   5.676  1.00  9.16           C  \\nANISOU  798  CA  ILE A 137     1173   1144   1160    -51   -142    -62       C  \\nATOM    799  C   ILE A 137      10.399  27.182   6.973  1.00  9.81           C  \\nANISOU  799  C   ILE A 137     1297   1157   1271      6    -57    -16       C  \\nATOM    800  O   ILE A 137      10.699  26.528   7.953  1.00 10.75           O  \\nANISOU  800  O   ILE A 137     1237   1510   1336     88    -76    117       O  \\nATOM    801  CB  ILE A 137      11.098  25.575   5.168  1.00 10.64           C  \\nANISOU  801  CB  ILE A 137     1317   1192   1531    -23   -124   -183       C  \\nATOM    802  CG1 ILE A 137      11.918  25.390   3.855  1.00 13.68           C  \\nANISOU  802  CG1 ILE A 137     1744   1806   1646   -193     42   -431       C  \\nATOM    803  CG2 ILE A 137       9.660  25.113   4.960  1.00 12.91           C  \\nANISOU  803  CG2 ILE A 137     1468   1576   1859   -375     76   -262       C  \\nATOM    804  CD1 ILE A 137      11.447  26.173   2.648  1.00 17.58           C  \\nANISOU  804  CD1 ILE A 137     2501   2579   1598   -359    -87   -295       C  \\nATOM    805  N   LEU A 138       9.344  28.017   6.923  1.00 10.00           N  \\nANISOU  805  N   LEU A 138     1310   1189   1298    116    -23     30       N  \\nATOM    806  CA  LEU A 138       8.411  28.131   8.041  1.00 10.19           C  \\nANISOU  806  CA  LEU A 138     1350   1217   1303     -6    -35    -65       C  \\nATOM    807  C   LEU A 138       7.403  26.987   7.967  1.00  9.61           C  \\nANISOU  807  C   LEU A 138     1316   1237   1097     21    -22    -51       C  \\nATOM    808  O   LEU A 138       6.697  26.854   6.957  1.00 12.39           O  \\nANISOU  808  O   LEU A 138     1621   1699   1387   -110   -391    218       O  \\nATOM    809  CB  LEU A 138       7.676  29.466   7.957  1.00 12.22           C  \\nANISOU  809  CB  LEU A 138     1605   1503   1535    250    -72      7       C  \\nATOM    810  CG  LEU A 138       6.581  29.648   9.000  1.00 13.38           C  \\nANISOU  810  CG  LEU A 138     1536   1619   1929    251     50   -163       C  \\nATOM    811  CD1 LEU A 138       7.101  29.722  10.386  1.00 14.98           C  \\nANISOU  811  CD1 LEU A 138     2208   1719   1764    208    197   -338       C  \\nATOM    812  CD2 LEU A 138       5.796  30.926   8.681  1.00 15.40           C  \\nANISOU  812  CD2 LEU A 138     1926   1935   1989    560   -161   -236       C  \\nATOM    813  N   PHE A 139       7.340  26.201   9.047  1.00  9.98           N  \\nANISOU  813  N   PHE A 139     1344   1262   1184    -19   -159      8       N  \\nATOM    814  CA  PHE A 139       6.334  25.151   9.148  1.00 10.43           C  \\nANISOU  814  CA  PHE A 139     1361   1118   1485      6   -178    -37       C  \\nATOM    815  C   PHE A 139       5.121  25.707   9.865  1.00 11.82           C  \\nANISOU  815  C   PHE A 139     1480   1504   1505     68   -146   -126       C  \\nATOM    816  O   PHE A 139       5.267  26.324  10.915  1.00 12.80           O  \\nANISOU  816  O   PHE A 139     1559   1836   1465    245   -102   -260       O  \\nATOM    817  CB  PHE A 139       6.859  23.953   9.938  1.00 11.48           C  \\nANISOU  817  CB  PHE A 139     1463   1380   1516     49   -188    147       C  \\nATOM    818  CG  PHE A 139       7.660  22.970   9.115  1.00 12.52           C  \\nANISOU  818  CG  PHE A 139     1409   1621   1727    156   -401    -84       C  \\nATOM    819  CD1 PHE A 139       8.838  23.329   8.475  1.00 13.68           C  \\nANISOU  819  CD1 PHE A 139     1646   1674   1875    296   -111     34       C  \\nATOM    820  CD2 PHE A 139       7.255  21.621   9.088  1.00 15.98           C  \\nANISOU  820  CD2 PHE A 139     1562   1851   2659    -68   -376   -693       C  \\nATOM    821  CE1 PHE A 139       9.552  22.396   7.715  1.00 14.85           C  \\nANISOU  821  CE1 PHE A 139     1603   1715   2323    347     60     13       C  \\nATOM    822  CE2 PHE A 139       7.977  20.671   8.379  1.00 18.52           C  \\nANISOU  822  CE2 PHE A 139     1496   2233   3305    262   -168   -642       C  \\nATOM    823  CZ  PHE A 139       9.117  21.072   7.676  1.00 16.51           C  \\nANISOU  823  CZ  PHE A 139     1623   1982   2665    225   -154   -684       C  \\nATOM    824  N   ILE A 140       3.946  25.457   9.308  1.00 11.60           N  \\nANISOU  824  N   ILE A 140     1387   1728   1289    -40     40   -270       N  \\nATOM    825  CA  ILE A 140       2.682  25.887   9.872  1.00 13.47           C  \\nANISOU  825  CA  ILE A 140     1342   2203   1571     98    -54   -283       C  \\nATOM    826  C   ILE A 140       1.814  24.660  10.106  1.00 13.09           C  \\nANISOU  826  C   ILE A 140     1177   2155   1638    111   -156   -244       C  \\nATOM    827  O   ILE A 140       1.565  23.920   9.163  1.00 13.54           O  \\nANISOU  827  O   ILE A 140     1533   2073   1538    -89    -88    -58       O  \\nATOM    828  CB  ILE A 140       1.987  26.868   8.917  1.00 16.68           C  \\nANISOU  828  CB  ILE A 140     1534   2127   2674    346   -272   -127       C  \\nATOM    829  CG1 ILE A 140       2.860  28.093   8.725  1.00 19.80           C  \\nANISOU  829  CG1 ILE A 140     1952   2020   3550    341      0     78       C  \\nATOM    830  CG2 ILE A 140       0.604  27.198   9.414  1.00 18.09           C  \\nANISOU  830  CG2 ILE A 140     1661   2251   2960    459   -191    150       C  \\nATOM    831  CD1 ILE A 140       2.270  29.181   7.878  1.00 24.51           C  \\nANISOU  831  CD1 ILE A 140     2643   2757   3912     88   -680    437       C  \\nATOM    832  N   VAL A 141       1.362  24.441  11.339  1.00 14.51           N  \\nANISOU  832  N   VAL A 141     1553   2426   1530     30   -323   -357       N  \\nATOM    833  CA  VAL A 141       0.448  23.346  11.711  1.00 16.70           C  \\nANISOU  833  CA  VAL A 141     1708   2785   1852   -248   -629   -133       C  \\nATOM    834  C   VAL A 141      -0.652  24.007  12.523  1.00 22.92           C  \\nANISOU  834  C   VAL A 141     2466   3590   2651    115    310    264       C  \\nATOM    835  O   VAL A 141      -1.796  23.784  12.263  1.00 37.89           O  \\nANISOU  835  O   VAL A 141     2763   5648   5984    178    322   -131       O  \\nATOM    836  CB  VAL A 141       1.086  22.283  12.643  1.00 23.19           C  \\nANISOU  836  CB  VAL A 141     3136   2974   2699     66   -536    546       C  \\nATOM    837  CG1 VAL A 141       0.316  20.973  12.582  1.00 25.66           C  \\nANISOU  837  CG1 VAL A 141     3028   3485   3237   -224   -896    553       C  \\nATOM    838  CG2 VAL A 141       2.551  22.087  12.399  1.00 26.05           C  \\nANISOU  838  CG2 VAL A 141     2941   3695   3260    229  -1211    -49       C  \\nATOM    839  N   GLN A 142      -0.202  24.729  13.537  1.00 30.17           N  \\nANISOU  839  N   GLN A 142     4351   4668   2442   -142    972   -386       N  \\nATOM    840  CA  GLN A 142      -0.849  25.863  14.249  1.00 43.44           C  \\nANISOU  840  CA  GLN A 142     5780   5341   5383    836    992   -581       C  \\nATOM    841  C   GLN A 142      -1.266  25.367  15.610  1.00 53.97           C  \\nANISOU  841  C   GLN A 142     6968   7519   6017    338    569   1055       C  \\nATOM    842  O   GLN A 142      -2.348  25.603  16.175  1.00 73.17           O  \\nANISOU  842  O   GLN A 142     7022  11526   9254   1465    999    695       O  \\nATOM    843  CB  GLN A 142      -1.971  26.585  13.489  1.00 51.01           C  \\nANISOU  843  CB  GLN A 142     7039   5947   6394    728    162    714       C  \\nATOM    844  CG  GLN A 142      -1.444  27.425  12.340  1.00 51.77           C  \\nANISOU  844  CG  GLN A 142     6756   6371   6540    543   1032    322       C  \\nATOM    845  CD  GLN A 142      -2.436  27.561  11.197  1.00 56.41           C  \\nANISOU  845  CD  GLN A 142     7133   7428   6871    597    522   -176       C  \\nATOM    846  OE1 GLN A 142      -3.152  26.612  10.853  1.00 68.90           O  \\nANISOU  846  OE1 GLN A 142     5619   8561  11997   1270    196  -1772       O  \\nATOM    847  NE2 GLN A 142      -2.473  28.741  10.588  1.00 44.85           N  \\nANISOU  847  NE2 GLN A 142     5119   5999   5921    714    934  -1790       N  \\nATOM    848  OXT GLN A 142      -0.406  24.689  16.162  1.00 41.66           O  \\nANISOU  848  OXT GLN A 142     6097   5135   4596   -391    602    308       O  \\nTER     849      GLN A 142                                                      \\nHETATM  850  O   HOH A 201      16.072  12.445  12.809  1.00 32.78           O  \\nANISOU  850  O   HOH A 201     3763   3016   5674    208    112   -239       O  \\nHETATM  851  O   HOH A 202      -3.461  23.570  13.871  1.00 38.54           O  \\nANISOU  851  O   HOH A 202     4713   5079   4851   -399    481   -248       O  \\nHETATM  852  O   HOH A 203       4.720  30.032  13.736  1.00 27.49           O  \\nANISOU  852  O   HOH A 203     3630   3243   3569    639    542     90       O  \\nHETATM  853  O   HOH A 204      17.540  13.609   8.791  1.00 30.95           O  \\nANISOU  853  O   HOH A 204     3823   3464   4472   1259   -221   -640       O  \\nHETATM  854  O   HOH A 205      -0.706  30.223   1.739  1.00 33.97           O  \\nANISOU  854  O   HOH A 205     4577   3928   4401   -361  -1438    768       O  \\nHETATM  855  O   HOH A 206       9.791  14.149  12.055  1.00 25.97           O  \\nANISOU  855  O   HOH A 206     3108   4134   2625   -457    -13   -514       O  \\nHETATM  856  O   HOH A 207       1.888  25.492  16.683  1.00 20.22           O  \\nANISOU  856  O   HOH A 207     2714   1804   3163    361   -280    -17       O  \\nHETATM  857  O   HOH A 208      -2.781  16.073   5.175  1.00 23.42           O  \\nANISOU  857  O   HOH A 208     2574   2223   4100    329  -1207   -107       O  \\nHETATM  858  O   HOH A 209      -0.808  22.151  19.753  1.00 33.56           O  \\nANISOU  858  O   HOH A 209     4257   4942   3552   1703   1388   1101       O  \\nHETATM  859  O   HOH A 210      23.504   9.909  15.510  1.00 34.86           O  \\nANISOU  859  O   HOH A 210     5222   2561   5463    298    -37    936       O  \\nHETATM  860  O   HOH A 211      14.469  10.626  15.427  1.00 29.66           O  \\nANISOU  860  O   HOH A 211     5111   2890   3267   -145    762   -369       O  \\nHETATM  861  O   HOH A 212       3.158   1.329  16.175  1.00 31.50           O  \\nANISOU  861  O   HOH A 212     3677   3446   4847  -1451   -989    686       O  \\nHETATM  862  O   HOH A 213      14.908  32.975  10.694  1.00 31.76           O  \\nANISOU  862  O   HOH A 213     3750   5383   2933    618   -999      4       O  \\nHETATM  863  O   HOH A 214      10.839   7.700   6.986  1.00 26.51           O  \\nANISOU  863  O   HOH A 214     3206   2844   4021    294   -732    370       O  \\nHETATM  864  O   HOH A 215      24.368  22.251   4.457  1.00 17.78           O  \\nANISOU  864  O   HOH A 215     1687   2308   2758    106   -348   -137       O  \\nHETATM  865  O   HOH A 216      16.914  14.945  -1.038  1.00 34.36           O  \\nANISOU  865  O   HOH A 216     3175   1563   8314  -1160    337   1236       O  \\nHETATM  866  O   HOH A 217      13.464  34.647   0.395  1.00 27.91           O  \\nANISOU  866  O   HOH A 217     2896   3347   4360   -325   -168   2221       O  \\nHETATM  867  O   HOH A 218       4.178  27.982  12.709  1.00 30.97           O  \\nANISOU  867  O   HOH A 218     3013   5670   3083   1866   -178  -2218       O  \\nHETATM  868  O   HOH A 219      -1.289   8.103  12.619  1.00 29.28           O  \\nANISOU  868  O   HOH A 219     3141   2584   5400   -964   -383    418       O  \\nHETATM  869  O   HOH A 220      -0.397  16.592  21.384  1.00 30.14           O  \\nANISOU  869  O   HOH A 220     3324   4517   3609   1035    515   -674       O  \\nHETATM  870  O   HOH A 221       1.135  20.720   2.443  1.00 19.47           O  \\nANISOU  870  O   HOH A 221     2827   2651   1920    120     59   -102       O  \\nHETATM  871  O   HOH A 222       3.774  26.227   0.588  1.00 38.84           O  \\nANISOU  871  O   HOH A 222     5627   6465   2665  -1404  -1173    795       O  \\nHETATM  872  O   HOH A 223       8.026  28.642  18.545  1.00 13.27           O  \\nANISOU  872  O   HOH A 223     1796   1718   1527    -23   -101   -227       O  \\nHETATM  873  O   HOH A 224       5.751  28.590  25.338  1.00 29.36           O  \\nANISOU  873  O   HOH A 224     3661   3634   3861    357    196   -396       O  \\nHETATM  874  O   HOH A 225      22.203  25.686   6.288  1.00 18.66           O  \\nANISOU  874  O   HOH A 225     2311   2068   2710   -458   -499    137       O  \\nHETATM  875  O   HOH A 226      16.926  22.028  19.155  1.00 15.42           O  \\nANISOU  875  O   HOH A 226     2170   1620   2066    -41   -397    -95       O  \\nHETATM  876  O   HOH A 227      -0.836  22.407   8.306  1.00 24.94           O  \\nANISOU  876  O   HOH A 227     2290   3537   3646   -161   -998   -141       O  \\nHETATM  877  O   HOH A 228      16.492  16.599  23.166  1.00 20.68           O  \\nANISOU  877  O   HOH A 228     2712   2417   2726    631  -1125    -20       O  \\nHETATM  878  O   HOH A 229      10.740   7.732  12.092  1.00 23.12           O  \\nANISOU  878  O   HOH A 229     3673   1908   3201   -215    366     74       O  \\nHETATM  879  O   HOH A 230      15.104  29.601  13.170  1.00 28.76           O  \\nANISOU  879  O   HOH A 230     4049   3421   3455    480   -318  -1521       O  \\nHETATM  880  O   HOH A 231       3.263  23.022  24.032  1.00 24.10           O  \\nANISOU  880  O   HOH A 231     4046   3846   1265   -256    832   -273       O  \\nHETATM  881  O   HOH A 232      18.204  13.929  24.992  1.00 30.09           O  \\nANISOU  881  O   HOH A 232     5447   3267   2718    552   -821    456       O  \\nHETATM  882  O   HOH A 233      15.048  19.640  -6.556  1.00 20.81           O  \\nANISOU  882  O   HOH A 233     2183   2233   3490    -83    563   -373       O  \\nHETATM  883  O   HOH A 234      11.699  25.820  23.720  1.00 17.71           O  \\nANISOU  883  O   HOH A 234     2473   2306   1950   -143   -366   -425       O  \\nHETATM  884  O   HOH A 235       5.091   9.419   3.166  1.00 30.28           O  \\nANISOU  884  O   HOH A 235     4821   3669   3012  -1389  -1450   -325       O  \\nHETATM  885  O   HOH A 236       1.821  13.127  19.026  1.00 15.51           O  \\nANISOU  885  O   HOH A 236     2229   2211   1450    406     17    227       O  \\nHETATM  886  O   HOH A 237       9.159  16.556  14.003  1.00 17.08           O  \\nANISOU  886  O   HOH A 237     2526   1621   2340    -39    -99    113       O  \\nHETATM  887  O   HOH A 238      10.039  21.296  25.088  1.00 33.21           O  \\nANISOU  887  O   HOH A 238     6336   3541   2739    250   1340    636       O  \\nHETATM  888  O   HOH A 239      -2.531  16.382  14.607  1.00 25.63           O  \\nANISOU  888  O   HOH A 239     2728   3401   3609   1128  -1073   -563       O  \\nHETATM  889  O   HOH A 240      23.242  25.115  -0.309  1.00 25.33           O  \\nANISOU  889  O   HOH A 240     3065   3076   3481    639    803    282       O  \\nHETATM  890  O   HOH A 241      -0.040   8.199   9.475  1.00 32.67           O  \\nANISOU  890  O   HOH A 241     4288   3271   4851    112  -1905    -58       O  \\nHETATM  891  O   HOH A 242      11.468   4.392  20.223  1.00 24.94           O  \\nANISOU  891  O   HOH A 242     2955   2079   4440   -260   -996   1032       O  \\nHETATM  892  O   HOH A 243      14.898  23.577  20.270  1.00 15.70           O  \\nANISOU  892  O   HOH A 243     2249   1757   1959    -53   -561   -231       O  \\nHETATM  893  O   HOH A 244      22.757  22.392  14.937  1.00 43.05           O  \\nANISOU  893  O   HOH A 244     3486   5789   7082    371  -2216  -3794       O  \\nHETATM  894  O   HOH A 245      18.667  30.834   9.570  1.00 27.14           O  \\nANISOU  894  O   HOH A 245     4430   2292   3588   -717   -402   -354       O  \\nHETATM  895  O   HOH A 246      17.023  25.782  18.004  1.00 27.14           O  \\nANISOU  895  O   HOH A 246     4492   2650   3170   -690    346   -805       O  \\nHETATM  896  O   HOH A 247       8.046   0.035  14.891  1.00 30.20           O  \\nANISOU  896  O   HOH A 247     5763   2139   3570    579   -144    254       O  \\nHETATM  897  O   HOH A 248       8.703  16.604  24.118  1.00 23.45           O  \\nANISOU  897  O   HOH A 248     2999   3397   2511    -68    -54    281       O  \\nHETATM  898  O   HOH A 249      16.986  17.529  16.105  1.00 14.10           O  \\nANISOU  898  O   HOH A 249     1550   1496   2311     31   -494   -116       O  \\nHETATM  899  O   HOH A 250      11.761  29.680  18.688  1.00 21.64           O  \\nANISOU  899  O   HOH A 250     3858   2302   2060   -883    140   -635       O  \\nHETATM  900  O   HOH A 251      18.781  17.698  24.105  1.00 25.53           O  \\nANISOU  900  O   HOH A 251     3199   3773   2727   -578   -713    807       O  \\nHETATM  901  O   HOH A 252       7.706  36.673  10.812  1.00 33.86           O  \\nANISOU  901  O   HOH A 252     6313   3042   3508   1431   -720   -270       O  \\nHETATM  902  O   HOH A 253      20.506  28.414   0.094  1.00 24.78           O  \\nANISOU  902  O   HOH A 253     4483   2336   2596    751    177    -37       O  \\nHETATM  903  O   HOH A 254      12.723  24.262  13.183  1.00 13.88           O  \\nANISOU  903  O   HOH A 254     2166   1493   1613    186   -208   -280       O  \\nHETATM  904  O   HOH A 255       2.598  10.572  19.644  1.00 18.30           O  \\nANISOU  904  O   HOH A 255     2518   1861   2573     49     27    576       O  \\nHETATM  905  O   HOH A 256       2.265  26.089  13.658  1.00 33.42           O  \\nANISOU  905  O   HOH A 256     2719   6246   3731   -741    405  -3000       O  \\nHETATM  906  O   HOH A 257       7.454  20.274  23.127  1.00 20.28           O  \\nANISOU  906  O   HOH A 257     3463   2510   1730    789    258    684       O  \\nHETATM  907  O   HOH A 258      11.910  11.238  13.304  1.00 26.47           O  \\nANISOU  907  O   HOH A 258     3006   3169   3881   -213    520   -534       O  \\nHETATM  908  O   HOH A 259      19.213  28.738  12.435  1.00 31.47           O  \\nANISOU  908  O   HOH A 259     3943   3328   4687   -363   -792    193       O  \\nHETATM  909  O   HOH A 260      24.402  13.298   8.952  1.00 38.57           O  \\nANISOU  909  O   HOH A 260     5443   3034   6176   1520   3487   1091       O  \\nHETATM  910  O   HOH A 261      13.079   4.068  16.144  1.00 23.20           O  \\nANISOU  910  O   HOH A 261     2634   2596   3582    160   -139   -171       O  \\nHETATM  911  O   HOH A 262       6.214  14.758  -0.208  1.00 19.80           O  \\nANISOU  911  O   HOH A 262     2431   3217   1873   -432   -250   -123       O  \\nHETATM  912  O   HOH A 263      23.271  19.734  18.796  1.00 39.14           O  \\nANISOU  912  O   HOH A 263     2750   7717   4402  -1641   -475   -697       O  \\nHETATM  913  O   HOH A 264       2.836  23.614   1.991  1.00 21.18           O  \\nANISOU  913  O   HOH A 264     1936   4404   1707     34    -49   -350       O  \\nHETATM  914  O   HOH A 265      12.482  21.621  12.225  1.00 13.37           O  \\nANISOU  914  O   HOH A 265     2166   1325   1590    355   -262     30       O  \\nHETATM  915  O   HOH A 266      13.672   6.473  17.276  1.00 20.90           O  \\nANISOU  915  O   HOH A 266     2571   2386   2984    492    397   -143       O  \\nHETATM  916  O   HOH A 267      14.689  14.637  12.770  1.00 17.31           O  \\nANISOU  916  O   HOH A 267     2330   1473   2773   -249   -359    -51       O  \\nHETATM  917  O   HOH A 268      13.051  13.763   6.369  1.00 22.40           O  \\nANISOU  917  O   HOH A 268     2331   3322   2857   -339   -239   -398       O  \\nHETATM  918  O   HOH A 269      16.346  26.483  -2.777  1.00 31.51           O  \\nANISOU  918  O   HOH A 269     4277   2709   4983    982  -1557    158       O  \\nHETATM  919  O   HOH A 270       6.127  15.898  23.802  1.00 30.32           O  \\nANISOU  919  O   HOH A 270     3865   4290   3365   -456   -413   -824       O  \\nHETATM  920  O   HOH A 271       2.996   2.102  13.555  1.00 25.82           O  \\nANISOU  920  O   HOH A 271     3884   2270   3654  -1102   -503    286       O  \\nHETATM  921  O   HOH A 272      14.438  25.854  18.825  1.00 18.08           O  \\nANISOU  921  O   HOH A 272     2326   2224   2317   -171   -624    -65       O  \\nHETATM  922  O   HOH A 273      11.956  16.571  24.846  1.00 25.64           O  \\nANISOU  922  O   HOH A 273     4536   3568   1638   -430   -436   -279       O  \\nHETATM  923  O   HOH A 274       8.709   0.860   8.438  1.00 29.62           O  \\nANISOU  923  O   HOH A 274     4409   2369   4474    486  -1324    -68       O  \\nHETATM  924  O   HOH A 275      27.033  20.327   9.682  1.00 36.23           O  \\nANISOU  924  O   HOH A 275     3657   5369   4740   -798   -508    414       O  \\nHETATM  925  O   HOH A 276      10.175  11.601  10.654  1.00 24.34           O  \\nANISOU  925  O   HOH A 276     3553   3257   2435   -284   -748    -14       O  \\nHETATM  926  O   HOH A 277       6.709  10.890  24.386  1.00 22.85           O  \\nANISOU  926  O   HOH A 277     4266   2440   1976    195    405    388       O  \\nHETATM  927  O   HOH A 278      26.310  14.622  11.205  1.00 28.28           O  \\nANISOU  927  O   HOH A 278     2395   4380   3970    832   1142   1348       O  \\nHETATM  928  O   HOH A 279      14.073  26.336  -4.385  1.00 31.42           O  \\nANISOU  928  O   HOH A 279     4903   3856   3180    491    239  -1028       O  \\nHETATM  929  O   HOH A 280      16.377  37.438   3.806  1.00 26.37           O  \\nANISOU  929  O   HOH A 280     3539   3334   3145   -962   -427    751       O  \\nHETATM  930  O   HOH A 281      12.100  14.073  13.451  1.00 28.03           O  \\nANISOU  930  O   HOH A 281     2371   5224   3053   -495    134  -1393       O  \\nHETATM  931  O   HOH A 282      17.746  33.433   9.224  1.00 26.35           O  \\nANISOU  931  O   HOH A 282     3158   4500   2352    305   -338   -781       O  \\nHETATM  932  O   HOH A 283      12.332  29.838  14.050  1.00 20.55           O  \\nANISOU  932  O   HOH A 283     3787   1940   2082   -168     76    -74       O  \\nHETATM  933  O   HOH A 284      15.492  15.212   9.923  1.00 16.33           O  \\nANISOU  933  O   HOH A 284     2209   1831   2163   -272     95   -152       O  \\nHETATM  934  O   HOH A 285      22.376  28.552   6.164  1.00 32.44           O  \\nANISOU  934  O   HOH A 285     4173   3476   4676   -414   -465   -198       O  \\nHETATM  935  O   HOH A 286      -4.524  16.181  12.875  1.00 36.39           O  \\nANISOU  935  O   HOH A 286     2979   5987   4860    -75     -7   2076       O  \\nHETATM  936  O   HOH A 287       9.518  12.846  -5.721  1.00 29.40           O  \\nANISOU  936  O   HOH A 287     3441   3644   4085    622     30   -211       O  \\nHETATM  937  O   HOH A 288      -2.563  12.458  16.875  1.00 31.88           O  \\nANISOU  937  O   HOH A 288     2880   4624   4607   -327    240    858       O  \\nHETATM  938  O   HOH A 289       6.385  30.464  17.119  1.00 22.22           O  \\nANISOU  938  O   HOH A 289     3974   2090   2378     28   -332   -146       O  \\nHETATM  939  O   HOH A 290      21.737   9.162  13.944  1.00 31.17           O  \\nANISOU  939  O   HOH A 290     5634   3125   3082   1100    399     12       O  \\nHETATM  940  O   HOH A 291      18.169  38.276   7.095  1.00 32.62           O  \\nANISOU  940  O   HOH A 291     4227   5713   2452     19     89   -168       O  \\nHETATM  941  O   HOH A 292       6.341  16.415  -2.481  1.00 32.31           O  \\nANISOU  941  O   HOH A 292     4621   3358   4297    -22  -1169   -434       O  \\nHETATM  942  O   HOH A 293       7.484  36.776   4.155  1.00 33.55           O  \\nANISOU  942  O   HOH A 293     4163   3127   5456     -4     84    859       O  \\nHETATM  943  O   HOH A 294      14.864   8.058  15.220  1.00 28.10           O  \\nANISOU  943  O   HOH A 294     3538   3461   3676   -667    -87    297       O  \\nHETATM  944  O   HOH A 295       1.707  28.226  16.735  1.00 26.12           O  \\nANISOU  944  O   HOH A 295     3503   2736   3686    142   -337   -272       O  \\nHETATM  945  O   HOH A 296       6.035  12.051  -0.584  1.00 37.96           O  \\nANISOU  945  O   HOH A 296     7336   3938   3146  -1766  -1978   -197       O  \\nHETATM  946  O   HOH A 297       3.744  30.089  16.276  1.00 23.14           O  \\nANISOU  946  O   HOH A 297     3008   2866   2917     52    -53   -145       O  \\nHETATM  947  O   HOH A 298      11.678  19.277  25.632  1.00 40.33           O  \\nANISOU  947  O   HOH A 298     6293   4875   4155    665  -2141   -778       O  \\nHETATM  948  O   HOH A 299      -0.971  12.784  19.363  1.00 25.96           O  \\nANISOU  948  O   HOH A 299     3094   4024   2744   -139    607    431       O  \\nHETATM  949  O   HOH A 300      17.256   7.664  15.084  1.00 34.89           O  \\nANISOU  949  O   HOH A 300     4123   5276   3856   -862   -126  -1003       O  \\nHETATM  950  O   HOH A 301      24.424  24.955   4.816  1.00 29.40           O  \\nANISOU  950  O   HOH A 301     4605   3102   3461  -1011    450   -446       O  \\nHETATM  951  O   HOH A 302       8.778  16.481  11.241  1.00 24.68           O  \\nANISOU  951  O   HOH A 302     3200   2868   3306    169   -216   -479       O  \\nHETATM  952  O   HOH A 303       2.308  28.141  -0.112  1.00 36.10           O  \\nANISOU  952  O   HOH A 303     2592   5656   5467   -556   -325   -568       O  \\nHETATM  953  O   HOH A 304      13.049  15.218   8.691  1.00 20.84           O  \\nANISOU  953  O   HOH A 304     2631   2990   2295    -61   -478   -306       O  \\nCONECT  254  525                                                                \\nCONECT  525  254                                                                \\nMASTER      309    0    0    3    5    0    0    6  943    1    2    9          \\nEND                                                                             \\n\");\n",
              "\tviewer_17261593772982423.setStyle({\"model\": -1, \"chain\": [\"A\", \"B\"]},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
              "\tviewer_17261593772982423.zoomTo();\n",
              "viewer_17261593772982423.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Sample sequence designs for a given structure"
      ],
      "metadata": {
        "id": "7yS_H2nvun9m"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "### Use the single chain backbone as model input for conditioning"
      ],
      "metadata": {
        "id": "cNQuKmHbbZGo"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "#### Load model"
      ],
      "metadata": {
        "id": "WASWHU-AZoRk"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import esm\n",
        "model, alphabet = esm.pretrained.esm_if1_gvp4_t16_142M_UR50()\n",
        "model = model.eval()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "9ZwrouYWunXE",
        "outputId": "87f2d67e-b17b-4f7a-fc4e-1aa06a12c24d"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/esm/pretrained.py:215: UserWarning: Regression weights not found, predicting contacts will not produce correct results.\n",
            "  warnings.warn(\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#### Load structure from PDB file"
      ],
      "metadata": {
        "id": "izZBH8p2acxD"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "fpath = 'data/5OMT.pdb'\n",
        "chain_id = 'A'\n",
        "structure = esm.inverse_folding.util.load_structure(fpath, chain_id)\n",
        "coords, native_seq = esm.inverse_folding.util.extract_coords_from_structure(structure)\n",
        "print(f'Native sequence: {native_seq}')\n",
        "print(f\"length of native sequence: {len(native_seq)}\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "hYKDY7Myuzw7",
        "outputId": "d3c78946-e054-405e-ed60-6c320d6cd175"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Native sequence: ARYDDVLYFPASRYPETGAHISDAIKAGHADVCTIERSGADKRRQESLKGIPTKPGFDRDEWPMAMCEEGGKGASVRYVSSSDNRGAGSWVGNRLNGYADGTRILFIVQ\n",
            "length of native sequence: 109\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#### Sample sequence and calculate sequence recovery"
      ],
      "metadata": {
        "id": "fg-8r5U6m12r"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import numpy as np\n",
        "\n",
        "sampled_seq = model.sample(coords, temperature=1)\n",
        "print('Sampled sequence:', sampled_seq)\n",
        "\n",
        "recovery = np.mean([(a==b) for a, b in zip(native_seq, sampled_seq)])\n",
        "print('Sequence recovery:', recovery)\n",
        "\n",
        "print(f\"length: {len(sampled_seq)}\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "bbAsJp-BGzcb",
        "outputId": "56fa4917-21f8-41a9-909e-6bcd258aa3fc"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Sampled sequence: DAFDATLTYPAATYPAAGAHIAAAITSGAPAVCTLARAGAAARRDESLAGIPARPGYTRDEWPLALCVEGGTGSSVAYVPAADNQGAGAWMLGTLSAYADGTRILQVVS\n",
            "Sequence recovery: 0.5504587155963303\n",
            "length: 109\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Lower sampling temperature typically results in higher sequence recovery but less diversity\n",
        "sampled_seq = model.sample(coords, temperature=1e-6)\n",
        "print('Sampled sequence:', sampled_seq)\n",
        "\n",
        "recovery = np.mean([(a==b) for a, b in zip(native_seq, sampled_seq)])\n",
        "print('Sequence recovery:', recovery)\n",
        "\n",
        "print(f\"length: {len(sampled_seq)}\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "RdtEwGGbm8R0",
        "outputId": "e577a7e8-7e28-4621-d6d9-da77ca23b107"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Sampled sequence: MDYDAVLYFPAATYPEAGAHIKAAIAAGHAAVCTLDRAGAAARRDQSLAGIPARPGYDRDEWPFAICREGGAGSAVRYVPAADNQGAGAWLRNTLAAYADGTRILFVVG\n",
            "Sequence recovery: 0.6422018348623854\n",
            "length: 109\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "The temperature parameter controls the sharpness of the probability distribution for sequence sampling. Higher sampling temperatures yield more diverse sequences but likely with lower native sequence recovery. The default sampling temperature is 1. To optimize for native sequence recovery, ESM-IF1 recommends sampling with low temperature such as 1e-6."
      ],
      "metadata": {
        "id": "kOwj4knQkYji"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "#### Conditional sequence log-likelihoods for given backbone coordinates"
      ],
      "metadata": {
        "id": "7lEemfptnFeg"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "The log-likelihood scores could be used to predict mutational effects."
      ],
      "metadata": {
        "id": "opsdxM5PnJF3"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "ll_fullseq, ll_withcoord = esm.inverse_folding.util.score_sequence(model, alphabet, coords, native_seq)\n",
        "\n",
        "print(f'average log-likelihood on entire sequence: {ll_fullseq:.2f} (perplexity {np.exp(-ll_fullseq):.2f})')\n",
        "print(f'average log-likelihood excluding missing coordinates: {ll_withcoord:.2f} (perplexity {np.exp(-ll_withcoord):.2f})')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Jtln6zLvnIzO",
        "outputId": "c37ca656-0d43-4574-c90a-0cca39682aa9"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "average log-likelihood on entire sequence: -0.99 (perplexity 2.70)\n",
            "average log-likelihood excluding missing coordinates: -0.99 (perplexity 2.70)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Batch sample and scoring"
      ],
      "metadata": {
        "id": "bfGcnj74rOuM"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!wget https://raw.githubusercontent.com/facebookresearch/esm/main/examples/inverse_folding/sample_sequences.py -P scripts/"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "J7iYdWtCnG1W",
        "outputId": "2fb04519-2ff6-4120-8ae7-243027b69b7b"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-09-12 16:44:18--  https://raw.githubusercontent.com/facebookresearch/esm/main/examples/inverse_folding/sample_sequences.py\n",
            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 4505 (4.4K) [text/plain]\n",
            "Saving to: ‘scripts/sample_sequences.py’\n",
            "\n",
            "sample_sequences.py 100%[===================>]   4.40K  --.-KB/s    in 0s      \n",
            "\n",
            "2024-09-12 16:44:19 (73.4 MB/s) - ‘scripts/sample_sequences.py’ saved [4505/4505]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!wget https://raw.githubusercontent.com/facebookresearch/esm/main/examples/inverse_folding/score_log_likelihoods.py -P scripts/"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "KhM4n2ey7pJw",
        "outputId": "218bb943-daae-42cf-a080-081269829309"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-09-12 16:44:22--  https://raw.githubusercontent.com/facebookresearch/esm/main/examples/inverse_folding/score_log_likelihoods.py\n",
            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 4661 (4.6K) [text/plain]\n",
            "Saving to: ‘scripts/score_log_likelihoods.py’\n",
            "\n",
            "score_log_likelihoo 100%[===================>]   4.55K  --.-KB/s    in 0s      \n",
            "\n",
            "2024-09-12 16:44:22 (70.7 MB/s) - ‘scripts/score_log_likelihoods.py’ saved [4661/4661]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### Sample sequence designs for a given structure"
      ],
      "metadata": {
        "id": "i9C1ke9ej66r"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "Use backbone of the specified target chain as model input for conditioning."
      ],
      "metadata": {
        "id": "pgAQyS8pkCfW"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!mkdir -p output"
      ],
      "metadata": {
        "id": "AP6dRcsJq0XR"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "PDBID = '5OMT'\n",
        "chain = 'A'\n",
        "temp = 1\n",
        "nsamples = 500\n",
        "fname_sampled_seqs = f'data/{PDBID}_{chain}_ESMIF1_{temp}_{nsamples}.fasta'\n",
        "fname_sampled_seqs"
      ],
      "metadata": {
        "id": "5sS4cMLfmM_e",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 35
        },
        "outputId": "f0a4c2ff-ca2b-4527-cdd0-0c825e1e2c58"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'data/5OMT_A_ESMIF1_1_500.fasta'"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            }
          },
          "metadata": {},
          "execution_count": 19
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python scripts/sample_sequences.py data/5OMT.pdb \\\n",
        "    --chain $chain --temperature $temp --num-samples $nsamples \\\n",
        "    --outpath $fname_sampled_seqs"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Qky5BDEF71Du",
        "outputId": "8b8cb0ab-a98b-4b93-9165-8552286c1ff8"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/esm/pretrained.py:215: UserWarning: Regression weights not found, predicting contacts will not produce correct results.\n",
            "  warnings.warn(\n",
            "Transferred model to GPU\n",
            "Native sequence loaded from structure file:\n",
            "ARYDDVLYFPASRYPETGAHISDAIKAGHADVCTIERSGADKRRQESLKGIPTKPGFDRDEWPMAMCEEGGKGASVRYVSSSDNRGAGSWVGNRLNGYADGTRILFIVQ\n",
            "Saving sampled sequences to data/5OMT_A_ESMIF1_1_500.fasta.\n",
            "\n",
            "Sampling.. (1 of 500)\n",
            "Sampled sequence:\n",
            "AWFDAEWFFPCDRFPESCAQIRQAIALGHAAVCTLDRDGARARNREALAGIPRRPGYERHEWPFSECREGGRGSLVKYVDARDNQGAGRYLRRHLARYANGTKLLGVVK\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (2 of 500)\n",
            "Sampled sequence:\n",
            "VAFDTTLQFPAAQFPESGQHIQDAIAEGHAAVCTLARQRAWQRRQEALAGIPQRAGYQRHEWPFSICTEGGAGSAVRYVSALDNSGMGSWLRNWLSQYADGTRIRLVVR\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (3 of 500)\n",
            "Sampled sequence:\n",
            "GIFDATMQQPVAEFPESGKMIKQAIAEGFADVCTLDRAARKERYTQSLKGIPEKPGYDRHEWPFAYCKEGGAGSKVVYVKHDDNQGFGRWLDNHLSKFNDGTRIKQVVK\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (4 of 500)\n",
            "Sampled sequence:\n",
            "MKYDVTWVFPRSKYPESGALIRDAIARGHAAVCTLARESQRLREDQALAGIPRRDGFERHEWPFAWCEEGGKGSFVRYVGKNDASGAGRWIDTTLAAYEDGTKVLFLVD\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (5 of 500)\n",
            "Sampled sequence:\n",
            "CNYDAVLYTPCNTFPDACRHIKAAIAQGHAAVCTLDRAGRKRKSDQALAGIPRKPGYERHEWPFASCVEGGRGSAVTYVPTSDNQGAGAWLRQYLAGYADGTRILFVID\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (6 of 500)\n",
            "Sampled sequence:\n",
            "MSFDVILDLPCATYPASCNLIKQAISEGHPAVCTLNRYGRRKRREQSLVGIPDRPGYDRHEWPFASCDEGGHGAKVQYVNRADNQGFGNWLTGYLDQYPDGTKIKMVVK\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (7 of 500)\n",
            "Sampled sequence:\n",
            "HTYDAELVFPCDEFPESCAMVKAAVASGHAAVCTLARAGQRTRRRQALKGIPNRPGYDRHEWPMDICVEGGRGSAVQYVPSQDNEGMGRWLKGHLSQYADGTRILFRVG\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (8 of 500)\n",
            "Sampled sequence:\n",
            "MEYDKKFEFPRAKFPESGNMIKDAIARGFSRVCTLARKESRKRQRQALKGIPDKPGYDRHEWPMAICEEGGKGSAVQYVPWQDNNGAGSWLSFELSKYKDGTKILFIAD\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (9 of 500)\n",
            "Sampled sequence:\n",
            "KKYDATLVFPCQLFPQACAMIKQAIAEGHAAVCTLDSAAADARREESLSGIPDKPGYERHEWPFAICLEGGKGARVEYVQYRDNRGAGKWLGKKLSQYADGTKILFIVD\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (10 of 500)\n",
            "Sampled sequence:\n",
            "MAYDVTLDFPAKKYPQSGKHVKDAIKNGHQSVCTIRRSSRSRRLQESLKGIPNKRGYTRDEWPFAICKEGGKGSSVRYVRSADNQGAGRWLKKKLNAYANGTRVKYVVR\n",
            "Sequence recovery: 0.5963302752293578\n",
            "\n",
            "Sampling.. (11 of 500)\n",
            "Sampled sequence:\n",
            "MEFTHSFSFPASAFPQSAAHIAAAIASGEAAVCTLSRGSQSTRRQESLAGIPNKSGYDRAEWPFAICTEGGSGAAVRYVDQSDNRGYGEWLTNELSQYADGTRILFIIS\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (12 of 500)\n",
            "Sampled sequence:\n",
            "SEFDVTWEFPTDTFPESGAHIRDAISRGHAAVCTLKRAGRQTRLEQALAGIAERPGFDRHEWPFANCVEGGKGASVRYVDVVDNQGFGKWASLTLSGFADGTKVLSLSG\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (13 of 500)\n",
            "Sampled sequence:\n",
            "EEYDTVVELPCSRYPEACGHIRDAIARGHAAVCTLDRSGQRQRADESLRGIKSRPGYGRHEWPFSFCREGGAGSSVRYVPMGDNNGFGTWLSQRLSAYSEGTRIRFVCP\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (14 of 500)\n",
            "Sampled sequence:\n",
            "ADHDVELQLPADKYPETFATLKAAIKKGFSSVCTLDRASFQKRREQALAGIPDKPGYQKHEWPFAICKEGGKGADVAYVPDNDNEGFGRYLSKTLSRYKDGTKVRFITN\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (15 of 500)\n",
            "Sampled sequence:\n",
            "MSFDAVWDFPVEQYPSSGAMIKAAIADGFAQVCTLDRAARSQRLAESLAGIPRRDGYDRGEWPFASCVEGGTGSRVEYVPRADNQGAGAWANQFLSRYSDGTRILVRAT\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (16 of 500)\n",
            "Sampled sequence:\n",
            "SEFDARVFMPFETFPEAATMIRTAIEEGHAAVCTLQRSEQRQRRRESLEGIPTKEGFDRHEWPFAICEEGGVGSKVTYVHRSDNQGMGQWLRRKLSAYQNGTRILFVVE\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (17 of 500)\n",
            "Sampled sequence:\n",
            "FTYDVTVTFPQTTYPYSGLMIKQAISYGYADVCTLDRSGRKSRQYQSLRGKPDKSGFQRHEWPYACCKEGGRGSRIKYVPAADMSGAAAWTRSWWSGYADGTRVLFTVA\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (18 of 500)\n",
            "Sampled sequence:\n",
            "MEFDVVIDFPAAEFPESGALVAAAIAEGYAAVCTLARAGQSARRKDSLAGIPGRPGYDRHEWPFALCTEGGTGARVAYVAVADNSGAGAWFSNTFSAYADGTRILFKVG\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (19 of 500)\n",
            "Sampled sequence:\n",
            "GHFDHTLVLPIVDFPDAGAHIAEAIAQGKAAVCTLHRAAATKRFQQSLKGIPKLAGYDRHHWPFSQCEEGGEGASVEYVPFDDNDGFGRWLSETLAAYKDGTRILLKVE\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (20 of 500)\n",
            "Sampled sequence:\n",
            "SEFDVVIRLPCTEFPQACAHIRRAMNKGFSAVCTLDRAGRDRRRIESLAGIPRKPGYDRHHWPLAICREGGKGSDVRYVDRADNRGFGAWLRRHLSAHADGAKVLVVAD\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (21 of 500)\n",
            "Sampled sequence:\n",
            "ADYSEEVHLPKAKYPATGATISEAIRQGFADVCTLSRSQSNQRRRESLSGIPNRSGYDRHEWPQSQCVEGGRGSRVRYVPTADNRGYGSWLAATLAKYSDGTRIKYVVR\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (22 of 500)\n",
            "Sampled sequence:\n",
            "AEYDVTLTYPDSSYPESGALIKQAIAEGHAAICTLDRAGQKARREESLRGIPKREGYDRHEWPFAICQEGGKGSKVQYVNAEDNQGAGTWLRNTLSKYADGTRILFKVE\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (23 of 500)\n",
            "Sampled sequence:\n",
            "HSYDERLFFPAGTYPAAGTHIARAIREGHAAVCTLSRRGELTRADQALAGIPAKEGFIRAEWPMAVCAEGGKGAAVRYVPAADGTGASAWLSATLAAYADGTRVLFEVE\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (24 of 500)\n",
            "Sampled sequence:\n",
            "QAYDVTVELPCATYPEACSHIRTAMAQGYAPVCTIQRGASAARRRQSLAGIPARSGFERDEWPMAGCTEGGTGSGVRYVSATDNRGFGTWLANTLARYADGTRVLFIVR\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (25 of 500)\n",
            "Sampled sequence:\n",
            "GAYDAQLVLPCAVYPQACQQIRQAISQGHAAVCTLERAGANARRDESLRGIPSRAGYMRFEWPFAMCREGGRGASVAYVQETDDTGFGAWLAAELAKYADGTRILFVIE\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (26 of 500)\n",
            "Sampled sequence:\n",
            "GMYDTNLITPCAQFPEACDMIAKAISQGHAQVCTLDRGGQKTRQKQSLKGIPSKEGYGRFEWPYSLCLEGGTGSAVQYVPKDDNRGMGRWLQNYLSQYSDGTRVLFTVK\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (27 of 500)\n",
            "Sampled sequence:\n",
            "CTYDTVITFPATTYPETGALITTAIAEGYSAVCTLDRAGQLIRRTQSLAGIPDRVGFDRHEWPMAMCTEGGTGSRVRYVNWNDNRGAGAWLSNWLNGYADGTRVLFTVT\n",
            "Sequence recovery: 0.6146788990825688\n",
            "\n",
            "Sampling.. (28 of 500)\n",
            "Sampled sequence:\n",
            "MSYDATVYLPCAAYPESCAHIAAAIAGGHAAVCTLDRAGRDARARQSLAGIPARDGYDRHEWPFASCLEGGAGAAVRYVPSADNQGFGRWLANFLNGHADGTRILIKVG\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (29 of 500)\n",
            "Sampled sequence:\n",
            "MEYDTQLIYPAAAYPHAGALIKQAVAEGHAAVCTLDRAGSETRRLQSLAGIPARPGYTRHEWPFSICVEGGQGSAVRYVPADDSQGMSAWISNYLSRYADGTRVLMKVN\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (30 of 500)\n",
            "Sampled sequence:\n",
            "VGYDCTLALPTATFPASGAHIAAAIAQGKAAVCTLARSGAAARANSSLIGIAAKAGFDRDEWPMAHCTEGGVGSAVQYVPVDDNRGMGRWLSSTLSKYADGTRVLTVVG\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (31 of 500)\n",
            "Sampled sequence:\n",
            "ADYDAKLYYPDDEYPQSGAMVRQAINEGHAAVCTLSRAQRSQRRRESLVGIDTKPGFDRHEWPVAICEEGGRGSAVRYVDFADNSGAGAWLRRHLDQYADGTRILFVTD\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (32 of 500)\n",
            "Sampled sequence:\n",
            "GDFDTTLSVPVTQFPRAGSEIRQAIAQGKAAVCTLERGAQKKRRDEALAGIPARPGYGRHEWPFAECREGGRGARVRYVGFEDNDGMGKWLRDNLSKHADGTRIKFIVD\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (33 of 500)\n",
            "Sampled sequence:\n",
            "SMFDATLVFPAATFPEAGALIAEAIREGHAAVCTLQRAGSDARRRESLAGIPSREGYDRHEWPFACCQEGGQGAVVRYVPWRDNAGAGSWQRQTLSAYDDGTRILLKVE\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (34 of 500)\n",
            "Sampled sequence:\n",
            "SDYDVRWHFPIADFPWSGELVKTAIAEGKAAVCTLDRSGRTERLRQSLAGIPKKPGYVRHEWPFADCREGGAGSRVEYVPREDGQGYARWLTDRLGRYDNGTRVLFVVE\n",
            "Sequence recovery: 0.45871559633027525\n",
            "\n",
            "Sampling.. (35 of 500)\n",
            "Sampled sequence:\n",
            "VSYDVVVTYPTALYPQAGSHIKQAIGQGHSSVCTLDYAGRERRRRESLQGIPRRTGYDRHEWPMAMCTEGGTGSSVRYVDRTDNQGMGTWILNWLAKYDDGTRVKTVVS\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (36 of 500)\n",
            "Sampled sequence:\n",
            "ADNDAEVFFPGSRFPQSAQHIAQAIAEGHAAVCTLSRSNEATRWNMSLKGIPARSGYQRHEWPFAQCTEGGSGSAVRYVDAADNQGQGNWTRSTLSAYADGTRLLYVVS\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (37 of 500)\n",
            "Sampled sequence:\n",
            "SEFDAEATFPCRQYPQSCALIARAIAEGFAAVCTLDRAGEHQRRREALRGIPSKPGYDRHEWPFAYCKEGGKGSAVAYVPKEDNSGAGAWFRREMSKYADGTRLLFRVQ\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (38 of 500)\n",
            "Sampled sequence:\n",
            "MLHDATLILPCAEYPQAGAHIRAAIAAGHSPICTLDRDGQRARQRESLKGIATQPGFHKQEWPFARCVEGGQGSAVQYVGWRDNEGMGRWLHRYLARYRDGTKIKMIVS\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (39 of 500)\n",
            "Sampled sequence:\n",
            "RDYDVSLDLPSEQFPESAAHIRQAIAQGHSAVCTLLSSSASRRRSQSLAGIPSRPGYVRSEWPMAICEEGGAGSAVQYVPRADYQGLEAWLRNHLAAYADGTRVLFRVG\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (40 of 500)\n",
            "Sampled sequence:\n",
            "ADYDATANFPCQKYPEACATIAAAISNGQAQVCTLSRAEQTRRYRESLKGIPARDGYGRHEWPMAFCVQGGNGSQVRYVNFSDQNGAGSWLESWLSQYKDGTKIKFVVK\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (41 of 500)\n",
            "Sampled sequence:\n",
            "EPYDVRQQVPCGSYPQTCAHVRQAIANGDAAVCTLSRGGQSTRRQQSLAGIPAREGYDRHEWPHDLCREGGAGSAVRYVQHSDHNGEGSWASSALSSFADGTKVLWVRD\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (42 of 500)\n",
            "Sampled sequence:\n",
            "MTYDTTLTFPCADYPQSCSHVKNAIAEGNAAVCTLNRDSQGRREDESLAGIPSRNGYDRHEWPFAICEEGGTGAGVRYVAKADNSGFGKWLRSFLDTFADGTRIKFVVE\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (43 of 500)\n",
            "Sampled sequence:\n",
            "MRYDAVVLFPEEIYPEAGRMISEAISRGYSSVCTLEREKEHVRRDQSLKGIPKKPGFTRHEWPFASCKEGGEGSYVKYVPVKDGNGFGEWLTNQLSRYKDGTRILFIGS\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (44 of 500)\n",
            "Sampled sequence:\n",
            "ASFDTCLNFPKTQFPFAGKMIEAAIEAGHAKVCTLNRAGRQSRLKQSLRGIPRKSGFHRHTWPFSICEEGGSGAMVQYVPVQDASGYGKWASKTLSRYKDGTKVLFVVL\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (45 of 500)\n",
            "Sampled sequence:\n",
            "ADWDAELTYPAEAFPNSGACARQAQSEGHAAVCTLDRAARRARRNESLRGIPERPGYVRHEWPFAWCLEGGRGARVTYVPAADGEGAARWLTRTLSGYGDGTRILVVIP\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (46 of 500)\n",
            "Sampled sequence:\n",
            "NSYDTTLLLPSKQFPQASAHVALAIAQGHSPVCTLDRAGSGIRRQQSLAGIPKKPGYTRDEWPFALCQQGGQGSVVRYVPTADNSGYGQWLQQYLAKYQNGTKILLEVK\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (47 of 500)\n",
            "Sampled sequence:\n",
            "SEFDATWDFPADEFPASGAQIAAAIEAGHDAVCTLDRAGMTTRRRESLAGIPEREGYDRHEWPFACCKEGGTGSAVEYVPSADNKGFGTWLHNKLGAYADGTRILVTVD\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (48 of 500)\n",
            "Sampled sequence:\n",
            "DDFDKKWEFPSDTYPESGKMIKDAIAEGHSKVCTLNRSERNRRLTESLKGIPSRPGFQRHEWPLAMCNEGGKGSKVQYVPAVDHNGAGGWLRETKSKYDDGTKILFIVQ\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (49 of 500)\n",
            "Sampled sequence:\n",
            "RAWDATLSLPGSTFPAAAAMIKTAIANGHASVCTIERAGETTRREQSLAGIPRQPGFGRAEWPFAFCLEGGLGALVDYVPITDNDGFGRWLGATLNKYADGTRIRFVVG\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (50 of 500)\n",
            "Sampled sequence:\n",
            "SDYDAVIDYPSSQFPEAGAHIKQSIALGHAAVCTLERSGSKGRRDQALKGIPQRSGYERHEWPFAYCREGGKGSEVKYVQRSDNQGAGAWLANKLSGYAEGTRLLMKVK\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (51 of 500)\n",
            "Sampled sequence:\n",
            "MAYDAVENVPTATYPESGADIKAAIAEGKAAVCTLDRAGADTRRTQALAGIPARSGYDRHEWPMAICREGGAGSRVQYVNEADNAGMGNWLAGHWDKYADGTRLLHTVG\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (52 of 500)\n",
            "Sampled sequence:\n",
            "HQFDAIWWFPAEKYPESGAHVAAAIEAGHSSVCTLEREDSAKRRQQALKGIPKKDGYIRHEWPFSFCREGGKGSSVEYVPSDDGNGFQSWMRQELSRFADGTRILWVNE\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (53 of 500)\n",
            "Sampled sequence:\n",
            "MAYDLEVYLPCQTYPDACAHIQKAIAQGHAAVCTLDRAGAKVRRRESLKGIPRKLGYNRDEWPFAVCKEGGRGSAVEYVPEADNNGMGGWLSGYLSAYADGTRILFNVK\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (54 of 500)\n",
            "Sampled sequence:\n",
            "GPYDDELYFPAATYPQSGNLIRQAIAEGQDIVCTLARAERATRYDQSLAGIPKKPGYGRFEWPFACCVEGGPGSRVQYVDYSDLDGAASWMRQRLAAYADGTRILFIID\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (55 of 500)\n",
            "Sampled sequence:\n",
            "AIFDKVLRFPAKNFPASASLVSQAISEGQASICTLNRAGAVVRRAQSLRGIPSKQGYDRHEWPAAFCLEGGKGAAVRYVPADDNQGAGTWVSQQLSAYSDGTKILIVNG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (56 of 500)\n",
            "Sampled sequence:\n",
            "CDFDEELLFPLKEYPYSGRVIADAIKQGFSAVCTLEREGAAKRRKQALAGIPGRKGFGRHEWPMSICKEGGSGAKVEYVPLADNNGFGTWLRKKLSKYANGTRIRFVID\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (57 of 500)\n",
            "Sampled sequence:\n",
            "DNYDAVYIFPCSVYPASCSHIRRAIAAGKAAVCTLHRAGQDARRRDSLKGIPERPGFDRHEWPMAFCQEGGRGASVEYVPTADNNGAGRDLRNALSAFADGTRILWIVR\n",
            "Sequence recovery: 0.5963302752293578\n",
            "\n",
            "Sampling.. (58 of 500)\n",
            "Sampled sequence:\n",
            "MSYDTTLHLPAAKYPQAAAHLKKAVQEGYAEVCTLEREAADARRRESLKGIPGKPGYGRDEWPFAICREGGKGSAVTYVNGADNQGFGGWLNGHLNNYDNGTRIKFVVG\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (59 of 500)\n",
            "Sampled sequence:\n",
            "STFDAEVLLPSSAFPHAARHLHDAINRGFAAICTLDRTGARLRRRQSLAGIPKKPGYSRHEWPMAYCVEGGKGASIEYVPINDNQGWGRWLSNWVSSYADGTRLLFVVR\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (60 of 500)\n",
            "Sampled sequence:\n",
            "PSHDDELIFPAAKFPQSGASIKQAIAEGYSAVCTLQRADRDNRRQQSLAGIPAKDGYDRHEWPFAMCEEGGRGSKVRYVERLDNQGAGAWVNGNLAGYSDGTRILLVVQ\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (61 of 500)\n",
            "Sampled sequence:\n",
            "ADYDVEVVLPCKEYPETCAMIKAAISEGKADVCTLDRAGSGKRRAESLKGIPARSGFGRHEWPMAICKEGGSGSNVRYVPYADNSGAGTWIANHLASYADGTKVLFKIS\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (62 of 500)\n",
            "Sampled sequence:\n",
            "MDYDTTLSLPCNTYPQACAHISKAISEGHASVCTLNREGSSQRRDQSLAGIPRKSGYDRHEWPFDFCLEGGVGSAVEYVDAADNQGFGSWLRQYLSAYSAGTRVKFVCP\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (63 of 500)\n",
            "Sampled sequence:\n",
            "ADYDAELEFPCKVYPEACGHIKKAIQLGYSAVCTLLRESARRRRNESLKGIPTRNGYHRHHWPMASCEEGGKGSSVQYVDAADNDGSGQWLQQWLSKYADGTRILFKCP\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (64 of 500)\n",
            "Sampled sequence:\n",
            "FDYDEEVKFPCEKYPEACQLIKKAIAEGHADVCTLLRSGQGQRKKEALNGIPTREGYTRHEWPFSMCKEGGAGSAVQYVPAEDNDGMGKWLYEFTSQFADGTKIKFIVE\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (65 of 500)\n",
            "Sampled sequence:\n",
            "MIYDATFQFAKASYPEAGAMIEKAIAEGHAAVCTLERTSSATRRAESLAGVPEKPGYIRFSWPMADCREGGKGSEIQYVNEDDGRGAGKWRNAALNQYADGTKIKFVAA\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (66 of 500)\n",
            "Sampled sequence:\n",
            "MSYDVRLDFPYREFPDSGAVIKEAISLGFSEVCTIATIGRGRRRQQSLRGIPKRPGYDRHEWPWDICLEGGEGALVRYVPRDDNAGFGRWLAEELAKYKDGTRVLFIVK\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (67 of 500)\n",
            "Sampled sequence:\n",
            "MAYDKTLVLPCSEYPQTCAMIKEAIAKGKAAVCTLDRSQFGKRFNEALKGIPKKTGYDRHEWPFASCVEGGKGSAVRYVKATDNRGAGKWFWSHTSKFADGTRILFIAR\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (68 of 500)\n",
            "Sampled sequence:\n",
            "GPYDATLNLPEAQYPESAALIKAAIAAGHASVCTLERAGQTQRRNESLKGIPAKPGYDRHEWPFATCVEGGKGAAVQYVDADDNQGMGRWLGTTLNGYADGTKLKITVE\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (69 of 500)\n",
            "Sampled sequence:\n",
            "WEFDTRVVFPDKTFPSSGKTIKQAIAQGHAKVCTLDRAGADTRRLESLAGIPKRSGYVRHEWPMSFCTEGGKGSAVTYVPAADGQGAGAWLSNYLAQYADGTRVLFIVS\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (70 of 500)\n",
            "Sampled sequence:\n",
            "MQYDHTLTFPVAQYPESGAHIKAAIAAGHASVCTLQRADAGRRREQSLAGIPDRPGFQREEWPFALCREGGSGSSVQYTPADDNQGAGSWLANTLSAFADGTRIKFITN\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (71 of 500)\n",
            "Sampled sequence:\n",
            "PAFDVQLQLPCRQYPGYCAMVRRAIAQGHAAVCTLARGGKAQRRQQSLQGIPRRAGYTRAEHPFADCLEGGAGSFVQYVAAADNNGAGAWLRNHLSGYQDGTKVLFVAR\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (72 of 500)\n",
            "Sampled sequence:\n",
            "AAYDTTWNYPTSTYPESGRMIKAAIAEGHAAVCTLRRSTATQRKNQSEAGIPAKSGYGRFEWPAAMCVEGGTGSRVAYVEQSDLNGAGTWYSNTYSKYADGTKIKNVVD\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (73 of 500)\n",
            "Sampled sequence:\n",
            "ADFDEVLHYPAEQYPQSGEHIRLAIERGFSSVCTLTRDGENQRRDQSLAGIPQRVGYMRHEWPFAHCSEGGLGSAVMYVDQADNSGMGAWLRGRLSAYGDGTRILWVIY\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (74 of 500)\n",
            "Sampled sequence:\n",
            "GDFDVTLTFPDDVYPEAGAMVEQAIAEGHAAVCTLRRLDHRARLKQSLAGIPRRRGYDRHEWPFSWCEEGGDGAAVRYVQREDNRGAGGWLSQTLSGYADGTRVLWTVG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (75 of 500)\n",
            "Sampled sequence:\n",
            "GKWDTTVQLPCATFPESCAHIKEAIAQGKAQVCTLARSGESTRRNQSLAGIPKKTGYSRHEWPYSRCTEGGAGSGVRYVPYDDNNGMGTWIGNTMSTYADGTKVKLVCG\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (76 of 500)\n",
            "Sampled sequence:\n",
            "HMHDVDLYLPIDQYPEAGAHIKQAISEGHAAICTLERAAQRERLKQSLEGIPQREGFDRFEWPIALCQEGGTGSSVRYVKTDDNQGAGSWLRQRLSFYDDGTKVKFIVE\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (77 of 500)\n",
            "Sampled sequence:\n",
            "STFDAVVYFPCERFPASCALVHTEISNGHASVCTLDRAGRAIRRNESLHGIPAREGYDRHEWPFAGCREGGAGAAVQYVPRADNKGFGAWLREWLGGYADGTKLRFVVS\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (78 of 500)\n",
            "Sampled sequence:\n",
            "MAYDVKVVYPKKTFPYWGDHVKRAIAAGHPEVCTLSRAARSQRYNEALEGIPKKPGYVRDEWPFATCEEGGEGSAVRYVPKEDDQGAGRWFQNYSSRYEDGTKILFEIG\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (79 of 500)\n",
            "Sampled sequence:\n",
            "ERFDAVMLLPNDEFPESGAHIRNAIAAGFSQVCTIDRATADRRREQALAGIPAEDGFDRHEWPLSTCKEGGEGAAVTYVPQADNRGYGRWLGAYLSGFADGTRILLVAA\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (80 of 500)\n",
            "Sampled sequence:\n",
            "MAYDAVLTFPAAAYPASGAQIKSAIAQGHAAVCTLDRSGAATRADQSLAGIPKKTGYVRDEWPMAVCVEGGTGSAVQYVPAADAQGAAAWLAQQLGAYADGTKILIVVD\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (81 of 500)\n",
            "Sampled sequence:\n",
            "ADYDAEVDLPRAQFPSAGKHIRTAIAQGKAAVCTIDRAGEAARRAQSLAGIPAKPGYDRDEWPFAICREGGNGSAVQYVPSDDNQGMGRWLGRLLSRYADGTRIRFVVK\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (82 of 500)\n",
            "Sampled sequence:\n",
            "MTYDFTLALPCSAFPAACAHVARAVSEGHAAVCTIEARAHAVKADEALVGIPGRAGYDRDEWPFAWCKEGGKGSGVEYVPADDNLGYAAWLAATLGAFADGTRVLFKVQ\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (83 of 500)\n",
            "Sampled sequence:\n",
            "YDFDVRVWLPCATYPEACAQVRRAIAEGHPRVCTLSRAGRRVRRDASLQGIPGRRGYDRHEWPFANCEEGGTGARVTYVPEADNAGMGRWTATRLAQYADGHRVLFVVE\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (84 of 500)\n",
            "Sampled sequence:\n",
            "TDHDTVWDLPCAEFPESCAMIRAAIEFGKSRVCTLARAGARARRQQSLAGIPSKRGFDRHEWPFAFCQEGGRGSLVRYVPYLDNQGMGNWFGASAARYADGTKVLYRVG\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (85 of 500)\n",
            "Sampled sequence:\n",
            "GEYDVVVQLPCDTYPEACRHIQSAINEGHAAVCTIERELASLRRGQALRGINNKRGFHRFEWPMAICQEGGRGSKVQYVPASDASGFGKWLHWKLSGYADGTRVRFVVE\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (86 of 500)\n",
            "Sampled sequence:\n",
            "ADYDIQVTVPADTYPESAAVIKQAIKEGFSDVCTLDRAGEKARADQSLKGIPRMTGYERHEWPFASCVEGGKGAQVQYVNMDDNNGFGEYMRKTLSAYADGTRVKFVID\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (87 of 500)\n",
            "Sampled sequence:\n",
            "NDFDDRLYFPLNTYPHAGNHIRKAIAEGKASVCTLKRAGQGKRRKASLRGIPTRPGYTRDEWPMAFCREGGRGSAVRYVPAADNNGMGSWLRNKLSKHADGTRILFVAQ\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (88 of 500)\n",
            "Sampled sequence:\n",
            "GTFDVELSLPCEKYPQACASIAQAIREGHAKVCTLDRAGRSARRDEALAGIPSKPGYVREEWPYASCKEGGAGSKVRYVANADNQGFGNWLRSELDQYEDGTRVLFKVG\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (89 of 500)\n",
            "Sampled sequence:\n",
            "WTFGVTWQYPADAFPQAGAQIREAISQGHSAVCTLKRASAASRADQALAGIPTRPGYGRHEWPASVCEEGGAGALVRYVPLADNRGAGTWIRRRLSSYANGTRIRQVVD\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (90 of 500)\n",
            "Sampled sequence:\n",
            "MDFDRELLFPCSQYPEACDHIRTAIKEGFSSVCTLDRAGQGKRYTESLQGIPQKSGHLRDEWPMATCLEGGVGSRVRYVPNPDGQGARNWLNTWLSGFKDGTKIKFICG\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (91 of 500)\n",
            "Sampled sequence:\n",
            "SEYDVELIFPAEVFPEAGAMVKSAIAQNFPEVCTLARDGQERRLDESLQGIPSKEGYDRHEWPTAFCCEGGTGAQVTYVKHTDNRGNGHWISGTLSNYEDGTKVLFKSK\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (92 of 500)\n",
            "Sampled sequence:\n",
            "SRFDATWVLPAAQFPQAAAHIRDAIADGHAAVCTLDRAGQTEREDESLAGIPRREGFDRDEWPFATCVEGGTGSAVRYVPKADNRGMGDWLNNSLAQFADGTRILLVVA\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (93 of 500)\n",
            "Sampled sequence:\n",
            "SDYDVTWTFPADTFPEAGAMIKAAIAAGHASVCTLDRAGQQQRRNESLAGIPNRDGFERHEWPMAQCVEGGKGSAVTYVPVDDNSGFGKWYQDTASNYANGTKILTIVD\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (94 of 500)\n",
            "Sampled sequence:\n",
            "GMYDAELVVPCETFPDACELIRAAVAEGHAEICTLQRERAGERRREALAGIPSREGYERHEWPFAMCREGGAGARVRYVPGEDHRGFGFWLVNELARFADGTRILFRVE\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (95 of 500)\n",
            "Sampled sequence:\n",
            "HLYDAVLHLPAATYPQTGAHVRRAIALGHAAVCTLSRAEAGARRRQSLAGIPQREGYVRQEWPMATCREGGLGAAVRYVPVEDGQGANQWLAGYLSGYADGTRILFVVD\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (96 of 500)\n",
            "Sampled sequence:\n",
            "MEYDATLYLPAEKYPESAAHISLAIREGKAKICTLSREGQRKRRDASLAGIPKKEGFERHEWPFSFCEQGGKGSSVKYVPQDDNQGFGRWLSNTLSNFADGTKILFVYT\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (97 of 500)\n",
            "Sampled sequence:\n",
            "MDFDATLVLPCDEFPESGAVIAHAIRQGHSAVCTLAREGRQKRRRESLAGIPTRTGYQRDEWPFAQCMEGGAGSEVQYVHHNDSSGFGAWLGNWLSQYADGTRIRFEVL\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (98 of 500)\n",
            "Sampled sequence:\n",
            "RDYDAVLYLPCKSYPEACATIRQAMAEGHAAVCTLDRAGEAVRRDEALMGIPSKPGYDRHEWPMASCREGGKGSRVRYVPWDDNRGFGAWLATHISGFADGTRIKFVAD\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (99 of 500)\n",
            "Sampled sequence:\n",
            "MSYDAQVVFPADRFPASAALIKTAIAEGHASVCTLDRAGRNARRNASLKNIPDRPGFDRHEWPFAWCREGGQGAKVEYVPCADNHGAGGWLSSWLSAYSDGTRIRIVAG\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (100 of 500)\n",
            "Sampled sequence:\n",
            "GAYDAEWKYPDKKYPESGNHIKEAIMKGFSKVCTLSRGSERQRREEALRGIPKKDGYDRHEWPAAICIEGGRGSDVRYVKSSDNQGAGSDERSWLAQYADGTRILDVIQ\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (101 of 500)\n",
            "Sampled sequence:\n",
            "TSFDAELELPCQEFPESCAHIRAAIAAGHAAVCTLDRHGASARRKESLKNIPDKKGYERHEWPFAWCKEGGKGSAVRYVKAKDNRGHGSWLKNTLNAYADGTRIKFTVP\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (102 of 500)\n",
            "Sampled sequence:\n",
            "TDHDVTWDYPSAQYPESGEQIRLAINLGYADVCTLDRAGQRKRDFEALSGIPNLPGFVRAEWPFAFCKEGGLGSWVAYVNASDAMGAARWASDQLASYADGTRVKVVVA\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (103 of 500)\n",
            "Sampled sequence:\n",
            "GSYDVVVYFPSSVYPQSAALLRQAIGQGHAWVCTLARKEADIRRAQSLKGIPQKEGYDRHEWPAAYCKEGGKGSGVRYVDHADNQGAGRWLADWLSRFADGTKVKIIAE\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (104 of 500)\n",
            "Sampled sequence:\n",
            "SMFDVVLEFPAKTFPQAGAMIKQAIAEGHSAVCTLQRARAKRRRVDSLRGIPGRPGYDRHHWPFASCLEGGQGSGVRYVEARDNRGAGSWIRQHLNRFADGTRVLFVVK\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (105 of 500)\n",
            "Sampled sequence:\n",
            "SNFDAVMHFPGDQFPQAAAMIRRAIAQGHSAVCTLDRAQQRKREDESLKGIPSKTGFVRFEWPMASCKEGGKGSSVEYVDAADAQGMGRYLAAKLGGFEDGTRIKFIKE\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (106 of 500)\n",
            "Sampled sequence:\n",
            "GDFDAVLDLPADQYPESGAHIKQAIADGKADICTLNRSKKKARQGESLKGIPKRQGYDRDEWPMASCIEGGKGSKVVYVKSTDNNGFGRWLSERLAQYKDGTRIKFVIR\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (107 of 500)\n",
            "Sampled sequence:\n",
            "PRYDVVMKHPCATYPQGCAQIAKAISEGHAAVCTLARSSAGTRRTQSLNGIPKRDGYDRHEWPAAWCKEGGLGSSVTYVNVSDNSGDGAWTSNKLSGYADGTRILFTVV\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (108 of 500)\n",
            "Sampled sequence:\n",
            "RNYDVEMNLPGDDFPYSAAHIQLAIQQGKAWVCTLERAGAEARREQSLAGIPERPGFERHEWPFAHCREGGAGASVMYVPAADNRGMGTWLANTLNAFADGTRVRFVVD\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (109 of 500)\n",
            "Sampled sequence:\n",
            "TNYDAELLLPCSEYPASCAQISAAIAEGKSKVCTLDRKSASQRKAEALRGIPARPGFIRHEWPFANCQEGGDGSAVKYVPSSDGQGFGKWLDQFLANYADGTRILFKVQ\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (110 of 500)\n",
            "Sampled sequence:\n",
            "MSYDAKLVIPCASYPESCAMIKRAIAQGHSAVCTLDREQQSRRRQESLAGIPAKPGYDRHEWPLAECIEGGKGAAVQYVPYQDNRGFGKWLWNHLAQFNDGTRIKIVVE\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (111 of 500)\n",
            "Sampled sequence:\n",
            "MGYDTRVKLPCKSFPQACAHIRDAISKGHAAICTLDRAQRSRRLQQALKGIPKKSGFTRHEWPQAFCKEGGKGSAVRYVKSDDANGFARWLSNRLRGYKDGTKVLHIVS\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (112 of 500)\n",
            "Sampled sequence:\n",
            "GEYDAVLEVPCEQFPEACKHIARAIRNGHESVCTLNRAQQKVRSDESLEGIPEREGYDRDEWPMSWCVEGGEGSSVAYIEASDNRGYGAWLSGELSQYDDGTKILWVVE\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (113 of 500)\n",
            "Sampled sequence:\n",
            "ADYDVTLTLPTSTYPEAGAMIARAIKQGYAAVCTLSRAEAAKRRAQALKGIPSKSGYNRHEWPMAQCKEGGKGSAVEYVPFSDNGGFGAWLRAELSRYADGTRVKLNVK\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (114 of 500)\n",
            "Sampled sequence:\n",
            "KDYDAEVHFPANKYPEAGRMIRIAIEEGHSDVCTLNRAGAARRRQASLKGIPTREGYDRHEWPMAYCEEGGDGSRVEYVNRADNRGMGHYLRNFMSGFADGTKILFIVD\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (115 of 500)\n",
            "Sampled sequence:\n",
            "ENYDAKLKYPVNQFPESGAMISEAIKQGYARTCTLDRAGARRRRKESLKGIPMKSGYIRAEWPFASCKEGGKGSYVRYVKTSDGQGAAAWLRKHLSQYANGTKILFIVK\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (116 of 500)\n",
            "Sampled sequence:\n",
            "SPFDATAIFPCEQFPQSCRQIASAIAEGYSSVCTLERWAFRERLNQSLKGIPRRDGYVRHEWPMSACREGGTGSRVEYVEESDSQGSARWMRTWSAQFPDGTRIQVKVG\n",
            "Sequence recovery: 0.4036697247706422\n",
            "\n",
            "Sampling.. (117 of 500)\n",
            "Sampled sequence:\n",
            "GSYDAELELPCRKYPETCAHVKAAIAEGFRAVCTLEREQESERKKESLWGIADKPGFDRHEWPFSLCQEGGKGAGVSYVPHDDNRGMGRWLSEKLSKYADGTKIKLVVR\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (118 of 500)\n",
            "Sampled sequence:\n",
            "GSFDTKLLYPANKYPNSGDHIALAISEGYADVCTLDRAKRANRRTESLRGIPSRSGYHRFEWPFAFCKEGGSGSSVRYVPADDGNGDATWLKNNLAKYADGTKVKFVVS\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (119 of 500)\n",
            "Sampled sequence:\n",
            "GSHHATVTLPVAQFPHSGAHIKQAIAEGKAAVCTLNRAGETTRRNQSLAGIPAKPGYDRDEWPFSLCVEGGAGSSVRYVPTSDNQGWGNWFKNFNNGYADGTKLLYTIG\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (120 of 500)\n",
            "Sampled sequence:\n",
            "MSFDTVINFPRVQYPDAGALIKSAINEGHSAVCTLDRAQQSKRLDAALLGIPDRNGYGRFCWPASNCVEGGAGSKVQYVPAADNNGAGNWLKNRLSSYADGTRVLGIVG\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (121 of 500)\n",
            "Sampled sequence:\n",
            "MNYDTTLTLPTAQYPEAGALIKAAISQGFSNVCTLERSSATQRRDESLAGIPGKAGFNRHEWPMAMCREGGKGSAVAYVESSDNQGMGKWLATTLSSYADGTRIKIVAS\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (122 of 500)\n",
            "Sampled sequence:\n",
            "KKYDVELVFPIAKFPESGAVIKNAIDEGHSKVCTLDRFSRAERRDQSLKGIPVKPGYGRHEWPFADCKEGGKGSRVEYVDYKDDSGYGKWLANTLSKYSDGTKVLMVVK\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (123 of 500)\n",
            "Sampled sequence:\n",
            "SMFDARLALPCETFPAACDHQRRAIAEGHAHICTLKRADAARRRRESLAGIPDRPGFERHEWPFALCEEGGAGSAVRYVPQEDNRGFGDWLRHTLAHYLDGTRILFEVD\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (124 of 500)\n",
            "Sampled sequence:\n",
            "ADYDVTVFLPCENFPAAGEVIKQAIAEGHSSICTLARASSARRRTESLSGIPEKEGFGRHEWPFASCEEGGLGAKVAYVPYEDINGANSWLRRHLARYADGTRVKFVVN\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (125 of 500)\n",
            "Sampled sequence:\n",
            "GNHDATRILPCARYPESCEHIRLAIAAGHAAVCTLDRLQARQRRKQSLAGIPRRSGFGRDEWPMAWCLEGGAGSAVWYVPLADNQGAGTWLANELSAYQDGTKILYVVQ\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (126 of 500)\n",
            "Sampled sequence:\n",
            "SDYNVKLKFPVSQFPEAGSVIKKAIYNGQARVCTIDRAGQRKRRDESLLGIPHRPGYKRHEWPFSSCREGGKGSTVKYVRATDSNGAGRWLNGFLAGYSDGTKVLFIVE\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (127 of 500)\n",
            "Sampled sequence:\n",
            "YDYDAELHVPGDVYPEAASHIRKAVQQGHASVCTLDREGASARRSQSLRGIPKKAGFQRHEWPFAFCKEGGKGSAVQYVAQADNSGFGSWLSNRLSRYADGTRILFVVD\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (128 of 500)\n",
            "Sampled sequence:\n",
            "ADYDVTWNFPTNSYPHAGAHIKQAIAQGFAAVCTLDRKNQSARRRQSLKGIPKRSGYDREEWPFAACQEGGKGSSVRYVPCADNGGAGKWLNGWLSGYADGTKIKFIIG\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (129 of 500)\n",
            "Sampled sequence:\n",
            "SDHDAVLELPCDRYPQAGAHVREAQRAGHAAVCTLDRAGRRRRLSQALAGIPARPGHHRDEWPMAECREGGRGAAVRYVDKADGQGLARWLARYLARFADGTRILTRVA\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (130 of 500)\n",
            "Sampled sequence:\n",
            "VDYDAVLNLPSSQFPYAGAHIKQAIKEGKSDVCTLERAGANQRRKESLKGIPSKSGYIRAEWPFAVCVEGGKGSSVRYVPAEDGNGYSAWLSRTLSGYADGTRIKFVVG\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (131 of 500)\n",
            "Sampled sequence:\n",
            "GSYDVEVIFPAESFPAAGAVIKKAIANGYAPVCTLNRSESGIRRNESLAGIPTKPGYTRHEWPFASCVEGGAGSYVQYVPKDDSDGMGQWLSKKLSAYADGTRVLFRVA\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (132 of 500)\n",
            "Sampled sequence:\n",
            "MAYDVTWFTPCEKYPQWCKMVKEAIARGHAAVCTLERAGRNKRREESLRGIPRKQGYGRQEWPMAICKEGGKGSAVRYVDYEDNNGAGRWLGRDLDQYADGTRVRVVVE\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (133 of 500)\n",
            "Sampled sequence:\n",
            "ALFDTTLVFPADTYPGSGAMIRNAISQGFAAICTLERAQSRSRRDQSLKGIPNSTGYNRHEWPFAICREGGTGSAVRYVPAADNGGAAGWAARQLSQYADGTRVLIVVQ\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (134 of 500)\n",
            "Sampled sequence:\n",
            "MTHDAVVLLPADTFPAASAMIRDAIQRGHAAVCSLAREGEKVRRRQSLAGIPSRPGFDRHEWPFSICTEGGQGSDVRYVPIADNQGMGRWLRATLAAYADGTRLLFEAQ\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (135 of 500)\n",
            "Sampled sequence:\n",
            "IDFDVTWKMPCDQYPQSCEHIRTAIKQGHADVCTLNRAGRHARRTEALKGIPEKPGYDRHEWPHACCVEGGFGSDVRYVYSIDNRGMGLWTATSEAKFRDGTRVRYVVE\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (136 of 500)\n",
            "Sampled sequence:\n",
            "TDYDTVLVYPGAEFPASAAMIRYAIAQGKADICTLQRAGADQRRSESLAGIPKRPGFDRDEWPFAWCVEGGHGSAIQYVPAADNQGAGAYLSKYLSGYKDGTRIKFVVL\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (137 of 500)\n",
            "Sampled sequence:\n",
            "GMYDAVLVLPCAEHPDACSHVRAAISAGKSRVCTLLRDDEQVRFEQSLLGIPTRPGFHRDEWPAAICEEGGAGSAVTYVPFDDNRGLGNWLSQYLSGYDDGTRILIVIG\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (138 of 500)\n",
            "Sampled sequence:\n",
            "SSWDVRLSLPIQQYPHYGKLVAEAIAMGHAAVCTLQRKGEVARKRSALQGIPRRSGFQRHHWPYSICLEGGMGSAVRYVPEEDSSGANRWLHQTLAGYADGTRILMEVE\n",
            "Sequence recovery: 0.45871559633027525\n",
            "\n",
            "Sampling.. (139 of 500)\n",
            "Sampled sequence:\n",
            "SLYDATLNYPTEKFPESGAQIAEAIQNGHASVCTLARAYAKQRREEALLGIPRKEGYDRAEWPFAICEEGGKGSRVEYVPARDNRGAAGWLGKELSKYKDGTRILFIVK\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (140 of 500)\n",
            "Sampled sequence:\n",
            "ISFDAEMYIPMRSFPASGAVIAKARAEGHAAVCTLDRAGQTKRRKQSLAGIPKKDGFVRHEWPFALCKEGGKGSRVEYVPSADGQGFKRWLAQTLSQFADGTKIRFVVK\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (141 of 500)\n",
            "Sampled sequence:\n",
            "MTYDKTLHYPTKKYPQSGATVKTAISRGHARVCTLKRKGAQTRRKQSLKGIPRKSGYKRFEWPMSYCQEGGKGSSVQYVDKADAHGAQVWLHNTLSKYKDGTKILFIIG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (142 of 500)\n",
            "Sampled sequence:\n",
            "GTYDATLVYPASTYPESGAMIKAAIAKGFSAVCTLDRSGTGNRRAQSLVGIPSKSGFGRHEWPFALCKEGGAGSGVVYVPLKDNSGAGTWLNNYLSSYNDGTKIKIVVD\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (143 of 500)\n",
            "Sampled sequence:\n",
            "GLHDVKLQMPGDRFPQSAEHIREAIARGHAAVCTLDRASQGQRRKQSLAGIPARPGFMRFEWPEADCVEGGLGAAVRYVDAADGQGFNKWLREELAQFADGTRVLFVVD\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (144 of 500)\n",
            "Sampled sequence:\n",
            "MTYDAKWYLPVKQYPQAGALIKKAMDAGHASVCTLDRSAQTQRSDQSLKGIPAKKGYDRHEWPFAICKEGGKGAAVQYVPATDNQGFGKWFTDQMSAYKDGTKILMIVN\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (145 of 500)\n",
            "Sampled sequence:\n",
            "GMHDVVQNWPCSAYPDACAMVKAATSAGKASVCTIERTGAATRRDEALRGIPGREGYERHEWPFAMCKEGGKGSVVRYVPAYDNQGDGKWLQNNLAAYKDGTKVLGVCD\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (146 of 500)\n",
            "Sampled sequence:\n",
            "SMYDKKLTFPSKQYPNAGATIREAIGQGHPSVCTLDRGGSGRRRDQSLKGIPNRSGYTRHEWPMAMCLEGGKGSRVRYVPTNDSQGAGMWAGKNLSKYADGTRILYVVQ\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (147 of 500)\n",
            "Sampled sequence:\n",
            "AEYDVVEIFPAMQFPHAAAQVKAAIAEGHAAVCTLDRAGQARRRKESLAGIPRKPGYDRHEWPMAICREGGAGSRVKYVPAADNNGMGAWLMQHLSRYDDGTRILYKVG\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (148 of 500)\n",
            "Sampled sequence:\n",
            "GEYDAVLEFPSSTFPQSGATIAQAISDGHAAVCTLDRAGARRRANESLAGIPDRPGYGRHEWPMAICEEGGVGAKVTYVDRSDNNGAGTWLSETLAGYADGTRIKFIVR\n",
            "Sequence recovery: 0.6055045871559633\n",
            "\n",
            "Sampling.. (149 of 500)\n",
            "Sampled sequence:\n",
            "PSYHVELLFPTSRFPEAGAHIRQAMAEGHASVCTLERGGWRTRRNQSLSGIPEKPGYDRDEWPFALCKEGGYGSSVRYVKRTDNQGFGSYLAQKLSGYADGTKVLFVVE\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (150 of 500)\n",
            "Sampled sequence:\n",
            "MAYDVTLNYPTAQFPASGAHIRQAIAQGHAAVCTLNRSGAAARRSASLQGIPTNSGFGRDEWPMSICTEGGTGSAVRYVPLADLNGAAAWLATTLSGYSDGTKVLFIVH\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (151 of 500)\n",
            "Sampled sequence:\n",
            "MSYDAVLTFPAQTYPESGATIKLAIALGHSAICTLQRADASTRRRQALAGIPACSGYDRHEWPFATCTEGGTGSMVQYVRASDNNGFGSWLRNELAKYADGTKILFTVS\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (152 of 500)\n",
            "Sampled sequence:\n",
            "VDYDVTVTVPVSRFPRAGAHRRRAIAQGHSDVCTLDRAGARTRADQSLAGINLRPGYDRMEWPMAFCVEGGRGAAVQYVRKADNSGYGAWLSKWLSTYEDGTRVRLVQK\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (153 of 500)\n",
            "Sampled sequence:\n",
            "MAYDDTWTFPEAQYPESGAHIKQAIAEGQAAVCTLDRAGQQKRFQESLAGIPKKDGFQRHEWPAAKCKEGGKGSSVQYVPAADNNGAGEWARNELSKYADGTKIKVVVN\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (154 of 500)\n",
            "Sampled sequence:\n",
            "CDFDATRTFPGSTFPESYAMIRQAISEGHASVCTLNRAGQRQRADDSLAGIPAREGYDRHEWPMSVCEEGGVGSSVRYIDVDDNQGFGAWLRNELNGYADGTKIKFVVE\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (155 of 500)\n",
            "Sampled sequence:\n",
            "ANYDTILVYPGARFPAAAEHIRAAIAMGHSAVCTLVRAAQEIRMRESLSGIPKRAGYDRHEWPMALCAEGGLGAAVRYVPSADNQGAGTWISRRLGRHKDGTRILFVVE\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (156 of 500)\n",
            "Sampled sequence:\n",
            "MEYDAKLYFPCRTYPESCLVIKQAIEDGHSDVCCLDRAGWNRRKDQSLQGIPSKTGYDRHEWPFAWCKEGGKGSKVTYVRSEDNNGFGRWAASHLSKYRDGTRILIVVE\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (157 of 500)\n",
            "Sampled sequence:\n",
            "RSYDAVVVFPCDTYPQSCAHVREAIASGHAAVCTLKRSAARTRQQESLAGIPAKSGYDRAEWPFSSCEEGGKGSSVEYVQSADNRGFGSWLGSYLSGFADGTRIKFVVD\n",
            "Sequence recovery: 0.5963302752293578\n",
            "\n",
            "Sampling.. (158 of 500)\n",
            "Sampled sequence:\n",
            "GGYDAEERFPMAQFPHAGATISAAIRSGHARVCTLARAGAKRRDSQSLKGIPARKGFDRHEWPMAICVEGGPGSAVQYVPADDNNGMGKWLADRLSHYADGTKIKWTVT\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (159 of 500)\n",
            "Sampled sequence:\n",
            "SNADVTVYLPCAAFPESCAQIKQAIAEGKASVCTLARASQNSRRNAALAGIPKKSGYLRHEWPMAICVEGGSGSDVRYVPTSDGSGYASWLASTLAAYKDGTKVLWVVN\n",
            "Sequence recovery: 0.45871559633027525\n",
            "\n",
            "Sampling.. (160 of 500)\n",
            "Sampled sequence:\n",
            "MAYDTTEILPATTYPEAAAHVRAAIAQGHDAVCTLARDGRKARRLASLAGIPDRPGFDRHEWPFSTCVEGGTGSAVQYVPKADNQGMGGFLQSRLAGFADGTKVLLVVS\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (161 of 500)\n",
            "Sampled sequence:\n",
            "MNYDVMAYLPEETYPQSGEHVRQAIAEGHAAVCTLDREGRYVRLKESLRGIPKKDGYDRHEWPFAACKEGGKGSSVRYVRADDNNGFGEWLGRRLSSYADGTRVKFVVR\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (162 of 500)\n",
            "Sampled sequence:\n",
            "MEYDKVWNLPCNNFPESCAHIKRAISEGHAAVATLDRKAQQQRRHQALQGIPSRKGYDRHEWPFAICKEGGRGSHVAYVPEDDNSGFGQWLVDFLSQYHDGTRILTVVA\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (163 of 500)\n",
            "Sampled sequence:\n",
            "GMFDAQLVYPGERFPDSAAAIERAMAQGHAAVCTLARAAEATRRDQALAGIPDLPGYQRHEWPYDSCLEGGEGSAVVYVPAADASGAAAWLGNSLSAYADGTRILFTVE\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (164 of 500)\n",
            "Sampled sequence:\n",
            "WSYDAVLALPCRRFPAACAHIAKAIANGHSAVCTLLRVAQALRRDESLIGIPTKPGYMRDEWPVARCEEGGVGASVAYVPSGDNAGYGRWLAKWLAQFADGTKIRFIVG\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (165 of 500)\n",
            "Sampled sequence:\n",
            "SDFDAKLYYPTAQFPESGSHVKKAMDEGHADVCTLKRSDQHKRRTQSLKGIPAKSGFERHEWPAARCVEGGTGSSVRYVPATDNRGAGAWWAGHLSKYNDGTRIKVIVG\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (166 of 500)\n",
            "Sampled sequence:\n",
            "HEFDATVAMPGPQFPASAQMIKDAQAQGYAAVCTLDRAGQRRRFKESLKGIPNKPGFDREEWPFACCKEGGQGAHVRYVPATDNQGYGNWYKKHLKNYADGTKLLISVS\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (167 of 500)\n",
            "Sampled sequence:\n",
            "GNSDAKLEFPCNKFPEACSMVAQAIKDGYSKVCTLDRAGQSARREQSLAGIPSKEGYGRHEWPFASCKEGGKGAHVKYVPFADNNGYGKYTAKELAKYADGTRILYDCS\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (168 of 500)\n",
            "Sampled sequence:\n",
            "SDYDVTLSFPCSKFPASCAHVAAAIKQGKASVCTIDRSGQRARREQSLRGIPTKSGYTRDEWPFAGCKEGGEGSKVRYVPFSDGQGAGRWLGAKLSAYNDGTKIKFVST\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (169 of 500)\n",
            "Sampled sequence:\n",
            "ASYDVTVLLPCNAYPQACAMVSKAISLGFSEVCTLQRSGAWARFNESLAGIPNRSGYDREEWPFAFCKEGGRGSYVQYVNKSDNQGFGRWLNQYLSTYADGTRILLVSG\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (170 of 500)\n",
            "Sampled sequence:\n",
            "MAYDAKLTFPVATYPESGSMIKSAISLGLPAVCTLERSSRTARFDQSLAGIPNKDGYDRHEWPLAICQEGGAGSAVVYVPRADSQGFGNWFNSWKSKYADGTKILIIVG\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (171 of 500)\n",
            "Sampled sequence:\n",
            "GSHDVELWFPASEFPEAGAVIKAAIADGFSAVCTLQRADENRRRLESLQGIPRREGYDREEWPFAMCEEGGKGSAVRYVEASDQYGAGRWLSNKLSKYDDGTRILFVAS\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (172 of 500)\n",
            "Sampled sequence:\n",
            "MTYDHRLEFPAERFPETGDLIKQAIEEGHSRVCTLNRAGQHTRHQQSLQGIPKREGYERHEWPHCLCEEGGTGSSVRYVPSNDGNGSSSWLRHHLGKYDDGTRILFTVD\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (173 of 500)\n",
            "Sampled sequence:\n",
            "MDYDVELHFPSDVYPQSGETIAAAIQDGQADVCTLDRAGENERRNQSLKGIPKKNGYHRHEWPFARCQEGGTGSKVRYVEASDHSGAGNWLRNKLSAYQDGTRIKLIVD\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (174 of 500)\n",
            "Sampled sequence:\n",
            "GSYDVELTFPRSAFPASGAMVRNAIAAGKAAVCTLDRAGADARRREALAGIATRTGYDRHEWPFALCREGGTGSSVQYVPHVDNHGMGSWLYTKLAQYADGTRVKVVVG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (175 of 500)\n",
            "Sampled sequence:\n",
            "MNFDVSVYTPAATFPEYAAHVRAAIAAGHADVCTLDRAGAQTRRAQSLAGIPSQAGYQRDEWPFASCVEGGQGSSVVYVPAADNTGMGSWLSSELAQFADGTKVRFTVT\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (176 of 500)\n",
            "Sampled sequence:\n",
            "ASYDVSLTFPCNTYPSSCRHIKRAISMGYAKICTISRAGANLRKQQSLRGIPSKPGYTKDEWPAARCKQGGTGSSVTYVPSNDANGAGAWLASTLNNYSDGTRILFTVG\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (177 of 500)\n",
            "Sampled sequence:\n",
            "ADYDVELDVPDRKFPESAALIKRAIELGHAAVCTLERRGRHIRHRESLRGVPARRGYMRHEWPFAQCKEGGRGSAVEYIPYDDGKGFGKWLSNTLSKYKDGTRVLFRMK\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (178 of 500)\n",
            "Sampled sequence:\n",
            "MSYDATLKFPCQKYPEACAQIKKAISEGHANVCTLERAGQTERRKASLKGIPDKAGYNRHEWPFALCKEGGTGSSVKYVPTDDQSGAGAWLKTKLAKYADGTKILQIVD\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (179 of 500)\n",
            "Sampled sequence:\n",
            "GDYDAEMRLPCAQFPFSCACIKASIGQGHADVCTLDRAGQKARSNQSLKGIPSKDGYERHEWPFAFCKEGGKGSTVQYVPRDDEQGYGAWLQNTLSQFQNGTRIKVIAD\n",
            "Sequence recovery: 0.45871559633027525\n",
            "\n",
            "Sampling.. (180 of 500)\n",
            "Sampled sequence:\n",
            "WEWDAVLTFPAKAFPESAAHVKAAIAAGKARVCTLDRAGHQQRQNEALAGIPGRPGFDRHEWPTAVCKEGGKGSAVKYVPAADNLGFGKWLANALAGYDDGTKILFQVS\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (181 of 500)\n",
            "Sampled sequence:\n",
            "SMEDTTLVYPASKYPAAGSHVKDAISDGQAAVCTLDRAGRKQRQRESLKGIPKRKGFTRHEWPMAFCEEGGKGASVRYVPADDSNGAGGWLSNKLSKYADGTKILFKIS\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (182 of 500)\n",
            "Sampled sequence:\n",
            "MDYDATLVFPCDHFPQSCAQIRQAIAEGHPAVCTLNRAQAKERRVQSLAGVPKRPGYARFEWPSAWCEEGGRGSRVTYVAQDDNMGYGKWFAWRTSAYADGTRILFVSS\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (183 of 500)\n",
            "Sampled sequence:\n",
            "AEYDAVHSFPGDIFPHSAATIRVAQARGHSSVCTLNRGAQNYRLSQALAGIPAKEGYDRHEWPMAICEEGGSGSLVRYVPERDNQGFGAWLRAHLARFADGTRIRYVVI\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (184 of 500)\n",
            "Sampled sequence:\n",
            "SDHTTVLYLPASTYPYSAQLIRQAIQEGHADVCSLDRGRQAERRRESLAGIPTRPGYVRAEWPFALCREGGSGARVQYVPTADGMGMKGWLGKYLSGYADGTKVLFRCD\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (185 of 500)\n",
            "Sampled sequence:\n",
            "CGHDATLYYPASEFPFAAAQMAKAIKEGHAAICTLARSSQGARRDQALAGIPGRPGFHRAEWPFSSCLEGGTGAKVTYVPAADNSGMGTWIRNYLARFADGTKIRFVVD\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (186 of 500)\n",
            "Sampled sequence:\n",
            "SDFDYRLHMPWQRFPEAGALINQAIAEGYAEVCTLDRKGSTTRRNQSLRGIPSRPGFGRFEWPMAACTQGGSGSAVRYVPLDDNNGYGAWLGRFLSAYADGTRILFTVS\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (187 of 500)\n",
            "Sampled sequence:\n",
            "TEFDAEVELPCAAFPEACAQIQAAIGEGHAAVCTLESGQSGRRRDESLEGIPEKPGYDRHEWPMAVCREGGVGSRVRYVPKDDNQGFGAWLSEQLSSYADGTKLRLVVG\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (188 of 500)\n",
            "Sampled sequence:\n",
            "RDYDATLIYPADQYPAAGAHIRKAIEQGHADICTLDRSGRRQRLREALRGIPRREGFERHHWPGANCVEGGTGSTIRYVPRSDHQGADRWLTDTLSMWANGVRIRVVTR\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (189 of 500)\n",
            "Sampled sequence:\n",
            "SKYDVVRHEPTKAFPQAGATIKAAVSQGHAAVCTLKRASAKARAKQSLKGIPKKRGYDRHEWPFASCSEGGKGAKVTYVRRKDNGGMGRWLDKTLAAYADGTKVEFRVK\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (190 of 500)\n",
            "Sampled sequence:\n",
            "MKYDAELNFPAADFPEAGALIKKAIADGKAAVCTLDRAGATQRRNESLKGIPNKKGFDRHEWPFAMCKEGGDGSAVQYVDSSDNAGAGTWLGATLSSYANGTKIKLVVD\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (191 of 500)\n",
            "Sampled sequence:\n",
            "GVWDHELSLPAAAFPESAALIARAISEGHAAVCTLDRAGSRSRRDQSLKGIPTRAGYDRHEWPFALCEEGGTGSAVEYVPESDNRGFGRWLSATLSQYADGTKILFNLK\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (192 of 500)\n",
            "Sampled sequence:\n",
            "MVHDARLYFPQSTYPAAGALIASAQSSGHAAVCTLARSAARRRRQEALLGIPARPGYDRHEWPFAHCREGGAGSAVRYVPAADNRGAGAWLRETLSRFADGTRILMIVP\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (193 of 500)\n",
            "Sampled sequence:\n",
            "AAHDAKLRFPASTFPAAGAHIAAAIAAGQAAVCTLNRAGAAQRRDEALRGIPSRAGYDRDEWPFAGCREGGQGATVRYVPAADNQGAGQWIAYTLSKYSDGTRILFQVA\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (194 of 500)\n",
            "Sampled sequence:\n",
            "DDFDEIVQFPSTVYPQAGAQIKRAIAEGHDAVCTLQRDSANQRRMESLAGIPKKSGYVRHEWPHSSCKQGGSGSAVTYIDAADGNGFNRWQGNWLSDYDDGTQILMVTE\n",
            "Sequence recovery: 0.44954128440366975\n",
            "\n",
            "Sampling.. (195 of 500)\n",
            "Sampled sequence:\n",
            "MTFDAVLKLPGKEFPESSAHMEKAVKEGHADICTLDRSAQKARRMEALKGIPGKEGFSRHEWPYASCLEGGKGSAVRYVQWSDNLGDGKWLAKHLSRFKDGTKIKYIID\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (196 of 500)\n",
            "Sampled sequence:\n",
            "MAYDAVLHFPGSEYPDSGAMIRAAIAAGHAEVCTLERSAQLDRQRQSLFGIPKKPGFDRHHWPESICREGGTGSAVAYVRRSDNNGFGRWLKDKLSRYADGTRILFIIP\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (197 of 500)\n",
            "Sampled sequence:\n",
            "SRYDEVWEFPAATYPASGAHIRAAIQQGHAAVCTLDRSSAEVRQIESLAGIPARPGYDRDEWPFAFCREGGQGAAVRYVDHADNQGAGQDIRQWLAQYAPGTRILAVVD\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (198 of 500)\n",
            "Sampled sequence:\n",
            "AKFDAKLAVPCKRFPEACAMIRAAIRQGHADVCTLARAEEKKRRKESLAGIPKKKGFDRHEWPFAWCQEGGRGSTVQYVKSRDNQGFGDWFKNKMASYADGTKILFIVE\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (199 of 500)\n",
            "Sampled sequence:\n",
            "GLFDASLVYPAATFPESGAMVKEAIAQGFSDVCTLERAAASTRRTQALAGIPSKPGFNRHEWPMADCTEGGQGSAVRYVPSQDSVGAQAWLQSTLSRYSDGTRIRVVVN\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (200 of 500)\n",
            "Sampled sequence:\n",
            "APYDATLTYPTSTYPAAGANVKTAIAQGHAAVCTLQRADANTRRHQSEAGIPAKAGYGRFEWPMALCTEGGAGASVAYVPSADRNGAEAWLRNRLAAYADGTKIRFVVG\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (201 of 500)\n",
            "Sampled sequence:\n",
            "MTFDVTLYLPDATYPQAGAHIAKAISEGKAAVCTLDRAGYAARRDQALAGIPAQNGFDRDEWPMDDCVQGGTGSDVRYVDAADNQGAGSWIRSTLAAYADGTRVRFVVV\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (202 of 500)\n",
            "Sampled sequence:\n",
            "RDHDDVCELPCSAYPQACEHIRAAIAEGHADVCTLQRSGREQRRQESLQGIPAKPGYDREEWPFAFCAEGGQGSSVKYVLQDDNRGFGRWLANRLSAYPDGTKIRFVCR\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (203 of 500)\n",
            "Sampled sequence:\n",
            "GDFDATLDLPDSTYPEAGAAIRKAIAEGHAAVCTLERATQSRRRDQALAGIPTKPGLIRMEWPFAFCKEGGKGSKVVYVAHADGSGFAEYLARYLSQFAEGTKIKFVVK\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (204 of 500)\n",
            "Sampled sequence:\n",
            "MTYDATLQYPASQYPESGAMIKKAIQAGKPAICTLDRAGENERRAESLAGIPRQSGYERHEWPHAFCLEGGQGSKVAYVNAADNQGAGQWMETYLSQYADGTKIKVVVG\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (205 of 500)\n",
            "Sampled sequence:\n",
            "VVFDVTYDFPAAEFPESGAHIAEAIKNGHSRICTLNKDSEKRRRIQALRGIPKRDGYDRFEWPFASCEEGGQGSSVRYVFKYDNQGAGHWLRQTLSKYADGTRIRVVVR\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (206 of 500)\n",
            "Sampled sequence:\n",
            "ATFDKRLHLPTSKYPESGSMVKAAIAQGHASVCTLKRSAQGSRRSESLAGIPKKSGYGRHEWPMSYCEEGGKGSDVRYVPGSDFDGFRSWLNSTLSGYADGTRILFTVD\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (207 of 500)\n",
            "Sampled sequence:\n",
            "ALFDVTDTFPASQYPQSASHIEAAILEGNAAVCTLNRAKERERANSSLQGIPRRPGYTRDEWPFCDCVEGGVGSAVRYVPATDNNGAGGHYSNHYAAYADGTRVLRVVA\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (208 of 500)\n",
            "Sampled sequence:\n",
            "SGADVRWYFPVDDYPEAGAQIAEAVRQGHAWVCTLDRAGQGRRRDESLGGIPARPGYIRQEWPFSLCQEGGAGSAVAYVPADDGNGAGSWLSRHLSRYADGTRVLTVVE\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (209 of 500)\n",
            "Sampled sequence:\n",
            "KKYDHTLQVPCDTYPESCAHIAAAIQAGHAAVCTLDRAGEKVRRKQSLAGIPAKPGYDRHEWPFASCREGGAGASVKYVPSTDNQGFGKWLKQTLAAFADGTRILFIVA\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (210 of 500)\n",
            "Sampled sequence:\n",
            "GEFDAELLFPDAEYPEAGAHVKKAIALGHSAVCTLERSKWRFRSHQALKGIPQRPGYGRFEWPMAQCVEGGKGSEVAYVPWDDLNGFRSWLASRLAVYPDGTRIRFTVQ\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (211 of 500)\n",
            "Sampled sequence:\n",
            "MDYDYTWTVPCSTYPQACAQYKQAIALGHSAVCTLDRAGQTARRDQSLLGIPSKPGYLRDEWPFALCREGGVGSRVEYVGGDDGTGMRGWLANYLSAFADGTKIKVIVG\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (212 of 500)\n",
            "Sampled sequence:\n",
            "HEFDTMILVPCADFPESCAMFKKAIQEGHSAVCTLDREQQKQREAESLQGIPQKSGYDRHEWPVAYCTEGGKGSHVMYVPREDNRGFGKWLDAELSQYADGTKVLLVVE\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (213 of 500)\n",
            "Sampled sequence:\n",
            "STFDATLQYPSSQYPQAGAQIKEAIAQGHAAVCTLARQGQKSRRKQALAGIPKRPGYQRHEWPFAICEEGGAGANVRYVPASDNMGAGTWLANTLAQYADGTRILFIVD\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (214 of 500)\n",
            "Sampled sequence:\n",
            "EWADTELSMPCTTFPYSCSEIQTAIAQGYSQVCTINRAGRSTRRNQSLNGIPSRVGYDRHEWPFAHCQEGGAGSRVEYVPIADNAGFGRWINDRLTAFADGTRILFRWT\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (215 of 500)\n",
            "Sampled sequence:\n",
            "HNFDVRLHFPSATFPQAGAQIAAAIRSGHAAVCTLDRARRGARMKESLAGIPLLPGYTREEWPFAFCREGGRGSDVRYVAEADNRGAGKWLRRHLSAFANGTRILFEIN\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (216 of 500)\n",
            "Sampled sequence:\n",
            "ASYDTVLKFPKKEFPASGAQVKTAIKNGKSAVCTIDRSGAAARRKESLKGIPKKKGYDRHEWPFANCKEGGKGSAVTYVPKADNNGAGSWMNHKLDQYADGTKILFKVK\n",
            "Sequence recovery: 0.5963302752293578\n",
            "\n",
            "Sampling.. (217 of 500)\n",
            "Sampled sequence:\n",
            "SNADATLVLPSKEYPQSGAGVKQAIAEGHSSVCTLDRSGEKERLNQSLSGIPAKDGFGRHEWPWAMCKEGGKGARVEYVRITDHDGFGAWLREHLAAFADGTKIKLIVK\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (218 of 500)\n",
            "Sampled sequence:\n",
            "SEFDATVNFPASQFPESAAQIRAAIADGHAAVCTLARSGANGRRTAALKGIPQKDGYDRHEWPMAWCTEGGEGANVTYVASADNRGAGAWIRNTYSRFEDGTKLLAVVN\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (219 of 500)\n",
            "Sampled sequence:\n",
            "QEYDAVLYLPIATFPEAGAMVKKAIEEGHAAVCTLERAGEKKRREEALAGIPHKEGYDRHEWPMAHCKEGGIGSAVRYVPHKDNRGMGHWLAEHLKNFDDGTRIKFVVE\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (220 of 500)\n",
            "Sampled sequence:\n",
            "ADFDVVRNYPCSEYPESCQMIQLAIKEGQPEVCTLSRSSAGTRRTQSLAGIPARSGFERFEWPMAICKEGGTGSAVKYVPHADNQGAGRWISNYLSAYADGTKVLFVVK\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (221 of 500)\n",
            "Sampled sequence:\n",
            "MNYDTTVYFPEETYPESGAHISEAIKEGHSKVCTLAREGRAERQRQSLQGIPTKPGYGRDEWPFAICLEGGEGSSVRYVNIWDNTGAGKWLRAKLSAYPDGTRVLFVTT\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (222 of 500)\n",
            "Sampled sequence:\n",
            "MAYDAELKLPCDKFPMACQHVKDAIARGYSAVCTLSRREQNLRRQESLKGIDSKPGYDREEWPFADCKQGGKGSAVQYVPSADNSGYGLWLREKLDAYQDGTKIRFIVE\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (223 of 500)\n",
            "Sampled sequence:\n",
            "GAYDTTLNYPVATYPASGAAIAKAQSQGHAAVCTLSRASATTRRTQALAGIPQRSGYQRFEWPLSICTEGGAGSMVWYVEASDNQGAAAWLSQTLSRYADGTRIRFQVK\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (224 of 500)\n",
            "Sampled sequence:\n",
            "AGYDAKVVFPSSSYPEAGAHIERAISKGRADVCTIDRSGEDQRRKDSLAGIPLKDGFIRHEWPFAGCKEGGTGSGVTYVKSSDGSGFGSWLSSWLSNYKDGTKILFEVA\n",
            "Sequence recovery: 0.5963302752293578\n",
            "\n",
            "Sampling.. (225 of 500)\n",
            "Sampled sequence:\n",
            "MQYDTELVFPAATFPNSGKTIAKAIQEGHADVCTLARASQGARSKESLLGIPDKEGFDRHEWPFAICKEGGKGSRINYVESADNQGFGKWLKDELSKYADGTRVLFKVG\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (226 of 500)\n",
            "Sampled sequence:\n",
            "ATYDAKLKLPAAKYPAAGAMIRKAISEGQSKVCTLARGNQYKRFQQSLAGIPKKSGYIRHEWPFALCQEGGKGAAVRYVPSADGQGFAKWLGRYLSKYKNGTKILFTVQ\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (227 of 500)\n",
            "Sampled sequence:\n",
            "EGYDVRLYFPDSKYPESGAQICEAISQGHAAVCTLDRAGQTRRRGESLAGIPTRDGYSRDEWPMALCVEGGRGAAVRYVPSDDNHGAGRWLANRLSRFDDGTRILFICN\n",
            "Sequence recovery: 0.6238532110091743\n",
            "\n",
            "Sampling.. (228 of 500)\n",
            "Sampled sequence:\n",
            "SEFDVRLFVPCDTYPEACEHLKRSIELGHASVCTIDSAGKQQRRKESLDGIPERTGFVRHEWPFSKCLEGGKGSAVQYVANADGEGLATWIGEHLAGYANGTKVLQVVD\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (229 of 500)\n",
            "Sampled sequence:\n",
            "MDYDVVLTFPKETYPESGLMIMAAISFGHADVCTLNRASAGVRRQESLAGIPAKSGYTRHEWPFAMCVEGGAGAFVQYVRASDGQGAGRWILNRLAGYADGTRVLGVSG\n",
            "Sequence recovery: 0.5963302752293578\n",
            "\n",
            "Sampling.. (230 of 500)\n",
            "Sampled sequence:\n",
            "GKYDYILYFPKKQYPAAAKHIKEAISRGYSKVCTIDRRGSGLRRKQSLRGISKKYGYDRHEWPFSSCREGGKGSSVKYVPSKDNTGFGKWLKSFLSAYKDGTRILFVVR\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (231 of 500)\n",
            "Sampled sequence:\n",
            "MSYDVKLTYPAAAFPHAGATIKKAIAQGFANVCTLKRAGASTRRQQALAGISSKPGYGRHEWPMAFCEQGGEGSDVRYVPWNDNTGAGKWLTNKLAKYADGTRILFVAG\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (232 of 500)\n",
            "Sampled sequence:\n",
            "ASYDVTLTFPCSSYPASCALVQSAIAQGYSAVCTLNRGGQSSRANQSLRGIPSRPGYQRHEWPFASCVEGGQGSMVQYVPSSDANGAGNWLANTLSAYADGTKVKFTVG\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (233 of 500)\n",
            "Sampled sequence:\n",
            "MSYNYELNLPDKKFPEAGATIKQAIAEGKDDVCTLDRKNSGRRSDQALKGIPKKKGFVRHEWPFDSCMEGGKGSKVTYVPSEDGQGFAKWLGSKLSAYKDGTRILMIVE\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (234 of 500)\n",
            "Sampled sequence:\n",
            "MKFDVTMAFPTATYPQSGAVIKKAISMGKDAVCTLSRGNAATRRAQSLAGIPDKAGYERHEWPVAVCLEGGQGSTVEYVQASDWQGMANWLSNYLSQYADGTRVLFVVD\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (235 of 500)\n",
            "Sampled sequence:\n",
            "SDYDAVFVDPCDTYPESCALIKQAIAEGYSAVCTLARSGATARRRQALAGIPARSGYGRAEWPAAYCTEGGRGSAVRYVPYRDYNGHSAWLTKALSAYADGTRIKIVMP\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (236 of 500)\n",
            "Sampled sequence:\n",
            "DDYDAKLVFPATVFPAAGAHVSTAINKGHAAVCTLLRASANIRRDESLTGIAAEAGYQRFEWPSSMCLEGGTGSAVTYVPILDGAGFGAWMSNVLWSYADGTRILLVVN\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (237 of 500)\n",
            "Sampled sequence:\n",
            "DDYDVTAELPCESYPESCDMIKKAISEGHSDVCTLERGKASIRNHEALRGIPGKPGFWRHEWPFAICKEGGTGSHVEYVDWQDNNGYGAWLAGTLSQYKDGTKILFTVS\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (238 of 500)\n",
            "Sampled sequence:\n",
            "MDFDVVEHFPCETYPQSGAHIKEAIALGHASICTLDRANWNARRIEALAGIPKKPGYDRHEWPMAVCMQGGAGASVRYVDKADQQGAGAWLAKHLSRFEDGTRILFIVE\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (239 of 500)\n",
            "Sampled sequence:\n",
            "AAYDATYYVPCAKFPNSCNHIRIAIAEGHAAICTLERTDQKKRRDESLAGIPAKEGYDRDEWPMACCKEGGKGSAVRYVPSADNQGMGTWLRQRLAAYADGTKIKFVAR\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (240 of 500)\n",
            "Sampled sequence:\n",
            "GDFDDVLQFPCGKFPESCALIRQAIAQGHSAICTLDRAGASKRREQALAGIPDRPGFKRQEWPMSWCKEGGYGSAVQYVPDDDNSGAGSWMSATLSQFNDGTRILFRVE\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (241 of 500)\n",
            "Sampled sequence:\n",
            "MDYDVTLTFPADEFPQSAATIAEAIKKGHAAVCTLDRAGRQTRRNDALAGIPEKDGYMRHEWPFATCQEGGKGSKVEYVDYYDENGAGTWLSNWLSGYGDGTRILFVVS\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (242 of 500)\n",
            "Sampled sequence:\n",
            "MAYDARLRYPEKTFPEAGAHLARAIEQGHAAVCTLARSAQDIRRDASLKGIPRRPGFGRFQWPMSFCEEGGRGAGVQYVDLKDNQGAGRWLSRTLAGYADGTRILMLVG\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (243 of 500)\n",
            "Sampled sequence:\n",
            "AVYDATLELPTETFPGSGAMVKDAIARGHNKICTLDRAHQQVRADQSLRGIASRPGYHRHEWPMDLCKEGGKGAAVTYVEKADNEGFAAWIRQKMNQYKDGTRILLRVS\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (244 of 500)\n",
            "Sampled sequence:\n",
            "GEFDVTLKLPCERFPESCRHIKNAISEGFSEVCTLKRSSQGKRAKESLKGIPKKEGYDRDEWPMAMCEEGGKGAKVRYVKSDDNQGMGKWLRNWLSQFRDGTRVLFVVD\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (245 of 500)\n",
            "Sampled sequence:\n",
            "ADYDTVLVLPCDTFPQACATIAAAIAEGHAEVCTLNRDSHGARANQSLAGIPDKPGYDRHEWPAASCEEGGSGSRVEYVEWLDMSGFAAWLSYYLSVYGDGTKVKFVVS\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (246 of 500)\n",
            "Sampled sequence:\n",
            "AEYDVELQLPCDRFPQSCEHIRSAIALGHARVCTLERAKSRERRDESLAGVPDKEGYDRHEWPFALCREGGAGSSVRYVDRADNGGFGAWLRRHLNVYADGTRVLFTVN\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (247 of 500)\n",
            "Sampled sequence:\n",
            "GKFDTVLELPCDMYPETSALIKDAIAKGFAAVCTLDRAGSSKRRAESLAGIPDKAGYIRAEWPFASAEQGGAGAAVEYVPSADGGGAGGYETEYLSSYADGTKVKFVVR\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (248 of 500)\n",
            "Sampled sequence:\n",
            "EGFDVKLDWPVEKYPQSGEMLRRAIAQGFSDVCTLERAQQGARRNESLAGIPAKEGYDRHEWPFASCLEGGKGAEVAYVDSSDNDGAGTWVRNTLSRYEDGTRVLFVNK\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (249 of 500)\n",
            "Sampled sequence:\n",
            "AAWDVELNLPCDKYPESCALISRAISEGHSNVCTLERASQKKRAEESLKGIPRKPGYGRHEWPFAMCVEGGKGSAVQYVPIDDINGFAAWYSRHLDKYEDGTRVKLVVG\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (250 of 500)\n",
            "Sampled sequence:\n",
            "SNADTVLVLPCDEFPQSCAQERAAIAEGHADVCTLARAGASQRRDESLRGIEARDGFTRDEWPMAVCEEGGEGSRVAYVNARDNSGFGRWLGETLANYDDGVRILVVCE\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (251 of 500)\n",
            "Sampled sequence:\n",
            "NAYDATCTFPRGAYPASGAMIAKAIAEGHAAVCTLTRADQLARRREALAGIPSRSGYERHEWPFAYCKEGGKGSAVQYVPRADNQGFGKWLKGTLAGYADGTRILFQVR\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (252 of 500)\n",
            "Sampled sequence:\n",
            "YGYDTTLTQPNDTYPDAGAHDAQAIAEGHADVCTLDRAGTSGRRNQSLLGIPTKEGFDRHEWPHCYCTEGGSGSSVQYVPRDDNQGWGTWLGKTLAGYSDGTKVKITVN\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (253 of 500)\n",
            "Sampled sequence:\n",
            "GSHDADLIFPAAEFPQAGAMVAEAMAAGHAAVCTLDRAGQSKRRRQSLKGIPRRQGYERHEWPFALCEEGGAGSRVEYVPAADNLGSGRWMAETLARYDNGTRIRLVVN\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (254 of 500)\n",
            "Sampled sequence:\n",
            "GAYDATLQLPGATYPQSAAVVKAAIAQGKSDVCTLKRDGQRERRTLSLRGIGEKPGYDRDEWPFAMCEEGGKGAAVRYVEKSDNAGYGRWLADHLAQYADGTKIKFIVN\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (255 of 500)\n",
            "Sampled sequence:\n",
            "EQYDATFTAPCKTFPASCAHIKAAIAQGHADVCTLDRAGETKRKTESLAGIPSKSGYGRHEWPFALCREGGKGSGVAYVPDADNSGYGSWLRDTLNQYKDGTKIKIVCR\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (256 of 500)\n",
            "Sampled sequence:\n",
            "SDFDVSLVFPSATYPQAGAHIKAAMAAGKPAVCTLLRASQTTRRNESLLGIPQLPGYIKDEWPFAYCQEGGQGSAVAYVAQADGQGASTWLSSTLSQYADGTKVLFIIE\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (257 of 500)\n",
            "Sampled sequence:\n",
            "VAYDVEVNFPTGTFPDSGQMVKEAIKDGHAQVCTLQRSGAEKRRTESLAGIPAKAGYQRHEWPFAECVEGGKGAKVMYVPEADNTGAGTWMGKKLSGYDDGTKVKFVVK\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (258 of 500)\n",
            "Sampled sequence:\n",
            "MAFDAVLAFPAARFPEAGAHIAKAIAQGHASVCTLARAHSRQRKRASLRGIPRRIGYIRQEWPFAQCEEGGTGASVEYVAKADGDGFGRWLKKHLASFSDGTRIKFIVG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (259 of 500)\n",
            "Sampled sequence:\n",
            "MAYDVERLLPCAEFPEACAVIEEAVNQGHDDVCTLERLQSSRRRQESLAGIPEKEGFDRHEWPFADCLEGGQGSRVKYVPTADNQGMGQWLSAYLSRYADGTRVKFIVQ\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (260 of 500)\n",
            "Sampled sequence:\n",
            "GSYDAVLLLPCREFPESCAHIRQAIQEGFSAVCTLERAGAGKRHNESLEGIPRKDGFIRHEWPLARCREGGRGAGVRYVRADDGAGYGSWLARTLAGYADGTRIRFICR\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (261 of 500)\n",
            "Sampled sequence:\n",
            "VDFDVVWDYPVDRYPESGAHIRQAVAEGQARVCTLDRECSSKRDKESLAGIPRKEGYDRDEWPMAKCKEGGKGASVRYVPQKDNRGAGRWFQEKYSEYQDGARILQVVR\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (262 of 500)\n",
            "Sampled sequence:\n",
            "GEYDVFMGTPCSQFPKSCEHIRTAIRNGHSAVCTLDREAEAERRTQSLKGIPTKDGYDRWTWPMAICKEGGKGAAVQYVDKKDNQGFGRWLSTTLSHYADGTRIRFIVK\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (263 of 500)\n",
            "Sampled sequence:\n",
            "LDFDHVLHLPCATYPESCAHIAAAIEQGHAAVCTLYAAGSGARRQQSLAGIPEKRGFVREEWPFAQCAEGGRGAAVRYVDSQDGQGFNRWLRRHLRQYADGTRILFVVP\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (264 of 500)\n",
            "Sampled sequence:\n",
            "WEYDTKLKLPCSTFPEACATIRRAISQGHAAVCTLDREGARQRRKESLKGVPLKEGYDRHEWPFAYAVEGGRGAQVAYVPKEDNGGFGRWFRNRLSQFKDGTRILFVIG\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (265 of 500)\n",
            "Sampled sequence:\n",
            "GMFDKRCYVPSKQFPESGQMIKIAIMKGHADVCTLDRKGSSKRRQQSLEGIPLKEGFERHEWPFANCLEGGSGADVRYVPADDNQGFGKWLGNHLSKYSDGTRIKYIVE\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (266 of 500)\n",
            "Sampled sequence:\n",
            "CEFDAVLDFPDDEFPESGREIATAIRNGHDAVCTLSRAGASSRRNASLAGIPARVGYDRFEWPMAFCSEGGVGAVVRYVPKADNSGFGRWLRATLSAYADGTRILLRRG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (267 of 500)\n",
            "Sampled sequence:\n",
            "MLFDTEILFPAATFPQAGAMIAQAIAEGKSAVCTLERAGQTERRKQSLAGIPSKAGYDRHEWPFAQCTEGGAGSAVRYVDFADNNGFGNWVSGYLSQFSDGTRVRLINA\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (268 of 500)\n",
            "Sampled sequence:\n",
            "TAYDVELNFPCATFPDACAQIKQAIQMGRAAVCTINRAGEHVRSKQSLAGIPDKSGYDRHEWPFALCTEGGTGSHVRYVPSSDNKGAGNWLRKYLSKYADGTRILMICY\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (269 of 500)\n",
            "Sampled sequence:\n",
            "GEYDNVLNLPISEFPLSGAHIATAISLGYSSICTLDRVGQKQRSTESLRGIPDKNGYQRDEWPFSSCTEGGTGAAVQYVDARDNQGFGKHMRTTLNQFDDGTRILFTVG\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (270 of 500)\n",
            "Sampled sequence:\n",
            "MTYDATVNYPSSQYPESGAMIRSSIASGKASVCTLSRGSRNQRLSQSLSGVPRRSGFDRHEWPFADCTEGGSGSAVQYVPDNDNSGAGNWLRRYLSGYANGTRILFIVN\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (271 of 500)\n",
            "Sampled sequence:\n",
            "LAFDVTLIFPWRTYPDAGDMVKRAIAEGHDWVCTLERGEQKRRRDEALAGIPSRQGYTRHEWPFSICREGGKGSAVRYVASADSQGAASWLQRRLGGFSEGTRVRFVVE\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (272 of 500)\n",
            "Sampled sequence:\n",
            "GQFDAVVQVPEDQFPQAGSHIKQSIEEGHADVCTLDRAAANERRAASLQGIPDRPGFDRHEWPFAICKEGGSGSSVRYVDKSDNSGFANWLANYMSAHADGTKILYISN\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (273 of 500)\n",
            "Sampled sequence:\n",
            "MQYDATLAFPAKRFPESAEHVRAAIADGKSSVCTLARDKRQARSQESLAGIPARKGFMRFEIPFAECEEGGKGSSVRYVPATDGAGARSWLRSNLSRYADGTRILFRVE\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (274 of 500)\n",
            "Sampled sequence:\n",
            "MEFDAVLTYPTSTYPASGAMVKQAIASGKAAVCTLNRSGQAAREAQALSGIPAKSGYHRFEWPFAICVEGGSGSAVKYVPSDDMQGYNKWLGEKLSTFADGTKILFLTG\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (275 of 500)\n",
            "Sampled sequence:\n",
            "MTFDAQLVLPGASFPASAQHIRTAIAEGHSAVCTLDRARQQRRRAESLLGIPKKRGFDRHEWPFDICNEGGQGSAVVYVPYRDNQGFGNWLRTHLDKYSDGTRILLRVG\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (276 of 500)\n",
            "Sampled sequence:\n",
            "GAFDAVLVVPCRTFPESCEQWSEAIANGHARVCTLDRASQDKRRDQSLAGIPGRRGFDRHEWPQSECVEGGAGSAVRYVDHDDNQGFGAWLRSHLDAFKDGTRILQVVY\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (277 of 500)\n",
            "Sampled sequence:\n",
            "TAYDATWYFPVLTFPASGAMVAQAVAQGHAAVCTLDRASRDARSRQSLAGIADRPGYDRHEWPFSFCLEGGAGSLVQYVPAADNQGAGTWLNKTFAAYADGTKILLIVP\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (278 of 500)\n",
            "Sampled sequence:\n",
            "GGYDVTLYWPSRSWPQSGAKVRSAVSEGHADVCTLDRGGQGERRKQSLAGIPRRSGYDRHEWVFAICSEGGKGSSVSYVPSRDNRGAGNWLSRRLSQYKDGTRVKIIGG\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (279 of 500)\n",
            "Sampled sequence:\n",
            "MAYDAKLTLPCSAFPAACAHIKKAIAEGHASVCTLARAGQGTRRRQSLAGIPAKQGYDRDEWPFADCVQGGAGSSVRYVPSADNNGFGQWMRNTLSGYSDGTKILFSLF\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (280 of 500)\n",
            "Sampled sequence:\n",
            "MAYDAVLKWPEARYPFSGEQIKAAIAAGHAAVCTLKRASQTRRREEALAGIPKRKGYDRHEWPWASCQEGGKGSKVRYVPTKDNQGAGRWLHATLARYADGTKIRFVVD\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (281 of 500)\n",
            "Sampled sequence:\n",
            "IVYDSTLYVPCNTYPNSCAHIKEAIAQGFAAVCTLERKGREARRKESLAGIPKKEGYERHEWPMALCKEGGKGASVRYVPKDDNQGFGKWLKDKLAGYKDGTKIKIICE\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (282 of 500)\n",
            "Sampled sequence:\n",
            "ADYDVEYSFPANSYPESGAHISSAMQQGHAAVCTINRSGEATRRAQSLAGIPSRTGYDRDEWPFATCTEGGTGSAVRYVPSADNRGMGTWTSNTLSRYADGTKVKLVVD\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (283 of 500)\n",
            "Sampled sequence:\n",
            "KTYDATLYYPAAAYPASAAHVAKAIAEGYASVCTIERAQQQQRRDQALAGIPQRQGYDRHEWPFDICLEGGRGSSVQYVPQSDNQGAGAWLSRQLAAYADGTRIKVIVD\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (284 of 500)\n",
            "Sampled sequence:\n",
            "PIYDVVQYLPCDKYPHACATIAKAQQQGFSSICTLDRASQSNRRRQSLAGIPERNGFMRHEWPMAFCQEGGKGSQVEYVPEQDHNGMSSWLSNRLSEFADGTRILFVCN\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (285 of 500)\n",
            "Sampled sequence:\n",
            "MTYDAVMMFPFREYPQSGDLIRAAIAKGYSDVCTLDRGNRGVRREQSLRGIPTKDGYERHEWPWDICLEGGVGAAVRYVDAKDNQGFGRWLASFLRGFADGTRILLQVR\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (286 of 500)\n",
            "Sampled sequence:\n",
            "MEYDAVLNFPTDTYPESGAHIKMAVSRGFSEVCTLDRASKQRRRTESLAGIPALSGYDRDEWPFSICLEGGAGSSVTYVSSSDNQGAGRWLASKLSAYADGTKILFVVQ\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (287 of 500)\n",
            "Sampled sequence:\n",
            "AGYDAELALPAQQYPDSASHISKAIRLGHAAVCTLERGNQAQRRKESLAGIPDKPGFDRHEWPLAMCREGGKGSAVRYVPWQDNRGFGSWLGNTLSFYKDGTRIKFIVS\n",
            "Sequence recovery: 0.6330275229357798\n",
            "\n",
            "Sampling.. (288 of 500)\n",
            "Sampled sequence:\n",
            "MEFDVELNWPRSRYPYAGEHIKQAIAEGHAAVCTLYRRGHRRRMEEALKGIPERSGYTRHEWPFALCEEGGKGSKVEYVPRSDNAGAGTWLRKRLSGYADGTRILWIVG\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (289 of 500)\n",
            "Sampled sequence:\n",
            "FDYDAVQDFPWKAYPQAGACIKAAIAQGHAAVCTLDRAGRAQRRNQALAGIPSKPGYGRHEWPFAMCKEGGKGSTVQYVDLADNQGMGKWLEKSWAAFADGTKIKQVVK\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (290 of 500)\n",
            "Sampled sequence:\n",
            "MAYDMRWNFPKSTYPYAGKLIANAIAEGKSSVCTLKRGAADLRRDQSLQGIPKKSGFDRHEWPFAMCTEGGVGSKVEYVPSSDNEGAGKWLGSKLSGYADGAKILVIVP\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (291 of 500)\n",
            "Sampled sequence:\n",
            "SSYDIIVDLPTNEFPYAGAHIAEAISKGHAAICTLNRSGASQRRTESLSGIPTKAGYDRDEWPMAGCKEGGTGSAVRYVPAADNRGFGKWLGGYLSRYADGTKVLFVVK\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (292 of 500)\n",
            "Sampled sequence:\n",
            "MDYDARLYFPCQAYPAAGAMIAQAISEGHSAVCTLDRAGSNSRANESLSGIPRRPGYDRHEWPMAICKEGGTGSRVVYVPADDNNGMGTYLSQRLSRYEAGTRILFVVD\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (293 of 500)\n",
            "Sampled sequence:\n",
            "DEFDTVINYPSSTFPFAGQQIRDAISQGFADVCTLDRKAQLQRRRESLAGIPERNGFDRHEWPFAWCLEGGKGAYVQYVPRADNTGAGAWLMEWLSQYRDGTKVLLRVE\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (294 of 500)\n",
            "Sampled sequence:\n",
            "TSYDVELLLPCSAYPEACAHIRTAIAQGYASVCTLDRAGARERRRESLAGIPKKQGYDRHEWPFAECKEGGAGSAVQYVPRDDNSGFGAWLTHTLDGFADGTRVRMTVG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (295 of 500)\n",
            "Sampled sequence:\n",
            "ADYDVTFSFPSDQYPQSGNQIKTAIASGFASVCTLERAASSVRRDQSLAGIPKKAGYMRDEWPYAYCRQGGAGSSVVYVDSADNQGAGSWLSTKLSAYADGTRILFKVS\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (296 of 500)\n",
            "Sampled sequence:\n",
            "MVYDAVMRFPSQTYPESGAHRKLAIELGHAKVCTLDRAGRVQRLIESLAGIPDKPGYIRDEWPFAECLEGGQGSAVRYVSSADGQGASRWARNHLAGYADGTRILFVDR\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (297 of 500)\n",
            "Sampled sequence:\n",
            "VVYDVTFQVPADTYPESGAHIAQAIANGYASVCTLNRAGRTIRRNQALQGIPNRIGYRRDEWPASYCTEGGLGSAVRYVDYRDNDGAGTWISNTLSGYLDGTRILFIVR\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (298 of 500)\n",
            "Sampled sequence:\n",
            "RLYDSVLTFPASTFPQSAAHIQQAIAEGHASICHLSRADATTRRRQSLAGIPAKAGYDRDEWPYALCTEGGTGSGVAYVNSADNQGAGAWFKSTFSAFADGTRIKSTVA\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (299 of 500)\n",
            "Sampled sequence:\n",
            "GEYDTKLFFPVATYPAAGAQIKEAIKMGYAEICTLRRGSAKQRRKESLKGIATRPGYDRHEWPFEPCVEGGLGSRVKYVPSVDNQGMGKWLHNFLANYADGTRIRFIVK\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (300 of 500)\n",
            "Sampled sequence:\n",
            "MTYDKTIETPCETYPEACAHIKAAIAAKHSPICTIERTGRRKRRNEALAGIPKREGYDRHEWPGSYCQEGGKGSSVTYVPDVDNQGFGNWLSETLSQFSDGTKIKLVVS\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (301 of 500)\n",
            "Sampled sequence:\n",
            "MAYDAVLLFPQNSFPQSGKHIKAAIARGKSAVCTLKRAGQSVRRDQSLSGIPSKSGYVRDEWPMAACLEGGSGADVQYVESSDNQGAGRWLRNNLSGYADGTRIKLVVG\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (302 of 500)\n",
            "Sampled sequence:\n",
            "SDHDVECEFPDTEYPESAAHIRDAIAAGHAAVCTLERVGAGQRADESLRGIPAKKGFQRAEWPHNSCVEGGRGSAVRYVPFADNQGAATWWRRTLAKYPDGTRVLMVAG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (303 of 500)\n",
            "Sampled sequence:\n",
            "EPYDRILVFPDDRYPESGALVQRAIAEGKAAVCTLDRASAQARADEALAGIPTKPGYDRHEWPAAECREGGAGSAVEYVPWRDNHGYGGWLRNQLSTYADGTKILVRVL\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (304 of 500)\n",
            "Sampled sequence:\n",
            "HVYDANVIVPAATYPEAGAQVRQAIAEGHAAVCTIDRAGAASRRDEALAGIPAKPGYNRQEWPFASCKEGGKGAKVTYVPTADNSGMGNWLGVSLNGYADGTKIKIVNN\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (305 of 500)\n",
            "Sampled sequence:\n",
            "MNYDAKMQFPAARFPEAGATIAAAIAAGHADVCTLDRASQKVRRDQALKGIPDRPGFDRHEWPHERCVEGGAGARVTYVASDDHQGFDAWLRLRLGGYADGTKILVVVK\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (306 of 500)\n",
            "Sampled sequence:\n",
            "MAYDATLITPANAYPNSGKLISDAIANGWASVWTLDRKSAGTRRNQSLEGIPTKSGFQRHEWPFACCKEGGLGSKVQYVPSADHSGFGGWIASYASQYEDGTKILIIVP\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (307 of 500)\n",
            "Sampled sequence:\n",
            "TSYDASFEFPAAEFPEAGAVIRQAIAEGHSSVCTLSRAGRQVRAKEALAGIPKKPGYDRHEWPMALCEEGGKGARVAYVGSTDNRGMGGWLATTLSAYADGTKILWKIS\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (308 of 500)\n",
            "Sampled sequence:\n",
            "GSYDATLWYPWATFPNSAAHIRRAISEGFSRVCTISRADENRRRRESRAGIPGKPGYIRQEWPFAICKEGGAGAAVSYVPDADGQGASRWLAGWTSRFADGTRILFRVD\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (309 of 500)\n",
            "Sampled sequence:\n",
            "SMYDAVLNFPKEKYPEAGQHIRTAIREGHADVCTLKRDGADSRKRQSLAGIPARTGFDRHEWPFSYCEEGGKGSAVRYVDKRDNMGSGRWLSRKLNKYSDGTKILFVVD\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (310 of 500)\n",
            "Sampled sequence:\n",
            "KSYDVVVEYPCSQFPEACEMIRDAISQGKSSVCQLERASSADRQTHSLQGIPSKSGYDRHHWPEAKCVEGGFGSQVRYVKTTDNQGFGNWIASDLAQYADGTKVLYKIS\n",
            "Sequence recovery: 0.45871559633027525\n",
            "\n",
            "Sampling.. (311 of 500)\n",
            "Sampled sequence:\n",
            "NLYDVELYLPVAAYPQAGQHIRAAINQGHAAVCTLNRAGQSRRKHESLQGIPKRRGYERFEWPASFCQEGGAGSRVTYVAAADNQGMGRWLASTLSRYADGTKVLFKVN\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (312 of 500)\n",
            "Sampled sequence:\n",
            "SDFDLEYHFPAAAYPAAGAQIKQAIAEGHAAVCTLDRAGEGNRLNQSLAGIPRRPGYDRHEWPFAYCSEGGQGAAVEYVPFDDNGGAGKWIQTTLSRFQDGTRVLFRVD\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (313 of 500)\n",
            "Sampled sequence:\n",
            "ADYDASIKFPAATYPQAAAHIARAIAQGYSSVCTLQRSGEGSRRMQSLAGIPAQNGYERHEWPFSICEEGGAGSSVRYVQSSDNRGFGRWLSGNLAGYSDGTRILIVVG\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (314 of 500)\n",
            "Sampled sequence:\n",
            "SDYDAVLYVPAAKYPEAAAQIRAAIAEGHSQVCTLDRAGQRARRRQSLAGIPNKPGYMRHEWPFAICREGGAGSSVEYVPSDDASGFSSWAQNRLSKYADGTKIKFITV\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (315 of 500)\n",
            "Sampled sequence:\n",
            "NEYDVQLELPAAAFPQAGALIARAIADGHPAVCTLERAQRGRRHDEALAGVPARPGYGREEWPFALCLEGGRGSMVEYVEESDNNGAGAYLRTYLAQFQDGTKVLITVD\n",
            "Sequence recovery: 0.45871559633027525\n",
            "\n",
            "Sampling.. (316 of 500)\n",
            "Sampled sequence:\n",
            "PSYDAVLSYPVDTYPASGAMIKQAIANGQSAICTIDRDSSSKRRDQALDGIPTKPGYGRAEWPMFACTQGGSGSTVQYVPFDDLSGAGRWIARVLSGYSDGTRIKFVVS\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (317 of 500)\n",
            "Sampled sequence:\n",
            "DDFDATLLFPCSEFPEACAQIRDAIERGHARTCTLDRERALERRRQALRGIPEREGFQRHEWAFALCREGGCGSAVAYVRSTDNAGAGTWLARTLNRYADGTRILFVCG\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (318 of 500)\n",
            "Sampled sequence:\n",
            "GEYDAELTFPESTYPHAAAHIAKAISEGHAAVCTLDRAGSGERRRQSLAGIPSRPGYRRDEWPFAMCREGGAGAAVEYVPRSDNDGAGAWARQYLAKYADGTKIKFIVG\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (319 of 500)\n",
            "Sampled sequence:\n",
            "MTYDARLIFPFNTYPNAGATIKAAIAKGHAAVCTIDRAGRAQRRKESLAGIPTKTGYDREEWPMAICKEGGKGATVQYVPGADNNGMGRYVLETMSKYKNGTKILFTVD\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (320 of 500)\n",
            "Sampled sequence:\n",
            "GVYDTVLTLPCDTYPEACAHIAVAISEGHSAVCTLDRAEAGRRRDEALHGIPTRPGYDRAEWPMAGCVEGGRGSSVTYVRTADNQGYGAYLRTHLSQHSDGTRILFVVR\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (321 of 500)\n",
            "Sampled sequence:\n",
            "MDFDAELFFPSEEFPESGAHIKQAIANGHAAVCTLERTGSSRRRREALAGIPEKDGFGRHEWPFAACKEGGEGSAVSYVNQNDNRGAGNWLSGYLDNFADGTKIKFIVD\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (322 of 500)\n",
            "Sampled sequence:\n",
            "ADFDAELVLPKKKYPASGAMIEQAIKDGYADVCTLERAGADERRKRSLKGIPAKSGYDRHEWPMAICKEGGTGSRVQYVPQADNNGMGRWLSQKLAAYADGTKIKMIVR\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (323 of 500)\n",
            "Sampled sequence:\n",
            "SDFDIELFFPYQQYPESAEQIKSAIQKGYSDICSLKRASRSERREQALKGIPARKGYMRHEWPSAICEEGGVGSDVKYVPQSDNSGAGTWLRAKLSSYQDGTRVLFKVQ\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (324 of 500)\n",
            "Sampled sequence:\n",
            "HAYDKTWQFPKTTYPKAGATIARAIAEGHAAVCTLSRAGAEARRKQALAGIPQKAGYGRHEWPFAACWEGGTGSYVAYVPGDDLNGMRTWIANRTAGYADGTKIKYIVA\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (325 of 500)\n",
            "Sampled sequence:\n",
            "GSFDATLTYPGNTFPESAATIKQAIENGHSSVCTLARGDQSTRRDQSLQGIPKQTGYDRHEWPFALCEEGGNGSYVAYVQTSDNSGFGAWLNGWLNAYDDGTKILFVVQ\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (326 of 500)\n",
            "Sampled sequence:\n",
            "MSFDTVLFVPCKQYPEACAMIKSSIAAGAADVCTLDRSGQSERRRQALKGIPSRDGFERHEWPFAMCKEGGAGATVEYVGFEDNRGMGRWLQTTLASFRDGTRVLFVVY\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (327 of 500)\n",
            "Sampled sequence:\n",
            "MDYDAKLLYPVEEFPASGALIKRAIEKGFPNVCTLKRADARVRRDQSLKGIPSRPGYQRGEWPFAICLEGGKGAQVEYVPFEDNNGSGGWLSGYLSGYPDGTRIKYIAD\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (328 of 500)\n",
            "Sampled sequence:\n",
            "VEFDNVEFFPQERFPESGAMLRDAMTRGHAAVCTLTREGADLRRQESLQGIPEKPGYIRHEWPFALCAEGGQGSLVVYVPSADGEGAGAWVRDRFSQYNDGTRILFVID\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (329 of 500)\n",
            "Sampled sequence:\n",
            "SDFDHKLKYPVKEYPEAGAVIKQAISEGFSDVCTIERKEAGKKRKESLKGIPTRPGYDRHEWPFASCVEGGKGSRVEYVRAVDNRGAGLWLKNKLSGYGNGTKIKFTVE\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (330 of 500)\n",
            "Sampled sequence:\n",
            "IDFDTEVHLPHQRFPQAAAHHAKAIRQGFAAVCTLDRASAERRADQSLAGIPSKPGFDRHEHPFAACKEGGKGSSVAYVPAADHHGFATWIRGHLARFADGTRVRFIAE\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (331 of 500)\n",
            "Sampled sequence:\n",
            "SDFDTELYYPSTTYPAAGTNIQTAIQSGYSAVCTLQRSAEGQRADEALQGIPSKSGYDRHEWPFAMCVEGGTGSEIKYVPTLDNQGAGKWAGRTLSNFADGTKVKFIVQ\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (332 of 500)\n",
            "Sampled sequence:\n",
            "MDYDTRIYLPCSTFPDSCRHVRSAIAEGYSAVCTLDRKSAATRRNQSLAGIPSKPGYDREEWPFAWCKEGGKGSSVRYVYSSDNQGYGNWMRSTLAAYADGTRILIVID\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (333 of 500)\n",
            "Sampled sequence:\n",
            "DAYDAEWHIPCEQYPEACAMIDEAIAQGHAAVCTLDRAGEGERLRDALAGIPAREGYGRHEWPFAYCLEGGRGASVRYVGVEDNNGLGAYLADSLAGYADGTRIKHVIE\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (334 of 500)\n",
            "Sampled sequence:\n",
            "LDFHAKLQFPVDTYPESGALVKSAIAAGYAKVCTIDRAGSGARRDASLAGIPRKVGYDRHEWPHAICLEGGTGSAVAYVAAADNQGFGKWFNNQMNAYADGTKILEIIE\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (335 of 500)\n",
            "Sampled sequence:\n",
            "ASYDVTLHVPSTQYPQYGTHVAAAIAQGHAAVCTLSRRNRGKRRNQSLAGIPDRPGYDRHEWPFSWCEEGGRGASVTYVNSSDNQGAGSWMSGYLSGYADGTRVLFVVD\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (336 of 500)\n",
            "Sampled sequence:\n",
            "MWYDAKLHFPVKKYPKAGALIKASIKRGFSSVCTLQRAEAAERRRQALAGIPKREGFDREEWPAALCKEGGKGSEVRYVPYADNRGAGMWIANRLASYDDGTRILFIVE\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (337 of 500)\n",
            "Sampled sequence:\n",
            "SVFDATLTFPESEFPEAGAHIRLAIAEGHAAVCTLERAGAKARRKQSLAGIPARPGFERHEWPFADCLEGGQGSAIVYVNKDDNRGASRWLSQVLSAYADGTRILFVVD\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (338 of 500)\n",
            "Sampled sequence:\n",
            "MEYDATLDLPCKTYPEACAHIKRAIALGHADVCTLDRAGEAARRKESLAGIPKKKGYERHEWPGASCKEGGKGSAVQYVEKADNRGFGSWLGKTLSQYKDGTKIYLRVG\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (339 of 500)\n",
            "Sampled sequence:\n",
            "MDFDAQLVFPSDEYPESGAMVKKAVAQGFADVCTLERNSASKRANESLAGIPGLPGYGRFQWPAANCKEGGSGSAVEYVDGADNNGAGEWFKNTLGGYADGTRIKFVVE\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (340 of 500)\n",
            "Sampled sequence:\n",
            "NAEDVKIKFPSKEFPNSSKMIADAIARGHAKVCTIDRASATTRRQQALKGIPKKPGYTREEWPLSFCKEGGKGSAVRYVPSWDNQGAGHWLSRRLSSFKNGTRVKFIAK\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (341 of 500)\n",
            "Sampled sequence:\n",
            "DTHDVTLDFPASSFPQAGQNIRTAVSEGRSDVCTLERSKRKIRLDQSRAGIPAKSGYQRHEWPMSFCKEGGTGSQVRYVNSSDSSGFGSWVRSYLAKYKDGTRVLFKIS\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (342 of 500)\n",
            "Sampled sequence:\n",
            "ADYDTILTFPSAEFPGSAAHIKQAIAQGFAEICTLNRAGAAIRRQQSLAGIPARDGYNRDEWPFALCAEGGQGSAVKYVDAADNNGAGTWLANYLKQYDDGTKILFVVD\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (343 of 500)\n",
            "Sampled sequence:\n",
            "MDYDARVYTPCKAYPQTCAHIRAAMAMGHAAVCTLQRAQESRRRKESLKGIPTKNGYDRHEWPFAWCVEGGKGALVQYVDAADNTGAGAWIRSWLSAYANGTRILFIAR\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (344 of 500)\n",
            "Sampled sequence:\n",
            "MRYTATWNYPAGSFPESAAQINRAVANGNAAVCTLARAERKKRLTQSLFGIPAKTGYNRHEWPMAACVEGGAGASVQYVGKTDNQGFGTWLRNTLAAYADGTRILTVVA\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (345 of 500)\n",
            "Sampled sequence:\n",
            "CKYDSVLYFPSSAYPASGKHVKNAISLGFSKICTLDRYSQSYRKQQSLRGIPARDGYIRHHWPHSLCVEGGWGSSVRYVPSDDGNGARQWLANHLSKYSDGTRIKFVTA\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (346 of 500)\n",
            "Sampled sequence:\n",
            "GMHDLTLNFPVATFPEAGAMIRTAIANGYAEVCTLDRQQSSKRSDQSLAGIPKRQGYERHEWPFNICLEGGQGSEVRYVPAADNQGAGAWLREELAQFNDGTRIRFIVG\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (347 of 500)\n",
            "Sampled sequence:\n",
            "ADYDVTLELPGEDYPEAAAMIKKAISQGFAAVCTLDRAGADRRADLSLAGIPARGGYQREEWPFAMCVEGGQGSAVQYVPFSDNLGFGNWMNRTLAGYADGTRVRFVVS\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (348 of 500)\n",
            "Sampled sequence:\n",
            "IAYDHIEIYPARKFPESGALIKAAIAAGKSAICTIQRETAAARRDESLKGIPDREGFTRHEWPFASCREGGKGSAVRYVPSEDNNGAGAWLSRRLSRYADGTRIRQVTS\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (349 of 500)\n",
            "Sampled sequence:\n",
            "MSYDVELKFPAKTFPQSAAHIAKAIKNGHSAVCTLNRSGQRTRKDQSLKGIPHKDGYDRDEWPFASCVQGGQGSDVRYVDHKDNRGAGTYASDELAQYADGTRVLFNVE\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (350 of 500)\n",
            "Sampled sequence:\n",
            "MPFDATLNFPAEQFPESGAVIRKAIANGKAAICTLAKGEQSVRRKESLAGIPKREGYVRHEWPFAMCAEGGKGSYVEYVPSEDGRGFGNWLTNKLAAYADGTRIKQVVG\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (351 of 500)\n",
            "Sampled sequence:\n",
            "CAYDVKWLYPAEAFPAAGAHIKAAIAEGKPAVCTLDRASQRLRRDQALRGIPAREGYDRDEWPFALCTEGGTGSFVRYVPRADNQGMGQYMRTTLAEFADGTKVLVEVQ\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (352 of 500)\n",
            "Sampled sequence:\n",
            "SDYDAVVYYPASAYPESGREIARAISEGHAAVCTLERARREERRRQSLAGIPSRPGYIRHEWPMSSCVEGGTGSAVAYVRAADGQGALAWISSYLSTYADGTRLLYVVW\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (353 of 500)\n",
            "Sampled sequence:\n",
            "MAYDVTLTYPASQYPQAGAMVAQAISEGQAAVCTLDRAGEAARRTQALQGIPERPGFDRDEWPASICREGGSGSAVRYVARSDNQGAGRWLRNHLSAYTDGTRVLIVVD\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (354 of 500)\n",
            "Sampled sequence:\n",
            "DTYDHVANYPTSQYPQSGAHVKQAISEGHSAVCTLDRSGQDQRRTQSLRGISKKSGYDRDEWPMASCIEGGSGSSVRYVDYQDNQGAGQWARQYFAKYADGTKIKFIVS\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (355 of 500)\n",
            "Sampled sequence:\n",
            "VDYDVRVDFPGKRFPEAASLVSRAIRQGHSAVCTLQREMSHVRQREALKGIFEKPGFVRHEWPMAHCKEGGKGSAIEYVRASDGEGYGKWLASTLSAYNDGTRVLFQVN\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (356 of 500)\n",
            "Sampled sequence:\n",
            "MPYDAVLLLPESQYPSAGAHIRAAIGNGHAAVCTLDRSGQGTRRGESLAGIPARPGYMRAEWPMSQCKEGGKGSSVQYVPSDDNAGYGKWLNNTLSGYADGTRILFIVK\n",
            "Sequence recovery: 0.6238532110091743\n",
            "\n",
            "Sampling.. (357 of 500)\n",
            "Sampled sequence:\n",
            "MAYDVTVHIPSATFPASAANIKAAVAEGHAAVCTLERSQQVVRREQSLKGIPAKPGYDRHEWPTAWCVEGGKGSRVVYVPSADNKGLGAWLDRNLRQYADGTKVLFVVG\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (358 of 500)\n",
            "Sampled sequence:\n",
            "GSYDVELTYPAATFPQSAAMVAAAIANGHASICTLRRADARTRRDLSLAGIPSRPGYHRLEWPFAACEEGGAGSAVAYVDNADHSGAGRWLKRTLSGYADGTKVRFIVN\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (359 of 500)\n",
            "Sampled sequence:\n",
            "MEFDVTLTIPCDTFPYSCSTIKTAIAEGFAAVCTLDRAGQGARRRASLKGIPSKEGYDRHEWPFANCLEGGKGSKVVYVPSDDNHGFGQWVNDELKKYADGTKILYTVS\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (360 of 500)\n",
            "Sampled sequence:\n",
            "AAYDHTLALPCSTFPQYCSHVAAAIKKGFAAVCTLDRAGRLARLKESLLGIPKKEGYDRHEWPFALCKEGGKGSAVEYVDKADNNGAGRWLLNRLSSYEDGTKILFSVW\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (361 of 500)\n",
            "Sampled sequence:\n",
            "CVFDTTLIFPAATFPEAAAHIANAISEGFAAVCTLSRAGANARFTQSLAGIPVRPGYGREEWPMARCVEGGTGSAVAYVPNADQAGYGEWLNGYLSAFADGTKVLIVVD\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (362 of 500)\n",
            "Sampled sequence:\n",
            "TVWDAKWYFECARFPESCAHVAWAIAHGHAEVCTLDRASQAERREESLKGIPNREGYERHEWPWAICKEGGAGAMVVYVIASDNRGAGRWVDRQRAQFADGTRIKSITD\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (363 of 500)\n",
            "Sampled sequence:\n",
            "GAADVTLNYPCDTYPEACSMIRTAMAQGHSAICTLQRSQYQSRRNQSLAGIPARSGYTREEWPHAQCLEGGTGSAVAYVGWADNSGAGQWMQNTLSKYSDGTRVLFVVR\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (364 of 500)\n",
            "Sampled sequence:\n",
            "ASYDATWKFPEATYPEAGAMIKTAISQGHAAICTIDRAGQDARRIESLAGIPAKPGFERHEWPFAFCIEGGQGSYVAYVDADDNNGAGKWLRTRLASFADGTKIKVTVS\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (365 of 500)\n",
            "Sampled sequence:\n",
            "GTYDCSLWYPSDKYPEAGAVIARAIAEGYSEVDTLDREAADRRRDEALRGIRNKEGYHRFEWPWALSKEGGKGSAVEYVYQSDGTGAGAYMRRTLDGYADGTKILFVTG\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (366 of 500)\n",
            "Sampled sequence:\n",
            "MDYDVELHYPADQFPQSGAVIAQAIREGHADICTLQRADDHKRRRESLAGIPEREGFDREEWPFAICLEGGQGARVEWVDENDNRGAGAWQRANLSAYEDGTRVKFVLP\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (367 of 500)\n",
            "Sampled sequence:\n",
            "MEYDVRLYTPMLQYPQYGNHLREAIQEGYSSICTLNRSGEQTRRRQSLLGIPNKEGYDRHEWPMADCIEGGEGSRVRYIGRSDNQGAGTWMSNELSRFKDGTKIKFIVN\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (368 of 500)\n",
            "Sampled sequence:\n",
            "ADYDFTLFNPTKNYPQSGAMIKSAISLGHAAVCTLNRAAENTRKEASLKGIPDKPGFVREEWPFDICLEGGKGSWVQYVKADDSRGMGAWLRERLSAFKNGTRIKFVID\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (369 of 500)\n",
            "Sampled sequence:\n",
            "GAYDAKLSMPIETFPEAGAVIRRAIAEGHASICTLARSGAGARRKASLKGIPNAPGFDRHEWPMAACIEGGAGATVEYVPALDNQGFGQWLRERLSRYSDGTRILFTVD\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (370 of 500)\n",
            "Sampled sequence:\n",
            "SDYDTTLYLPCATFPQTCAHIKKAIQEGFDSVETLERASATLRRQQALAGIPAKQGFERHEWPFAYSKEGGTGASVSYVDQDDNRGYGRWLATTLSGYADGTKIKFVVQ\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (371 of 500)\n",
            "Sampled sequence:\n",
            "HMFDTTLTLPTSTFPWMGKHIQDAIASGHNAVCTLNRAGADTRRKESLAGIPGKPGYTRDEWPMSLCTEGGTGSRVRYVPSADNEGAGTWVKNHLSAFDDGTRVLFVVK\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (372 of 500)\n",
            "Sampled sequence:\n",
            "TGFDAVWNYPSSSYPMAGSSVAQAISEGFSKVCTLDRASRGARRTQSLSGIPSKSGYDRHEWPSAMCTEGGTGSAVTYVPVTDNSGAGTWLVNTTAAYTDGTKILFKVG\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (373 of 500)\n",
            "Sampled sequence:\n",
            "KDYDAILRLPCKTFPESCAHIRKAIKQGHAAVCTLDRAAADARRRESLKGIPARPGFDRHHWPGAVCREGGKGAGVAYVPTGDNQGMGALLGRTLATYKDGTRIRIECG\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (374 of 500)\n",
            "Sampled sequence:\n",
            "MSYDATVNYPMETYPQSGAHIKSAIANGHAAVCTLDRANASKRRKEALKGVPAKPGFIRHEWPFAHCKEGGVGSSVTYVDWKDGAGANSWLAARLSAYADGTRLKFIVS\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (375 of 500)\n",
            "Sampled sequence:\n",
            "ADYDTRIAFPGARFPASAAHISKAIAEGHADVCTLNRAGADFRKDQALRGIKAKHGYGRQEWPFAACTEGGKGAAVTYVPLHDNSGAGHWLAKQLSGYKDGTKVLFTVA\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (376 of 500)\n",
            "Sampled sequence:\n",
            "MAYDTTLQLPGAQYPESAAHIAQAIAQGKAAVCTLDRAGQGARADAALAGISHRVGFGREEWPFSQCFEGGQGSAVQYVPWEDNNGMGAWLGRTLSSYADGTRVLFQVH\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (377 of 500)\n",
            "Sampled sequence:\n",
            "EKYDVTVNLPVAKYPDSGAMIKAAIEEGQAAVCTLGRAGQQQRRKEALAGIPTKEGYQRHEWPFAVCKEGGKGSAVRYVPSADNTGMGTWLKKTLAKFADGTKVLFVID\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (378 of 500)\n",
            "Sampled sequence:\n",
            "NSYDAVLIFPAATFPESAACIKKAIGEGHAAVCTLERDNAGMRRQESLAGIPERPGYVRHEWPFAICAEGGAGSHVEYVPAADAEGAAAWFGRELSKYDNGTRILFVVR\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (379 of 500)\n",
            "Sampled sequence:\n",
            "SMYDVKLYLPSLQFPQSGALIRQAMEEGHAAVCTLSRAEANRRRNESQKGIPKRRGYDRHEWPASWCVEGGRGSKVQYVPKADNQGFASWKGRYLNQYADGTRILIVVE\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (380 of 500)\n",
            "Sampled sequence:\n",
            "GHHHVVFLAPESKFPQSAAMIRRAIAKGHASTCTLDRAGQSERRTQALAGIPKRDGFTRHEWPFAMCREGGTGSSVAYVDRADQEGFGAWLARTLSKYADGTKVQVVVF\n",
            "Sequence recovery: 0.44954128440366975\n",
            "\n",
            "Sampling.. (381 of 500)\n",
            "Sampled sequence:\n",
            "PDYDAMLRLPCKTYPQACAHIQQAISEGHAAVCTLFRWGQAARRNSSLKGIPTREGYSRAEWPFALCTEGGEGAAVKYVNKDDNQGFGAWLAKQLAKYDDGTRILFVNP\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (382 of 500)\n",
            "Sampled sequence:\n",
            "SNADVIWHFPCSEFPESCAHIAQAIDQGHAAVCTLERAKAGTRRKQSLAGIPQRDGFDRHEWPFAICSEGGIGSSVTYVPQNDNRGAGRWLMGELSLYSDGTRILQVVG\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (383 of 500)\n",
            "Sampled sequence:\n",
            "GAYDQTCTFPSDTFPEYGAQVAAAIANGHSAVCTLRRADAAVKRDESLQGIAQRPGFDRDEWPMAYCLEGGTGSAVQYVEYTDNRGAGAWLAAWLAGYADGTKILWIVD\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (384 of 500)\n",
            "Sampled sequence:\n",
            "SNFDASLAFPIDEHPQAGATLKKAESEGHAQVCTLDQAGRTLRRKQSLAGIAKKKGFGRHHWPFALCKEGGKGSRVEYVPADDQQAAGKWLAKTLDRFNNGTKILFTTG\n",
            "Sequence recovery: 0.43119266055045874\n",
            "\n",
            "Sampling.. (385 of 500)\n",
            "Sampled sequence:\n",
            "FEFDARVHLPDQQYPAAGAHIKAAIAAGHPAVCTLDRDQSRERRRQALEGIPTKPGFERHEWPFAFCKEGGTGSDVQYVPADDNQGWGRWLANSLASYADGTKILFEAA\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (386 of 500)\n",
            "Sampled sequence:\n",
            "VAYDAELLFPCDTFPHSCRLLAVAIAEGHASVCTLDRAGRTRRRQQSLAGIPRRDGYDRHEWPMAMCKEGGDGAAVEYVPKKDNRGFGKWLGERLGEYADGTKIKVINR\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (387 of 500)\n",
            "Sampled sequence:\n",
            "KDYDCEIFFPRNTFPHSGGLIKKAIADGHSSVCTLRRSGARKRREEALLGIPKRSGFDRHEWPHSNCYEGGAGARVEYVPAADNRGAGQWLKNTLSVFAEGTRIKFVVG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (388 of 500)\n",
            "Sampled sequence:\n",
            "MSYSATMNLPVSAFPESGAMVAQAINEGHANVCTLDRANRHVRLMQALAGIPARNGYVRFEWPFDSCVEGGTGSKVKYVPAQDGDGWDAWLRNQLAAYSDGTRIKFVVT\n",
            "Sequence recovery: 0.43119266055045874\n",
            "\n",
            "Sampling.. (389 of 500)\n",
            "Sampled sequence:\n",
            "AKFDVELLFPSATFPQSGAHVKAAIAAGHSDVCTLERAGATARRQQSLAGIPKKPGYDRDEWAMAMCKEGGQGSAVRYVFRADNQGAGNWMGKTLAQYADGTRILFKAG\n",
            "Sequence recovery: 0.6146788990825688\n",
            "\n",
            "Sampling.. (390 of 500)\n",
            "Sampled sequence:\n",
            "NTYDTECVLPCATYPHSCGLIQTAISEGKAAVCTLTRAESTKRRNESLAGIPRKSGYERHHWPFAECTEGGTGAAVAYVPEADNQGFGTWISGWSAAYADGTRIKFKTY\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (391 of 500)\n",
            "Sampled sequence:\n",
            "MTYDVRVWLPAERFPEAGALIREAIRRGHAGVCTLDREGAKRRQAQSLRGIPDRPGFQRQEWPMAECVQGGAGSWVRYVAGADGRGFNAWLGQTLSGYADGTRVLLVVD\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (392 of 500)\n",
            "Sampled sequence:\n",
            "ADYDVQLNFPTEKYPSAGSQIKEAIEQGHAAICTLKRETRGERFDQALEGIPEREGYDRHEWPHALCEEGGEGAYVKYVPEDDHNGAGSWTRYYLSGYEDGTKVLFIVE\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (393 of 500)\n",
            "Sampled sequence:\n",
            "GKYDVEEILPCSAYPDHCAMIKTAIYWGHSAVQTLSRDTQQHRRKQSLKGIPQRDGYGRHEWPFAVSKEGGTGSYIKYVPQDDLDGAGRWLTSRLHKYQNGTKVLFKVR\n",
            "Sequence recovery: 0.44036697247706424\n",
            "\n",
            "Sampling.. (394 of 500)\n",
            "Sampled sequence:\n",
            "SLFDEVLILPCETYPETCAAIAAAIALGHAAVCTLDRSGSARRRDESLKGIPARPGYQRHEWPASHCREGGRGASVEYVPAEDNRGDGGWLRLRLSAFPDGTRILVAVG\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (395 of 500)\n",
            "Sampled sequence:\n",
            "CQYDVTLNFPHTTYPQAGAVIRKAIAQGHAAVCTLARSGANARRAQSLNGIPSRAGYDRHEWPMAICREGGTGSAVQYVDSADNNGAGGWLRSRLSQFKDGTRVKWVVD\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (396 of 500)\n",
            "Sampled sequence:\n",
            "CGYDVTWQTPCATYPHSCAHIRAAVAAGHAAVCTLDRSGAKQRRFESLFGIPAKSGYDRHEWPFSFCTEGGQGSAVAYVPWADNNGFGTALGKALAGYADGTRVRFVVG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (397 of 500)\n",
            "Sampled sequence:\n",
            "QDYDAVIQLPCATFPASCAQIKYAIGQGYAAVCTLQRETQKHRRMQSLEGIAVKDGYNRHEWPFSVCAEGGSGSYVQYVGATDNSGMGSWLSGFLGNYADGTKILFVVG\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (398 of 500)\n",
            "Sampled sequence:\n",
            "MDFDVVLSLPVDMYPEAGAQIRHAIARGHADVCTLDRASANVRRNQSLAGIPRRPGFGRFEWPLDLCSEGGHGSSVTYVPYADNDGLGAWLRNHLSAYKEGTRILFVDG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (399 of 500)\n",
            "Sampled sequence:\n",
            "TTFSAVWKAPCTKYPESCNHIKNAINKGHAAICTIDRSGRGKRRDESLKGIPKKDGYGRDEWPYAWCREGGTGSDVRYVNVADNQGMGRWIKTRLSKYSDGTRILEKCK\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (400 of 500)\n",
            "Sampled sequence:\n",
            "PDYDTELVFPSSTYPEAGAMIKAAISEGHSDVCTLQRAESNKRRNQSLAGIPTKQGFTRGEWPYAKCEEGGAGSKVQYVKQSDNQGAGKWLTAKLAGYEDGTKIKFIVK\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (401 of 500)\n",
            "Sampled sequence:\n",
            "VTYDATLTFPAAQYPNSAAHIATAIAQGHAPVCTLARAAAGTRRTQSLQGIPARPGYERHEWPMARCTEGGAGASVRYVPLADNRGAGSWLSSTLSGFADGTKIRFICQ\n",
            "Sequence recovery: 0.5963302752293578\n",
            "\n",
            "Sampling.. (402 of 500)\n",
            "Sampled sequence:\n",
            "MDFDFVLVNPSAQFPEAAAHIREAVSQGHAAVCTLERGGQEQRRKESLAGIPENEGYERFEWPMAWCVEGGRGSAVRYVPREDNQGFAAWLSKALDAYEDGTRILFVVG\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (403 of 500)\n",
            "Sampled sequence:\n",
            "MTYDTEVKVPSSQFPQAGSHIRKAILLGFSKVCTLDRVGQGARHKESLAGIPNRQGYVRDEWPFAQCREGGQGSGVQYVHSDDNQGMGRWLRNELSRYADGTRIRFIVE\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (404 of 500)\n",
            "Sampled sequence:\n",
            "MKYDAILYLPAEKYPQSARTIEQAIALGRPAVCTLIRAGRNQRRREALAGIPLRHGYHRHEWPFAICQEGGRGSNVEYVDFVDARGYGRWLARTLSQYADGTRILFVVE\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (405 of 500)\n",
            "Sampled sequence:\n",
            "EEYDVVLNFPTDTFPEAGAHISTAISNGFASVCTLQRTGQRQRRNESLNGIPRKEGYSRSEWPMAVCVEGGQGSAVQYVPQDDNSGAGTWLANTLSAYPDGTRVLFVID\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (406 of 500)\n",
            "Sampled sequence:\n",
            "GDYDVKLTLPASQFPEAGQHIKQAISEGKAEVCTLDRAGYHLRYRQSLRGIPKKDGYERQEWPMSRCVEGGKGANVRYVPVQDNRGFGAWIANYLSGYADGTRVKFVVQ\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (407 of 500)\n",
            "Sampled sequence:\n",
            "SSFDAIIIYPQDAYPESGKVIRDAIKAGKAAICTLDRAGQTQRANESLKGIPDRTGYVRLEWPFAFCVQGGKGSYVEYVPARDGQGSGQWLRNYLSAYAEGTKIKLVID\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (408 of 500)\n",
            "Sampled sequence:\n",
            "AGFDAELILPETTYPQSSSHIRKAIADGHAAVCTLNRASANTRRDESLRGIPQKLGYDRDEWPMAMCLEGGTGSAVQYVPWKDNRGMGTWLNNTLDAYADGTKIKFVVK\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (409 of 500)\n",
            "Sampled sequence:\n",
            "STFDVVVQFPAAKFPESAAHIKTAIGQGKAAICTLLRSGRKARRSQALAGIPKRSGYIRDEWPLAYCLEGGAGSAVQYVAEGDGDGAGSWVRSYLAAYADGTKVLFVVA\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (410 of 500)\n",
            "Sampled sequence:\n",
            "MKYDAILNFPADSYPQSGAHIKAAIAEGHSDVCTLERAQQSTRRDQSLAGIPRRAGYARDEWPMASCVEGGQGARVTYVKAADNDGAGAWLSQTLSHYADGTKIKVVVK\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (411 of 500)\n",
            "Sampled sequence:\n",
            "MSFHATWIFPSSVFPQSGAMIRQAISEGFSEVCTLDRESREKRRDESLRGIPARPGYERHEWPFALCREGGTGSAVRYVGNADNRGAGAWISNALSAYANGTRILTIVE\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (412 of 500)\n",
            "Sampled sequence:\n",
            "PAYDTTWVFPCDTFPESCAMIKVAMSKGHSAICTLQRADARKRRNQSLKGIPDRDGYTRHEWPFADCIEGGEGAWVEYVSSSDNIGFGIWLKNKLAQYEDGTRIKEIVT\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (413 of 500)\n",
            "Sampled sequence:\n",
            "IPYDAVVELPDADYPEAGATIRRAIAEGHSRVCTIERAARGRRQDESLRGIPRRSGYDRHEWPFACCVEGGTGSTVAYVGSADNQGHGRWLAQYLAAFNDGTRILTKVR\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (414 of 500)\n",
            "Sampled sequence:\n",
            "MEYDATLEYPSATYPESGQMVKAAIAKGHASVCTLDRAGEKERREQSLAGIPEREGFSRQEWPFSMCREGGAGAEVRYVARKDNQGAGKWLNNYLAGYQDGTRIKFVVE\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (415 of 500)\n",
            "Sampled sequence:\n",
            "VSYDTTLSLPYRKYPEASAHIRQAISEGHSDICTLERRAALKRRQQSLRGIPKRRGYDRHEWPFAICEEGGSGSSVRYVPKADNSGMGSWLRTKLSRFADGTKVRFVVD\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (416 of 500)\n",
            "Sampled sequence:\n",
            "SEFDTILYLPCAQYPESCAHIRQAVSEGYAAVCTLERAKAKIRRDQALQGIPAKEGYDRHEWPFSHCKEGGKGSSVRFVPTVDNQGYGSWLHTTLAKYADGTKILLVYA\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (417 of 500)\n",
            "Sampled sequence:\n",
            "GEYDATLVFPKDEYPDSGDMVKKAIEEGNSAICTLDRSGRQKRRKESLKGIPAKEGYVRHEWPFAICKEGGKGSKVQYVPEDDGRGAGRWLHNTLSQYGDGTKIRFVIK\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (418 of 500)\n",
            "Sampled sequence:\n",
            "GPYDAEWVFPSKQFPQSGRQVKTAIAQGHAAVCTLERIGSHARSVQALAGIPQKSGHERMEWPMALCKEGGRGSAVRYVKRADNSGMGGFMRTEQSRFADGTRILRRVR\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (419 of 500)\n",
            "Sampled sequence:\n",
            "QTHDVEVNFPCKTYPESCKQITAAIKAGFSKVCTLDRAQYQKRLDQALKGIPKRSGYDRHEWPFALCKEGGKGSHVAYVDYKDNGGAGSWMWNWLSQYADGTKILFKVS\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (420 of 500)\n",
            "Sampled sequence:\n",
            "HMFDVTLDFPCATYPDACAHIAQAIELGHAAICTLERDSEKARKKESLAGIPGKDGYDKHEWPFSICKEGGLGSHVTYVPKDDNAGAGAWLKSHLMRFNDGTRILYRAA\n",
            "Sequence recovery: 0.47706422018348627\n",
            "\n",
            "Sampling.. (421 of 500)\n",
            "Sampled sequence:\n",
            "GDHDATLTVPCDTFPHACAHIKAAIAEGHAAVCTLERQQQNQRYLESLEGIPRKRGYDRDEWPFSLCLEGGKGSSVQYVPSYDSQGFGAWAANWLNEYADGTRILFRVG\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (422 of 500)\n",
            "Sampled sequence:\n",
            "MLYDYTLIFPKTRFPESGAHISDCIRQGHAAVCTLSRSLHQRRRIESLRGIPSAPGYDRDEWPFAMCREGGKGSCVRYVPWSDNRGAGAWAGGFLSRYGDGTRILFIVG\n",
            "Sequence recovery: 0.6513761467889908\n",
            "\n",
            "Sampling.. (423 of 500)\n",
            "Sampled sequence:\n",
            "HTFDVKLTLPCSTYPESCDTIKDAIADGKAAICTLSRSSSSAKRNESLSGIPAKSGYDRHEWPFSLCKQGGAGSAVTYVPINDNQGFGRWLGNELSGYADGTRVLFTVE\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (424 of 500)\n",
            "Sampled sequence:\n",
            "AEYDATLDFPDAEFPASGQHIRNAISKGFADVCTLDRAGAKQRKKQSLANIPTKEGYVRAEWPMSICKEGGQGSSVQYVPADDGEGFAQWLKNKLAGYKDGTRIKIVVS\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (425 of 500)\n",
            "Sampled sequence:\n",
            "SLYDTTLHFPDSQFPEAGAMVDSAIREGHADVCTLQRAQAEERRQESLHGIPERDGYDRHEWPFAFCEEGGKGSKVRYVRLSDNQGFGTWLGQRLSVFDDGTKVKIVVQ\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (426 of 500)\n",
            "Sampled sequence:\n",
            "APYDVELVFPCRRYPEAGAHIRDAIQQGHPAVCTLDRASSADRRRASLANIPEKHGYERHEWPMSWCKEGGAGSAVRYVPAADNRGAGAWMQRRLAEFDDGTKVLQRVG\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (427 of 500)\n",
            "Sampled sequence:\n",
            "ASYDIVQQLPCKQFPASCDHIKQCIAANKDAVCTLERNAQESRRDQSLQGIPHQTGYDRHEWPMAVCQEGGTGSAVQYVPENDNQGFGQWLSGFLSQYADGTRVKFVSI\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (428 of 500)\n",
            "Sampled sequence:\n",
            "ATYDVEWRLPCARYPAACAHIAAANRAGHAAVCTLERAEQDVRLRQSLFGIPERPGYERHEWPFAWCREGGAGSAVTYVPRTDQQGFGAFEKSALARYADGTRIKRVVL\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (429 of 500)\n",
            "Sampled sequence:\n",
            "GEFDAKAFYPEAKYPESGALVKMAIAKGYAAVCTLERSGSLARRKDSLKGVPAKPGFDKHEWPFSMCKEGGKGSAVEYVPCADNQGAGKWLGEKLSQYADGTRILYVVD\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (430 of 500)\n",
            "Sampled sequence:\n",
            "ASYDAVLYFPTQKYPESGKHIQTAISQGKSSICTLDRAGEKKRRLESLQGIPNKAGYERHEWPFSQCTEGGKGSSVRYVPSDDNQGAGKWLGEKLSKYQDGTKILFVVR\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (431 of 500)\n",
            "Sampled sequence:\n",
            "DGHDVELIFPASQFPKSGAVIKEAISLGKANVCTLARKEESERKRQSLHGIPEKTGFDRHEWPFASCEEGGKGSRVEYVDFRDNQGMGDWLKRKLSGFSDGTKILFRVE\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (432 of 500)\n",
            "Sampled sequence:\n",
            "SDYDAVETFPSSTYPQAGSHVKAAISEGQAAVCTLSRDSADTRRNQALVGIPSKSGYGRDEWPMASCVEGGAGSAVRYVDAADQNGAGAWLRTTLSGYQDGTKIKIVTN\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (433 of 500)\n",
            "Sampled sequence:\n",
            "MVYDAVFSFPADEFPEAGAMIRQAIEEGYSAICTLKRDAKSARLDESLAGIPAKEGFNRHEWPHAQCEEGGAGSAVRYVNSDDNSGAGHWLSEYLKQYEDGTRILFIVA\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (434 of 500)\n",
            "Sampled sequence:\n",
            "VDYDAILNTPCDTYPQACNHIKRAISEGHAEVCTLDRAGAAARREQSLKGIPAKPGFDRHEWPFAICVEGGKGASVAYVEQSDNRGMGAWLRTTLSKYADGTRILLIVG\n",
            "Sequence recovery: 0.6055045871559633\n",
            "\n",
            "Sampling.. (435 of 500)\n",
            "Sampled sequence:\n",
            "GDYDAELDLPCEAFPEACQSIREAIADGHAAVCTLQRAEAGKRRTESLRGIPSREGYDRFEWPHSYCVEGGRGARVDYVRSVDNQGFGKWLQKTLGAFADGTRIRFTVR\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (436 of 500)\n",
            "Sampled sequence:\n",
            "MQYDVMVHLPCQRYPFYCDLVRSAIADGHSEICTLDRAGEAERRNQSLAGIPTREGYDRHEWPFALCKEGGQGSRVEYVPWQDNRGAGGWVRTYLAQYADGTRILFIVR\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (437 of 500)\n",
            "Sampled sequence:\n",
            "GDWDATYDFPCEKYPESCEMVRKAIDDGHSRVCTIERSGAKTRRRQSLKGIPTRPGYMRAEWPFAFCKEGGKGSMVKYVPASDNSGAGRWFQNKLSQYADGTKILEVVV\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (438 of 500)\n",
            "Sampled sequence:\n",
            "HDHDVVLQFPQAEFPQAGETIAKCIELGFASVCTLERAGQQERFEQSLKGIPKKDGYDRHEWPFALCKEGGKGACTRYVPSADNRGMGRWLNKTLGRFADGTKVLFQTR\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (439 of 500)\n",
            "Sampled sequence:\n",
            "GDFDDSLVVPVAEFPELGEHIREAIARGHAAVCTLDRAGSRERSRQALEGIPNKPGYVREEWPFAACREGGEGASVRYVPEDDGSGANEWLRRWLDAYDDGTKILFVVE\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (440 of 500)\n",
            "Sampled sequence:\n",
            "MNYDTVLRLPIKTYPESGAHIKKAIANGHAAVCTLCREQREQRLDEALAGIPQREGFTRDEWPAAWCVEGGLGSAVSYVDQYDSEGYGMWLSNTLAQYADGTRVLFVIN\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (441 of 500)\n",
            "Sampled sequence:\n",
            "GLYDAVLYYPKQAFPQAGAMIAAAIAAGHAAVCTLKRAGANVRRTQSLEGIASKVGYDRHEWPFSICQEGGQGSSVQYVPSRDNRGAGTWLANTLSRYKDGTKIKFVIN\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (442 of 500)\n",
            "Sampled sequence:\n",
            "AHYDTTLTFPCKAYPQSCAHIKNAIKKGHADVCTLQRSGWGARRRQSLKGIPKKRGYDRHEWPMSICREGGKGSEVQYVGIDDNKGAGAWWMKETSRYADGTRIKFVVG\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (443 of 500)\n",
            "Sampled sequence:\n",
            "ADYDAVLQLPCADFPQACRMISHAVRQGQSDVCTLERGAAEARRDQSLKGIPTRPGYTRHEWPFDLCREGGAGSWVEYVDEADGSGLAAWLDQTLSAFDSGTRIKMIVK\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (444 of 500)\n",
            "Sampled sequence:\n",
            "GAYDATLKLPVFAYPQSGAHIRQAILLGWPKVCTLDRAKRQQRRQQSLAGIAEKPGYVRDEWPMAICKEGGKGSAVQYVPEADSDGFGHWLFSTLAQYADGTKILFTID\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (445 of 500)\n",
            "Sampled sequence:\n",
            "PSYDKIIILPGDKYPEAAAVIKAAISEGHPKVCTLAREGQLARLQESLAGIPSKPGFDKHEWPFAGCVEGGKGARVSYVSSADNNGFGTWLKNTLSIFADGTRILFKVG\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (446 of 500)\n",
            "Sampled sequence:\n",
            "GKYDVRMKFPRNEFPEAGKMVKTAIKLGHAAVCTLDRSSHTARRNQALAGIPSKPGYQRHEWPFALCKEGGQGAKVEYVPAKDSQGFGKWLARELAKYKNGTKVKFTVE\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (447 of 500)\n",
            "Sampled sequence:\n",
            "TVYDAIENFPQSKYPQAGAQVKAAIELGHASVCTLDRAQSRSRRQASLAGIPGKPGYDRHEWPMAMCKQGGTGSNVQYVPVADNRGAGSWLYNKLAQYADGTKILVVVR\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (448 of 500)\n",
            "Sampled sequence:\n",
            "GDYDVKLIYPIAQYPESGAMIAKAISEGRAAVCTLTRDSADARRDQALKGIPSKVGFMRDEWPFALCKEGGKGSAVKYVKASDGQGAGTWLNQTLSRFKDGTKVLFEVK\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (449 of 500)\n",
            "Sampled sequence:\n",
            "MTFDVEWDFPGEKFPQSAAAIKAAIAEGHAAVCTLERSGENVRREQSLAGIPLKEGYDRHEWPMAKCKQGGKGSRVEYVKAADHQGAGTWVANHLNQYADGTKVLFKVG\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (450 of 500)\n",
            "Sampled sequence:\n",
            "GLYDADLRFPHDAFPEAGAHIAEAISQGQASVCTIDRAGHGERLTQSLAGIPGRSGYQRAEWPFAMCSEGGAGSAVQYVLAADNSGMGLWIGTTLSRYADGTRIKMVTE\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (451 of 500)\n",
            "Sampled sequence:\n",
            "SDFDRVFQYPAEEYPEAGAMIRAAIRKGYADVATLRREGSKKRQDESLAGIPDKRGFVRAEWPFAWAEEGGKGSAVQYVRWADGDGAGKYMSRRLSRYADGTKILFVVT\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (452 of 500)\n",
            "Sampled sequence:\n",
            "PTWDVTLTLPVATYPEAGAMIRKAIALGHPAVCTLERAQADQRRRQSLRGIPPKSGYVRHEWPFALCKEGGQGAAVAYVPAADSAGFGAWLSNVLNRYANGTRIKFVVE\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (453 of 500)\n",
            "Sampled sequence:\n",
            "GAFDRVLQFPGAEFPESAAHVAAAIAGGHAAVCTLERAGSRKRRAQALAGIPARKGYNRDEWPFAICREGGAGSAVRYVRNIDNIAAGGWLKGWLAAYPDGTRIKMVVG\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (454 of 500)\n",
            "Sampled sequence:\n",
            "TDYDVTFDFPADTYPQSGAMVKAAIEEGKAAIATLDRSGADARRAQSLQGIPKKDGYGRFEWPMAMAEEGGQGSKVVYVPFQDNTGAGTWLKNTLGRFKDGTKVKFVVS\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (455 of 500)\n",
            "Sampled sequence:\n",
            "ARYDTVMTFPASQYPQSAAAIKQAIAEGMSSVWTLNRGSETKRRVQSLAGIAAKTGYDRFEWPAAYCEEGGAGSLVRYVPSSDNQGAGRWLNNTLSRYADGTRILWVVG\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (456 of 500)\n",
            "Sampled sequence:\n",
            "MEYDAEMQYPAERFPESGAHIKKAIAEGHSKVCTLARAGANQRRNEALAGIPAQPGFDRHHWPFAICVEGGHGSDVRYVPHDDNRGAGAWMAGRLSSFADGTRIRFVVK\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (457 of 500)\n",
            "Sampled sequence:\n",
            "ATYDAVWYVPTSTYPHAGAVIKAAIAEGHADVCTLSRADADKRRVEALAGIPAKSGYERHEWPFALCQEGGKGSKVVYVPTHDHRGFSSWANRALSGYADGKRIRVVTG\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (458 of 500)\n",
            "Sampled sequence:\n",
            "MSYDVELNLPCATFPQACAQVKQAIAEGQAAICTLDRSGLEKRNAESLAGIPKRDGYSRHEWPFALCTEGGKGSNVVYVDINDNQGFGKWLWDTLSQYEDGTRIKFVAD\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (459 of 500)\n",
            "Sampled sequence:\n",
            "MEYDTTLTFPADKYPEAGAVIATAISEGKAAVCTLNRAGFRERRRESLAGVPSKPGYERHEWPFASCEEGGRGSKVEYVKSDDNRGAGTWLSDYLSGYDDGTRVLQVVS\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (460 of 500)\n",
            "Sampled sequence:\n",
            "VVYDAELVFPSAKYPQAAAHIKAAISEGKAAVCTLNRAAQKERRKQSLAGIPTKAGYDRHEWPFSICSEGGMGASVKYVPLQDNQGFGRYMANRLSRYSDGTKILFKVG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (461 of 500)\n",
            "Sampled sequence:\n",
            "MRYDAVLDIPGNLFPHAAHHWKIAIEQGKSAICTLDRARQYHRRKQALAGIPRRFGYIRDEWPAAECKEGGWGSAVRYVPAADGKGLGSWRKNWASRYSDGTRLLFIVH\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (462 of 500)\n",
            "Sampled sequence:\n",
            "GMYDTVVYLPESTYPESGAVIKAAISEGHAAVCTLDRAGRRERQNQSLKGIPGKVGFERHEWPMSSCVEGGQGSRVEYVKISDNQGMGSWLKQYLSQYADGTRVLYIVK\n",
            "Sequence recovery: 0.5688073394495413\n",
            "\n",
            "Sampling.. (463 of 500)\n",
            "Sampled sequence:\n",
            "TGYDQVMILPRATYPASGAHIAKAIADGHAAVCTLARGGRRKREDEALRGIPARPGYGRDEWPMAVCREGGTGAAVRYVPLADNRGFGRWLASQRSRYADGTKILYLVR\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (464 of 500)\n",
            "Sampled sequence:\n",
            "MQHDAELNLPAEQFPASAAHIKKAIEEGHAAICTLDRDGARQRKRESLKGVPHRPGYDRHEWPFSMCLEGGAGSSVKYVPSKDNQGFGKWLARWLSAYDDGTRIRHIVS\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (465 of 500)\n",
            "Sampled sequence:\n",
            "GAFDAILIFPATQYPASAKTIMKAIANGYAAVCTLDRAGRKSRSIASLAGIPQRPGFDRHEWPASYCVEGGSGSAVEYVPSADNNGAGSWTSSKLSAFADGSKILFTAS\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (466 of 500)\n",
            "Sampled sequence:\n",
            "MDYDWTLEVPCQQFPESCAHIREAIAAGHAEVCTLDRKRQRVRRRQALRGIPHRPGYTREEWPFAYCSEGGRGSAVRYVETGDSTGMGGWLRRRLNQHADGTRILFRVR\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (467 of 500)\n",
            "Sampled sequence:\n",
            "MSYDVELRFPKDKFPAAGEMIRVAIANGHANVCTLDRAAANARYDQSLAGIPAREGFWRDEWPFATCLEGGTGSAVRYVPGDDNSGFGLWLNNHLSRYADGTRVLFIVE\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (468 of 500)\n",
            "Sampled sequence:\n",
            "GEYDFEVHLPIHTFPQSGEQIRAAISQGHAAVCTLNRAGQKKREKESLRGIPDREGYERHHWPFSCCVEGGKGATVVYVDRKDHRGFARWWGRKSNAFSDGTRIKVILQ\n",
            "Sequence recovery: 0.46788990825688076\n",
            "\n",
            "Sampling.. (469 of 500)\n",
            "Sampled sequence:\n",
            "CSFDAELNFPCDEFPESGAHIKKAIEEGYSSVCTLDRAGAGERRKASLFGIADKQGFRRGEWPFALCEEGGKGSRVNYVPADDNIGAGRYLGEYLDRYEDGTKIKFVCP\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (470 of 500)\n",
            "Sampled sequence:\n",
            "SNYDATLNYPCEQYPEACQMIRDAIKEGYSDVCTLYRDGADVRREESLKGIPEKPGYHRHEWPMAICMEGGKGSRVEYVPFADQDGAGRWLSNHLAKYADGTRIKFVVE\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (471 of 500)\n",
            "Sampled sequence:\n",
            "MSYDARLLFPADRYPESGQHIRLAIARGHAAVCTLARAGARERRDQALAGIPERTGFHRFQWPMAICEEGGAGAAVRYVPRSDAEGFGAWLAGHLSAYADGTRILFVVA\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (472 of 500)\n",
            "Sampled sequence:\n",
            "GHWDAEWQVPCARYPEACELIRRAIADGHASVCTLDRASEHERLRQSLAGIPERPGYDRHEWPLAICREGGLGARVEYVPRQDHQGFHKWLARTLARYRDGTRVLVRVR\n",
            "Sequence recovery: 0.45871559633027525\n",
            "\n",
            "Sampling.. (473 of 500)\n",
            "Sampled sequence:\n",
            "AEYDAVLLFPTALYPEAGAVVERAISEGHAAVCTLERSSSAERRDQSLAGIPGKPGYDRHEWPFDFCKEGGQGSLVEYVKTSDNSGFGSWLKGTLSKYADGTRILFVVE\n",
            "Sequence recovery: 0.5871559633027523\n",
            "\n",
            "Sampling.. (474 of 500)\n",
            "Sampled sequence:\n",
            "MTYDARLDFPAERFPEAGAHIAAAIAKGFASICTLDRAGDKERRDQALAGIPSREGYDRDEWPFAWCVEGGKGSSVRYVPHRDNQGAGTWLSNKLSGYRDGTRILLTVN\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (475 of 500)\n",
            "Sampled sequence:\n",
            "MEYDAILHLPCDKYPNTCALIKRAIEQGKADVCTLDRKGQRERRDESLKGIPPRPGFERHEWPASYCKEGGKGSEVEYVPIRDNQGAGRWAGKRLSRYKDGTRIKFVVS\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (476 of 500)\n",
            "Sampled sequence:\n",
            "MSYDAELFFPCARFPQAGAMIKDAIARGHADVCTISRADSNLRKQESLQGIPKRPGYVRFEWPAAICVEGGRGSAVRYVKQDDAEGAGRWLASRLSSFADGTKILFVVS\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (477 of 500)\n",
            "Sampled sequence:\n",
            "GTYDAELSYPGAAFPASAAHVREACAGGKAKVCTLERAGAAARRDQALRGIQKKPGYHRDEWPMASCTEGGEGSAVRYVPTEDNRGAGRWLAAELGQYADGTRILFKIG\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (478 of 500)\n",
            "Sampled sequence:\n",
            "ADFDATLHFPCDTYPASGTLIRAAIAEGHASVCTRDRAGRSKRHTQALDGIPERPGYERHEWPMAMCREGGRGAKVVYVPADDNQGFGRWLRTTLAAFADGTKIKFVVN\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (479 of 500)\n",
            "Sampled sequence:\n",
            "KEYDAEVVYPCEEYPESCAMIREAISEGFSSVCTLERASRSRRRDQSLEGIPNRVGYGRHEWPLSTCVEGGEGSRVRYVPVNDHNGAGEWLGNALSGYADGTKILFKVE\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (480 of 500)\n",
            "Sampled sequence:\n",
            "TLYDVVEIFPARQYPESGAQIRQAIAEGYSSVCTIDRASANTRRREALSGIPTKQGFNRHEWPMAMCEEGGKGAQVRYVPQADHQGAGAWLRRRFSAYADGTKVLFIVE\n",
            "Sequence recovery: 0.6055045871559633\n",
            "\n",
            "Sampling.. (481 of 500)\n",
            "Sampled sequence:\n",
            "GPYDAKLALPLQLFPEAGAHIKAAIAAGFSSVCTLDRAGAKARRKESLSGIPKKPGYTRHEWPMAICREGGRGSSVTYVDSTDDHGYGAWLGDKLKKYADGTKILLSGG\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (482 of 500)\n",
            "Sampled sequence:\n",
            "TAFDVTVDLPVEKYPAAGAHIAAAIAEGHDAVCTLDRAGQGARRLEALAGIPAKDGYDRHEWPFAWCVQGGTGAAVRYVPENDNRGFGRWLEEELGAYADGTRILFRVR\n",
            "Sequence recovery: 0.5321100917431193\n",
            "\n",
            "Sampling.. (483 of 500)\n",
            "Sampled sequence:\n",
            "KSYDVVLSFPIEQYPETGAMIRAAIANGFAAVCTIDRAGAGARRAEALLGIPTKRGFERHEWPFALCREGGLGSAVQYVPKKDNNGAGRWLSNYLSRYNEGTRVLFCVN\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (484 of 500)\n",
            "Sampled sequence:\n",
            "HMHDATLTVPGEAFPAAAAHIAAAIASGHAAVCTLQRTTAATRRDQSLTGIPAKDGYDRHEWPFSICTEGGAGAAVTYVPTDDNRGFGAWLNAYLSAYADGTRILFVVG\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (485 of 500)\n",
            "Sampled sequence:\n",
            "TTYDVKVIFPYKQYPESGAHVKKAIQQGFAAVCTLERSGQQKRRKQSLKGIPKKPGYDRHEWPFANCKEGGKGAAVQYVKSADNRGAGSWLSKKLSQYSDGTKVLFNIK\n",
            "Sequence recovery: 0.5963302752293578\n",
            "\n",
            "Sampling.. (486 of 500)\n",
            "Sampled sequence:\n",
            "MEFDWELVFPGETYPESCSHIAQAIRQGHASVCTLWRAGWRERRRESLEGIPEREGFERFEWPFALCREGGIGAAVKYVGKEDMLGFGQWLERTLSGYPDGTRILFKIE\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (487 of 500)\n",
            "Sampled sequence:\n",
            "VTYDYVLYLPAAQYPEASAQVKAAIAEGHAAVCTLDRWGHTRRKQASLRGIPGRPGYDREEWPFSWCKEGGAGSSVRYVPYQDNRGFGSWLAFTLSNYANGTRILYIIE\n",
            "Sequence recovery: 0.5596330275229358\n",
            "\n",
            "Sampling.. (488 of 500)\n",
            "Sampled sequence:\n",
            "DNFDVVEVYPSLVYPDSGSHISLAIAKGYSDVCTLERSGSQRRRDQSLEGIPGRPGFDRHHWPSSVCVQGGQGSAVRWVPTQDNSGFGSWLRGHLAGFADGTRVLVVVP\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (489 of 500)\n",
            "Sampled sequence:\n",
            "FEYDAKLSLPCTAFPKYCAEIKAAIAQGKPNVCTLNRTAQSRRRNQALAGIPDKSGYDKHEWPFALCVEGGTGSEVRYVPSADNQGAGKWLKNTLSKYADGTKILFTIN\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (490 of 500)\n",
            "Sampled sequence:\n",
            "GAFDTQLYFPCATYPDACAQIKDAIAGGHEAVCTLDRAGSRTRRREALRGIPNRPGYARFEWPQSICREGGRGSDVRYVPRDDHNGAARWLGNRLSAYSDGTKVLFVVD\n",
            "Sequence recovery: 0.5137614678899083\n",
            "\n",
            "Sampling.. (491 of 500)\n",
            "Sampled sequence:\n",
            "AEYDVNLAFPCSEYPEACSQIRTAIQQGHADVCTLERSAAATRRDQALAGIPSRPGYDREEWPFAYCTEGGTGSYVEYVLTSDNQGAGRWLGSTLAGYGDGTRVLFTLD\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (492 of 500)\n",
            "Sampled sequence:\n",
            "ADYDAIWNVPVATYPQAGALIKLAIQQGAAAVCTLDRAGQDARRNASLAGIPTKPGYRRQEWPMAVCVEGGTGAAVQYVDSNDNTGFGSWLGNTLSSYADGTRIKFVTS\n",
            "Sequence recovery: 0.5779816513761468\n",
            "\n",
            "Sampling.. (493 of 500)\n",
            "Sampled sequence:\n",
            "FKYDATFDFPVAQYPQAGELIRTAISEGKASVCTLERAGSNKRSDESLAGIPSKAGYDRHEWPMAICTEGGQGARVRYVAHADNRGAGTWFDKELSGYEDGTKIRLTVS\n",
            "Sequence recovery: 0.5504587155963303\n",
            "\n",
            "Sampling.. (494 of 500)\n",
            "Sampled sequence:\n",
            "DYYDVVVYFPARAFPESGALVKRSIREGFADVCTLDRAGRKRRRTQSLWGIPAKDGYTRHEWPFSDCEQGGTGSRVEYVRWRDADGARRWLRSKLAAYDDGTRVKFQVP\n",
            "Sequence recovery: 0.48623853211009177\n",
            "\n",
            "Sampling.. (495 of 500)\n",
            "Sampled sequence:\n",
            "ASYDATLEFPGAEFPQAGNQIRTAIERGYAAVCTLERRSENRRRNESLQGIPKRPGFIREEWPFAFCEEGGRGAAVQYVEATDGEGAAEWLNRRLSRYQDGTRILFEVD\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (496 of 500)\n",
            "Sampled sequence:\n",
            "EDYDATLVFPVAQYPHSGRLIAKAIHQGKASVCTLARAERNKRREESLAGIPGKQGFVRHEWPHSRCVEGGEGAAVRYVPADDGRGASSWMAAALAAYADGTRIKFVVE\n",
            "Sequence recovery: 0.5412844036697247\n",
            "\n",
            "Sampling.. (497 of 500)\n",
            "Sampled sequence:\n",
            "IDYDVELEFPCEKYPHACAVIKKAIAEGHASVCTLDRIGEGSRRTQSLANIPSRKGYGRHEWPMASCVEGGKGASVAYVPGTDLQGFASWLGNYLNGFKDGTKVKMTIK\n",
            "Sequence recovery: 0.5045871559633027\n",
            "\n",
            "Sampling.. (498 of 500)\n",
            "Sampled sequence:\n",
            "SDYDVTVHFPDQQFPQSGAMVRQAVQSGHASVCTLDRATQGARRQQSLAGIPERSGFDRHEWPFSLCVEGGSGAVVRYVGSADHRGFTAWLAQKLSSYADGTRVLYVVD\n",
            "Sequence recovery: 0.4954128440366973\n",
            "\n",
            "Sampling.. (499 of 500)\n",
            "Sampled sequence:\n",
            "MSYDRTLDLPCSTYPHTCSQIERAIAQGHPAVCTLQRSAAERRRRQSLRGIPKRSGYGRFEWPMAICQEGGAGSAVQYVPLDDLRGFNRWLQLTLAGYADGTRIRFRVQ\n",
            "Sequence recovery: 0.5229357798165137\n",
            "\n",
            "Sampling.. (500 of 500)\n",
            "Sampled sequence:\n",
            "ADYDAELDFPVAEFPEAGKMISKAIREGHSEVCTLQRAGATKRRNESLAGIPAKEGFVRHEWPMSMCEEGGRGAAVTYIGEADGQGAALWIGNRLDKYADGTKIKFNIN\n",
            "Sequence recovery: 0.5779816513761468\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### Score sequences"
      ],
      "metadata": {
        "id": "l0GYuvnISO3h"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "The output values are the average log-likelihoods averaged over all amino acids in a sequence."
      ],
      "metadata": {
        "id": "0BijzGWJnEfz"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "fname_sampled_seqs_score = f'output/{PDBID}_{chain}_ESMIF1_{temp}_{nsamples}_score.csv'\n",
        "fname_sampled_seqs_score"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 35
        },
        "id": "0hawkT1boR_m",
        "outputId": "2f50725b-faa2-43f4-d832-6b44971e43a7"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'output/5OMT_A_ESMIF1_1_500_score.csv'"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            }
          },
          "metadata": {},
          "execution_count": 21
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python scripts/score_log_likelihoods.py data/5OMT.pdb \\\n",
        "    data/5OMT_A_ESMIF1_1_500.fasta --chain A \\\n",
        "    --outpath output/5OMT_A_ESMIF1_1_500_score.csv"
      ],
      "metadata": {
        "id": "k12uzd1hY0oA",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9827ccb3-fb85-4e81-f2ba-9c3f94fdb22e"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/esm/pretrained.py:215: UserWarning: Regression weights not found, predicting contacts will not produce correct results.\n",
            "  warnings.warn(\n",
            "Transferred model to GPU\n",
            "Native sequence loaded from structure file:\n",
            "ARYDDVLYFPASRYPETGAHISDAIKAGHADVCTIERSGADKRRQESLKGIPTKPGFDRDEWPMAMCEEGGKGASVRYVSSSDNRGAGSWVGNRLNGYADGTRILFIVQ\n",
            "\n",
            "\n",
            "Native sequence\n",
            "Log likelihood: -0.99\n",
            "Perplexity: 2.70\n",
            "\n",
            "Scoring variant sequences from sequence file..\n",
            "\n",
            "100% 500/500 [00:26<00:00, 18.95it/s]\n",
            "Results saved to output/5OMT_A_ESMIF1_1_500_score.csv\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd\n",
        "sampled_seqs_df = pd.read_csv(fname_sampled_seqs_score)\n",
        "sampled_seqs_df.head()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 206
        },
        "id": "JiiRSMTWXzTb",
        "outputId": "9895171a-e2fb-44b5-a2d6-ce85ce88c1f8"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "           seqid  log_likelihood\n",
              "0  sampled_seq_1       -1.067197\n",
              "1  sampled_seq_2       -1.029135\n",
              "2  sampled_seq_3       -1.038908\n",
              "3  sampled_seq_4       -1.094937\n",
              "4  sampled_seq_5       -0.958626"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-0371b5ac-0b72-4194-883e-f808a2511eb8\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>seqid</th>\n",
              "      <th>log_likelihood</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>sampled_seq_1</td>\n",
              "      <td>-1.067197</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>sampled_seq_2</td>\n",
              "      <td>-1.029135</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>sampled_seq_3</td>\n",
              "      <td>-1.038908</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>sampled_seq_4</td>\n",
              "      <td>-1.094937</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>sampled_seq_5</td>\n",
              "      <td>-0.958626</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-0371b5ac-0b72-4194-883e-f808a2511eb8')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-0371b5ac-0b72-4194-883e-f808a2511eb8 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-0371b5ac-0b72-4194-883e-f808a2511eb8');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-c9c04acc-c049-4d72-b5ac-04b2eaf499ff\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-c9c04acc-c049-4d72-b5ac-04b2eaf499ff')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-c9c04acc-c049-4d72-b5ac-04b2eaf499ff button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "sampled_seqs_df",
              "summary": "{\n  \"name\": \"sampled_seqs_df\",\n  \"rows\": 500,\n  \"fields\": [\n    {\n      \"column\": \"seqid\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 500,\n        \"samples\": [\n          \"sampled_seq_362\",\n          \"sampled_seq_74\",\n          \"sampled_seq_375\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"log_likelihood\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0.10502884715848494,\n        \"min\": -1.4153408785478785,\n        \"max\": -0.7746016476132455,\n        \"num_unique_values\": 500,\n        \"samples\": [\n          -1.3204783868352208,\n          -0.9781881209907182,\n          -1.1459114004712585\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 24
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "sns.histplot(sampled_seqs_df.log_likelihood, kde=True)\n",
        "plt.xlabel('log likelihood of sequences')\n",
        "plt.ylabel('Num. sequences');"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 460
        },
        "id": "mzNNVrAZYmkH",
        "outputId": "d5579c84-5341-4997-85ed-d52f440c2218"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 640x480 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAj8AAAG7CAYAAAAsZfRkAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAB71ElEQVR4nO3dd3xT5f4H8E+SNk3TvfeiQFllj7KnooyKgDJkq3ARAS844F5BRFHByxXFAQqXC6IoKIi4AAXKLpsyyu5uKd07SZuc3x/c5mdtCm2a9iTN5/165aU858nJJ6dt+u05z3keiSAIAoiIiIishFTsAERERESNicUPERERWRUWP0RERGRVWPwQERGRVWHxQ0RERFaFxQ8RERFZFRY/REREZFVsxA5gjrKzs7F3716EhobC3t5e7DhERERUC2VlZUhMTMTQoUPh6elZYz8WPwbs3bsXkyZNEjsGERERGWHr1q145plnatzO4seA0NBQAPcPXuvWrcUNQ0RERLUSHx+PSZMm6X+P14TFjwGVl7pat26Nzp07i5yGiIiI6uJhQ1Y44JmIiIisCosfIiIisiosfoiIiMiqsPghIiIiq2J2xc+yZcsgkUge+EhLS9P3r6iowMqVKxEREQE7Ozv4+/tj9uzZyMnJEfFdEBERkbkyu7u9Ro8ejebNm1drT0pKwuuvv47OnTsjICBA3z59+nRs3boVI0aMwMsvv4yEhASsWbMGR48excmTJ+Hg4NCY8YmIiMjMmV3x0759e7Rv375a+5IlSwAAM2fO1LcdOHAAW7duRXR0NHbv3q1v79KlC8aOHYvVq1dj6dKlDR+aiIiILIbZXfYyRKvVYtOmTXBwcMDEiRP17Vu2bAEALFiwoEr/MWPGIDQ0VL+diIiIqJJFFD+//vor0tLSMG7cODg5OenbY2NjIZVKERUVVe05PXv2xO3bt5Gbm9uYUYmIiMjMmd1lL0O++OILAFUveQFAamoqPD09YWdnV+05gYGB+j7u7u4G95uRkYGMjIxq7fHx8fWNTERERGbK7IufjIwM/Pzzz4iMjESPHj2qbCstLYWbm5vB5ykUCn2fmqxfvx5vvvmm6cISERGR2TP74mfTpk3QarV4/vnnq21TKpVQq9UGn6dSqfR9ajJr1ixER0dXa69cGI2IiIiaHrMufgRBwMaNG2Fvb4/JkydX2x4YGIgbN25ArVZXu/SVmpqq71MTPz8/+Pn5mTY0ERERmTWzHvD8xx9/4M6dOxg7dixcXV2rbe/evTt0Oh1iY2OrbTtx4gTCw8NrHO9DRERE1smsz/xs2LABAAxe8gKAyZMnY8uWLVi9ejX69eunb9+5cycSExOxbNmyxohJRGQ0lUoFjUbTYPuXy+X6MZBEdJ/ZFj/Z2dnYtWsXWrVqhb59+xrsM2TIEEyYMAHbtm3DyJEj8cQTTyAhIQEffPAB2rRpg4ULFzZyaiKi2lOpVAgODUVWZmaDvYaXjw+SExNZABH9idkWP1u2bIFGo6nxrE+lzZs3IzIyEps2bcKcOXPg7u6OyZMnY8WKFXB0dGyktEREdafRaJCVmYmlX8VAoTT955WqtBjLn+kPjUbD4ofoT8y2+FmwYEG1mZsNsbW1xeLFi7F48eJGSEVEZHoKpSMUDvxjjaixmPWAZyIiIiJTY/FDREREVoXFDxEREVkVFj9ERERkVVj8EBERkVVh8UNERERWhcUPERERWRUWP0RERGRVWPwQERGRVWHxQ0RERFaFxQ8RERFZFRY/REREZFVY/BAREZFVYfFDREREVsVG7ABERJZOXaHFrXvFuJNVgsxCFRS2MjgpbNDazxktvB0hkUjEjkhEf8Lih4jISGcSc/Hd2VT8cikDhaoKg33cHeQY0tob03qFoY2/cyMnJCJDWPwQEdXRrXvFePeXePxx7R4AoLWfM6a29kZzb0f4u9pDU6FDXqkGF5Lzcfx2DrafScX2M6no28ITb4xsg+beTiK/AyLrxuKHiKiWBEHA54fvYNXe69DqBIxo74d5g1ugpY/hYmZEe38AQFxqPjYeTcCPF9Px+IdH8Lf+4Zg7qEVjRieiP2HxQ0RUCwVl5Xh5x0Xsv5qJ5t6OWDW2PToHu9Xque0DXfHh+E6Y2isU/9h5CWsP3MLJOzlYFd2ygVMTkSG824uI6CFySzSY8PlJ7L+aiSc6+mP3nN61Lnz+rHOwG/bM7YNn+4ThdGIexm06D1vvZg2QmIgehMUPEdEDZBWpMeHzk7iaUYiFj7TEmnEd4WBn/ElzW5kUS0a0wb+f7oD80nL4THgH94o1JkxMRA/Dy15EZPFUKhU0GtMXEMXqCkzdchE3s0qxYFAYpnbzQVFRkUn2PaS5M1aOCMP8767i5/hcjFLYw8/F3iT7JqIHY/FDRBZNpVIhODQUWZmZpt2xRAqvMUugDO+GvJjNmL9yB+ab9hUAAHbBkQh85l3svpCOp7sGwd1B3gCvQkR/xuKHiCyaRqNBVmYmln4VA4XS0WT7PZZYgMt3S1Ectx+zZz4Hl9cWmWzflQqyM7Hy+WEY2MwJv98uwu4LaRjXLQhKOT+aiRoSf8KIqElQKB2hcDBN8XMjswiX75bCx0GGpL2fQDFtnMn2/Weq0mIAQICzLQZFeOOPa/ew52IGxnQOgI2MQzKJGgp/uoiI/qSwrBx/XLsHe1sZ+gcrAJ3hmZtNrV2AC7qGuOFuoQqHb2Y3ymsSWSsWP0RE/6MTBOy9cheaCh0eaeMDe9vG/YjsGe6BQDd7XEorwLW7hY362kTWhMUPEdH/nE/OR3qBCh0DXRHm6dDory+VSPBYW18o5TIcuHYPeSW8BZ6oIbD4ISLC/ctdJ+/kwMXeFr2be4iWw8HOBo+380W5VsDeq3eh0wmiZSFqqlj8EJHVEwQBB67fQ4VOwKBW3qIPNg50U6JzsCsyC9U4k5QnahaipojFDxFZvZv3ipGUU4pWvk4IdleKHQcA0LOZBzwc5IhNyEFWkVrsOERNCosfIrJqFVodjtzMhp2NFH1beIodR89GJsWjbXwAAL/HZ/LyF5EJsfghIqt2LiUfxeoK9AhzN7vJBb2dFegU7IZ7RWpcTM0XOw5Rk8Hih4isVom6AmcSc+Fib4v2ga5ixzGoR5g7nBU2OHEnB0WqcrHjEDUJLH6IyGqdvJODcq2APs09IZNKxI5jkK1MioER3ijXCoi5kSV2HKImgcUPEVml/FINrmQUwt9FgXCvxp/Tpy5CPR3Q3NsRt7NKkJRTInYcIovH4oeIrNKpxFwIwv1ZlSUS8zzr82d9/3d26sjNbA5+JqonFj9EZHXySjW4llGEQDd7BLqZx63tD+Nsb4suwW7IKdHgUlqB2HGILJrZFj+FhYV4/fXX0bp1a9jb28Pd3R09evTA1q1bq/QrLS3FokWLEBoaCjs7O4SGhmLx4sUoLS0VKTkRmbvYhFwIAKKaiTeTszG6hrrB0e7+4GdVuVbsOEQWy7zu6/yftLQ0DBw4ENnZ2Zg2bRratm2LkpIS3LhxA0lJSfp+Wq0Ww4YNQ0xMDCZPnox+/frh4sWLeP/993Hq1Cns378fUqnZ1ndEJIK8Ug2u3y1CsLsSAa72YsepE1uZFL3DPbD3aibOJOWhT3PzmZeIyJKYZfEzZcoUFBUV4eLFiwgKCqqx3+bNmxETE4O5c+fio48+0reHhobi5ZdfxtatWzFlypTGiExEFuLs/5aL6B7mLnIS40T4OuFMch4upOSjY5ArHO3M8mOcyKyZ3WmRY8eO4cCBA3jttdcQFBQErVaL4uJig323bNkCAFi4cGGV9hdeeAH29vb67UREAFCsrsC1jCL4uSgs7qxPJYlEgl7hHtDqBJxKyBU7DpFFMrs/GX7++WcAQHh4OMaMGYM9e/agvLwcfn5+eOGFF7B48WLIZDIIgoDTp0/D398fISEhVfZhb2+Pjh074vTp0w98rYyMDGRkZFRrj4+PN90bIiKzcSE5H1pBQNcQN7Gj1EuYhwP8XBS4nF6ATsGucFPKxY5EZFHMrvipLDyeffZZhIWFYcOGDZBIJPj000+xZMkSJCcn4/PPP0dubi5KS0vRrl07g/sJDAzEiRMnUFhYCGdnZ4N91q9fjzfffLPB3gsRmQ9VuRZxafnwcJAjzNO85/V5GIlEgt7NPfHd2VScvJODx9v5iR2JyKKYXfFTVFQEAHBwcMDhw4dhZ2cHABg3bhzatGmDDRs2YOHChVAq79+eWrn9rxQKBYD7d4PVVPzMmjUL0dHR1drj4+MxadKker8XIjIfl9MKUK4V0CXEzSLm9XmYAFd7hHoocSOzGF1CVPB2UogdichimN2YH3v7+9fhJ06cWKWwkcvleOaZZyAIAg4ePKgvftRqtcH9qFQqAND3M8TPzw+dO3eu9mjdurWp3g4RmQGtTsDF1AI4yGVo6eMkdhyT6RV+/26v47dzRE5CZFnMrvipvLvLz6/6adzKttzcXLi7u0OpVCI1NdXgflJTU+Hs7FzjWR8ish63s4pRrK5A+0BXs13DyxheTnaI8HVCUk4pUvM4txlRbZld8RMVFQUASElJqbatss3HxwcSiQRdu3ZFenp6lbl/AKCsrAwXLlxAt27dGj4wEZm9Cyn5kEklaBfQ9P4Y6tnMA1LJ/bM/gsBlL4hqw+yKnyeeeAKurq748ssv9eN/AKC4uBibN2+Gra0tHn30UQDA5MmTAQCrV6+uso/PPvsMZWVl+u1EZL3uFqiQUaBChI8TlHKzG+ZYby72tmjt54yMAhVS88rEjkNkEczuk8DFxQUffvghpk6dim7duuHZZ5+FRCLBf/7zH6SlpWHFihX6S2PTp0/Hli1bsHbtWhQUFOhneP70008xYMAADlomIlxMzQcAdAxyFTVHQ+oW6o6rGYU4lZCLIHfLWKuMSExmV/wA92d49vLywnvvvYc333wTOp0OkZGR2LZtG8aPH6/vJ5PJ8Msvv2D58uX49ttvsW3bNvj5+WHhwoVYunQpZDKZiO+CiMRWqqnAzcxi+Lsq4OVk+M7QpsDF3hatfJ0Qn1GEtLwyBLhZ5gSORI3FLIsfAHj88cfx+OOPP7Sfo6MjVq1ahVWrVjVCKiKyJFfTC6EVBLQPcBU7SoPrFuqOaxlFiE3MwWi3QLHjEJk1sxvzQ0RkCoIg4FJaAextZWju7Sh2nAbnppQjwtcJKbllSM/n2B+iB2HxQ0RNUlJOKQpVFWgX4Nykbm9/kG6h9xdrPZXINb+IHoTFDxE1SXFpBZAAaOfvInaURuPuIEdLH0ck5ZTibqFK7DhEZovFDxE1OcWqCiRmlyDU0wHO9rZix2lU3SvP/nDFd6IasfghoibnakYhBABt/ZvepIYP4+Foh+bejkjILkF2SbnYcYjMEosfImpSBEHA1YxCKOUyhHpY9urtxqo8+3MhvVjkJETmicUPETUpqXllKCgrR2s/6xno/FdeTnYI8VDiTo4KNq7V10kksnYsfoioSbmaUQgAaONnfZe8/qxbiDsEAM49RosdhcjssPghoiZDXa7FzXvF8HNRwN1BLnYcUfm7KuDjaAvHdkNwr0gtdhwis8Lih4iajOuZRdDqBKsc6PxXEokEHQMcIbGxxdbT6WLHITIrLH6IqMm4kl4IW5kELbydxI5iFkJc7aDJSsL2cxkoKOWdX0SVWPwQUZOQU1KOe0VqtPRxgtyGH23A/bM/hSd3oFSjxZYTiWLHITIb/IQgoibhWlYpAOuc2+dBSq4dgb+LHTYdT0SZRit2HCKzwOKHiCyfzBY3s8rgrpTD11khdhrzotNiWo9A5JZo8O3pZLHTEJkFFj9EZPGULaKg1t4f6CyRWOfcPg/yRAcfeDjI8cWRBJRrdWLHIRIdix8isngO7QZBAiDClwOdDbG3lWFGnzCk5Zfhxwu884uIxQ8RWbScYg3swzojyNUODnY2YscxW5OiQuBoZ4N1Mbeh0wlixyESFYsfIrJov1zNgkQqQwtPe7GjmDUXe1s8ExWMm/eK8Xt8pthxiETF4oeILNpPl+9Bpy5FqDsHOj/Ms73DILeR4tNDtyEIPPtD1ovFDxFZrJuZRYi/W4zS60dhY6WLmNaFt7MCY7sE4kJKPk7eyRU7DpFoWPwQkcXaeT4NAFB8+aDISSzHrH7NIJUAnx66JXYUItGw+CEii6TTCdh9Pg1+znZQp1wWO47FCPFwwPD2/jhyMxuX0wrEjkMkChY/RGSRTibkIL1AhWHtvAFw/EpdzO4fDgD47NBtkZMQiYPFDxFZpJ3n7l/yGtnOW+QklqeNvzMGRHjhl8sZuJNVLHYcokbH4oeILE6ZRotfL2UgMsAFzTyVYsexSC8MaA5BANbH3BE7ClGjY/FDRBZn39W7KNFoMbpzgNhRLFb3MHd0DXHDzvOpyCgoEzsOUaNi8UNEFmfX+TTIpBKM7OAvdhSLNmdgc5RrBXxxOEHsKESNisUPEVmUrCI1jtzMRv+WXvB0tBM7jkUbEOGF1n7O2HYqGbklGrHjEDUaFj9EZFF+vJgOrU7Ak514yau+JBIJXhgQjrJyLf57PFHsOESNhsUPEVmUXedT4WRng0fa+IgdpUkYFumHUA8l/nssAcXqCrHjEDUKkxc/eXl5KCkpMfVuiYhwI7MIl9MKMSzSDwpbmdhxmgSZVIJZ/cNRqKrA17FJYschahRGFT9//PEHXn31VeTl5enb7t27h/79+8PT0xPu7u5YsGCByUISEQH/P7fPk7zLy6RGdw6Aj7MdNhxJgKpcK3YcogZnVPGzdu1a7Ny5E25ubvq2l19+GUeOHEF4eDg8PDzw4YcfYvv27SYLSkTWTacTsPtCGgJc7dE91F3sOE2KnY0Mz/dthntFanx/LlXsOEQNzqji5+LFi+jTp4/+32VlZfjuu+/wyCOP4MaNG7h+/TqCgoKwbt06kwUlIut28k4OMgpUGNXJH1Ku4G5yE7oHw1Vpi/Uxd1Ch1Ykdh6hBGVX83Lt3D/7+/z+/RmxsLFQqFaZNmwYAcHJywogRI3D9+nWThCQiqlzB/clOgSInaZoc7GwwrVcoknNL8fOlDLHjEDUoo4ofOzs7lJX9/4ygR44cgUQiQb9+/fRtzs7OyM3NrX9CIrJ6lctZtA90QXNvR7HjNFnTeoVCKZfhs0O3IQhcLJaaLqOKn7CwMBw4cED/7++//x4tWrRAQMD/D0JMSUmBp6dn/RMSkdXTL2fBuX0alKtSjmd6BOPa3SIcuHZP7DhEDcao4mfq1Km4dOkSevTogb59++LSpUuYOHFilT5xcXGIiIgwKpREIqnxcfny5Sp9KyoqsHLlSkRERMDOzg7+/v6YPXs2cnJyjHptIjI/O8+lwYbLWTSK5/o2g1wmxScHb/HsDzVZNsY8afbs2Th58iS+/fZbCIKAkSNH4rXXXtNvv3z5Mi5duoTly5cbHaxv376YOXNmtfagoKAq/54+fTq2bt2KESNG4OWXX0ZCQgLWrFmDo0eP4uTJk3BwcDA6AxGJ716RCkduZmFghDc8uJxFg/NxVmBMl0BsO5WM2IRcRDXzEDsSkckZVfzY2tri66+/xrp16yCRSODk5FRlu6+vL86fP4/Q0FCjgzVr1gyTJk16YJ8DBw5g69atiI6Oxu7du/XtXbp0wdixY7F69WosXbrU6AxEJL4fL6RDJ3Bun8b0t/7N8O3pZHxy8BaLH2qS6jXDs7Ozc7XCBwA8PT3RoUMHuLi41Gf3KC8vR1FRUY3bt2zZAgDVJlQcM2YMQkND9duJyHLtPJcGJ4UNhrTmchaNJcTDAcPb++PIzWxcSi0QOw6RydWr+MnKysK6deswf/58PPfcc1XaT506VeWOsLr67rvvYG9vD2dnZ7i6umLSpElITEys0ic2NhZSqRRRUVHVnt+zZ0/cvn37gXecZWRk4Ny5c9Ue8fHxRucmItOJzyjE1YxCjGjP5Swa2wsDwgEAn8XcEjkJkekZddkLADZu3Ih58+ZBpVJBEARIJBJs2LABAJCZmYmePXvi888/x7PPPlvnfXft2hVjxoxBy5YtoVarcfToUXzxxRf49ddfcezYMbRq1QoAkJqaCk9PT9jZVR8HEBgYqO/j7m54Ntj169fjzTffrHM+Imocu/43t8/ozpzbp7G19nPGoFbe+PXyXdy6V8wpBqhJMerMz/79+zFz5ky0bNkSu3btwuzZs6tsb9euHdq2bYsffvjBqFCnT5/GokWLMHr0aEyYMAGffPIJdu/ejdzcXLz00kv6fqWlpQYLHwBQKBT6PjWZNWsWzp49W+2xdetWo3ITkelUaHXYdT4NQe726Bri9vAnkMnNGRgOQQDWx9wWOwqRSRl15mflypXw8/NDTEwMnJ2dcf78+Wp92rdvjxMnTtQ7YKXHH38cPXr0wB9//AGVSgWFQgGlUgm1Wm2wv0qlAgAolcoa9+nn5wc/Pz+TZSQi0zl2OwdZRWrMH9wCEgmXsxBDlxB39Ahzx67zaXjpkZYIcLUXOxKRSRh15ufMmTMYMWIEnJ2da+wTGBiIu3fvGh3MkLCwMFRUVOjH8QQGBiI7O9tgAZSamqrvQ0SWZ+f/Ftgczbu8RDVnYHNU6AR8cfiO2FGITMaoMz8ajeah8+fk5+dDJjPtAMUbN27A1tYWHh73b73s3r07rl27htjY2CpLawDAiRMnEB4eXuN4HyIyX0Wqcuy9chddQ9wQ4sG5uuqrsLDQ6Od28JGjja8jtp1KxtRuPvBwkFfZLpfL9cMMiCyFUWd+QkNDcfbs2Qf2iY2NNWqG55pmZt62bRvOnTuHxx57TD/OZ/LkyQCA1atXV+m7c+dOJCYm6rcTkWX59fJdqMp1HOhcT+UaNSCRIigoCC4uLkY9XF1dceizxVBX6NB5/MJq24NDQ/XDDIgshVFnfp544gmsWrUKO3bswFNPPVVt+6ZNmxAXF4cVK1bUed9vv/02jh07hkGDBiE4OBgajQbHjh3D999/Dz8/P6xZs0bfd8iQIZgwYQK2bduGkSNH4oknnkBCQgI++OADtGnTBgsXLjTm7RGRyHaeS4XcRorhkRyTVx/ainJA0GHRpn1wdjV+skJBELD9YhbkfcZh/ty5sLO5/3ezqrQYy5/pD41Gw7M/ZFGMKn5effVVfPPNN5gwYQK+++47FBTcnwTr448/xpEjR7Bz5060aNECc+fOrfO+Bw4ciGvXruGrr75CdnY2BEFAaGgo/v73v+O1116Dt7d3lf6bN29GZGQkNm3ahDlz5sDd3R2TJ0/GihUr4OjIWzOJLE1qXilO3snF8Eg/uChtxY7TJNgpHaFwqN/nYfdmAvZdzcT1vAp0D+VwArJsRhU/bm5uiImJwZQpU7Bjxw59+7x58wDcX5fr66+/NmpdrejoaERHR9e6v62tLRYvXozFixfX+bWIyPz8oJ/bhwOdzUlLHyecuJODC8n56BTkCltZvebIJRKV0ZMcBgcH49ChQ4iLi8OJEyeQk5MDFxcXREVFoUuXLqbMSERWQhAE7DyXBg8HOfq19BI7Dv2JTCpBlxA3HLqehSvphegY5Cp2JCKjGV38VGrfvj3at29viixEZOUupOTjTnYJpvcO5ZkFM9TWzxmnEnJxNikPkQH1W7uRSExGfbqUlZUhOTkZGo3G4Ha1Wo3k5GTeAUBEdbLz3P1LXmN4l5dZspFJ0SnYFcXqCly7a/zt80RiM6r4Wb58OSIiIlBcXGxwe0lJCVq1aoV33nmnXuGIyHqoK7TYE5eOlj6OaOtf8wSqJK7IABfY2UhxJjEPOkEQOw6RUYwqfn799VcMGTKkxgkE3d3dMWTIEPz000/1CkdE1uPgtSzkl5ZjdOdALmdhxuxsZOgQ6Ir8snIk5PLsPlkmo4qfxMREtGzZ8oF9WrZsicTERGN2T0RWaOe5VEgkwKiOvMvL3HUMcoWNVILzaYbP/hOZO6OKn/LyckilD36qRCLhmB8iqpXcEg0OXr+HPs094evCyfLMnb1chsgAF+SUVkDRjHf3kuUxqvhp1qwZYmJiHtjn0KFDCAkJMSoUEVmXPRfTUa4VOLePBekU7AqpBHCJelrsKER1ZlTxEx0djbNnz2LVqlUGt7/33ns4d+4cRo0aVZ9sRGQltp9JgZOdDR5ry+UsLIWTwhYtveyhCGqLs8kFYschqhOj5vl5+eWX8dVXX2Hx4sXYvn07Hn30UQQEBCAtLQ179+7FhQsXEBwcjFdffdXUeYmoibmcVoAr6YWY2CMY9nKZ2HGoDjr6OSL+bjE2HE/BwHZBYschqjWjl7c4dOgQJk6ciJMnT+LcuXOQSCQQ/nfbY69evbB161a4ubmZNCwRNT3fnU0FADzdlb88LY2LvQ1Krx3FMWl/XE4rQDtOfEgWwugZnkNDQ3H8+HGcO3cOJ0+eRH5+PlxdXREVFYXOnTubMiMRNVGqci12nU9DSx9HdAjkL05LVHByBxza9Menh27h02c4+JksQ72Xt+jcuTOLHSIyyu/xmSgoK8eLA5tzbh8LVZ6ViH7N3fHr5bu4da8Yzb3rt3o8UWPg4jlEJJrtZ1JhI5VgVCfe5WXJnu8VBEEA1sXcFjsKUa0YfeanvLwcu3fvxqlTp5CXlwetVlutj0QiwcaNG+sVkIiapvT8Mhy5mYVHWvvAy8lO7DhUDx0CnRHVzB0/nE/DS0NaINBNKXYkogcyqvhJT0/HI488gmvXrukHORvC4oeIarLzXCoEgQOdm4o5A5tj8sZT+OLwHbz5RDux4xA9kFHFz8KFCxEfH48JEybg+eefR1BQEGxs6j18iIishE4nYPuZVHg52WFAhJfYccgE+jT3RGSAC749k4L5Q1rC3UEudiSiGhlVsezbtw/9+vXDV199Zeo8RGQFTiXmIjm3FLP6N4ONjEMPmwKJRIKZ/Zph7rbz+PJEEuYPaSF2JKIaGfWpo1Kp0KNHD1NnISIrsf1MCgDgqS685NWUPN7OF4Fu9th8IhGq8urjQInMhVHFT7t27ZCUlGTqLERkBQrKyvHLpQx0DnblbdFNjI1Miuf6hCG3RIMd/5u8ksgcGVX8vPLKK/jxxx9x9epVU+choiZu94U0qMp1mNA9WOwo1ACe7hYEV6UtNhy5A62u5htiiMRk1Jgfb29vjBw5Er169cL8+fPRpUsXuLq6Guzbr1+/+uQjoiZEEAR8HZsMJ4UNRrT3FzsONQCl3AZTokLw0YFb2HflLh6P5GK1ZH6MKn4GDBigX8vrrbfeeuDMrIbm/yEi63QxtQDX7hZhclQIFzFtwqb0CsX6w3ew7vAdPNbOl7N3k9kxqvhZunQpv5mJqM6+OZUMALzk1cR5OtphTJdAfB2bjFMJuejRzEPsSERVGFX8LFu2zMQxiKipK1ZX4MeL6egQ6II2/s5ix6EG9nzfZth2KhmfH77D4ofMDifYIKJG8eOFdJRqtDzrYyXCPB0wtI0v/rh2Dzczi8SOQ1RFvaZlLi8vxx9//IH4+HgUFxdjyZIlAO7PA1RYWAhPT09IpayviAjYdioZDnIZRnbgQOemprCw0GD7xC4++O3KXaw/eANLh9V90kO5XA6FQlHfeETVGF38/Pbbb3j22Wdx9+5dCIIAiUSiL34uXLiA3r17Y+vWrZgwYYLJwhKRZbqcVoBLaQWY0D0YDnZcCqepKNeoAYkUQUE1T1bpO3k1tper8cGMQdCVGS6SauLl44PkxEQWQGRyRn0KnTlzBqNGjYKnpyc++OADnDp1Ctu2bdNvj4qKQlhYGHbt2sXih4iwTT/QmTM6NyXainJA0GHRpn1wdjU8rudWdhn+uJWP0e//iM4BTrXet6q0GMuf6Q+NRsPih0zOqOLnrbfeglKpxJkzZ+Dr64s333yzWp9u3brh3Llz9Q5IRJatRF2B3RfS0dbfGZEBLmLHoQZgp3SEwsHwbN2t7R0Qm1KMq5ll6NHcFzIp7xQm8Rk1IOfYsWMYNWoUfH19a+wTFBSEjIwMo4MRUdPwU1w6itUVGN89mFNkWCGZVIIOQS4o0WhxgwOfyUwYdeanuLgYnp6eD+xTWloKnU5nVCgialwqlQoajcbk+xUEAf89lgClrQyjOnKgs7Vq5++C2Du5OJ+Sj1a+TiyCSXRGFT8BAQG4cuXKA/tcuHABzZo1MyoUETUelUqF4NBQZGVmmnzfcr+W8Jvyb5RfOwhbDABga/LXIPOnsJWhjb8z4lILkJZfhkA3pdiRyMoZVfw8/vjjWLduHY4ePYo+ffpU2/7rr7/i+PHjWLRoUb0DElHD0mg0yMrMxNKvYqBQmnaV9YO38nEjuwxZx7+HRrOUA1etWMcgV8SlFuB8cj6LHxKdUcXP4sWL8c033+DRRx/F3LlzkZiYCAD4+eefcfjwYXzyySfw8/PDggULTJmViBqQ4gGDVo2hKtfidu5d+DrZIikr0WT7JcvkppQjzNMBd7JLkF+qgatSLnYksmJGX/bat28fnn76abz//vv69ujoaAiCgPDwcOzcufOh44KIqOm6mlEIrU5AGx8HxIodhsxCpyBXJGSX4EJKPgZEeIsdh6yY0bONde7cGdevX8fPP/+MEydOICcnBy4uLoiKisITTzwBGxtOZEZkrQRBQFxqAextZWjmfv9SV02zANdXQ+2XTC/QzR6ejnJczShEz3AP2NnIxI5EVqpeFYpMJkN0dDSio6NNlccgnU6HXr16ITY2FoMHD8bvv/9eZXtpaSmWL1+Ob775BhkZGfDz88OECROwZMkSKJW8tkzU2JJzS1FQVo4uIW7QVWgeOguwKVRoKxp0/1R/EokEHQJd8ce1e7iWUYQOQa5iRyIrZRGnZ9asWVPj3WVarRbDhg1DTEwMJk+ejH79+uHixYt4//33cerUKezfv5/rixE1sktpBQCAyAAXaItzHjoLcH0UZGdi5fPDoNVqTb5vMr0IXyccvZWNi6n5aB/owtveSRRGFT9btmypdd8pU6YY8xJ6d+7cwZIlS/DOO+/gpZdeqrZ98+bNiImJwdy5c/HRRx/p20NDQ/Hyyy9j69at9c5ARLVXpCrHnawShHoo4WJvi/zi++0PmgW4PlSlxSbfJzUcW5kUbf2dcS45H8m5pQjxcBA7Elkho4qfadOmPbRar1zstL6Fx/PPP4+2bdti7ty5BoufykJs4cKFVdpfeOEFLFmyBFu2bGHxQ9SILqcVQgAQGcilLMiw9oGuOJecj7jUAhY/JAqjip9NmzYZbM/Pz8fp06fxzTffYMyYMRg+fHi9wn3xxRc4fPgwzpw5Y/DSlSAIOH36NPz9/RESElJlm729PTp27IjTp0/XuP+MjAyDS3DEx8fXKzeRtdLqBFxOL4CTwgah/KVGNXCxt9Xf9l5QVg4Xe05+SY3LqOJn6tSpD9w+ffp0DB8+HPPmzTMqFACkp6fjlVdewcKFC9GhQweDfXJzc1FaWop27doZ3B4YGIgTJ06gsLAQzs7O1bavX7/e4KKsRGScO1nFKNVo0SvcA1KO5aAH6BDogoTsElxKLUCfFpwWhRpXgwx4Hjx4MB577DEsXboUBw4cMGofs2fPhqenJ954440a+5SWlgIA7OzsDG6vnE22tLTUYPEza9Ysg3eqxcfHY9KkScbEJrJqcWkFkEqAtv7Vf96I/izYXQk3pS0upxegRzN32Mp4Ywo1nga726tly5ZYt26dUc/95ptv8OOPP2L//v2wt7evsV/lbexqtdrgdpVKVaXfX/n5+cHPz8+ojERUVW6JBql5ZWjp4wil3CJuJCURSSQStA90RcyNLFzPLEI7f44Ro8bTYJ9QV69eNeoWRrVajXnz5uHRRx9FaGgobt26VWV7WVkZbt26BScnJ3h7e0OpVCI1NdXgvlJTU+Hs7GzwrA8Rmdal1Pu3t7cPdBU3CFmM1n5OOH47GxdT8tHWz5m3vVOjMel5Rp1Oh6SkJLz++uv49ddf0bdv3zrvo6ysDFlZWdi3bx9atGhR5QEAx48fR4sWLTB//nxIJBJ07doV6enpSEpKqrafCxcuoFu3biZ5b0RUs3KtDlczCuHhIIe/Cxcvpdqxs5GhtZ8zsos1SC9QiR2HrIhRZ36kUukDK3RBEODh4VFl3a/acnBwwI4dOwxue+qppxAZGYmlS5fqZ4udPHkyDh8+jNWrV1eZ5+ezzz5DWVkZJk+eXOcMRFQ31+8WQaPVcdI6qrP2AS6ISy3ApbQCBLjWPMyByJSMKn769etn8ANOKpXCzc0N3bt3x/Tp0+Hl5VXnfdva2mLs2LE1bvf29q6yffr06diyZQvWrl2LgoIC/QzPn376KQYMGMCBy0QNTBAExKUVwFYmQStfXmKmuvFwtEOAqz1uZRajrIUW9nKu90UNz6ji59ChQyaOYTyZTIZffvkFy5cvx7fffott27bBz88PCxcuxNKlSyGT8QeJqCFlFqqRVaRGZIAL5Da8Y4fqLjLABWn5ZbiaUYguIW5ixyErYFG3ZAiCYLDd0dERq1atwqpVqxo5ERHFpeYDANpzRmcyUri3A+xvyHAprQCdg1156ZQaHP9MIyKjlZVrceNeMfxdFPB0NDzfFtHD2EilaOPvjIKyciTnloodh6yAUWd+ZsyYYdSLSSQSbNy40ajnEpH5uZpeCK1O4DpeVG+RAS44m5SHS2lc74sanlHFz3//+1/9aUlDl6IkEkmN7Sx+iJoGQRBwKa0A9rYyNPc2/WrtZF1c7G0R4q7EnewSFKsrLGtMBlkcoy573b59GyNHjoSrqyveeOMNHDx4EPHx8Th48CCWLl0KV1dXPPHEE7h9+zYSEhL0jzt37pg6PxGJJDm3FAVl5Wjr7wwbAwsPE9VVZKALBAG4klYgdhRq4owqrn/44QccOXIE586dq7KaekREBPr374+pU6eiS5cu2L17N1566SVTZSUiMxL3vxmdIwN4yYtMI8zDAY52NricXohIby52Sg3HqD/XPv/8czz99NNVCp8/CwsLw1NPPYXPP/+8XuGIyDwVqcqRkF2CUA8lnO1txY5DTYRUKkFbf2cUqyuQnGd4zUYiUzCq+ElMTISrq+sD+7i5uSExMdGY3RORmbucVggBXMeLTK+dvwskEuDqPd71RQ3HqOLH09MTe/furXG7IAjYu3cvPDw8jA5GROZJqxNwOb0ATgobhHgoxY5DTYyjwgbNPB2Qkq+GjYuP2HGoiTKq+Hnqqadw4cIFPP3000hISKiyLSEhAePGjUNcXBzGjRtnkpBEZD7uZBWjVKNFZIALpJyMjhpA5Tgyxw6PiZyEmiqjBjwvX74cR48exXfffYddu3YhICAAPj4+yMzMRFpaGrRaLbp164Zly5aZOC4RiS0utQBSCdDWn+t4UcMIdlfC2U4GbftHUK7ViR2HmiCjzvw4Ojri6NGjePvttxEaGork5GScPn0aycnJCAsLw4oVK3DkyBE4OnLuD6KmJLdEg9T8MrTwdoJSzplYqGFIJBK09lFC5uCKP67niB2HmiCjJ+eQy+X4xz/+gZs3b6KwsBApKSkoLCzEjRs3sHjxYsjlclPmJCIzULmOF2d0poYW4WUPoaIcO85liB2FmiCTzEzm6OiIgIAAnukhasLKtTrEZxTBw0EOfxeF2HGoibO3laH0+jGcTi7ArXvFYsehJqZexU9WVhbWrVuH+fPn47nnnqvSfurUKZSVldU7IBGZh+t3i6DR6tA+0IWrblOjKLrwKwBg26lkkZNQU2N08bNx40aEhoZizpw5WLt2LTZt2qTflpmZiZ49e+Lrr782SUgiEpcgCIhLK4CtTIJWvhzoTI1DnXoFYR72+P5cKlTlWrHjUBNiVPGzf/9+zJw5Ey1btsSuXbswe/bsKtvbtWuHtm3b4ocffjBFRiISWWahGllFarTydYbchut4UeMZ28kP+aXl+O3yXbGjUBNi1KfYypUr4efnh5iYGERHR8Pb27tan/bt2+Pq1av1DkhE4tMPdOY6XtTIRrbzhtxGiq956YtMyKji58yZMxgxYgScnWs+/R0YGIi7d1mpE1m6snItbtwrhp+LAl5OdmLHISvjqrTFsHa+OJWQi1v3isSOQ02EUcWPRqOBg4PDA/vk5+dDJpMZFYqIzEd8eiG0OgHteXs7iWRC92AAwLZTKSInoabCqOInNDQUZ8+efWCf2NhYREREGBWKiMxD5UBne1sZmntzKgsSR/cwd4R7OXDgM5mMUcXPE088gSNHjmDHjh0Gt2/atAlxcXEYM2ZMvcIRkbiSc0tRUFaONv7OsJFyoDOJQyKRYEL3YA58JpMx6tPs1VdfRXBwMCZMmIBx48bhxIkTAICPP/4Y48aNw8yZM9GiRQvMnTvXpGGJqHHFpRYA4EBnEt+YzoH3Bz7HcuAz1Z9Ri/O4ubkhJiYGU6ZMqXL2Z968eQCAvn374uuvv37ouCAiMl9FqnIkZJcg1EMJF3tbseOQlXNzkGNYO1/8cCEdt+4Vobm3k9iRyIIZvTJhcHAwDh06hLi4OJw4cQI5OTlwcXFBVFQUunTpYsqMRCSCy2mFEMB1vMh8TOwRgh8upOPr2BQsHdlG7Dhkweq9LHP79u3Rvn17U2QhIjOh1Qm4nF4AJ4UNQj14BpfMQ7dQNzT3dsT351Lx6mMRUNjyjmIyjklHMObk5GDXrl3Yu3cvtFqOyCeyVHeyilGq0SIywAVSruNFZqJy4HNBWTl+vczV3sl4RhU/n332GXr06IHc3Fx929mzZ9GqVSuMHTsWw4YNQ69evVBSUmKyoETUeOJSCyCVAG39uY4XmZcxnQMgt5FiWyzn/CHjGVX8fPvtt5BIJHB3d9e3vfLKK8jLy8P06dMxbNgwnD59GuvWrTNZUCJqHLklGqTml6G5tyOU8npfGScyKVelHMMj/XAqMRc3MznjMxnHqOLn5s2bVcb5ZGdnIyYmBs8++yw2bNiAPXv2oFu3blzVncgCXfrf7e3tA1zFDUJUA874TPVlVPGTk5NTZTHTY8eOAQCefPJJfVvfvn2RlJRUz3hE1JjKtTpczSiEh4Mc/q4KseMQGfTngc+c8ZmMYVTx4+7ujuzsbP2/Y2JiIJVK0atXL32bRCKBSqWqf0IiajTX7xZBo9UhMtAFEg50JjPFgc9UX0YVP61bt8aePXuQk5OD/Px8fPPNN+jWrVuVVd4TExPh6+trsqBE1LAq1/GylUnQypcTyJF5qxz4zBmfyRhGFT/z589HRkYGAgMDERQUhMzMTLzwwgtV+pw8eRIdOnQwSUgianj3isuRVaRGK19n2Nlw/hQyb5UDn08n5nHgM9WZUcVPdHQ01q1bh7Zt2yIiIgL/+te/MGnSJP32Q4cOobi4GEOHDjVZUCJqWFczSwFwHS+yHBN73B/4/PUpnv2hujH6PtaZM2di5syZBrcNGDAAeXl5RociosYlVTjhdk4Z/FwU8HKyEzsOUa10Dbk/8HnnuTS89lgrzvhMtWbSGZ6JyDI5RA6BVgDacx0vsiASiQQT/zfw+ZdLHPhMtcfih8jK6QQBTp0eh8JGiubejmLHIaqT0Rz4TEYwu+InOzsbM2bMQIcOHeDh4QGFQoGwsDCMHz8e586dq9a/oqICK1euREREBOzs7ODv74/Zs2cjJydHhPREludkQj5s3fzRytseNlKz+0ggeiBXpRwjIv1wJikPNzjwmWrJ7D7p8vPzce3aNQwZMgRvvPEGPvnkE0yaNAnHjh1Djx49sHfv3ir9p0+fjkWLFqFly5b4+OOPMW3aNGzevBkDBgzg2mJEtbD9XAYEQYfW3kqxoxAZZULlwGee/aFaMruFe5o3b47jx49Xa589ezaCg4OxcuVK/V1kBw4cwNatWxEdHY3du3fr+3bp0gVjx47F6tWrsXTp0kbLTmRp0vPLcOhmDlR3zsG550ix4xAZpWuIG1p4O2LnuVQsepwDn+nhzO7MT018fHxgb2+P/Px8fduWLVsAAAsWLKjSd8yYMQgNDdVvJyLDvjmVDJ0AFJ3/WewoREarnPG5UFWBn+M48JkezuzO/FQqLy9HQUEBKioqkJycjNWrV6O4uBgjRozQ94mNjYVUKkVUVFS15/fs2RPbtm1Dbm5uldXn/ywjIwMZGdV/UOLj4033RojMVLlWh22nU+DvYoekO2fFjkNUL6M7B2Dlb9ew7VQyxnQJFDsOmTmzLX6OHTuGgQMH6v/t4uKC1157rcplrNTUVHh6esLOrvq8JIGBgfo+NRU/69evx5tvvmni5ESWYd+VTGQVqTG3fwhOCDqx4xDVS+WMzzvPp+FGZhFa+nCJFqpZgxQ/OTk5+OSTTyCRSLBkyRKj9tGhQwfs378farUaN27cwJdffomioiKo1WrY2NyPXVpaCjc3N4PPVygU+j41mTVrFqKjo6u1x8fHV5mxmqgp2noyCbYyCUZ38MXLYochMoGJPYKx83wavo5NxrLotmLHITPWIMVPdnY2li1bVq/ix83NDUOGDAEADB8+HNOmTUOHDh1w584d/PrrrwAApVIJtVpt8PmVK8orlTXfweLn5wc/Pz+j8hFZslv3inDiTg5GdvCHh6Nc7DhEJtGFA5+plhpkwLOnpyeWLl1q0jut3NzcEB0djd9++w2JiYkA7l/ays7ONlgApaam6vsQUVVbT96/JXhyVIjISYhMRyKRYGIPDnymh2uQMz8eHh5YtmyZyfdbVlYGAMjLy0NoaCi6d++Oa9euITY2Fv369avS98SJEwgPD69xvA+RtSrVVOD7s6lo6eOIbqFuKCrixHBkvgoLC+vU/5HmznjPRoovTyTgkRbOD+wrl8v1QyTIupjdre6ZmZkG2xMTE/HDDz/AxcUFrVu3BgBMnjwZALB69eoqfXfu3InExET9diL6fz9eSEeRugKTokIgkUjEjkNkULlGDUikCAoKgouLS60fQb6eyDm/HxdSC+EZ3v6BfYNDQ/VDJMi6mN3dXu+++y7279+PYcOGITQ0FBKJBPHx8diyZQuKi4uxefNmfaU+ZMgQTJgwAdu2bcPIkSPxxBNPICEhAR988AHatGmDhQsXivxuiMyLIAj48mQSlHIZnuwUIHYcohppK8oBQYdFm/bB2dWjTs+9W6TB7is5ePSfm9A71PBivarSYix/pj80Gg3P/lihehU/e/bswYULF5Camory8vJq2yUSCTZu3FinfY4YMQJpaWn47rvvcO/ePVRUVMDPzw8jRozA/Pnz0b179yr9N2/ejMjISGzatAlz5syBu7s7Jk+ejBUrVsDRkYs0Ev3ZhZR8XEkvxMQewXBS2Iodh+ih7JSOUDjU7bM8RCnAI7EIN7NV6NfKD7Yys7vIQSIzqvhJSkrCyJEjceXKFQiCUGM/Y4qfIUOG6O/yqg1bW1ssXrwYixcvrtPrEFmjyoHOk3pwoDM1XRKJBO0CXBBzIws37xWjjd+Dx/6Q9TGq+Jk3bx4uX76MGTNmYMqUKQgICNDPvUNE5im3RIM9cenoEuKGNv78ZUBNWytfJxy9lY3LaQUsfqgaoyqWAwcOYOjQodiwYYOp8xBRA9l+JgWaCh2m9ORZH2r6FLYytPRxRHxGEe4VqeDtxHE99P+MuhBqa2uLyMhIU2chogai1Qn4KjYJno5yPNbOV+w4RI2iQ6ArACAutUDcIGR2jCp+evfujcuXL5s6CxE1kJgb95CSW4Zx3YJgZ8NZb8k6+Dgr4OuswPW7RVCVa8WOQ2bEqOJn+fLlOHz4ML755htT5yGiBrDlRBKkEmAiBzqTlekQ6IIKnYCr6XWbLJGaNqPG/HTq1Al//PEHhg8fjvXr16Nz585wcak+l0J91vYiItNIyilBzI0sPNLaBwGu9mLHIWpUzX0ccfhmNuLSCtAp2JUTexIAI4ufgoIC/OMf/0Bubi5iYmIQExNjsB+LHyLxbT2ZBEEApvQMFTsKUaOzkUrRLsAZpxPzkJhTijBPB7EjkRkwqvj5+9//joMHD2LIkCGYPHky/P39eas7kRkq02ix/Uwqmnk5oHfzus2SS9RURAa44ExiHi6m5rP4IQBGFj8//fQTevXqhX379pk6DxGZ0J6L6SgoK8dLQ1rwdD9ZLSeFLZp5OeB2VgnySzVwVcrFjkQiM2rAc1lZGXr16mXqLERkQoIgYMvJRCjlMozpEih2HCJR8bZ3+jOjip9OnTrhzp07ps5CRCZ0ISUfl9MKMapTAJy5jhdZuUA3e7gr5biaUYhyrU7sOCQyo4qfJUuWYM+ePTh69Kip8xCRiXx5IgkAOKMzEe7fgNM+0AXqCh2u3y0SOw6JzKgxPxkZGRgxYgQGDRqEiRMnokuXLgZvdQeAKVOm1CsgEdVdTrEaP8VloHuoO1r5cl0jIgBo7eeM47dzcDE1H+Eu7mLHIREZVfxMmzYNEonk/piCLVuwZcuWaoMpBUGARCJh8UMkgm/PpECj1WEyz/oQ6cltpGjt54SLqQW4W1QudhwSkVHFz6ZNm0ydg4hMRKsT8NXJZHg52WFoW67jRfRn7QNdcTG1AFcyS8SOQiIyqviZOnWqqXMQkYkcuHYPafllmDeoOeQ2Rg3rI2qy3B3kCHKzR0JuGWQObigsbLhlL+RyORQKriZvjjgzIVETs+VEImRSCSb0CBY7CpFZ6hDkipS8Mjh2GoagoKAGex0vHx8kJyayADJDLH6ImpAbmUU4cjMbI9r7wc+F63gRGRLm4QAHWwm0HR7DzOlT4ObuafLXUJUWY/kz/aHRaFj8mCGjip9mzZrVqp9EIsHt27eNeQkiMsKmYwkAgBl9wkROQmS+pFIJWnnKcbbcDelqOfwcHMWORI3MqAEBOp0OgiBUe+Tl5SExMRGJiYnQaDTQ6TiRFFFjyS3RYOe5NHQMckXnYDex4xCZtZbuttBpynD1ngqCIIgdhxqZUWd+EhMTa9x269YtzJs3DyUlJdi7d6+xuYiojradSoa6QsezPkS1IJdJUHzpd0i7jERafhkC3ZRiR6JGZPJbQZo3b46dO3ciLS0Nb775pql3T0QGaCp02HIiEb7OCjzejre3E9VG0dk9AIDzyfniBqFG1yD3wSoUCjzyyCPYtm1bQ+yeiP7i18sZyCxUY0qvENjKeHs7UW1U5KUjyMUWd7Lvr/ZO1qPBPiVtbGxw9+7dhto9Ef2PIAjYeDQBClspJnTj7e1EddHG+/6dWBdTuNq7NWmQ4ic7Oxu7du1q0PkTiOi+c8l5iEstwOjOgXBzkIsdh8ii+DrawNNRjisZBVBXaMWOQ43EqAHPy5cvN9heUVGBlJQU7N69GwUFBXj33XfrFY6IHu4/RxMBANN7hYqag8gSSSQSdAxyxe/x93AlvZB3SloJo4qfZcuWPXC7s7MzXn/9dbz66qvG7J6Iaik1rxS/XbmLfi290MLHSew4RBYpwscJx27l4EJKPjoGukIqlTz8SWTRjCp+Dh48aLBdKpXCzc0NrVq1go0NJ48mamgbjiRAqxPwHG9vJzKajUyK9oEuiE3Ixe3sYrTw5h8STZ1RFUr//v1NnYOI6ii3RINvTiejrb8z+rYw/fT8RNYkMsAFZxLzcD45n8WPFeA9sUQWavPxRKjKdZjVPxwSCU/TE9WHg50NWvo6IqNAhbuFKrHjUAOrdfGj0+mMehCR6ZVqKrD5RCKC3O0xjJMaEplEp6D7g50vcNLDJq/Wl71sbW3rvHOJRIKKioo6P4+IHuzb0ynILy3HwkcjYMNJDYlMwsvJDoFu9rh5rwh9VJ5wVHDsalNV669sUFBQrU+tFxcXIycnx+hQRFSzcq0OG44kwNNRjqe6BIodh6hJ6RTkitS8MlxMzUfv5hxL11TVuvh50GKmlcrLy7F27VqsWLECABAaGmpsLiKqwU9x6UjLL8PLj7aEwlYmdhyiJiXM0wGu9ra4lFaAbqHukNvwzGpTZLKv6o4dO9C6dWu88sorEAQBq1atQnx8vKl2T0S4v5TF+pg7cJDLMDkqVOw4RE2ORCJB52A3qCt0uJLOJS+aqnoXP8ePH0evXr0wfvx4pKSkYN68ebh9+zZefvllyOWcap/IlA5dz8K1u0WY0D0YLsq6j8Mjoodr7ecEe1sZzqfkQ6sTxI5DDcDo0Vy3b9/Ga6+9hl27dkEQBIwdOxbvvvsuwsPDTZmPiP6nrKwMH/1+HTZSCZ7u6IXCwkKT7NdU+yFqKmxkUnQIcsHJO7m4da8YEb6c96epqfOZn9zcXMyfPx9t27bFzp07ERUVhePHj2P79u0mKXxu3ryJZcuWoXfv3vD19YWDgwPatGmDefPmISMjo1r/iooKrFy5EhEREbCzs4O/vz9mz57NAdfUpKhUKoT1Go7zqYXIO/cLIoK84eLiYpJH5QLEFVremUlUqX2gK2ykEpxNzoMg8OxPU1PrMz8ajQZr1qzBe++9h/z8fISHh+O9997DmDFjTBpo48aN+PjjjzFy5Eg8/fTTsLe3x8mTJ/Hpp59i69atOH78OFq1aqXvP336dGzduhUjRozAyy+/jISEBKxZswZHjx7FyZMn4eDgYNJ8RGJQq9VAu2GQSoBZ06fA6W8zTLbvguxMrHx+GLRarmhNVMneVoY2/s6ISy1ASl4Zgt2VYkciE6p18RMREYHk5GS4u7tjzZo1mDNnDmQy099pMnbsWCxatAiurq76tpkzZyIqKgqzZs3C0qVLsX37dgDAgQMHsHXrVkRHR2P37t36/l26dMHYsWOxevVqLF261OQZiRrb8YR8KAJaI8JLCS93V5PuW1VabNL9ETUVnYPdcCm1AOeS81j8NDG1vuyVlJQE4P7dJv/6178QFhaG4ODgBz5CQkLqHKhr165VCp9K48ePBwDExcXp27Zs2QIAWLBgQZW+Y8aMQWhoqH47kSUTBAGfHk6CoC1HpwBHseMQWQ0Xe1s093ZEUk4psovVYschE6rTgGdBEJCbm4vc3NyGylOjtLQ0AICPj4++LTY2FlKpFFFRUdX69+zZE9u2bUNubi7c3d0N7jMjI8PgOCLeok/mJOZGFi6lF6E4bj+cej8rdhwiq9I5xA037xXjXFIeHm3LpWSailoXP2Kv07VkyRIA98f4VEpNTYWnpyfs7Oyq9Q8MDNT3qan4Wb9+Pd58880GSEtkGoIg4IPfb8JWJkHBie3AbBY/RI3J11mBAFd7XM8sQs9wDzgpOMVEU2ARC5e88847+P777zFq1ChMnTpV315aWgo3NzeDz1EoFPo+NZk1axaio6OrtcfHx2PSpEn1TE1Uf4euZ+FiSj7GdfbDqqJsseMQWaXOIa7Yc7EMF1MK0KcFl7xoCsy++Pnwww/xz3/+EwMGDMBXX31VZX0xpVJ5/y4YA1Qqlb5PTfz8/ODn52fawEQmcv+szw3IZVI82ysIq8QORGSlwjwc4K6U31/yIswNdjZcVsbSmfWiJf/+97/x0ksvYfDgwfj555+rFTKBgYHIzs42WAClpqbq+xBZon1XMxGXWoDx3YPg61z90i4RNQ6JRILOIa7QaHW4nMZJQZsCsy1+Vq5ciYULF+Kxxx7DTz/9ZPAMTvfu3aHT6RAbG1tt24kTJxAeHl7jeB8ic1au1eG9X69BKZfhxUHNxY5DZPUifJ2glMtwgUteNAlmWfy88847WLRoEUaMGIEffvhBP37nryZPngwAWL16dZX2nTt3IjExUb+dyNJ8HZuMhOwS/K1/OLydDH//E1HjsZFK0THIFcXqCtzILBI7DtWT2Y35+eSTT/DPf/4TPj4+GD16NHbs2FFlu6OjI0aNGgUAGDJkCCZMmIBt27Zh5MiReOKJJ5CQkIAPPvgAbdq0wcKFC0V4B0T1U6gqx4d/3ISPsx2e6xsmdhwi+p/IABecTszF2eQ8tPJ1qjIGlSyL2RU/p0+fBgBkZmZixozqU/iHhIToix8A2Lx5MyIjI7Fp0ybMmTMH7u7umDx5MlasWAFHR04IR5bns0O3kVuiwaqx7aGUm92PKJHVUtjK0NbfBRdS8pGcW4oQDy6fZKnM7rLXf//7XwiCUOMjMTGxSn9bW1ssXrwYN27cgFqtRkZGBtavXw9PT96OSJYnLb8MG48moJWvE8Z05mB9InPTKcgVEglwNjlP7ChUD2ZX/BBZs3/tvQ5NhQ7/GNYaMilPqROZG2d7W7TwdkRKbhnuFanEjkNGYvFDZCYupRZg1/k09GvphX4tvcSOQ0Q16BJyf3Lds0k8+2OpWPwQmQFBEPDWT1chlQD/GNZK7DhE9ADeTgoEuytxM7MYBWXlYschI7D4ITIDP1xIw6nEXIzvHoxWvs5ixyGih+ga4gYBPPtjqVj8EImsUFWOFT9fg5vSFq8OjRA7DhHVQqCbPXyc7XA1oxAl6gqx41AdsfghEtkH+28gu1iNRY+3gqtSLnYcIqoFiUSCriHu0OoEXEjJFzsO1RGLHyIRXU4rwObjiegU7IqnugSJHYeI6iDcywFuSlvEpRZAXaEVOw7VAYsfIpGUa3V49bs4SCUSvPNkJKS8tZ3IokgkEnQJcYNGq8OltAKx41AdsPghEsnGowm4mlGIv/UPR2s/DnImskStfJ3haGeD88n5qNDqxI5DtcTih0gECdkl+GD/DTTzdOCq7UQWTCaVoHOwK0o1WsRncMFTS8Hih6iRaXUCXt5xERqtDu+OjoTCViZ2JCKqh7b+LlDYSHE2OQ86nSB2HKoFFj9EjeyLI3dwNikPM3qHoUczD7HjEFE9yW2k6BDkioKyctzKKhY7DtUCix+iRnT9bhH+ve8Gmnk54BXO6UPUZHQIdIWNVIIziXkQBJ79MXcsfogaiapci/nfnIdWEPDvpzvychdRE2Ivl6FdgAuyitVIyi0VOw49BIsfokbyzi/xuHa3CC8NboGOQa5ixyEiE+sc7AqpBDiTyCUvzJ2N2AGIGotKpYJGo2mw/cvlcigUCoPb9l/NxJYTSegR5o4XBvLuLqKmyElhiwhfJ8RnFOFukVLsOPQALH7IKqhUKgSHhiIrM7PBXsPLxwfJiYnVCqCU3FK8vOMiXJW2WDO+I2SczJCoyeoa4o74jCJcTOfAZ3PG4oesgkajQVZmJpZ+FQOF0tHk+1eVFmP5M/2h0WiqFD+qci1mf3UWhapy/GdqN/i52Jv8tYnIfLg7yBHu5YDbWSWw9eCSNeaKxQ9ZFYXSEQoH0xc/NXlzzxVcTivEvMEtMLCVd6O9LhGJp0uIG25nlcC5xxixo1ANOOCZqIF8eTIJ206loG8LT8wf3ELsOETUSPxc7OHvLIdDmwHIKFCJHYcMYPFD1ACO38rGsh+vIMRDibUTOnGcD5GV6ejvCInMBptj08SOQgaw+CEysYTsEsz+6hyUtjJsnNoVrkq52JGIqJEFusihzriJ7y/cxb0inv0xNyx+iEwot0SDaZtOoUhVjo8mdEJzbyexIxGRCCQSCQpOfAt1hQ4bjySIHYf+gsUPkYlIbOzw4o6rSMopxVuj2nGAM5GVK7sZi+ZeSnx5Mgm5JQ03xxjVHYsfIhPQ6gR4jlqEy+lFeGFAOJ7pESJ2JCISnYCZvYNRqtFi0zGe/TEnLH6I6kmrE/D7zTwow7thVHsfLlhKRHqPtPJEMy8H/PdYIgrKysWOQ//DeX6I6kEnCNh39S4S89QouXoIf39hHoqKikz+OoWFhSbfJxE1PJlUgjkDmmPhjovYcjwRcznthVlg8UNkJEEQ8Ht8Jm5kFiPExRaHf/kQoSH/atDXrNBWNOj+icj0ojv6Y80fN7DxWAKm9wmDox1/9YqNXwEiIwiCgIPXsxCfUYRQDyX6+ElxWFuORZv2wdnVw+SvV5CdiZXPD4NWqzX5vomoYdnKpHhhQHMs3nkJW08m4W/9w8WOZPVY/BDVkU4n4PdrmYjPKEKQmz2GR/qhOPceAMCugZbPUJVykUQiSza6cwA++uMmNhy5g6k9Q2Evl4kdyapxwDNRHVRodfj5Uob+jM/IDv6wkfHHiIgezM5GhtkDwpFdrMHWk0lix7F6/NQmqiV1hRa7L6bjTnYJInycMKK9P2xZ+BBRLT3dNQh+Lgqsi7mNEjXH74mJn9xEtVCqqcDOc2lIzStDh0AXDG3rw/W6iKhOFLYyzBnYHDklGmw5wbM/YmLxQ/QQuSUabD+TintFavQIc0f/ll6QSFj4EFHdPd01CAGu9lh/+DaKVJz3RywsfogeICmnBN+eSUGhqhwDI7wQ1cyDhQ8RGU1uI8XcQc2RX1qOzccTxY5jtVj8ENXgYmo+dl9MhwTAqI4BaB/oKnYkImoCxnQJRLC7Ep8fvoNCnv0RBYsfor/Q6gQcun4Ph65nwVlhi3FdgxDsrhQ7FhE1EbYyKeYNboFCVQX+c5RrfonBLIuf9957D+PGjUOLFi0glUphY/Pg6YgqKiqwcuVKREREwM7ODv7+/pg9ezZycnIaKTE1FcXqCuw8n4qLqQUIdLXHuG5BcHOQix2LiJqYUR39EebpgI1HEpBfyhXfG5tZFj+LFy/Gvn37EBQUBB8fn4f2nz59OhYtWoSWLVvi448/xrRp07B582YMGDAAJSUljZCYmoLUvFJsO5WM9HwVOga5YlSnANjbciIyIjI9G5kU8we3QJG6Al8cuSN2HKtjljM837p1C+Hh96f/HjBgALKysmrse+DAAWzduhXR0dHYvXu3vr1Lly4YO3YsVq9ejaVLlzZ4ZrJcgiDgbFIejt/OgY1Mgsfb+aKlj5PYsYioiRvZwR+fHLyF/xxNxNSeofB2VogdyWqY5ZmfysKnNrZs2QIAWLBgQZX2MWPGIDQ0VL+dyJBSTQX2xGXg2O0cuDvIMb5bMAsfImoUMqkErz3WCmXlWnzw+02x41gVszzzUxexsbGQSqWIioqqtq1nz57Ytm0bcnNz4e7uXm17RkYGMjIyqrXHx8c3SFYyL4k5Jdh/NROlGi1a+TphUCtvzthMRI1qcGtvdA91x/YzKXi2Tyiae/OPr8Zg8cVPamoqPD09YWdnV21bYGCgvo+h4mf9+vV48803GzwjmZcKrQ7HbuXgQmo+5DZSPNbWFxG+/MAhosYnkUiwaFgrjP70OFb+dh1fTOkqdiSrYPHFT2lpKdzc3AxuUygU+j6GzJo1C9HR0dXa4+PjMWnSJNOFJLORXVKOQ5dSkFOigb+rAkPb+MLZ3lbsWERkxToHu2FYpC9+uXQXZxJz0TW0+h/rZFoWX/wolUqo1WqD21Qqlb6PIX5+fvDz82uwbGQ+NBU6uPadhJ2XsiGRAD2beaBrqBuknK2ZiMzAK0NbYd+VTLzzSzy+n92LM8k3MIsf4BAYGIjs7GyDBVBqaqq+D1mvCyn5GPef83DpNR7uShuM7xaM7mHuLHyIyGyEeTpgQvdgnEvOx94rmWLHafIsvvjp3r07dDodYmNjq207ceIEwsPDDY73oaavRF2Bd36Jx+hPjyEptwz5h7/Ek+084eVUfXwYEZHY5g1uAQe5DKt+u4ZyrU7sOE2axRc/kydPBgCsXr26SvvOnTuRmJio307WQxAE/Hb5Lh75dww+P3wHkQEu+HZGJxSc+BYyKc/2EJF58nKyw8x+4biTXYIvTySJHadJM8sxP19++SWSku5/4ZOSkiAIAt5++2399tdff13//0OGDMGECROwbds2jBw5Ek888QQSEhLwwQcfoE2bNli4cGGj5yfxJOeU4o0fL+Pg9Sw4KWzw1hNtMbFHCEqKi8SORkT0UDP7NcP2Myn4YP8NjOjgB28nTnzYEMyy+Nm4cSNiYmKqtC1ZskT//38ufgBg8+bNiIyMxKZNmzBnzhy4u7tj8uTJWLFiBRwdHRslM4mrRF2B9TG3sf7wHagrdHiyUwAWD2vFDw4isij2chmWjmyDWV+exXu/XMO/x3UUO1KTZJbFz6FDh+rU39bWFosXL8bixYsbJhCZLZ1OwK7zaVi19xoyC9Vo4e2IN59oi17hnmJHIyIyyqNtfNC/pRd2nk/D+O73b9Ag07L4MT9kvU4n5mLUp8ewcMdFaCp0eOuJtvh1fl8WPkRk0SQSCZZFt4VcJsXS3ZdRwcHPJmeWZ36IHuRKegH+tfc6Dl7Pgo1Uguf6hGHuoBZwUXKyQiJqGsI8HTCzXzN8fPAWvjyZhOm9w8SO1KSw+CGLkZBdgn/vv4E9F9MB3F8RecEjLRHm6SByMiIi05szsDl2nU/Dv/fdwIj2/pymw4RY/JDZu1ugwod/3MT2MynQ6gQMjPDCy0Mj0NbfRexoREQNxl4uw5IRrfG3refwzi/x+ICDn02GxQ+ZrbwSDT6LuY3NxxOhrtChW6gbXhnaioP/iMhqDG3ri4ERXth1Pg3RHf0xMMJb7EhNAosfMjvF6gpsPJKAL47cQbG6Aq39nPHq0AgMiPDiejdEZFEKCwvrvY/Fj4ThVEIuFn13EbtmdoGj3f1f3XK5XL+AN9UNix8yG6pyLb6KTcanB28hp0SDUA8l3hkdiRGRfpByZmYisiDlGjUgkSIoKMgk+3PsMBQej81F5OSlyN37CQDAy8cHyYmJLICMwOKHRFeh1WHnuTSs+f0G0gtU8HVW4J0nI/FU10DYyjgbAxFZHm1FOSDosGjTPji7etR7f4Ig4Kf4XKR3fBxPj5sIb7sKLH+mPzQaDYsfI7D4IdEIgoB9VzOx6rdruJ1VAjelLf45rDUm9wyBwlYmdjwionqzUzpC4WCalQaGtlPgq1PJiEkoxNjI+hdU1ozFD4nidGIu3v0lHueS86GUyzBvUHM8368ZnBScq4eIyBBne1sMjPDC3iuZOHynQOw4Fo3FD9WJSqWCRqMx+vm3skrw4aFExNzMhY1UgnGd/TCrTzA8HeWwhRYAix8iopq08nVGQnYJbmQWw7Hj42LHsVgsfqjWVCoVgkNDkZWZWefnyhzd4dp3EhzaDYZEKkNJ/GHkH9mKVXnpWPW/Phy8R0T0cIMivHE3vwzC4Jm4klGEns7OYkeyOCx+qNY0Gg2yMjOx9KsYKJS1u4ZdoRNwKaME59KKUaET4O8sR49gJ3hHjQOmj9P3U5UWc/AeEVEt2NnKMKSlG74/n46FO+Px63xvLu9TRyx+qM4UtRjAJwgCbmeV4MjNbBSqKuBib4t+LTwR5unAuXqIiOrJy8EWub9/DsljL2LuN+fxn6ldYcO7Y2uNR4pMLrtYjZ3n0/DzpQyoynXo09wTk6KC0czLkYUPEZGJFF/8DU928MHhG1l499drYsexKDzzQyZTptHixJ0cXE4rgACgrb8zejbzgINd7b/NTDEbamPul4hITK8/1hxpBeXYeDQBLbwdMb57sNiRLAKLH6o3rU7ApbQCnLyTA3WFDv4uCvRv6QVv59qP3TH1bKg1qdBWNOj+iYgak61Mis8mdcaoT4/hnz9chpeTHQa39hE7ltlj8UP1kpRTgsM3spFbqoGjnQ0GtfJGC++6X94y9Wyof1WQnYmVzw+DVqs1+b6JiMTk4WiH/07vjqfWncALX53DV8/1QNdQLgD9ICx+yCh5pRocuZmNhOwS2Egl6BHmji4hbvVejsKUs6H+maq02OT7JCIyF+Fejtg0rRsmfHESM/57Gluf64H2ga5ixzJbHPBMdSKRK3EiqRBbTyYhIbsELX0cMblnCKKaeXAdLiIiEXUIcsUXU7pCXaHDMxticTElX+xIZou/rahWKrQ6fHs2HQGzvkBcRgk8He0wtksgHm/nB2cuSUFEZBZ6N/fEf6Z1Q7lWh0kbYnEmMVfsSGaJxQ891KHr9/D4h0ewYu9tCNoKDAh3wfhuQQhwtRc7GhER/UVlAaQVBDyzIRZ7r9wVO5LZYfFDNbqRWYQp/zmFaZtOIyWvFLP6BCP9i5mI8FJyvh4iIjPWK9wT387sCSeFDWZvPYv/HE2AIAhixzIbHPBM1aTnl2HtgZv49nQKdAIwulMAXnksAg6ScvyjXC12PCIiqoXIQBfsnN0b0zadwvKfruJSWgHeeTIS9nKZ2NFEx+KH9LKK1Pjk4C18HZsMjVaH7mHueH14a/0dA4WF5eIGJCKiOgn2UOKHF3tjwbcXsOt8GuIzCvHRhE5o6eMkdjRRsfghZBersfFoAv57LBFl5Vq0D3TBwkcj0K+FJy9vERFZOGeFLT6f3BUfH7yFNb/fwIi1R7H48VaY2jMUUql1fsaz+LFiyTml+OLIHWw/kwJ1hQ4RPk5Y8GhLPNrGh0UPEVETIpVKMG9wC/Ru7om/f3sBb+65ij0X0/HO6Ei08nUWO16jY/FjhS6lFuCLI3fwU1w6dALQPtAFs/uH49G2vpBZ6V8BRETWoEuIG36Z3xer913H5uOJGPHRUUyKCsG8wS3g7iAXO16jYfFjJUo1FdhzMR1fxSYjLrUAANC3hSdm9w9Hz3APnukhIrISjnY2eGNkW4zuFIgluy/jv8cT8f3ZVDzXtxmm9Q6Fi33Tn7uNxU8TJggC4lILsPNcKnaeT0ORqgL2tjJM6B6ESVEhaOvvInZEIiISSWSgC3a90Au/XLqLVXuv4YPfb2DDkTt4JioEz/QIRpC7UuyIDYbFTxN0614RfryQjh8vpiMxpxQA0NLHEa8MDcGoTgGckZmIiAAAEokEw9v7YWhbH/x4MR0fH7yFdTG3sf7wbfRr4YWJPYIxuJU3bJrY8kUsfhqZSqWCRqMx6T4rdAIuphbiyO1cxNzMxe3s+wWPp4Mtnunmj8fbeCHS3+n+pS1NGQo1ZUa9TmFhoSljExFRPZnyc3lIc2cMCu+EEwn52HkhEwduZCPmRhZ8nRUY0d4Pj7XzRedgtyZxhxiLn0akUqkQHBqKrMzMeu/LxtUXiuBIKEI6QBHWBTL7+3M2aEvyUXorFqVXY5CUchlnBR3eqferVVWhrTDxHomIqC7KNWpAIkVQUFCD7N/Lxwen465j96V7+PZ0CjYcTcCGownwdLTDo2198EgbH/QIc4dSbpllhGWmtlAajQZZmZlY+lUMFErHWj9PqxOQW1aBrGIN7haVI71QjRKNTr/d08EWwa528JCUYMPcyVi0aS+cXaeZPH9BdiZWPj8MWq3W5PsmIqLa01aUA4IOizbtg7Orh0n3rSotxvJn+sNNcf/2+LmDmuNyWiF+u5KBvVcy8XVsMr6OTYatTIJOQW7o1dwDvZt7on2gC+xsLGP2aBY/IlAoHaFwMFz86AQB+aXlyCxU/e+hRlaxGlrd/6/J4u4gR7iXPQLd7OHvag8Hu/tfxvysuwAE2D1g//WhKi02+T6JiMh4DfV5/2cSiQSRgS6IDHTBK0Nb4da9Ihy6noVjt7IRm5CLU4m5WPP7TchlUrT2d0anIFd0CnZFxyBXBLub51qQLH5EIggCitUVyCnWIKdEg5xiNXJKNMgt0aDiT4WOwkaKQFd7+Dgr4ONsB18XhcWeZiQiIsvX3NsJzb2d8FzfZijX6nAxJR/Hb+fgfHIezqfk42JKPv57/H5fdwc5OgS6oF2AC9r4OaONvzOC3JSijxvib9FG9NPle3Af+iJ+uJyNvLJMaLS6Ktsd7GTwd7WHh6McPk4K+Loo4KywMcuqmYiImrbaDqZu6W6Dlu4+mNbNB4IgICVPhUvpRbiUXoS4tEIcuZmNg9ez9P0d5DJ0CHTBV89Hifb7rckUPzt37sSqVatw6dIlyOVy9O3bF++88w7atWsndjS9369lw6njYyhQVcDbSQF3Rzk8HOTwcLCDh6McClvLuFZKRERNl8kHU0ttYOsRBLlPGOTezaDyboYjCTKop3SCQqEwzWvUUZMofjZu3IjnnnsO7dq1w8qVK6FSqbB27Vr06tULx44dQ2RkpNgRAQAvD2mGzXMfx1tbf4e9o3WvqEtEROapIQdTA/8/oFrzyd9Y/BgrLy8PCxYsQGBgII4dOwZn5/sLtD399NNo06YN5s+fjwMHDoic8r5AVwV0pfm8jEVERGavMQZTi8Xip2zcvXs3CgsL8dxzz+kLHwAIDg7G2LFjcfDgQaSkpIiYkIiIiMyJxZ/5iY2NBQD06tWr2rZevXph8+bNOH36tMFrlxkZGcjIyKjWfuHCBQBAfHy8SbMWF9+/VTzhyjnIFfYm3TcAFOfnAgCSrp6Hg5Pp1+1qyP1bcnZL378lZ2/o/VtydkvfvyVnt/T9N3R2jer+KgMXLlyAo6NpzyxV/t4uK3vISgaChRsxYoQAQLh69Wq1bT///LMAQPjwww8NPveNN94QAPDBBx988MEHH03osXXr1gfWDhZ/5qe09P46VnZ2dtW2VQ6kquzzV7NmzUJ0dHS19ry8PMTHx6NTp06wtzf9GZraiI+Px6RJk7B161a0bt1alAzmiMelZjw2hvG4GMbjUjMeG8Ms4biUlZUhMTERQ4cOfWA/iy9+lEolAECtVlfbplKpqvT5Kz8/P/j5+RncNnjwYBMlrJ/WrVujc+fOYscwOzwuNeOxMYzHxTAel5rx2Bhm7seld+/eD+1j8QOeAwMDAQCpqanVtlW2VfYhIiIisvjip3v37gCAEydOVNtW2datW7dGzURERETmy+KLn1GjRsHJyQlffPFFlam4k5OTsWPHDgwYMMB0s1QSERGRxbP44sfNzQ3vv/8+UlNT0bt3b3z88cdYvXo1+vXrB4lEgjVr1ogdkYiIiMyIxRc/wP27tnbs2AGlUolXX30Vb731FiIjI3Hs2DF06NBB7HhG8fPzwxtvvFHjgGxrxeNSMx4bw3hcDONxqRmPjWFN6bhIBEEQxA5BRERE1FiaxJkfIiIiotpi8UNERERWhcUPERERWRUWP0RERGRVWPyYif3792P27NmIioqCUqmERCLB1q1b67VPnU6HqKgoSCQSDBkyxERJG5cpjkt2djZmzJiBDh06wMPDAwqFAmFhYRg/fjzOnTvXQMkblimOy82bN7Fs2TL07t0bvr6+cHBwQJs2bTBv3jxkZGQ0UPKGZ6qfpffeew/jxo1DixYtIJVKYWNj2asBmfIzZufOnYiKioKDgwPc3NwQHR2Ny5cvmzhx40lKSsLEiRPh5eUFe3t7dOzYERs2bKjTPrKysjBnzhyEhIRALpfD398fzz//PO7evdtAqRuHKY5NYWGh/i5sJycneHh4oFu3bvjkk09QXl7eQMkfwgQLq5MJTJ06VbCxsRHat28vdO/eXQAgfPnll/Xa5+rVqwVHR0cBgDB48GATJW1cpjguN2/eFHr27CksWLBA+PDDD4UNGzYIr7/+uhAYGCjY2NgIv/32WwOlbzimOC6vvfaa4ODgIIwfP15Ys2aNsH79emH69OmCTCYT3NzchPj4+AZK37BM9bMEQHB1dRUGDhwo+Pr6CjKZrAHSNh5THZcNGzYIAIR27doJa9euFd5//30hODhYcHJyEuLi4hogecNKSUkRfH19BXt7e2Hx4sXC559/LgwfPlwAICxbtqxW+7h3754QFhYmSCQSYerUqcK6deuE1157TXB0dBRCQ0OFzMzMBn4XDcMUx6a8vFzo2rWrIJVKhenTpwvr1q0TPvzwQ6Ffv34CAOGZZ55p4HdhGIsfM5GamiqUlZUJgiAImzZtqnfxc/v2bUGpVApr1qyx6OLH1Mflz9LS0gSZTCYMHDjQJPtrTKY4LqdPnxby8vKqta9fv14AIDz11FOmiNroTPU9c+vWLf3/9+/f3+KLH1Mcl9zcXMHZ2VkIDAwUCgoK9O1JSUmCg4ODRf4sTZ48WQAgfP/991XaR44cKdjY2Ai3b99+6D5eeuklAYDwzjvvVGk/duyYIJFIhOeff96kmRuLKY7N/v37BQDCwoULq7RXVFQIHTt2FKRSqVBYWGjS3LXBy15mIiAgAAqFwmT7e/7559G2bVvMnTvXZPsUg6mPy5/5+PjA3t4e+fn5DbL/hmSK49K1a1e4urpWax8/fjwAIC4url77F4upvmfCw8NNkMZ8mOK47N69G4WFhXjuuefg7Oysbw8ODsbYsWNx8OBBpKSk1DdqoyktLcV3332HsLAwjB49usq2BQsWoKKiAl9//fVD93PgwAEAwPTp06u09+rVCy1atMC2bdugUqlMF7wRmOrYFBQUAAD8/f2rtMtkMvj6+kImk0Eul5sueC2x+GmCvvjiCxw+fBhffPEFpFJ+iSuVl5cjOzsbd+/exalTpzBx4kQUFxdjxIgRYkczK2lpaQDuF4dEfxYbGwvg/i/1v6psO336dKNmqo9Lly6hrKwMPXv2rLatZ8+ekEgkOHXq1EP3o1arAQBKpbLaNqVSieLiYosbE2WqY9O7d284ODjgvffew/bt25GcnIwbN27g7bffxt69e7F06VLY2dk1xFt4IMsewUfVpKen45VXXsHChQstdmmPhnLs2DEMHDhQ/28XFxe89tprWLp0qYipzM+SJUsAVP8rlig1NRUAEBgYWG1bZVtlH0vwoPdjZ2cHT0/PWr2ftm3b4vr16zhw4ABGjRqlb8/IyMC1a9cA3F9su2vXrqYJ3ghMdWx8fX2xe/duzJ49G+PGjdO3KxQKbNy4UbTPGRY/JrRs2bJa9x0wYAAGDBhg8gyzZ8+Gp6cn3njjDZPv21jmcFwAoEOHDti/fz/UajVu3LiBL7/8EkVFRVCr1aLcyWMux+XP3nnnHXz//fcYNWoUpk6d2uCvVxNzPDbmQOzjUlpaCgAG/1KvvKRW2acxGXtcHvR+gPvvqTbvZ8GCBfpf8Gq1GlFRUUhKSsIrr7wCnU5X5bUam9jHBrj/h2ZERAQGDBiARx99FKWlpdi8eTOef/55SCQSTJs2rdYZTYXFjwm9+eabdepv6g+mb775Bj/++CP2798Pe3t7k+67PsQ+LpXc3Nz0t/wPHz4c06ZNQ4cOHXDnzh38+uuvDfKaD2Iux6XShx9+iH/+858YMGAAvvrqK0gkkgZ9vQcxt2NjLsQ+LpWXdSov8/xZ5ZgWQ5d+Gpqxx+VB7we4/548PT0fur/evXtjx44dmDt3rn7MnEQiwdixY9G1a1d8+umnVcZINSaxj83FixfRp08fvPTSS3jvvff07ZMmTULv3r0xZ84cDB8+HF5eXnXKWV8sfkxIEHGNWLVajXnz5uHRRx9FaGgobt26VWV7WVkZbt26BScnp0YfyyHmcXmQyvlJPvnkEyQmJiI0NLRRX9+cjsu///1vLFy4EIMHD8aPP/4oyi+wPzOnY2NOxD4uf7601bp16yrbHnSZpKEZe1wedKlOrVYjOzsbUVFRtdrXk08+iejoaFy9ehV5eXkIDw9HQEAAnn76aQCodrwai9jH5sMPP4RarcZTTz1VpV0qlWLs2LE4efIkTp06heHDhxuV01gcDdtElJWVISsrC/v27UOLFi2qPADg+PHjaNGiBebPny9yUvNSVlYGAMjLyxM5iXhWrlyJhQsX4rHHHsNPP/0keuFD5qt79+4AgBMnTlTbVtnWrVu3Rs1UH5GRkVAoFAbfz8mTJyEIgv4914ZMJkNkZCT69euHgIAAqNVqHDhwoMpnsaUw1bGpvIFCq9VW21ZRUVHlv42JxY+Fun37tn4gHQA4ODhgx44dBh/A/W/kHTt24O9//7tYkRvFX48LAGRmZhrsm5iYiB9++AEuLi6i/VXWWAwdF+D+GJ9FixZhxIgR+OGHHxpsWgFzVtOxsXaGjsuoUaPg5OSEL774AoWFhfr25ORk7NixAwMGDEBQUFBjRzWaUqnEmDFjkJCQgJ07d1bZtnr1atjY2GDChAlV2uvy/fKPf/wDOTk5eP31102WubGY6ti0bdsWAPDf//63Snt5eTm+/vpryGQyUQaC87KXmYiLi8OPP/4IADh//jyA+3NqJCYmAgCio6PRvn17ff/BgwcjKSlJf0rT1tYWY8eOrXH/3t7eD9xurup7XADg3Xffxf79+zFs2DCEhoZCIpEgPj4eW7ZsQXFxMTZv3mxxv/RNcVw++eQT/POf/4SPjw9Gjx6tL5QrOTo6VrlzxVKY4tgAwJdffomkpCQA0G9/++239dst7ReaKY6Lm5sb3n//ffztb39D7969MWvWLKjVaqxduxYSiQRr1qxptPdjKu+88w5+//13TJ48GWfPnkVYWBh2796Nn376CUuWLKk231NN3y+tWrVCdHQ0mjdvjrKyMuzatQsxMTF44YUXMGXKlMZ8SyZjimPz0ksv4csvv8Rnn32G1NRUDB06FKWlpdi6dSvi4uKwYMECBAQENPZb4/IW5qJyxtWaHps2barSPyQkRKjtlw8WPMOzKY7L/v37hbFjxwqhoaGCUqkU5HK5EBISIkycOFGIjY1txHdjOqY4LlOnTn3gPkJCQhrvDZmQqX6W+vfv/8D9WBpTfsbs2LFD6N69u2Bvby+4uLgII0aMEC5evNgI76Jh3LlzRxg/frzg4eEh2NnZCZGRkcL69esN9q3puEydOlUIDw8XFAqF4OzsLPTv31/Yvn17Q0dvcKY4NgkJCcKMGTOEoKAgwcbGRlAqlUL37t2FDRs2CDqdrqHfgkESQeDIQiIiIrIeHPNDREREVoXFDxEREVkVFj9ERERkVVj8EBERkVVh8UNERERWhcUPERERWRUWP0RERGRVWPwQERGRVWHxQ0RERFaFxQ81WYmJiZBIJJg2bZrYUfQOHToEiUSCZcuWVWkfMGAAJBJJrfqaikQiwYABA6q0LVu2DBKJBIcOHXpoX3NW0/swpcLCQsybNw+hoaGwsbGBRCLBhQsXGuz1iMh0uLApEZERXn31Vaxfvx4jRozA5MmTIZPJ4OvrK3YsIqoFFj9EZmDLli0oLS0VOwZefPFFjB8/HsHBwWJHMXs//fQTWrZsiT179ogdhYjqiMUPkRkwl2LD09MTnp6eYsewCOnp6ejXr5/YMYjICBzzQ1YpIyMDc+bMQWhoKORyOby8vDB69GicPXvWYP+CggK89NJLCAwMhEKhQKtWrfDvf/8bd+7cMcm4IkNjfmqiUqkwduxYSCQSzJkzBzqdTr9t27ZtGDhwIFxdXaFQKNC6dWu8/fbbUKvVtdr3w8bKZGdnY+bMmfDz84OdnR3atm2LTZs2Geyr0+mwbt06dOvWDY6OjnBwcEC3bt3w2WefVcn8Z3/88Qcee+wxuLu7w87ODi1btsSiRYtQUFBgsP/Zs2fx2GOPwcnJCc7OzhgyZAhOnDhRq/f6V7X9nqj8WgmCgJiYGEgkklqPiYqLi8OECRMQGhoKOzs7eHl5oXPnznjppZdQXl5epW9FRQU+/fRTREVFwdnZGUqlEp06dcLHH39s8PgJgoCPP/4Ybdu2hUKhQEBAAF588UUUFBQgNDQUoaGhVfo/6Gv9oPFypaWlePfdd9GxY0c4ODjA0dERPXv2xLZt26r1/fO4tQsXLmD48OFwdXWFUqlE//79cfz4cYPHSavVYt26dejduzdcXFxgb2+P5s2b47nnnsPNmzfrdZx+/PFHDB48WP897O/vj/79++PTTz81mIWaJp75IauTkJCAPn36ID09HYMGDcKECROQkpKCHTt24Oeff8b333+PESNG6PurVCoMGjQI586dQ6dOnfDMM8+goKAAK1aswJEjRxo1e15eHqKjo3Hs2DG8++67WLRokX7bjBkzsGnTJgQGBmLMmDFwdXXFyZMnsWTJEvzxxx/Yv38/bGyM/5HPz89H7969IZfLMXbsWKjVauzYsQMzZsyAVCrF1KlTq/SfPHkyvv76awQFBeG5556DRCLBrl278MILL+Do0aP46quvqvRfv349Zs+eDQcHBzz11FPw9vbGoUOHsHLlSuzZswfHjh2Dq6urvv/x48cxZMgQaDQajB49Gs2bN8eFCxcwYMAADBo0qE7vrS7fE9OmTcOAAQPw5ptvIiQkRF8g/LW4+Ku4uDj06NEDEokE0dHRCAsLQ2FhIW7duoVPP/0Ub7/9NmxtbQEA5eXlGDlyJPbu3YuIiAhMnDgRCoUCBw8exNy5cxEbG4svv/yyyv5feuklfPTRR/Dz88PMmTNha2uL3bt3IzY2FhqNBnK5vE7HxJD8/HwMGjQI58+fR+fOnTFjxgzodDrs3bsXEydOxJUrV/D2229Xe96ZM2ewatUq9OzZE8899xySk5Px/fffY/Dgwbhw4QIiIiL0fTUaDUaMGIH9+/cjKCgIEydOhLOzMxITE7Fr1y706dMHLVq0MOo4ff7555g1axZ8fX0xcuRIeHp64t69e4iLi8OmTZvwwgsv1PsYkYUQiJqohIQEAYAwderUKu2PPvqoAEB4++23q7QfO3ZMkMlkgru7u1BUVKRvX758uQBAGD9+vKDT6fTtycnJgqenp8HXqMnBgwcFAMIbb7xRpb1///7CX38c/9o3MTFRaN26tWBrayts3bq1St9NmzYJAIQnn3xSKC0trbLtjTfeEAAIa9asqdIOQOjfv7/BvgcPHqzWF4Dw7LPPChUVFfr2K1euCDKZTGjdunWV/l9//bUAQOjUqVOVY1lcXCx06dJFACB89dVX+vbExERBLpcLTk5OQnx8fJV9zZ49WwAgPP/88/o2nU4nRERECACEH374oUr/NWvW6PP+9X3UpK7fE5XH5K/H70EWLFhgMK8gCEJubq6g1Wr1/678Orz44otVjndFRYUwY8aMavs5duyYAEAIDw8XcnJy9O1lZWVCVFSUAEAICQmp8po1fa0FoeafnalTpwoAhJUrV1ZpLysrE4YOHSpIJBLh/Pnz+vbK72EAwqZNm6o8Z926dQIAYfbs2VXaFy9eLAAQRo4cKahUqirbVCqVcO/ePaOPU+fOnQW5XC5kZmZWe89ZWVnV2qjpYvFDTZahD/CUlBQBgBAcHCxoNJpqz5k0aZIAQNi8ebO+LTw8XJBKpUJCQkK1/m+//XajFD/nz58X/Pz8BGdnZ+H333+vtt+OHTsKNjY2Ql5eXrVtFRUVgoeHh9CtW7cq7XUtfpRKpVBQUFBt//369RMAVCkOhgwZIgAQ9u7dW63/77//LgAQBg4cqG+rPI6LFy+u1j83N1dwcnISFAqF/pfh0aNHBQBCv379DL7f8PDwWhc/xnxPCILxxY+hY/JnWq1WcHd3F3x9fYXy8vJq2/Py8gSJRCI89dRT+rbnnntOACD85z//qda/8vuovsVPdna2IJPJhK5duxrMfeHCBQGA8Morr1R77d69e1frr9FoBBsbG6FLly76toqKCsHFxUWwt7cX0tLSDL5OJWOOU+fOnQWlUink5uY+cN/U9PGyF1mV8+fPAwD69u2rv8TwZ4MGDcLWrVtx/vx5TJkyBYWFhbh9+zaCgoIMXtbo06dPQ0fG0aNH8e9//xtOTk44fPgwOnToUGV7aWkpLl68CE9PT6xZs8bgPuzs7BAfH1+vHC1atICzs3O19qCgIAD3L8k5OjoCAM6dOwepVGpwHEz//v0hk8n0X4vK/gAMXq5yc3NDp06dcPjwYVy7dg0dOnTQ9+/fv3+1/jKZDH369MHt27dr9b7q+j1hrHHjxuHDDz/EqFGjMHbsWAwZMgS9e/dGeHh4lX43btxAbm4uWrRoYfASEgDY29tX+Xo+6Hj06dMHMpnM6NyVTp8+Da1WW+PcU5Vjlgx9n3Xt2rVam62tLXx8fJCXl6dvu3btGgoKCtCjRw/4+/s/MI8xx+mZZ57BwoUL0aZNG4wfPx79+/dH79694eXl9cDXoqaHxQ9ZlcqBs35+fga3V7bn5+cDuD+RHQD4+PgY7F9TuymdP38eRUVF6NWrF1q1alVte15eHgRBQFZWFt58880Gy/Hn8TZ/VjmOSKvV6tsKCgrg7u5ucJyJjY2NfqzFn/sDtf+6VPav6fjXZb6dur62sbp3744jR45gxYoV+O677/RjUSIiIvDGG29gwoQJAICcnBwAwM2bNx/49SwuLq72Hgwdj8rjXV+VuU6fPo3Tp0/XKlelB33v/Pn7pvIYBwQE1DpPXY7TggUL4OnpiU8//RQfffQR1qxZA4lEgv79++P99983WKRR08S7vciquLi4AADu3r1rcHtGRkaVfpVnOjIzMw32r6ndlF588UX87W9/w969exEdHY2ysrIq2yuzdurUCcL9S9k1PhqLi4sLcnNzq93BBNy/Oyc7O7vKWaS6fl0q/1vT8a9pPzVlrctr10fPnj3x008/IS8vD8eOHcOSJUuQmZmJiRMn4vfff6/yOk8++eQDv5YJCQnV3oOh41F5vP9KKpXqt/+VoUKv8jX+/ve/PzDXwYMH63hU/l9lkZSWlvbQvsYcJwCYMmUKTp48iZycHPz888949tlncfjwYQwdOhRZWVlGZyfLwuKHrEqnTp0A3L+UZOhDv/KDu3PnzgDuFz/NmjVDWloaEhMTq/U/evRow4X9H4lEgs8++wwvvfQS9u3bh+HDh6OkpES/3dHREW3btsWVK1eQm5vb4Hlqo1OnTtDpdDh8+HC1bYcPH4ZWq9Uf48r+AAzedp2fn48LFy7ob90H/v/rExMTU62/Vqut09elrt8TpmBnZ4devXph+fLl+OijjwAAu3fvBgC0atVKf6eeoeLRkAcdj6NHj1Y5u1LJzc0NAJCSklJt25kzZ6q1de/eHVKptEHvcKx873FxcUhPT69V37ocpz9zdXXFsGHD8MUXX2DatGnIzc01+P1KTROLH7IqgYGBeOSRR5CYmFhtfExsbCy+/vpruLm54cknn9S3T5kyBTqdDosXL65y9iQlJaXGMTYN4YMPPsDixYtx8OBBDB06VH9JDrh/Ol+j0WDGjBkG/2rPy8vTjwtpDDNmzAAALF68uMrM1aWlpfrb85999ll9+6RJk2Bra4u1a9fi1q1bVfa1ZMkSFBYWYtKkSbCzswMA9OrVCxERETh8+LC+aKj08ccf13q8D2Dc94Qxjh8/Xu2sHfD/Z2uUSiWA+5eC5s6di4yMDMybN8/gczIyMnD16lX9vytvt1+xYkWVAlilUmHx4sUG83Tv3h0AsGnTpipFX0pKCpYvX16tv7e3N5555hmcOXMGb731lsGC6vbt29XOtNSFTCbDCy+8gLKyMvztb3+rNj+VRqPRn50x5jgdPHjQ4BnQykuwlV8Davo45oesTuXkaa+88gr27duHrl276ud0kUql2LRpE5ycnPT9X331Vfzwww/45ptvcP36dTz66KMoKCjA9u3b0a9fP/zwww/6SwgN7Z133oFCocAbb7yBRx55BL/99hvc3NwwY8YMnD17Fp9++inCw8MxdOhQBAcHIzc3FwkJCTh8+DCmT5+OdevWNUrOiRMnYvfu3di+fTvatm2LUaNGQSKR4IcffkBCQgLGjRuHZ555Rt8/NDQUa9aswZw5c9C5c2c8/fTT8PLyQkxMDE6cOIFWrVph5cqV+v4SiQQbN27EI488gjFjxlSZ56dyosTffvut1nnr+j1hjFWrVuHAgQPo27cvwsLC4OjoiCtXruDXX3+Fm5sbZs6cqe+7ZMkSXLx4EevWrcOePXswaNAgBAQE4N69e7h58yaOHTuGFStWoE2bNgCA3r17Y+7cuVi7di3atWuHsWPH6uf5cXNzMzieqUePHujXrx8OHz6M7t27Y9CgQcjMzMSePXswdOhQg2eEPv74Y9y8eRNLly7Fl19+iT59+sDHxwfp6emIj4/H6dOnsW3bNoSFhRl9nN544w3ExsZiz549aNmyJUaMGAEnJyekpKRg3759eP/99/XFXl2P05NPPglHR0dERUUhNDQUgiDgyJEjOH36NLp06YIhQ4YYnZssTIPfT0YkkprmKhEEQUhNTRX+9re/CcHBwYKtra3g4eEhPPHEE8KpU6cM7isvL0+YO3eu4OfnJ8jlciEiIkL417/+JcTGxgoAhPnz59cqU33m+fmzVatW6efR+fP8JHv27BGGDx8ueHl5Cba2toKPj4/QrVs34Z///Ge1+XNQx1vda7qtu3Lul79OBaDVaoVPPvlE6NKli2Bvby/Y29sLnTt3Fj7++OMqc9r82d69e4VHHnlEcHV1FeRyuRAeHi688sorBm/hFwRBOHPmjDB06FDB0dFRcHR0FAYPHiwcP378gbdx16Su3xMPOiY1vbdp06YJrVu3FpydnQWlUim0bNlSmDt3rpCYmFitv06nE7Zs2SIMGjRIcHNzE2xtbQV/f3+hd+/ewooVK4Tk5ORq/deuXSu0atVKkMvlgp+fn/DCCy8I+fn5QkhISLVb3QXh/vf1c889J3h5eQlyuVxo27atsH79+gf+7KjVamHt2rVCz549BWdnZ0EulwtBQUHCoEGDhA8++EDIzs7W933Q97AgCDXmKi8vF9auXSt069ZNcHBwEJRKpdC8eXPh+eefF27evGn0cfrss8+EUaNGCWFhYYK9vb3g5uYmdOzYUVi5cqVQWFhoMCM1TRJBaMRRkERNzBdffIGZM2di3bp1mDVrlthxiAyqnKbB0Lg1ImvEMT9EtWBo8GVycjLeeust2NjYYOTIkSKkIiIiY3DMD1EtjBkzBuXl5ejSpQtcXV2RmJiIn376Sb/I48MmZCMiIvPB4oeoFiZPnowvv/wS33//PQoKCuDo6IgePXrgxRdfxOjRo8WOR0REdcAxP0RERGRVOOaHiIiIrAqLHyIiIrIqLH6IiIjIqrD4ISIiIqvC4oeIiIisCosfIiIisiosfoiIiMiqsPghIiIiq/J/J1YDAph2ZwAAAAAASUVORK5CYII=\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Sampled sequence -> full length sequence & mutational distance"
      ],
      "metadata": {
        "id": "WuK7bJym-QYo"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "Get full length sequence of target protein from uniprot."
      ],
      "metadata": {
        "id": "_k04ta5B-h5f"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import requests\n",
        "from Bio import SeqIO, Entrez\n",
        "from io import StringIO\n",
        "\n",
        "def download_protein_seq(protein_id, output_dir=\"data\"):\n",
        "\n",
        "    query_url =  f\"https://rest.uniprot.org/uniprotkb/{protein_id}.fasta\"\n",
        "    # Download the FASTA file content from the web link\n",
        "    response = requests.get(query_url)\n",
        "\n",
        "    if response.status_code == 200:\n",
        "\n",
        "        # File path for saving the FASTA file\n",
        "        output_fpath = f\"{output_dir}/{protein_id}.fasta\"\n",
        "\n",
        "        # Parse the content using Biopython's SeqIO\n",
        "        fasta_content = StringIO(response.text)\n",
        "        refseq = next(SeqIO.parse(fasta_content, 'fasta'))\n",
        "\n",
        "        # Writing the sequence record to a file\n",
        "        with open(output_fpath, 'w') as refseq_out:\n",
        "            SeqIO.write(refseq, refseq_out, \"fasta-2line\")\n",
        "\n",
        "        return refseq\n",
        "    else:\n",
        "        print(f\"Failed to retrieve data. HTTP Status Code: {response.status_code}\")"
      ],
      "metadata": {
        "id": "UJhYDfiPZRbN"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "refseq = download_protein_seq(\"F1BV52\")\n",
        "print(f\"Protein Sequence: {refseq.seq}\")\n",
        "print(f\"Protein Sequence Length: {len(refseq.seq)}\")"
      ],
      "metadata": {
        "id": "8pydOCqj-mZx",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b05fc06d-eff6-4e62-8e06-cbf6bd4f2c17"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Protein Sequence: MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAARYDDVLYFPASRYPETGAHISDAIKAGHADVCTIERSGADKRRQESLKGIPTKPGFDRDEWPMAMCEEGGKGASVRYVSSSDNRGAGSWVGNRLNGYADGTRILFIVQ\n",
            "Protein Sequence Length: 142\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Compare the refseq sequence to native sequence from PDB file."
      ],
      "metadata": {
        "id": "6ls1mHyI-wij"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "def find_missing_sequence(refseq, pdb_seq):\n",
        "    \"\"\"\n",
        "    Find missing sequences in the PDB sequence compared to the reference sequence,\n",
        "    handling missing parts on both the left and right sides.\n",
        "\n",
        "    Parameters:\n",
        "        refseq (str): The reference sequence (full sequence).\n",
        "        pdb_seq (str): The PDB sequence (possibly missing parts).\n",
        "\n",
        "    Returns:\n",
        "        missing_left (str): The missing sequence on the left side of the PDB sequence.\n",
        "        missing_right (str): The missing sequence on the right side of the PDB sequence.\n",
        "        start_pos (int): The start position of the PDB sequence in refseq.\n",
        "    \"\"\"\n",
        "    # Make sure both sequences are upper case\n",
        "    refseq = refseq.upper()\n",
        "    pdb_seq = pdb_seq.upper()\n",
        "\n",
        "    # Find where the PDB sequence starts in the reference sequence\n",
        "    start_pos = refseq.find(pdb_seq)\n",
        "\n",
        "    if start_pos == -1:\n",
        "        raise ValueError(\"PDB sequence is not found within the reference sequence.\")\n",
        "\n",
        "    # Missing on the left: everything before the start of the PDB sequence in the refseq\n",
        "    missing_left = refseq[:start_pos]\n",
        "\n",
        "    # Missing on the right: everything after the end of the PDB sequence in the refseq\n",
        "    end_pos = start_pos + len(pdb_seq)\n",
        "    missing_right = refseq[end_pos:]\n",
        "\n",
        "    return missing_left, missing_right, start_pos"
      ],
      "metadata": {
        "id": "9mKM-_gvTm5_"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "missing_left, missing_right, start_pos = find_missing_sequence(refseq.seq, native_seq)\n",
        "print(f\"Missing sequence on the left: {missing_left}\")\n",
        "print(f\"Missing sequence on the right: {missing_right}\")\n",
        "print(f\"Start position of the PDB sequence in the reference sequence: {start_pos}\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "uLC7ey8UNAvz",
        "outputId": "0a57690c-0605-4b7a-d95f-8f90d9269210"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Missing sequence on the left: MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGA\n",
            "Missing sequence on the right: \n",
            "Start position of the PDB sequence in the reference sequence: 33\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Append missing sequences in ESM-IF1 sampled sequences."
      ],
      "metadata": {
        "id": "syO1mfFz-2G-"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "from Bio.SeqRecord import SeqRecord\n",
        "from Bio.Seq import Seq\n",
        "def append_missing_sequence_to_fasta(in_path, out_path, missing_left, missing_right):\n",
        "    \"\"\"\n",
        "    Append missing sequences to both sides of each sequence in the FASTA file.\n",
        "\n",
        "    Parameters:\n",
        "        in_path (str): Path to the input FASTA file (e.g., ESM-IF1 results).\n",
        "        out_path (str): Path to the output FASTA file with missing sequence appended.\n",
        "        missing_left (str): The missing sequence to be appended to the left.\n",
        "        missing_right (str): The missing sequence to be appended to the right.\n",
        "\n",
        "    Returns:\n",
        "        None\n",
        "    \"\"\"\n",
        "    # Load the sequences from the FASTA file\n",
        "    sequences = list(SeqIO.parse(in_path, \"fasta\"))\n",
        "\n",
        "    # Iterate through each sequence and append the missing sequences\n",
        "    updated_sequences = []\n",
        "    for record in sequences:\n",
        "        # Append the missing sequences to the left and right of the original sequence\n",
        "        new_seq = missing_left + str(record.seq) + missing_right\n",
        "        updated_record = SeqRecord(Seq(new_seq), id=record.id, description=record.description)\n",
        "        updated_sequences.append(updated_record)\n",
        "\n",
        "    # Write the updated sequences to the output FASTA file\n",
        "    SeqIO.write(updated_sequences, out_path, \"fasta\")\n",
        "\n",
        "    print(f\"Missing sequences appended and saved to {out_path}\")"
      ],
      "metadata": {
        "id": "6gF46C8ETvnZ"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "fname_sampled_seqs_full = f'output/{PDBID}_{chain}_ESMIF1_{temp}_{nsamples}_full.fasta'\n",
        "fname_sampled_seqs_full"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 35
        },
        "id": "OEMQxO3kT2vy",
        "outputId": "159ad39a-8cbc-495a-9370-f6d9d3ec7112"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'output/5OMT_A_ESMIF1_1_500_full.fasta'"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            }
          },
          "metadata": {},
          "execution_count": 40
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "append_missing_sequence_to_fasta(fname_sampled_seqs, fname_sampled_seqs_full,\n",
        "                                 missing_left, missing_right)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "dTtoBoU0OHer",
        "outputId": "99f92bd3-0682-4d13-8b4b-be6d2c5fb51f"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Missing sequences appended and saved to output/5OMT_A_ESMIF1_1_500_full.fasta\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!head -n 10 $fname_sampled_seqs_full"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "EkH4UQcNN47W",
        "outputId": "360cedb9-7c87-4c6c-9b47-5cf4b384fec6"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            ">sampled_seq_1\n",
            "MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAAWFDAEWFFPCDRFPESCAQIRQAIAL\n",
            "GHAAVCTLDRDGARARNREALAGIPRRPGYERHEWPFSECREGGRGSLVKYVDARDNQGA\n",
            "GRYLRRHLARYANGTKLLGVVK\n",
            ">sampled_seq_2\n",
            "MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAVAFDTTLQFPAAQFPESGQHIQDAIAE\n",
            "GHAAVCTLARQRAWQRRQEALAGIPQRAGYQRHEWPFSICTEGGAGSAVRYVSALDNSGM\n",
            "GSWLRNWLSQYADGTRIRLVVR\n",
            ">sampled_seq_3\n",
            "MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAGIFDATMQQPVAEFPESGKMIKQAIAE\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Save full length sampled sequence corresponding log_likelihood, and mutational distance from refernce sequence in csv file."
      ],
      "metadata": {
        "id": "gHJdLjEvU0Nj"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "from Levenshtein import distance\n",
        "# Load FASTA file into a dictionary\n",
        "sampled_seqs_full = {record.id: str(record.seq) for record in SeqIO.parse(fname_sampled_seqs_full, \"fasta\")}\n",
        "\n",
        "# Map sequences from FASTA to CSV\n",
        "sampled_seqs_df['sequence'] = sampled_seqs_df['seqid'].map(sampled_seqs_full)\n",
        "\n",
        "# Calculate the Levenshtein (edit) distance for each sequence\n",
        "sampled_seqs_df['num_muts'] = sampled_seqs_df['sequence'].apply(lambda seq:\n",
        "                                                                distance(seq, refseq.seq))"
      ],
      "metadata": {
        "id": "xy3WPjenQo85"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "sampled_seqs_df.head()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 206
        },
        "id": "Y1fac9DtQ7aQ",
        "outputId": "fefc01d2-98d6-4455-aeba-da4241922a89"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "           seqid  log_likelihood  \\\n",
              "0  sampled_seq_1       -1.067197   \n",
              "1  sampled_seq_2       -1.029135   \n",
              "2  sampled_seq_3       -1.038908   \n",
              "3  sampled_seq_4       -1.094937   \n",
              "4  sampled_seq_5       -0.958626   \n",
              "\n",
              "                                            sequence  num_muts  \n",
              "0  MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAAWFDAEWFFPCDR...        57  \n",
              "1  MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAVAFDTTLQFPAAQ...        49  \n",
              "2  MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAGIFDATMQQPVAE...        56  \n",
              "3  MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAMKYDVTWVFPRSK...        52  \n",
              "4  MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGACNYDAVLYTPCNT...        51  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-ad4d1fc2-2e8f-487b-a364-4edef4b261c8\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>seqid</th>\n",
              "      <th>log_likelihood</th>\n",
              "      <th>sequence</th>\n",
              "      <th>num_muts</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>sampled_seq_1</td>\n",
              "      <td>-1.067197</td>\n",
              "      <td>MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAAWFDAEWFFPCDR...</td>\n",
              "      <td>57</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>sampled_seq_2</td>\n",
              "      <td>-1.029135</td>\n",
              "      <td>MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAVAFDTTLQFPAAQ...</td>\n",
              "      <td>49</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>sampled_seq_3</td>\n",
              "      <td>-1.038908</td>\n",
              "      <td>MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAGIFDATMQQPVAE...</td>\n",
              "      <td>56</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>sampled_seq_4</td>\n",
              "      <td>-1.094937</td>\n",
              "      <td>MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAMKYDVTWVFPRSK...</td>\n",
              "      <td>52</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>sampled_seq_5</td>\n",
              "      <td>-0.958626</td>\n",
              "      <td>MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGACNYDAVLYTPCNT...</td>\n",
              "      <td>51</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-ad4d1fc2-2e8f-487b-a364-4edef4b261c8')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-ad4d1fc2-2e8f-487b-a364-4edef4b261c8 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-ad4d1fc2-2e8f-487b-a364-4edef4b261c8');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-79d39379-d694-4b4f-8385-f87a6640b351\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-79d39379-d694-4b4f-8385-f87a6640b351')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-79d39379-d694-4b4f-8385-f87a6640b351 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "sampled_seqs_df",
              "summary": "{\n  \"name\": \"sampled_seqs_df\",\n  \"rows\": 500,\n  \"fields\": [\n    {\n      \"column\": \"seqid\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 500,\n        \"samples\": [\n          \"sampled_seq_362\",\n          \"sampled_seq_74\",\n          \"sampled_seq_375\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"log_likelihood\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0.10502884715848494,\n        \"min\": -1.4153408785478785,\n        \"max\": -0.7746016476132455,\n        \"num_unique_values\": 500,\n        \"samples\": [\n          -1.3204783868352208,\n          -0.9781881209907182,\n          -1.1459114004712585\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"sequence\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 500,\n        \"samples\": [\n          \"MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGATVWDAKWYFECARFPESCAHVAWAIAHGHAEVCTLDRASQAERREESLKGIPNREGYERHEWPWAICKEGGAGAMVVYVIASDNRGAGRWVDRQRAQFADGTRIKSITD\",\n          \"MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAGDFDVTLTFPDDVYPEAGAMVEQAIAEGHAAVCTLRRLDHRARLKQSLAGIPRRRGYDRHEWPFSWCEEGGDGAAVRYVQREDNRGAGGWLSQTLSGYADGTRVLWTVG\",\n          \"MIKKWAVHLLFSALVLLGLSGGAAYSPQHAEGAADYDTRIAFPGARFPASAAHISKAIAEGHADVCTLNRAGADFRKDQALRGIKAKHGYGRQEWPFAACTEGGKGAAVTYVPLHDNSGAGHWLAKQLSGYKDGTKVLFTVA\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"num_muts\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 4,\n        \"min\": 38,\n        \"max\": 65,\n        \"num_unique_values\": 25,\n        \"samples\": [\n          55,\n          65,\n          57\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 56
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "sns.histplot(sampled_seqs_df['num_muts'].values, kde=True);\n",
        "plt.xlabel('Edit distance to wild-type')\n",
        "plt.ylabel('Num. sequences')\n",
        "plt.title(\"Levenstein distance distribution for sequences\");"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 484
        },
        "id": "dQ5nxyBRT5b7",
        "outputId": "d5f6ee19-ddac-46de-90c0-b618c9bcbde0"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 640x480 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAj8AAAHTCAYAAADIwnlTAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAACGuElEQVR4nO3dd3gUVdsG8Ht2s9n0Tnol9N4JofdeBBSlFwErCIiKShE76ieWF0VEBBFELGAB6YQOobfQSUgjpPdkk93z/cGbfYnZtM0mu8nev+vKBZk5O/PMyezss2fOOSMJIQSIiIiIzITM2AEQERER1SQmP0RERGRWmPwQERGRWWHyQ0RERGaFyQ8RERGZFSY/REREZFaY/BAREZFZYfJDREREZoXJDxEREZkVJj9Ua33//feQJAnff/+9Ufa/bNkySJKEgwcPFlsuSRJ69epllJjqqsjISEiShKlTpxZbPnXqVEiShMjISKPEZcrngLHrpjSnT59G//794ebmBkmS0KZNG2OHRGaIyU81kCQJkiQZOwyTY6oXY1NS2oc8VY9evXrV2vdqaYmXKcvIyMDQoUNx6tQpPPnkk1i6dCmeeeYZY4dFZsjC2AEQ6euxxx5DSEgIvLy8jB1KMREREbCxsTF2GGbh/fffx2uvvQYfHx+j7P+FF17Ak08+CX9/f6PsvyzGrhtdTp06hQcPHuDdd9/F66+/buxwyIwx+aFay9HREY6OjsYOo4QmTZoYOwSz4eXlZdTk183NDW5ubkbbf1mMXTe6xMXFAQC8vb2NHAmZO972MgGbN29G79694eTkBCsrKzRt2hTvvPMO8vPztWViY2Mhl8vRtm3bUrczePBgSJKEy5cvF1t+8uRJjB07Fp6enrC0tISfnx9mz56tvRA9qug2QGFhId577z00bNgQSqUSfn5+ePXVV6FSqUq85vDhwxg+fDh8fX2hVCrh6emJkJAQvPXWW9oykiRh/fr1AICgoCDtrcHAwMBi20pJScGiRYvQtGlTWFtbw9HREX379sXu3btL7Le0Pj+BgYEIDAxEdnY2Fi5cCH9/fyiVSjRo0AAffvghhBCl1qEuZ86cwaBBg2Bvbw8HBwf069cPx48fL7W8rv4emZmZePvtt9GiRQs4ODjA3t4ewcHBGDduHM6cOQPg4W2MoKAgAMD69eu1dfToMapUKnz55ZcYMmQIAgICoFQq4eLign79+mHnzp0649G3Pk6dOoVx48bBx8cHSqUSXl5eGDBgAH7++ecSZStzjpUlMzMT8+fPh6+vL6ysrNCkSRP83//9HzQajc7ypd1K/eOPP9C3b194eXlBqVTC29sbPXv2xKpVqwD87/ZiWFgYABSr60f/dkV1l5GRgfnz5yMwMBAKhQLLli0DUP6tp7i4OEyaNAnu7u6wtrZG+/btsWnTphLlyuu/piuuovdX7969i8VfXt0AwM8//4wePXrA0dER1tbWaNmyJd5///1i15x/10FV3k9F9T1lyhQAwLRp00qc2wAQHx+P559/HoGBgbC0tES9evUwevRo7XuktDr7559/0KtXLzg6OlboNmZCQgJefvllNG7cGLa2tnByckLjxo0xdepU3Llzp0T5Xbt2YciQIXBzc4NSqURwcDAWLlyItLQ0ndvfu3cvunfvDltbW7i4uGDUqFG4du2azr/JwYMHIUmS9pz6t6L616Uinx1Fis6hpKQkzJo1S/veaN68OdatW1dqXe3evRvDhw+Hu7u79rNg5MiR2Lt3b5Xq6eLFi3jqqacQGBgIpVKJevXqoV27dnjppZdQUFBQajyGwpYfI5s+fTrWrVsHX19fjBkzBk5OTjhx4gQWL16Mffv2Yc+ePbCwsICPjw/69euH3bt349KlS2jZsmWx7cTHx2PPnj1o3749WrRooV3+3XffYdasWVAqlRgxYgT8/Pxw8+ZNfPvtt/jzzz9x4sQJnU3248ePx+HDhzF48GA4ODhgx44dWLFiBR48eFDsjfLPP/9g6NChcHBwwIgRI+Dj44OUlBRERERg1apVWLp0KQBg6dKl2LZtGy5cuIC5c+fCyckJALT/AkBUVBR69eqFyMhIdO/eHYMGDUJ2djb++usvDBo0CKtXr8bMmTMrVK8FBQUYOHAg4uLiMHjwYFhYWGDbtm147bXXkJeXp42rPMeOHUO/fv2gUqkwevRoNGjQAOfPn0evXr3Qp0+fCm1DCIFBgwbh2LFj6NKlC55++mlYWFggJiYGBw4cQPfu3dG+fXv06tULaWlp+Oyzz9C6dWuMGjVKu42iTqEpKSmYO3cuQkND0b9/f9SrVw/x8fH4888/MWTIEKxZswZPP/10letjzZo1ePbZZyGXyzFixAg0bNgQDx48wOnTp7Fq1So88cQT2rL6nmP/lp+fj759+yI8PBytW7fGhAkTkJaWhrffflubpFTEN998g9mzZ8PT0xPDhw+Hm5sbHjx4gIsXL2LdunV47rnn4OTkhKVLl+L7779HVFRUseP/9weNSqVCnz59kJKSggEDBsDBwUGbpJYlNTUVoaGhcHJywrRp05CWloaff/4ZEyZMQGxsLBYuXFjhY/q3l156Cdu2bUNYWBimTJlS6oejLq+//jref/99uLm5Yfz48bCzs8POnTvx+uuvY9euXdi9ezcsLS2Lvaaq76ei+j5//jy2b9+OkSNHas/pon/v3r2Lbt26IS4uDn369MFTTz2F6OhobN26FX///Td+/fVXDBs2rMS2f/nlF/zzzz8YPHgwnnnmGURFRZUZS05ODrp27Yrbt2+jf//+GD58OIQQiIqKwvbt2zF27FjUr19fW/6tt97CsmXL4OLigmHDhsHd3R0XL17Exx9/jB07duD48eNwcHAoFs+4ceNgaWmJcePGwcvLC0eOHEGXLl3QqlWrMmOrjIp+djwqLS0NXbt2haWlJcaOHYv8/Hxs3boV06dPh0wm0yanRZYuXYrly5fDzs4Oo0aNgp+fH+Li4nDs2DFs3LgR/fr106ueLl68iM6dO0OSJIwYMQJBQUHIyMjArVu3sGrVKrzzzjtQKBQGqyudBBkcAFGRql23bp0AIB577DGRk5NTbN3SpUsFALFy5Urtsk2bNgkAYsGCBSW2tWLFCgFAfP7559pl169fFwqFQgQHB4uYmJhi5ffu3StkMpkYNWpUseU9e/YUAES7du1EcnKydnlWVpYIDg4WMplMxMfHa5ePHj1aABDnz58vEVNiYmKx36dMmSIAiLt37+qsj549ewpJksTmzZuLLU9NTRWtW7cWVlZW4v79+9rlRfW3bt26YuUDAgIEADF48OBi9ZqQkCAcHR2Fo6OjUKlUOmN4lEajEY0bNxYAxLZt24qtW7lypfbvfODAgWLrAIiePXtqf7948aIAUKKuhRBCrVaLlJQU7e93794VAMSUKVN0xpSXlyeio6NLLE9LSxPNmzcXzs7OJc6lytbHlStXhIWFhXB2dhaXL18usa9H96/POVaad999VwAQo0ePFmq1Wrv8zp07wtnZWWe96Dqn2rVrJywtLUVCQkKJffz7nCw630tTVHd9+/YVWVlZJdYXvU91nQMAxOOPP67zWBQKhbh9+7Z2eWnn8qPbe/ScKmvfRXTVzbFjxwQA4efnV+x9XFBQIIYNGyYAiHfffVdnHVT1/VTecQ4YMEAAEO+8806x5UePHhVyuVy4uLiIzMzMEtuSJEns3LmzQvsXQog//vhDABAvvfRSiXX5+fkiIyND+/v+/fsFANGlSxeRmpqq81ge3U5mZqZwcXERFhYWIjw8vFj5l156SXtePPo3OXDggAAgli5dqjPegIAAERAQoHPfFf3sEOJ/5+SMGTNEYWGhdvmVK1eEXC4XTZs2LVZ+165dAoAICgoq8d4Wovh1oLL1NH/+fJ3XVSGESElJKfaeqS5MfqpBRZOfNm3aCAsLixInixBCFBYWCldXV9GxY0ftspycHOHo6Cg8PT2LnbxCCNG8eXOhUCiKXdyL3mx//fWXzv2PGjVKyOXyYm/2og+DPXv2lCi/ZMkSAUD8+eef2mVFyc/169fLPd6ykp/z588LAGLs2LE6X7tt2zYBQPznP//RLisv+bl582aJ7UyePFkAEJcuXSo33iNHjggAokePHiXWFRYWiuDg4EolP0899VS5+ywv+SnLJ598IgCIsLCwYssrWx8vvPCCACD+7//+r9x96nOOlaZBgwZCJpOJW7dulVhXdEGvaPJjY2NTLKksTUWTH13J/aNx6ToH5HK5uHPnTqmvWbZsmXZZTSU/Tz/9tAAgVq9eXaL89evXhUwmE0FBQcWWG+r9JETpxxkdHS0ACH9/f52J1MSJEwUAsX79+hLbqmhyXaQo+Vm0aFG5ZUeNGiUA6PwSIMTDa3i9evW0v2/cuFEAEJMnTy5RNi0tTTg6Ohok+ansZ4cQD88hGxsbkZ6eXuI1PXr0EACKJZdFyfBvv/2mM65HVbaeipKfXbt2lbvt6sLbXkaSk5ODCxcuwM3NDStXrtRZRqlUIiIiQvu7tbU1nnjiCaxZs0Z7bxV42CflypUreOyxx4p1vizqlxIWFobw8PAS23/w4AHUajVu3LiB9u3bF1vXoUOHEuX9/PwAPGzOLzJhwgT89ttv6Ny5M8aNG4fevXuja9eu8PX1rWBNFI81PT1d573vxMREAChWH2VxdHREgwYNKnQMpTl79iwAoGfPniXWyeVydOvWDbdv3y53O82aNUObNm2wefNmREVFYeTIkejWrRs6dOhQ4vZCRVy5cgUfffQRDh06hPj4eOTl5RVbHxsbW+I1lamPEydOAHjYh6w8VTnHHpWZmYlbt27Bz88PwcHBJdb36tWrWB+yskyYMAELFixAs2bN8OSTT6Jnz57o2rUr6tWrV6HX/5uVlZVetyv8/f113h4rOpZz587pFU9VFJ3Tum7ZNmrUCL6+vrh79y7S09OLDSYwxPupLEV10b17d523O/r06YONGzfi3LlzmDx5crF1nTp1qtS+evbsCR8fH3zwwQc4e/YshgwZgq5du6JNmzaQy+XFyh4/fhwKhQJbt27F1q1bS2xLpVIhMTERycnJcHV1LfOa4ejoiDZt2lTqFq4u+nx2FGnYsGGxW3RFHv072tnZAXh4HZAkCYMGDSo3psrW07hx4/DZZ59h1KhRGDt2LPr164euXbvqfO9XFyY/RpKamgohBBITEyt8UQcedmJcs2YN1q9fr01+ijoS//t+bXJyMgDgo48+KnObWVlZJZY92henSNH9Y7VarV02evRo/PXXX/jkk0/w3XffYfXq1QCA9u3b4/3330f//v0rdFxFse7Zswd79uypVKy66Iof0H0MpUlPTwcAeHh46Fzv6elZoVjkcjn279+P5cuX45dffsGrr74KALC3t8eUKVPw/vvvay845Tlx4gT69OmDwsJC9O3bFyNGjICDgwNkMpm2P4Wuzo6VqY+izokVGSJdlXPsUYaqawCYP38+3NzcsGrVKnz++edYuXIlJElCz5498dFHH+lM7Mvi7u6u11xA5R1L0THXpKJ9ljYKzMvLC/fu3UNaWlqx5McQ76eqxgVAZ8fZypwbAODg4IATJ05g6dKl+OOPP7Br1y4AD0fuPffcc3jzzTe1CVhycjIKCwvLvUZnZWXB1dXVoOdxafT97AAqfx1wdnaGtbV1udutbD116tQJhw8fxrvvvotffvkFP/zwAwCgcePGWLp0KZ566qkKHpH+ONrLSIouLG3btoV4ePux1J9HhYaGomHDhvjjjz+QlpaGgoICbN68GW5ubtpk6N/7SE9PL3P7ur6lVMbQoUOxf/9+pKamYt++fZg3bx6uXLmCYcOG4erVq5Wqj88++6zMWMsalWBoRTElJCToXH///v0Kb8vZ2RmffvopoqOjtZ2BmzRpgi+//BLPPvtshbfzzjvvIDc3F7t378bOnTuxcuVKLF++HMuWLUPnzp0rvJ2yFF0gdbUg/ZuhzjFD1jUATJ48GSdOnEBycjL+/vtvzJgxA4cOHcLAgQO1rYgVpe8kiOUdy6PJhUz28FJcWFhYonxpI4r0UbTP0uozPj6+RGw1oSpx6fP38fX1xdq1a/HgwQNcvnwZn3/+OVxdXbF8+XIsX768WFzOzs7lXqMDAgKKxVeZ87isvz1Q8u+v72dHZTk5OSE1NRW5ubnllq1sPQFAly5d8NdffyE1NRVHjx7F4sWLkZCQgPHjx+scSWZoTH6MxM7ODs2bN8eVK1eQkpJSqddOmTIFeXl52LJlC/7++28kJSVh/PjxJZqLQ0JCADwcil4TbG1t0adPH/zf//0fXn/9dahUqmLDr4ualHV9S6zpWCuiXbt2AKCzmVqtVuPIkSN6bbdBgwaYMWMGwsLCYGdnh+3bt2vXlVVHAHDr1i24uLjofHRCVZvTixT9LUobOq+rbFX/bvb29mjQoAFiY2N13krUdxZjJycn7Si4qVOnIiUlBYcOHdKuL6++q+LevXs6h5kXHcuj01Y4OzsDAKKjo0uUP336tM7t6xN70T511eetW7cQExODoKCgUlsIqktRXEeOHNGZBBw4cADA/96ThiJJEpo3b44XX3xR2+K8bds27fqQkBCkpqbiypUrFdpeWdeM9PR0nD9/vsTysv72t27dKtFCWJXPjsoICQmBEAL//PNPhcpWpp4epVQqERoaiuXLl+Pzzz8HgGLXxOrC5MeI5s+fD5VKhenTp+v8dpeamqq9h/yoyZMnQyaTYcOGDdiwYQMA6HwcwgsvvACFQoF58+bhxo0bJdarVKoqf2gdOnRI58Wq6JvPozMdu7q6Anj4ofBvHTp0QPfu3fHbb7/hu+++07mvS5cu4cGDB1WKtzJCQ0PRuHFjHDp0qMSb8csvv6xQfx/g4RBeXXOHpKamIj8/v1izsrOzMyRJ0llHwMNh2CkpKbh48WKx5WvXrtU231fVs88+CwsLC7z99ts6W+5iYmK0/zfkOTZt2jRoNBq8+uqrxeb1uXv3rvaiWBEHDhzQ+a236Nyp6DlZVWq1utRjsbCwwMSJE7XLO3ToAJlMhk2bNiEnJ0e7PCUlBa+88orO7esT+/Tp0wE8bEF8tAVMrVbj5ZdfhkajwYwZMyq8PUPx9fVF//79ERkZWaIfy8mTJ7Fp0yY4Ozvjscceq/K+rly5orNlRtc1a968eQCAmTNn6pyzKjs7W9tHDgBGjhwJZ2dnbNq0qUTSumzZMp23Ops0aQIHBwds37692PUtNzcXc+bM0XkM+n52VMaLL74IAFiwYIHOVuBHl1W2no4dO6azRUnX36C6sM9PNSrr+UyrVq3C9OnTcebMGaxatQrBwcEYOHAg/P39kZKSgrt37+LQoUOYNm0avv7662Kv9fPzQ+/evbFv3z5YWFigZcuWOic/bNKkCb777jtMnz4dzZs3x6BBg9CoUSMUFBTg3r17OHz4MOrVq4dr167pfYxz5sxBbGwsunbtqp2Y7MyZM9i/fz8CAgLw5JNPasv27dsXH330EWbOnIkxY8bA3t4eTk5OeOGFFwAAmzZtQp8+fTBjxgx8/vnn6Ny5M5ycnBATE4OLFy/i8uXLOH78ONzd3fWOtzIkScLatWvRv39/jBkzptg8P/v27cOgQYMq9K3owoULGD16NDp27IimTZvC29sbiYmJ2L59OwoKCrR9gICH3+o6d+6Mw4cPY8KECWjUqJF2rp1WrVrhpZdewq5du9CtWzc88cQTcHR0xOnTp3HkyBGMHTsWv/zyS5WPu1mzZli1ahWeeeYZtG3bFiNHjkTDhg2RnJyM8PBwODg4aL+JG/IcW7BgAbZt24Zff/0V7dq1w8CBA7Vz4/To0QN//PFHheJ/7LHHYGdnh5CQEAQGBkIIgcOHDyM8PBzt27cvNjdJ3759sXXrVowePRpDhgyBtbU1AgICMGnSJP0q7xGtWrXCyZMn0b59ewwYMEB7LGlpaVixYkWxzp1eXl6YMGECfvjhB7Rp0wZDhw5FRkYGduzYgR49eujsHN27d2/IZDIsWrQIly9f1rYgvPnmm6XGFBoaildeeQUrVqxAixYtMHbsWNja2mLnzp24fPkyunXrVqX5h6ri66+/RteuXbFw4ULs3r0bHTp00M7zI5PJsG7dOtjb21d5P3v27MHChQvRpUsXNGrUCO7u7oiJicH27dshk8mKHX/fvn3xwQcfYNGiRWjYsCGGDBmCoKAgZGVlISoqCmFhYejWrZv2OmBnZ4dvvvkG48aNQ/fu3YvN83P58mX06NGjWMsjACgUCsydOxdvv/022rZti8ceewyFhYXYs2cPvL29dc6Gre9nR2UMGDAAb775Jt555x00bdpUO89PQkICjhw5gpCQEO0ElZWtpxUrVmD//v3o3r07goKCYGdnhytXrmDnzp1wdnbGrFmz9I67wgw1bIz+B/8d6l7Wz6NDFP/8808xdOhQUa9ePaFQKISHh4fo2LGjeOONN0RERITOffzwww/abX388cdlxnPx4kUxZcoU4e/vLywtLYWzs7No3ry5mDVrlti3b1+xsmUN/dU1THXLli3iySefFA0aNBC2trbC3t5eNG/eXLz++uviwYMHJbbxySefiCZNmghLS0sBoMQQzoyMDPHuu++Kdu3aCVtbW2FlZSUCAwPFkCFDxOrVq4vNtVLWUPd/b7dIecODdTl9+rQYOHCgsLOzE3Z2dqJv377i2LFjZQ5zfnRYcnR0tFi0aJEIDQ0VHh4ewtLSUvj4+IhBgwaJHTt2lNjfzZs3xbBhw4SLi4uQJKnEMf7555+ic+fOws7OTjg6Oor+/fuLsLAwg9fHsWPHxOjRo7XnpZeXlxg4cKDYunVribKVOcfKkp6eLubNmye8vb2FUqkUjRs3Fh9//LG4fft2hYe6f/XVV2LUqFEiKChIWFtbC2dnZ9GmTRvx4YcflhhyX1hYKBYtWiSCgoKEhYVFib9dWXUnRNlD3Xv27CliY2PFhAkTRL169YRSqRRt27YVP/74o85t5eXliZdffln4+Pho50567733REFBgc6h7kI8vA4UzYFVdD0oq26KbN68WXTt2lXY2dkJpVIpmjVrJt555x2Rm5tboqwh30/lDemPiYkRzzzzjPD39xcKhUK4urqKkSNHilOnTlV6W6W5evWqmDdvnmjfvr1wc3MTlpaWIiAgQIwZM0YcPXpU52sOHz4sHn/8ceHl5SUUCoVwc3MTrVu3FvPmzSsxn48QQuzevVt07dpVWFtbCycnJzFixAgRERFR6t9Eo9GI999/X9SvX18oFArh5+cnFi5cKLKzs8us/8p8dpR2DglR9rny999/i4EDBwpnZ2dhaWkpfH19xahRo3S+rytaT7t27RJTp04VTZs2FQ4ODsLGxkY0atRIvPjiiyIyMlJnjIYmCVHFXlFERERUrqlTp2L9+vW4e/dupWblJsNjnx8iIiIyK0x+iIiIyKww+SEiIiKzwj4/REREZFbY8kNERERmhckPERERmRVOcqhDUlISdu3ahcDAwAo91I2IiIiMLzc3F5GRkRg4cCDc3NxKLcfkR4ddu3YVm3qeiIiIao+NGzdiwoQJpa5n8qND0eRTGzduRNOmTY0bDBEREVVIREQEJk6cWO4kkkx+dCi61dW0aVODP0WYiIiIqld5XVbY4ZmIiIjMCpMfIiIiMitMfoiIiMisMPkhIiIis8Lkh4iIiMwKkx8iIiIyK0x+iIiIyKww+SEiIiKzwuSHiIiIzAqTHyIiIjIrTH6IiIjIrDD5ISIiIrPC5IeIiIjMCpMfIiIiMisWxg6AiOhReXl5UKlU1boPS0tLWFlZVes+iMh0MfkhIpORl5cH/8BAJCYkVOt+6nl44F5kJBMgIjPF5IeITIZKpUJiQgKW/BgGKxu7atlHXk4Wlk/oCZVKxeSHyEwx+SEik2NlYwcr2+pJfoiI2OGZiIiIzAqTHyIiIjIrTH6IiIjIrDD5ISIiIrPC5IeIiIjMCpMfIiIiMitMfoiIiMisMPkhIiIis8Lkh4iIiMwKkx8iIiIyK0x+iIiIyKww+SEiIiKzwuSHiIiIzAqTHyIiIjIrTH6IiIjIrDD5ISIiIrPC5IeIiIjMCpMfIiIiMitMfoiIiMisMPkhIiIis8Lkh4iIiMwKkx8iIiIyK0x+iIiIyKww+SEiIiKzwuSHiIiIzAqTHyIiIjIrTH6IiIjIrDD5ISIiIrPC5IeIiIjMCpMfIiIiMitMfoiIiMisWBg7ACKiuigvLw8qlapa92FpaQkrK6tq3QdRXcTkh4jIwPLy8uAfGIjEhIRq3U89Dw/ci4xkAkRUSSaZ/GRkZOCzzz7Dzz//jMjISFhaWqJ+/fqYOnUqZs2aBYVCoS2bk5OD5cuX46effkJ8fDy8vLzw1FNPYfHixbCxsTHiURCRuVKpVEhMSMCSH8NgZWNXLfvIy8nC8gk9oVKpmPwQVZLJJT+FhYXo27cvzp49iylTpuCFF15Afn4+fv31V7zwwgs4fvw4Nm7cCABQq9UYMmQIwsLCMGnSJPTo0QMXLlzARx99hFOnTmHPnj2QyditiYiMw8rGDla21ZP8EJH+TC75OXjwIE6fPo0FCxbg448/1i5//vnn0aFDB2zevBlfffUV7O3tsX79eoSFheHFF1/E559/ri0bGBiIl19+GRs3bsTkyZONcRhERERkokyuWSQ9PR0A4O3tXWy5XC6Hp6cn5HI5LC0tAQAbNmwAACxYsKBY2eeeew7W1tba9URERERFTK7lp2vXrrC1tcUHH3wAX19fhISEIC8vDz///DN27dqF5cuXQ6lUQgiB8PBweHt7IyAgoNg2rK2t0aZNG4SHhxvpKIiIiMhUmVzy4+npie3bt+PZZ5/FuHHjtMutrKywdu1aTJs2DQCQkpKCnJwctGjRQud2fH19cfz4cWRkZMDBwUFnmfj4eMTHx5dYHhERYYAjISIiIlNkcskPADg6OqJx48bo1asXBgwYgJycHKxfvx4zZ86EJEmYOnUqcnJyAABKpVLnNopGP+Tk5JSa/KxevRpvvfVW9RwEERERmSSTS34uXLiAbt264aWXXsIHH3ygXT5x4kR07doVzz//PIYOHaodxp6fn69zO3l5eQBQ5nD32bNnY8SIESWWR0REYOLEiVU5DCIiIjJRJpf8fPbZZ8jPz8fjjz9ebLlMJsPYsWNx4sQJnDp1CkOGDIGNjQ1iYmJ0bicmJgYODg6ltvoAgJeXF7y8vAwaPxEREZk2kxvtFRsbC+DhHD7/VlhYqP1XkiR06NABcXFxiIqKKlYuNzcX58+fR8eOHas/YCIiIqpVTC75ad68OQDg+++/L7a8oKAAmzZtglwuR4cOHQAAkyZNAgB88sknxcp+9dVXyM3N1a4nIiIiKmJyt71eeukl/PDDD/jqq68QExODgQMHIicnBxs3bsTFixcxf/58+Pj4AACmTZuGDRs24IsvvkB6erp2hudVq1ahV69e7LdDREREJZhc8uPv74/w8HC8/fbb2LNnD3bu3AlLS0u0aNEC3377LaZPn64tK5fLsWPHDixfvhxbtmzB5s2b4eXlhQULFmDJkiWQy+VGPBIiIiIyRSaX/AAPH0+xdu3aCpW1s7PDihUrsGLFimqOioiIiOoCk+vzQ0RERFSdmPwQERGRWWHyQ0RERGaFyQ8RERGZFSY/REREZFaY/BAREZFZYfJDREREZoXJDxEREZkVJj9ERERkVpj8EBERkVlh8kNERERmhckPERERmRUmP0RERGRWmPwQERGRWWHyQ0RERGaFyQ8RERGZFSY/REREZFaY/BAREZFZYfJDREREZoXJDxEREZkVJj9ERERkVpj8EBERkVlh8kNERERmhckPERERmRUmP0RERGRWmPwQERGRWWHyQ0RERGaFyQ8RERGZFSY/REREZFaY/BAREZFZYfJDREREZoXJDxEREZkVJj9ERERkVpj8EBERkVlh8kNERERmxeDJT2pqKrKzsw29WSIiIiKD0Cv52bdvH1555RWkpqZqlz148AA9e/aEm5sbXFxcMH/+fIMFSURERGQoFvq86IsvvsDly5exYsUK7bKXX34Zhw8fRoMGDZCVlYXPPvsMISEheOKJJwwWLBFRWQrUGqTlFCAzvwDZ+WrkF6pRoBYQQkAmSbCQS1BoCmEV1A6xaXmwt7eHJEnGDpuIapheyc+FCxfQs2dP7e+5ubn45Zdf0L9/f+zatQuZmZlo2bIlvv76ayY/RFQthBB4kJmP2LRc3E/PQ0JGHjLyCiv0Wo8nlmPwqnDYKy3Qys8RXRu4oVcjdzT1YjJEZA70Sn4ePHgAb29v7e8nT55EXl4epk6dCgCwt7fHsGHD8PvvvxskSCIiAFBrBKJTcnDjQSYik3KQW6DWrnO1tUQjdzs421rC3soCtkoLWCnkUMgkyCQJaiFQqBZIy8zCps/fwYtvvItbyXk4HZmKo7eSseKf62jsYY+x7X3xREc/OForjHikRFSd9Ep+lEolcnNztb8fPnwYkiShR48e2mUODg5ISUmpeoREZPbScwtwKSYdV+MztAmPq50lmnk7wM/ZGp6OVlBayCu0LSeLQmSd34nXBvwEBwcH5BWocSYqFf9cvo8/LsTh3R0R+HzfTUztGoinu9WHow2TIKK6Rq/kJygoCPv379f+/uuvv6Jhw4bw8fHRLouOjoabm1vVIyQis/UgMw+n7qbgduLDEaRO1gq08XNCIw87ONlYGmQfVgo5ujZwQ9cGbnhzWFP8dSEe/zlwC1/sv4VNJ+/hjaFN8VhbH94OI6pD9Ep+pkyZgpdeegmdO3eGpaUlLl26hKVLlxYrc/HiRTRu3NggQRKReUnPLcTe23HapCfIzRZt/Jzg52xdrUmI0kKOMe19MaqtD34/F4v3d0Rg/s8X8Pu5WHw6rg3c7JTVtm8iqjl6JT/PPvssTpw4gS1btkAIgeHDh+PVV1/Vrr98+TIuXbqE5cuXGyxQIqr7clRqOPWahp8vJkIjgOB6tugc5Ip69jWbdMhlEsa290W/pu54b0cEfj4dgyGfHcaX49uhU5BLjcZCRIanV/KjUCiwadMmfP3115AkCfb29sXWe3p64ty5cwgMDDREjERkBo7dTsIrWy/AsfMYuNgo0KuJB3ycrI0ak5ONJVaMbY2uDdzw2q+XMH7NCXz0eCs81tbXqHERUdVUaYZnBweHEokPALi5uaF169ZwdHSsyuaJyAwUqjX4YOc1jF9zEklZKqTs/QaPtXA1euLzqJFtfLD9ha5wt1di3pYLWHvkrrFDIqIqqFLyk5iYiK+//hpz587F008/XWz5qVOnio0IIyL6t4SMPIxfcxJfh91Gaz8n/PJ0O2Se+QMyE+xc3MjDHr88G4oG7nZ4+6+rWHXwlrFDIiI96Z38rF27FoGBgXj++efxxRdfYN26ddp1CQkJ6NKlCzZt2mSQIImo7rkal4GRXx7FqcgUTO8ahK2zu8DfxXRae3TxdrLG1tld0MTTHiv+uY5NJ+8ZOyQi0oNeyc+ePXswa9YsNGrUCL///jueffbZYutbtGiB5s2bY9u2bYaIkYjqmIPXH+Dxr48hJVuFlePaYMnwZrC0MPhzlquFs60lNszohABXG7yx7RJ2XIo3dkhEVEl6XW0+/PBDeHl5ISwsDCNGjIC7u3uJMq1atcLVq1erHCAR1S3/XL6Pp9efhsJChh9mdMKotj7lv8jEuNtbYeOMznCzU2L+z+dxOTbd2CERUSXolfycPn0aw4YNg4ODQ6llfH19cf/+fb0DI6K656+LcXh+01k42Vji59ld0Lm+q7FD0pufiw1WT2oPjQaY/cMZJGflGzskIqogvZIflUoFW1vbMsukpaVBLq/YdPNEVPftuZqAuT+dh5udJbbMDkEjj5IjRWubdv7OeGdUC8Sm5eKFTeeg1ghjh0REFaBX8hMYGIgzZ86UWebkyZOc4ZmIAACn7qbghU1n4WStwOaZIQiuZ2fskAzmiY5+mBjij+N3kvF12G1jh0NEFaBX8jNy5EgcPnwYW7du1bl+3bp1uHjxIsaMGVOl4Iio9ruRkIkZ68NhIZPw/bROqF+HEp8ibw5thobudvh0zw1ciE4zdjhEVA69kp9XXnkF/v7+eOqppzBu3DgcP34cAPDll19i3LhxmDVrFho2bIgXX3zRoMESUe2Smq3CjPXhyFWp8c3kDmjpWzcnPrVSyPHZk20hkyS8tOU8clRqY4dERGXQ6/EWzs7OCAsLw+TJk4u1/syZMwcA0L17d2zatKncfkFEVHcVqDV47seziE7JxXuPtUTXBm7GDqlaNfN2wCuDGuOdvyOw6lCUscMhojLolfwAgL+/Pw4ePIiLFy/i+PHjSE5OhqOjI0JCQtC+fXtDxkhEtdB7OyJw/E4yJncJwPjO/sYOp0ZM6xqEvy7GY2N4LCw9Gxo7HCIqhd7JT5FWrVqhVatWhoiFiPSUl5cHlUpVrfuwtLSElZVVhcr+c/k+1h2NRKdAFywe1qxa4zIlcpmED8e0wtDPD8N18ByO/iIyUXolP7m5uUhMTISnpycsLS1LrM/Pz0dCQgLc3d0rfLEkIv3k5eXBPzAQiQkJ1bqfeh4euBcZWe57OjolB6/8cgEutpb4/Km2UMhrx8zNhtLY0x4zuvhi9VGBS/ezEaLj4c9EZFx6JT/Lly/HypUrERsbCxcXlxLrs7Oz0aRJE7z88stYvnx5lYMkotKpVCokJiRgyY9hsLKpnpFUeTlZWD6hJ1QqVZnJT6Fagzk/nUNGXiHWTesIT0fz/PIzs6s/vvzrJM7KfNDCvxB2yio3shORAen1lWznzp3o16+fzsQHAFxcXNCvXz/89ddfVQqOiCrOysYOVrbV9FPBpOrrsNs4dy8NM7sHoXfjko+9MReWFjKk7luDAo3A0VtJxg6HiP5Fr+QnMjISjRo1KrNMo0aNEBkZqc/miagWuhKXjs/23UQjDzssGMAJTnNvh8PPSYlr9zMRl5Zr7HCI6BF6JT8FBQWQycp+qSRJyMvL0ysoIqpd8gvVWPDzBQgB/N8TbWCl4KNtACA0wAEyCQi7kQgh2PmZyFTolfzUr18fYWFhZZY5ePAgAgIC9AqKiGqXrw/ewbX7mXihTwO08KmbExnqw8naAq18nfAgMx83ErKMHQ4R/Zdeyc+IESNw5swZrFixQuf6Dz74AGfPnsWoUaP0DiwjIwNvvvkmmjZtCmtra7i4uKBz587YuHFjsXI5OTl47bXXEBgYCKVSicDAQCxatAg5OTl675uIKu7Wgyz858AtNPKww3O9Ghg7HJPTKcgFlhYyHLudhEKNxtjhEBH0HO318ssv48cff8SiRYvw888/Y8CAAfDx8UFsbCx27dqF8+fPw9/fH6+88opeQcXGxqJ3795ISkrC1KlT0bx5c2RnZ+PGjRuIivrfzKlqtRpDhgxBWFgYJk2ahB49euDChQv46KOPcOrUKezZs6fc23NEpD+NRuD13y+hQKPB+6NbwdKC77d/s1bI0THAGUdvJ+NSTDra+jsbOyQis6f34y0OHjyI8ePH48SJEzh79iwkSdLe0w4NDcXGjRvh7Kzfm3zy5MnIzMzEhQsX4OfnV2q59evXIywsDC+++CI+//xz7fLAwEC8/PLL2LhxIyZPnqxXDERUvl/OxODU3RRMDPFH+wB+qJemjZ8TLsSk49TdFDTzdoDSgn2iiIxJ78knAgMDcezYMZw9exYnTpxAWloanJycEBISgnbt2ukd0NGjR7F//358+umn8PPzg1qtRm5uLuzsSg613bBhAwBgwYIFxZY/99xzWLx4MTZs2MDkh6iapOcU4MN/rsHNTolXBjUxdjgmzUIuQ+f6LtgX8QAXotPRKUj3NCFEVDOqPPNWu3btqpTs/Nvff/8NAAgODsaYMWPw559/oqCgAF5eXnjuueewaNEiyOVyCCEQHh4Ob2/vEh2rra2t0aZNG4SHhxssLiIq7tO9N5CcrcInj7eGg5XC2OGYvKaeDjgdmYqz91LR2tcRSo6IIzIak5t2NCIiAgAwY8YMBAUF4dtvv4UkSVi1ahUWL16Me/fu4ZtvvkFKSgpycnLQokULndvx9fXF8ePHkZGRAQcHB51l4uPjER8fX2oMRKRbRHwGNhyPRPsAZzzW1sfY4dQKcpmETkEu2HM1Aeei0xBS39XYIRGZLb2Tn4KCAmzfvh2nTp1Camoq1Gp1iTKSJGHt2rWV2m5mZiYAwNbWFocOHYJSqQQAjBs3Ds2aNcO3336LBQsWwMbGBgC06/+taAr+nJycUpOf1atX46233qpUfETmTgiBt/68AgHgrRHNIZNJxg6p1mjiYY/wuyk4dy8NbfycOB8SkZHolfzExcWhf//+uHbtWpkTd+mT/FhbWwMAxo8fXyyxsbS0xIQJE7B8+XIcOHAAjz/+OICHD1HVpWiCxaIkSZfZs2djxIgRJZZHRERg4sSJlYqbyFzsi3iAE3dS8GRHP87pU0kymYTOQS7YdTUB5+6loUswW3+IjEGv5GfBggWIiIjAU089hZkzZ8LPzw8WFoa5g1Y0usvLy6vEuqJlKSkpcHFxgY2NDWJiYnRuJyYmBg4ODqW2+hRtT9d+iEi3Qo3A+zsjYK2QY37/sh9xQ7o18rTHqcgUnItORRt/J1iz9YeoxumVsezevRs9evTAjz/+aOh4EBISgq+++grR0dEl1hUt8/DwgCRJ6NChAw4dOoSoqKhinZ5zc3Nx/vx5hIaGGjw+InP2+/n7uJ2YjTl9G8LdwTyf2F5VMklC5yBX/HPlPs5GpaJrAzdjh0RkdvSakSwvLw+dO3c2dCwAgJEjR8LJyQk//PCDtv8PAGRlZWH9+vVQKBQYMGAAAGDSpEkAgE8++aTYNr766ivk5uZq1xNR1UmW1lh1OApudkrM7lHf2OHUag097OBqa4kLMWnIURUaOxwis6NXy0+LFi2KzbRsSI6Ojvjss88wZcoUdOzYETNmzIAkSfjuu+8QGxuLd999V3trbNq0adiwYQO++OILpKena2d4XrVqFXr16sV+O0QG5NDpMSRnF+Ddx1rAVmlyA0VrlYetPy7Ycfk+zkaloVtDtv4Q1SS9Wn4WLlyIP/74A1evXjV0PAAezvC8Y8cOeHh44K233sKSJUtgb2+PzZs34/XXX9eWk8vl2LFjBxYuXIiDBw/i2Wefxfbt27FgwQL89ddfkMt5L53IELJVajh0HI36rtYY16H0Wdep4hq4P2z9uRibhlxVydGyRFR99Pr65u7ujuHDhyM0NBRz585F+/bt4eTkpLNsjx499Aps8ODBGDx4cLnl7OzssGLFilIfskpEVXc6JhMySyvM6xMECzmf32UIkvRw3p+dl+/jXHQqQoPZ+kNUU/RKfnr16qV9ltfbb78NSSp9ng9d8/8QUe2RnJWP6w9ykRd1ET0adDN2OHVKA3c7ONsocCE6He38nTnvD1EN0Sv5WbJkSZkJDxHVHcfvJEMASD24DpL0vLHDqVNkkoSOgS7YfTUB5znrM1GN0Sv5WbZsmYHDICJTlJCRh9uJ2Qh0ViLq/k1jh1MnNfawx8m7KTgfnYa2/k584jtRDeDNeyIq1fE7yQCADn72Ro6k7pLJJHQMdEZ+oQYXYtKNHQ6RWahS8lNQUIB//vkHn376Kd5++23t8ry8PDx48AAajabKARKRccSm5SIqOQeNPOzgasOntlenJp4OsLeywLl7qVAV8rpJVN30Tn7++ecfBAYGYujQoViwYEGxW2Hnz5+Hl5cXtmzZYogYiaiGCSFw/HYyJAnsh1ID5DIJHQNckFegwaVYtv4QVTe9kp/Tp09j1KhRkCQJn376KcaPH19sfUhICIKCgvD7778bJEgiqln3UnIQm5aLZl4OcLaxNHY4ZqGptz3slBY4E5WKAjVbf4iqk17Jz9tvvw0bGxucPn0ac+bMQcOGDUuU6dixIy5cuFDlAImoZgkhcPxOMuT/nYeGaoaFTIb2Ac7ILVDjMlt/iKqVXsnP0aNHMWrUKHh6epZaxs/PD/Hx8XoHRkTGcScpGwkZ+Wjh4wAHK/b1qUktvB1gYynHmXupKGTrD1G10Sv5ycrKgptb2bOR5uTksMMzUS1T1OpjIXs4/wzVLAv5w9af7Hw1rsZnGDscojpLr+THx8cHV65cKbPM+fPnUb8+n/xMVJvcfJCF5CwVWvs58eGlRtLSxxHWCjnCI1Oh1ghjh0NUJ+mV/AwePBi7du3CkSNHdK7fuXMnjh07hmHDhlUpOCKqORohcPJOChRyCe39nY0djtlSyGVo6++ErPxCRLD1h6ha6JX8LFq0CE5OThgwYABeffVV7dPd//77b7z66qt4/PHH4eXlhfnz5xs0WCKqPjfuZyIlR4U2fk6wtuQsw8bU2tcJVhYyhEemsPWHqBro1a7t4+OD3bt344knnsBHH32kXT5ixAgIIRAcHIzffvut3H5BRGQaNBqBE3dTYGkhQzu2+hidpYUMbfydcOJOCq4nZKKZl4OxQyKqU/S+qd+uXTtcv34df//9N44fP47k5GQ4OjoiJCQEI0eOhIUF+wsQ1RYR9zOQnluAkCAXPlncRLTxdcLZqDSER6agiac9ZHyYNJHBVClDkcvlGDFiBEaMGGGoeIiohqk1AifvpsDqv60NZBqUCjna+DnhVGQKbiZkobEnn69GZCh8sCmRmbsSl47MvEK0C3DmE8VNTFt/JyjkEk5FpkAI9v0hMhS9Wn42bNhQ4bKTJ0/WZxdEVAMK1RqER6bCWiFHa18nY4dD/2KlkKOVrxPORKXi1oMsNPRg6w+RIeiV/EydOhVSOfefhRCQJInJD5EJuxyXgaz8QnRv6AZLCzYEm6J2/k64EJ2GU5EpaOBuV+61l4jKp1fys27dOp3L09LSEB4ejp9++gljxozB0KFDqxQcEVWfArUG4ZEpsLGUo6WPo7HDoVLYWFqgpa8jzt1Lw92kbNSvZ2fskIhqPb2SnylTppS5ftq0aRg6dCjmzJmjV1BEVP0uxqQjR6VGz0b1oJCz1ceUtfd3xsWYdJy8m4IgN1u2/hBVUbVc8fr27YtBgwZhyZIl1bF5IqoiVaEGZ6JSYae0QAtvziFj6mz/+3d6kJmPqOQcY4dDVOtV29e9Ro0a4fTp09W1eSKqgvMxacgtUKNToAss2OpTK7QPcIZcknDyLkd+EVVVtV31rl69yqZZIhOUX6jG2ahUOFhZoBlbfWoNeysFmnrb435GHqJTc40dDlGtZtDkR6PRICoqCm+++SZ27tyJ7t27G3LzRGQA5+6lIb9Qg05BLpDL+AWlNukY4AKZBJy6m2LsUIhqNb06PMtksjJbdYQQcHV1LfbcLyIyvrwCNc7dS4OTtQJNPdnqU9s4WCvQxNMBV+MzEJNubexwiGotvZKfHj166Ex+ZDIZnJ2d0alTJ0ybNg316tWrcoBEZDhnolKhUmvQu349yNjqUyt1DnLBtfsZOHUv09ihENVaeiU/Bw8eNHAYRFTdsvMLcT46DS42lmjEmYJrLQdrBVr5OOF8TBpsGoUaOxyiWonDPIjMxKnIFBRqBLoEu/IJ4bVcxyBnKGQSnHpMQqGGI7+IKovJD5EZSM8twOXYdHg6WCG4nq2xw6EqsrG0QCsvWyhc/fDHxQRjh0NU6+h122v69Ol67UySJKxdu1av1xKR/o7fSYZGAF0buHIKijqilZctTt2MxVeHo/Bkl2BYKeTGDomo1tAr+fn++++1F1Bdk21JklTqciY/RDUrMTMf1+9nIsDFBr7ONsYOhwzE0kKG9ONbIO87CxuOR2JWj2Bjh0RUa+h12+v27dsYPnw4nJycsHTpUhw4cAARERE4cOAAlixZAicnJ4wcORK3b9/G3bt3tT937twxdPxEVI5jt5MAAKENXI0cCRla5rmd8HJQ4j8HbiM9t8DY4RDVGnq1/Gzbtg2HDx/G2bNnERAQoF3euHFj9OzZE1OmTEH79u2xfft2vPTSS4aKlYgqKTY1F5HJOWjkYQd3eytjh0OGpi7Acz0CsPivG/jm0G0sHNjE2BER1Qp6tfx88803eOKJJ4olPo8KCgrC448/jm+++aZKwRGR/oQQOHo7CTIJ6FKfrT511bAW7mjsYY9vD99FXBofe0FUEXolP5GRkXByciqzjLOzMyIjI/XZPBEZwN2kbMSn56G5tyOcbCyNHQ5VE7lMwutDmyK/UIOPdl03djhEtYJeyY+bmxt27dpV6nohBHbt2gVXV37bJDIGtUbgyK0kKOQSOge5GDscqmY9G9VDz0b18Pu5WFyITjN2OEQmT6/k5/HHH8f58+fxxBNP4O7du8XW3b17F+PGjcPFixcxbtw4gwRJRJVzOS4dqTkFaO/vDFulXl37dMrIyKj2H9LPG0ObQiYB7/4doXO0LRH9j15XxeXLl+PIkSP45Zdf8Pvvv8PHxwceHh5ISEhAbGws1Go1OnbsiGXLlhk4XCIqT36hGifvpMDWUo52Ac4G2WaBKh+QZPDz8zPI9spTqC6skf3UJY087PFUJ3/8ePIedl25j0EtvIwdEpHJ0iv5sbOzw5EjR/Dxxx9j3bp1uH37Nu7duwcAaNCgAaZNm4YFCxbA0pL9DIhq2unIVOQWqNGvqTsUcsNM4q4uLACEBq+t2w0Hp+q7nZ2elIAPZw6BWq2utn3UZfP6N8L283F4f+c19G7iDqUFJz4k0kXv9nBLS0u8/vrreP3115GVlYX09HQ4OjrCzs7OkPERUSVk5BXgXHQaXO0s0dTLweDbV9rYwcq2+t7jeTlZ1bZtc+Bmp8RzvYOx4p/r+OF4FJ7uXt/YIRGZJIN8LbSzs4OPjw8THyIjO347GWqNQPcGbnx4qZma3jUIPk7W+HzfTaRmq4wdDpFJqlLyk5iYiK+//hpz587F008/XWz5qVOnkJvLOSeIakpidgGu/fcxFgGufHipubJSyPHq4CbIyCvEp3tvGDscIpOkd/Kzdu1aBAYG4vnnn8cXX3yBdevWadclJCSgS5cu2LRpk0GCJKLynYjKgASgW0M3Y4dCRja8lRfaBzjjx5P3cCMh09jhEJkcvZKfPXv2YNasWWjUqBF+//13PPvss8XWt2jRAs2bN8e2bdsMESMRlcO6URfEZajQzNsBbnZKY4dDRiZJEpYMawa1RuDtv65y6DvRv+jV4fnDDz+El5cXwsLC4ODggHPnzpUo06pVKxw/frzKARJR2fILNXDpPQOWcomPsSCt1n5OGNPOF7+ejcG+iAfo18xDr+3k5eVBparevkOWlpawsuKz56jm6JX8nD59Gk8++SQcHEofTeLr64v79+/rHRgRVcyGkzGwcPJEOx87g05oSLXfK4MaY+fleLy7IwI9GtWDpUXlGvvz8vLgHxiIxISEaorwoXoeHrgXGckEiGqMXldKlUoFW9uyO1SmpaVBLuccE0TV6X56Hr49Fo2C5Bi06NTB2OGQifFwsMLzvRvgo13XseF4ZKWHvqtUKiQmJGDJj2Gwsqme0bx5OVlYPqEnVCoVkx+qMXolP4GBgThz5kyZZU6ePInGjRvrFRQRVcyH/1xDboEGKfu/hXxoR2OHQyZoRrcgbD51D5/tu4nH2vrAVY8+YVbVPL8TUU3Tq8PzyJEjcfjwYWzdulXn+nXr1uHixYsYM2ZMlYIjotKdiUrF7+di0S3YGXl3Ths7HDJRVgo5Xh/SFJl5hfhkD4e+EwF6Jj+vvPIK/P398dRTT2HcuHHajs1ffvklxo0bh1mzZqFhw4Z48cUXDRosET2k0Qgs//MKLGQSFvblLL5UtsEtPNEpyAU/nbqHiHg+PJZIr+TH2dkZYWFh6NatG7Zu3Yrdu3dDCIE5c+Zg69atCA0Nxb59+8rtF0RE+vntXCwuxKRjSmgggtxsjB0Ombiioe8CwPI/OfSdSO+hIf7+/jh48CAuXryI48ePIzk5GY6OjggJCUH79u0NGSMRPSIrvxAf/nMNrraWmNO3IVDAmdSpfC18HPFEez9sOR2NXVcSMKiFp7FDIjKaKo+LbdWqFVq1amWIWIioAr7cfwuJmfl4f3RLOForkMHkhyro5YGN8feleLy3IwK9m9TjU9/JbBnkwaZFkpOT8fvvv2PXrl1Qq9WG3DQRAYhMysZ3R+6imZcDnujgZ+xwqJapZ6/EC30a4F5KDtYdjTR2OERGo1fy89VXX6Fz585ISUnRLjtz5gyaNGmCsWPHYsiQIQgNDUV2drbBAiUi4N0dEVCpNVg6vBnkMj61nSpvWtdABLjaaFsQicyRXsnPli1bIEkSXFxctMsWLlyI1NRUTJs2DUOGDEF4eDi+/vprgwVKZO4O30zEnqsJGNrKC535GAvSk9Li4dD3rPxCfLzrurHDITIKvZKfmzdvFuvnk5SUhLCwMMyYMQPffvst/vzzT3Ts2JFPdScykAK1Bsv/vAqlhQyLBjcxdjhUyw1o5oEu9V3x85loXI5NN3Y4RDVOr+QnOTkZ7u7u2t+PHj0KAHjssce0y7p3746oqKgqhkdEAPDjiSjcfJCF2T2D4evMoe1UNZIkYcnwZpDAoe9knvRKflxcXJCUlKT9PSwsDDKZDKGhodplkiQhLy+v6hESmbnUbBU+3XsTXo5WeLZnsLHDoTqiqZcDnuzkj1ORKdhxiQ+hJvOi11D3pk2b4s8//8S7774LuVyOn376CR07diz2lPfIyEh4enIeCaKqWrn3BtJzC7B8ZHNYW3JosqFkZFTfTMfVuW1DWtC/Ef68EIf3dkSgb1N3WCl4fpF50Cv5mTt3LkaNGgVfX19YWFggJycHK1asKFbmxIkT6NSpk0GCJDJXNxMysfHkPbT1d8KI1t7GDqdOKFDlA5IMfn7VP1VAobqw2vdRFa52Sszt2xDv/B2BtUfu4vneDYwdElGN0Cv5GTFiBL7++mt88803AIAJEyZg4sSJ2vUHDx5EVlYWBg4caJgoiczUO39HQK0RWDq8OSSJQ9sNQV1YAAgNXlu3Gw5O1TNqLj0pAR/OHFIr5jub3CUQP568h/8cuIWx7X3h4WBl7JCIqp3ekxzOmjULp0+fxunTpzFv3rxi63r16oXU1FTMmjWrygECgEajQUhICCRJQr9+/Uqsz8nJwWuvvYbAwEAolUoEBgZi0aJFyMnJMcj+iYzhwPUHCLuRiNFtfdDGz8nY4dQ5Shs7WNlWz4/SpvY819DSQoY3hjRFjkqNFf9w6DuZB4PO8FxdVq5ciStXruhcp1arMWTIEHz44Yfo0aMH/vOf/2D48OH46KOPMHz4cGg0mhqOlqjqCtQavPPXVVgr5Fg4qLGxw6E6rm9Td3Rv6IZfz8bgQnSascMhqnYmn/zcuXMHixcvxjvvvKNz/fr16xEWFoYXX3wRGzZswNNPP40vvvgCH374Ifbv34+NGzfWcMREVbfxRBRuJ2bjmZ7B8HK0NnY4VMdJkoTFwx7OGv72Xxz6TnWfySc/M2fORPPmzfHiiy/qXL9hwwYAwIIFC4otf+6552Btba1dT1RbpGarsPK/Q9tn9ahv7HDITDTysMeTHf1wOioVeyMeGDscompl0snPmjVrcOjQIaxZswYyWclQhRAIDw+Ht7c3AgICiq2ztrZGmzZtEB4eXlPhEhnEZ/tuIj23AK8NbsKh7VSj5vZtCGuFHCv+uQa1hq0/VHfpNdqrJsTFxWHhwoVYsGABWrdurbNMSkoKcnJy0KJFC53rfX19cfz4cWRkZBSbg6hIfHw84uPjSyyPiIioWvBEerr1IBM/nIji0HYyCncHK8zoFoQvD9zCr2djMKiRo7FDIqoWJpv8PPvss3Bzc8PSpUtLLVM0mkupVOpcb2VlpS2nK/lZvXo13nrrLQNES2QYRUPblwxrxqHtZBSzetbHjyejsHLPDfSu397Y4RBVC5NMfn766Sf88ccf2LNnD6ytS+/saWPz8BlH+fn5OtcXPV6jqNy/zZ49GyNGjCixPCIioti8RUQ14cD1Bzh4PRGPtfVBW39nY4dDZsrBSoHnezfAO39H4KczccYOh6hamFzyk5+fjzlz5mDAgAEIDAzErVu3iq3Pzc3FrVu3YG9vD3d3d9jY2CAmJkbntmJiYuDg4KCz1QcAvLy84OXlZfBjIKostUbgvb8jYKWQ4RUObScjm9QlAOuORmLN0WhIytozZxFRRVVLh+fk5GQsX74cb7/9dqVfm5ubi8TEROzevRsNGzYs9gMAx44dQ8OGDTF37lxIkoQOHTogLi6uxBPkc3Nzcf78eXTs2NEgx0RUnX49G4ObD7Iws3t9Dm0no1NayDG/fyNk5BXCsfMYY4dDZHDV0vKTlJSEZcuWPZw7YvHiSr3W1tYWW7du1bnu8ccfR8uWLbFkyRLtc3kmTZqEQ4cO4ZNPPsHnn3+uLfvVV18hNzcXkyZN0v9AiGpAXoEan+65AWcbBYe2k8kY1dYHXx+8iesdRiBbpYYVG4CoDqmW5MfNzQ1LlizRq8OmQqHA2LFjS13v7u5ebP20adOwYcMGfPHFF0hPT0ePHj1w4cIFrFq1Cr169WLfHTJ5649FIj49D0uGNYO9lcLY4RABAOQyCXN7B+GFn3NwNjYL/Z058ovqjmpJflxdXbFs2bLq2HQJcrkcO3bswPLly7FlyxZs3rwZXl5eWLBgAZYsWQK5nPOkkOlKzynAfw7cgq+zNSaE+Bs7HKJiugc7Iz/2Gq5JTdA5twAO1kzOqW4wuQ7PZSltynU7OzusWLECK1asqOGIiKpmVdgtZOQV4q2RzaG0YKJOpkWSJKQd/gHKJ9/FqcgU9GvqYeyQiAzCpGd4JqrL4tNz8f3RSDT1csDI1j7GDodIp7yoC/Cyt8TV+Ayk5aiMHQ6RQVSp5efPP//E+fPnERMTg4KCghLrJUnC2rVrq7ILojpr5Z6byC/U4NVBjSGTcUJDMl0d/ezxx9VknLybgoHNPY0dDlGV6ZX8REVFYfjw4bhy5UqZT/9l8kOk260Hmdh6Jhpd6ruiZ6N6xg6HqExeDpbwd7HBtfuZ6BDgDFc73bPqE9UWeiU/c+bMweXLlzF9+nRMnjwZPj4+sLCoVd2HiIzq0703oRHAK4Ma8zEWVCt0qe+Keyk5OHk3BUNacnJYqt30ylj279+PgQMH4ttvvzV0PER13vX7mdhxKR59mrjzMRZUa3g6WiHIzRY3H2QhMTMf9ezZ+kO1l14dnhUKBVq2bGnoWIjMwmf7bkAI4KV+DY0dClGlhNR3AQCcuJNs5EiIqkav5Kdr1664fPmyoWMhqvMi4jOw49J99Gvqjla+TsYOh6hS3O2tEFzPFneSsvEgM8/Y4RDpTa/kZ/ny5Th06BB++uknQ8dDVKd9tvcmAOClfo2MHAmRfjoHuQIATt1NMXIkRPrTq89P27ZtsW/fPgwdOhSrV69Gu3bt4OhYcupzfZ7tRVRXXYlLxz9X7qN/Mw+08OGjAqh2qmevRH03W9xOzEZSVj7cOPKLaiG9kp/09HS8/vrrSElJQVhYGMLCwnSWY/JD9D8rta0+7OtDtVunIBfcScpG+N0UDObIL6qF9Ep+5s2bhwMHDqBfv36YNGkSvL29OdSdqAyXY9Ox52oCBjX3RHNvtvpQ7ebhYIVAVxvceJCFztkquNhaGjskokrRK2P566+/EBoait27dxs6HqI6qajVZy5bfaiO6BTkgsjkHJyKTMEgzvpMtYxeHZ5zc3MRGhpq6FiI6qRr9zOwN+Jhq09TLwdjh0NkEF6O1vB3scGN+5lI5TO/qJbRK/lp27Yt7ty5Y+hYiOqk1WEP3yvP9go2ciREhtUpyAUCQHgkR35R7aJX8rN48WL8+eefOHLkiKHjIapTolNy8MeFOHRt4IrWfk7GDofIoHycrOHrbI1r9zORnlvy4dZEpkqvPj/x8fEYNmwY+vTpg/Hjx6N9+/Y6h7oDwOTJk6sUIFFttubwHag1As/2bGDsUIiqRecgF/x6NhbhkSno19TD2OEQVYheyc/UqVMhSRKEENiwYQM2bNhQ4uGMQghIksTkh8xWUlY+toRHo5WvI7o2cDV2OETVwsfJGt5OVoiIz0CnQBc4WCuMHRJRufRKftatW2foOIjqnHVH7yK/UINnewbzye1UZ0mShE6BLth2Pg6no1LRp4m7sUMiKpdeyc+UKVMMHQeRUeTl5UGlMvxIlaz8Qmw4FokAF2v0CHYy+PaJTIm/iw08HaxwNS4DHQOdYW/F1h8ybZyZkMxWXl4e/AMDkZiQYPBtO3QaDefe0xG57QME/d9F3IuMhJWVlcH3Q2QKJElC5yAXbL8QhzNRqejVmK0/ZNqY/JDZUqlUSExIwJIfw2BlY2ew7RZqBDafewBJAiYufAPvTuoFlUrF5IfqtABXG7jbK3E5LgOdglxgY8mPFzJdep2d9evXr1A5SZJw+/ZtfXZBVGOsbOxgZWu45OdybDpyCjTo3sANtnZs/ifzIEkSOgQ6Y8el+zgfnYbQYDdjh0RUKr2SH41Go7MDZ1paGtLT0wEA3t7eUCh44SfzIoTA+eg0WMplaO7jAJGfa+yQqI7LyMgwmW03qGcHZxsFLsSko32AM5QW8mqKjKhq9Ep+IiMjS11369YtzJkzB9nZ2di1a5e+cRHVSvdScpCcrUJbPycoLeTIyzd2RFRXFajyAUkGPz+/at9XobqwQuUkSUL7AGfsjXiAizHp6BjoUs2REenH4DdlGzRogN9++w0tWrTAW2+9hffff9/QuyAyWeei0yABnM2Zqp26sAAQGry2bjccnKpnHqn0pAR8OHMI1Gp1hV/TxNMBJ+6k4Ny9NLT1c4KFXK8HCRBVq2rpkWZlZYX+/ftj8+bNTH7IbKRkqxCVnIPgerZw5ERvVEOUBu6z9qi8nKxKv0Yue9j6E3YjEVfiM9Da18nwgRFVUbWl5BYWFrh//351bZ7I5JyPTgMAtPV3Nm4gREbW3NsB1go5zkSlQq0Rxg6HqIRqSX6SkpLw+++/18i9aCJTkFegRkR8BtztlfB25JB2Mm8KuQxt/JyQmVeIGwmZxg6HqAS9bnstX75c5/LCwkJER0dj+/btSE9P5y0vMhuXY9NRqBFo6+/ER1kQAWjt64gzUak4HZmKJp72fF+QSdEr+Vm2bFmZ6x0cHPDmm2/ilVde0WfzRLWKWiNwISYdtpZyNHS3N3Y4RCZBqZCj5X8ToDtJ2QiuVz39koj0oVfyc+DAAZ3LZTIZnJ2d0aRJE1hYcHZPMg+3HmQhK78QXYJdIZfx2y1RkbZ+TjgfnYbwyBTUd7Nl6w+ZDL0ylJ49exo6DqJa63x0GuQyCS29HY0dCpFJsVVaoJmXAy7FpiM6NRf+LjbGDokIQDWO9iIyBwkZebifkYcmnvawtuRstkT/1j7AGZIEnI5MMXYoRFoVbvnRaDR67UAmY35FddfFmIePc+FcJkS6OVor0NjDHtfuZ+J+eh48ORqSTECFMxOFQlHpH0tLy+qMnciocgvUuJ6QCS9HK9SzVxo7HCKT1SHg4dxXp6PY+kOmocItP35+fhXurJaVlYXk5GS9gyKqDSLiMqDWCLTyZV8forK42ilR380WtxOzkZyVD1c7flkg46pw8lPWw0yLFBQU4IsvvsC7774LAAgMDNQ3LiKTJoTAxdh0WCvkaODOIbxE5ekY6II7Sdk4E5WKAc09jR0OmTmDdcjZunUrmjZtioULF0IIgRUrViAiIsJQmycyKVEpOUjPLUBzbwdYsF8bUbk8Ha3g62yNawmZyMgtMHY4ZOaqfNU+duwYQkND8eSTTyI6Ohpz5szB7du38fLLL7PPD9VZRR2dW/rwlhdRRXUMdIEQwJl7qcYOhcyc3snP7du3MXbsWHTv3h0nTpzAmDFjcPXqVXz66adwduaDHanuysgtwN2kbAS52cKBT28nqjA/Z2u42ytxNS4DOapCY4dDZqzSyU9KSgrmzp2L5s2b47fffkNISAiOHTuGn3/+GcHBwdURI5FJuRRbNLydrT5ElSFJEjoEOKPwv4+EITKWCic/KpUKK1asQIMGDfDFF1/Az88PW7duxdGjRxESElKdMRKZjEK1BlfiMuBoreBstUR6CHa3g6O1Ahej01Cg1m/+OKKqqvBor8aNG+PevXtwcXHBypUr8fzzz0Mu54y2ZF5uPchCboEaHQKd+ZwiIj3IJAnt/J1w4HoirsRloIkLnwNJNa/CZ11UVBQkSYIQAh9//DE+/vjjcl8jSRKioqKqFCCRKbkUmw65TEIzLwdjh0JUazXzcsCJOyk4ey8VDZ3cjB0OmaFKpdxCCKSkpCAlhbN0kvlJzspHXPrD53hZKdjqSaQvC7kMbfyccPxOMu6k5Bk7HDJD1f5sL6K64nJcBgCghR7D2zMyMgwdTo1sm6i6tPJ1RHhkCi7EZRk7FDJDvNlKVAGFag0i4jPgYmMJ70o8mLFAlQ9IMvj5+VVjdA8Vqjl0mGoPK4UcLXwccT46DVaBbY0dDpkZJj9EFXDzQRbyCzXoHORQqY7O6sICQGjw2rrdcHByrZbY0pMS8OHMIVCr1dWyfaLq0tbfCRei0+DQeYyxQyEzw+SHqAIu/7ejc1M9OzorbexgZVs9zwDLy+FtA6qdHKwUaOBmjZtog6vxmQhx4EACqhl8KBFROYo6Ojd0t2NHZyIDa+1lCwBYdyLGyJGQOWHyQ1SOqnR0JqKyudoqkHvnNPZcS0JUcraxwyEzweSHqAz6dnQmoopLP/ErNAL49vBdY4dCZoLJD1EZijo6t/CpXEdnIqq4/OhLaOFlh59PRyMpK9/Y4ZAZYPJDVIaqdnQmooqZGuKL/EINNhyLNHYoZAaY/BCVgh2diWpO38ZuCHS1wfrjUcjO55xVVL2Y/BCVgh2diWqOXCZhZo/6SM8twM+no40dDtVxTH6IdNB2dLZlR2eimjKmnS/c7Czx7eG7KFDzkUpUfZj8EOmg7ejszY7ORDXFSiHH1NBAxKbl4u+L8cYOh+owJj9EOlxiR2cio5gYEgAbSzm+DrsNIYSxw6E6iskP0b8kZ+Ujnh2diYzCycYST3b0x7X7mTh0M8nY4VAdxeSH6F8uxaYDAFqyozORUUzvFgi5TMI3h24bOxSqo5j8ED2iQK1BxP1MuNpawosdnYmMwtfZBsNbeeHorWRc/u+XESJDYvJD9IibCVlQFWrQwseRHZ2JjGhWj2AAwOpDd4wcCdVFTH6IHnEpNh0WMglNPe2NHQqRWWvm7YDuDd2w41I8olNyjB0O1TEml/zcvHkTy5YtQ9euXeHp6QlbW1s0a9YMc+bMQXx8yaGPhYWF+PDDD9G4cWMolUp4e3vj2WefRXJyshGip9osMTMf9zPy0NDDDkp2dCYyutk9gqHWCKw9wgeekmGZXPKzdu1afPzxx/D398eiRYvw6aefIiQkBKtWrULz5s1x7dq1YuWnTZuG1157DY0aNcKXX36JqVOnYv369ejVqxeys7ONdBRUG11mR2cik9K1gSuaeTlgS3g0UrNVxg6H6hALYwfwb2PHjsVrr70GJycn7bJZs2YhJCQEs2fPxpIlS/Dzzz8DAPbv34+NGzdixIgR2L59u7Z8+/btMXbsWHzyySdYsmRJTR8C1UIFag2u3c+Em50lPB3Y0ZnIFEiShNk962PuT+ex8UQUXuzb0NghUR1hci0/HTp0KJb4FHnyyScBABcvXtQu27BhAwBg/vz5xcqOGTMGgYGB2vVE5bmekAmVmh2diUzN0JZe8HGyxvfHIpFXoDZ2OFRHmFzyU5rY2FgAgIeHh3bZyZMnIZPJEBISUqJ8ly5dcPv2baSkpNRYjFR7XY5Nh0IuoQk7OhOZFAu5DDO6BSE5W4Vfz8YYOxyqI0zutldpFi9eDOBhH58iMTExcHNzg1KpLFHe19dXW8bFxUXnNuPj43V2oo6IiDBEyFRLJGYXICEjH829HaC0YEdnIlMzrqMfPtt3E2sO3cGTHf0hl7F1lqqmViQ/7733Hn799VeMGjUKU6ZM0S7PycmBs7OzztdYWVlpy5Rm9erVeOuttwwbLNU6EQkPzxF2dCYyTbZKC0wKCcCXB25hz9X7GNTCy9ghUS1n8snPZ599hjfeeAO9evXCjz/+WKw/ho2NDfLz83W+Li8vT1umNLNnz8aIESNKLI+IiMDEiROrGDnVBpKlNW4l5cLdXgkPdnQmMllTQgPxzeE7+DrsDgY292TfPKoSk05+/u///g8LFixA37598ccff5RIZHx9fXHjxg3k5+eXuPUVExOjLVMaLy8veHnxG4Q5s23aEwUagRZs9SEyafXslRjTzhebT91DeGQqOgXp7s5AVBEm2+H5ww8/xIIFCzBo0CD89ddfOltwOnXqBI1Gg5MnT5ZYd/z4cQQHB5fa34dICAG7NoOgkElo7MGOzkSmbmb3IEgS+MBTqjKTTH7ee+89vPbaaxg2bBi2bdum7b/zb5MmTQIAfPLJJ8WW//bbb4iMjNSuJ9LlSnwWlJ4N0MDNGpYWJvlWIKJH1K9nhwHNPLA34gFuJmQaOxyqxUzuttd//vMfvPHGG/Dw8MDo0aOxdevWYuvt7OwwatQoAEC/fv3w1FNPYfPmzRg+fDhGjhyJu3fv4tNPP0WzZs2wYMECIxwB1RZbzz0c6dfMo/R+YURkWmb1CMauKwlYc/gOVoxtbexwqJYyueQnPDwcAJCQkIDp06eXWB8QEKBNfgBg/fr1aNmyJdatW4fnn38eLi4umDRpEt59913Y2dnVVNhUy6TnFmDn1UTkx92AWwj7fREZW0ZGRoXKNXSWo62vA34/F4tZXbzhbl9yqhNdLC0tS72LQObH5JKf77//Ht9//32FyysUCixatAiLFi2qvqCoztl6Ohp5BRpknvsLGN3T2OEQma0CVT4gyeDn51fh11g36AT3MUvQccJCpIWtr9Br6nl44F5kJBMgAmCCyQ9RddNoBH44EQVnawtERRw2djhEZk1dWAAIDV5btxsOTq4Veo0QAj9fSIRl1yfw0gsvlNtnLy8nC8sn9IRKpWLyQwCY/JAZOnjjAaKSc/B0qB/OqwuMHQ4RAVDa2MHKtuJdFdoHabAv4gFupanRLsChGiOjuohDXMjsrDsaCblMwhPt2NeHqLZq4mkPG0s5zkWnQa0Rxg6HahkmP2RWbidm4fDNJAxs7gFPh4p1lCQi02Mhk6GNnxOy8gtxg8PeqZKY/JBZ2XAsEgAwpUugUeMgoqpr5eMIhVzCmXupEIKtP1RxTH7IbGTmFeCXMzFo4mnPqfGJ6gClQo4W3o5IzlIhKqX0h1gT/RuTHzIbv56JQbZKjamhgXwoIlEd0cbfCTIJOBOVauxQqBZh8kNmQaMR2HA8Co7WCoxs42PscIjIQBysFGjkYY+Y1FwkZOQZOxyqJZj8kFk4fCsJd5Ky8WRHP1hbyo0dDhEZUDt/ZwBs/aGKY/JDZuHbw3cgk4CJIQHGDoWIDKyevRIBLja49SAL6bmcu4vKx+SH6rzLsek4fDMJQ1t5w8+FDzElqovaBzhDADgdlWLsUKgWYPJDdd7qQ3cAALN71DdyJERUXXydreHpYIWIuExk5rH1h8rG5IfqtHvJOfj7Yhy6N3RDCx9HY4dDRNVEkiR0CnKBWgj2/aFyMfmhOu3bI3egEcAzPYONHQoRVbNAVxu42ytxOS4D2fmFxg6HTBiTH6qzkrPy8fPpaLT0cURocMWeFk1EtZe29UcjcPYeW3+odEx+qM5afywSeQUazO5Zn5MaEpmJ+m62cLOzxMWYdOSo2PpDujH5oTopO78Q649HIcDVBoNb8OntROZCkiR0CnRBoUbg3L00Y4dDJorJD9VJW8KjkZ5bgJnd60MuY6sPkTlp4G4HF5uHrT95BWpjh0MmiMkP1TmqQg3WHrkLNztLjG3va+xwiKiGSZKEjkHOUKk1OB+dZuxwyAQx+aE659ezMYhNy8W0rkGwUvBRFkTmqJG7PRytFTgXnYa8Qo2xwyETw+SH6hRVoQZf7r8FZxsFpoQGGjscIjISmUxCSH0XqAo1uBCXZexwyMQw+aE65efT0YhNy8XMHvVhp7QwdjhEZESNPezhZmeJS/HZkNtxugv6HyY/VGfkF6rxnwO34GJriSldAo0dDhEZmSRJCA12g1oAjqFPGjscMiFMfqjO2HzyHuLT8zCrR33YstWHiPBw1mdPewXsWg9AVEquscMhE8Hkh+qEzLwCfL7/FjwclGz1ISItSZLQyc8BkkyO/xyKMnY4ZCKY/FCdsObQHaRkqzCvXyNYW3KEFxH9j5eDJXJuncI/VxNxOTbd2OGQCWDyQ7Xeg4w8rDl8F8H1bDmvDxHplHZoAyQAH+26buxQyAQw+aFa79O9N5FboMarg5rAQs5TmohKKkiMxODm9RB2IxHHbycbOxwyMn5SUK12OTYdP4XfQ6cgF/Rv5mHscIjIhL3QIwCWchne/usq1Bph7HDIiJj8UK0lhMCyP65AArBseHM+uZ2IyuTrbI3p3YJwNT4DP5+ONnY4ZERMfqjW+uNCHE5HpWJC5wA083YwdjhEVAu80KcB6tkr8fGu68jIKzB2OGQkTH6oVsrIK8B7OyLgaK3A/P6NjB0OEdUSdkoLvDKwMZKzVfhs701jh0NGwuSHaqUV/1xDQkY+XhvcBM62lsYOh4hqkTHtfNHazwnfH4vE1bgMY4dDRsDkh2qd05Ep2HjiYSfncR38jB0OEdUyMpmEd0e1gBACr/9+iZ2fzRCTH6pV8gvVeO23S7C0kOH90S0hk7GTMxFVXgsfR0zrGoTz0WnYdOqescOhGsbkh2qVj3ddx60HWZjTpwGC69kZOxwiqsXm928EL0crrNh5DfHpfO6XOWHyQ7XGsVtJ+PbIXbT1d8IzPYONHQ4R1XK2Sgu8+1gLZOYX4tVfL0EI3v4yF0x+qFZIzynAgq0XYK2Q49Mn2nAmZyIyiD5NPPB4e18cupGILeGc+8dc8BOETJ5GI7Bg6wXEp+dhybBmCHSzNXZIRFSHvDmsGbwcrfDO3xGITskxdjhUA5j8kMn7+tBt7I1IwMg23hjXkaO7iMiwHK0VWDG2FbLyC/Hi5nMoUGuMHRJVMyY/ZNKO3krCx7uuo7GHPd4f3ZKPsCCiatG9YT3M7lkf56PT8PFuPvm9rmPyQybr1oNMPLvxDGwsLfDVxHawsbQwdkhEVIe9PKAx2vg5YXXYHRy49sDY4VA1YvJDJikxMx9T14UjW6XGqgntUJ/D2omominkMnzxVFs4Wisw56dzuJOYZeyQqJow+SGTk5lXgKfXhyMmNRfvjmqBHo3qGTskIjITfi42+HJ8W2TnF+LpDaf58NM6iskPmZTs/EJMXReOCzHpmNOnAZ7s5G/skIjIzHRvWA9vDG2GO4nZeGETO0DXRexEQSYjO78Q078Px5moVMzuUR/z+LR2IjKgjIyKP8R0bEsXXInxxG/n7+OlTafx7ojGkJUz4MLS0hJWVlZVDZNqAJMfMglJWfmY/n04LsakY1rXQLw2uAlHdhGRQRSo8gFJBj+/Sk6VIclQb9Qi/I0u2Lz+W6TuW1Nm8XoeHrgXGckEqBZg8kNGF5WcjanrwnE3KRtz+jbEvH4NmfgQkcGoCwsAocFr63bDwcm1Uq8t1Aj8HZECdBiJrsPGo0uAvc7rU15OFpZP6AmVSsXkpxZg8kNGdeD6A8zdfA6Z+YV4e2RzTOoSaOyQiKiOUtrYwcq28iNHH2tni+0XYnHpfjYgt0DvxvX4Ba2WY4dnMopCtQYr997A9O/DIUkS1k3tyMSHiEySpYUMo9r4wNfZGpdi07Hj8n0UshN0rcaWH6pxd5OyMW/LeZyPTkNTLwesntge/q42xg6LiKhUCrkMI1t7Y/fVBNx8kIXMvAIMb+UNWyU/Rmsj/tWoxuQXqrHm0B18sf8WVGoNZveoj/kDGkFpITd2aERE5bKQyzC4hSec7iQjPDIVm07dw8DmnvB34Ze32obJD+klLy8PKpWqQmWFENh/IxkrD0QiKiUXQa7WWDyoAToEOCE/Jxv5pbyOw0aJyNRIkoTQYDe42SmxL+IBfj8Xiw4BzmjtoTR2aFQJTH6o0vLy8uAfGIjEhIRyy1oFtYNTt/FQejeBRpWL9GNbEBW+DQdfKSz3tRw2SkSmqpGHPdztldh5+T5OR6XiZoIcSt/mxg6LKojJD1WaSqVCYkIClvwYBiubkiMn1BqBOyl5uBCXheScQsgkoKm7Ddr5uMOmxyIAi8rdB4eNEpGpc7KxxBMd/HDuXipO3EmG54QPseC3CLwxrAUC3WyNHR6VgckP6c3qX8NGk7Lyce1+Jq7GZSC3QA0LmYQ2vk5o6+8EB2uFESMlIqoecpmEDoEu8LOT4butf2APuuDAjTA81tYHz/QKRjAfymySmPxQlWTkFuBGQiauJ2QiKethHyBHawXaBzijmbcDrBVV68xcmenoTWnbRGReHK0tkPj7u9h/6R6+OhKDrWdi8MvZGPRt4o7xnf3Rs5E75DLODWQqmPxQpWg0AlfjM+EY+iR+u5SExOyHTzxWWsjQwtsBjT3t4eNkXeUJwPSejl4Phery+x8REVVEe39H/PyMH05HpuDrsDvYfy0BeyMewN1eiYHNPTG4hSc6BbnAQs5p9oyJyQ+VKyEjD8dvJ+PY7SQcvJ6IB5n5cOo+Eel5hWjkYYfGHvYIcLU16LeaqkxHX1HpSQn4cOYQqNXqatk+EZmvDoEu+DbQBbFpudgSHo0/L8ThhxNR+OFEFJxtFOjfzAM9GtVD5yBX1LPnSLGaxuSHSkjMzMeJO8k4ficZJ24n405StnZdA3c7DGleDx/MmYg3V34PW3v7ao1F3+noKyIvJ6tatktEVMTHyRrz+zfCvH4NcT0hEzsv3cc/l+/j59Mx+Pl0DICH19WQ+i7oGOiC1r5OCHC14eMzqhmTHzOn1gjcSMjEuXtpOHsvFWfvpeJO4v+SHR8na4xt74su9V3RJdgV3k7WyMjIwFvRl3n/moiogiRJQhNPBzTxdMC8/o1wLzkHJ+4ka382nriHjSfuAQAcrCzQ0tcRLX2c0NrXES19HbXdCSozx5q+zGGONSY/ZkQIgdi0XFyNy8DFmHScvZeKC9FpyFb977aPn4s1Rrf1QUiwK7rUd4UfZy4lIjI4f1cb+Lva4ImOfhBCIDolF2fvpeJiTDouxqThbFQajt5K1pZ3slGgkbstju7YivTIqyh4cAcFydEQhYZPhMxhjjUmP3VUXoEatxOzcDUuA1fjMxARn4GrcRnIyPtf515rhRytfB3R1t8Z7fyd0MbfCe72dfdkJyIyRZIkaZOhUW19ADx8+PPtxGxcjEnDxZh0XI3PwOW4DCia9Ydbs/4PX4eHo8xcbSzgaqOAy3//tbWU6X3bzFzmWKszyc9vv/2GFStW4NKlS7C0tET37t3x3nvvoUWLFsYOrdpk5xciPj0XkUk5iEzOxp2kbET+9ycuPa9YWTulBZp62aOZlwOaeTugubcjmnjac8QBEZEJspDL0NjTHo097fF4h4ejXtPS01EvoDGm/d9vSC+QkJSVj6QsFW4n5+F28v+u+VYWMrjZKeFmp4SzrQLONpZwsbWEjaWcfYn+q04kP2vXrsXTTz+NFi1a4MMPP0ReXh6++OILhIaG4ujRo2jZsqWxQ9Qq635tgVqDjLxCZOQWPvw3rxDpeYVIzy1AUpYKD7JUeJCpwoPMfDzIVBW7XVXExlIOf2crtPR2Q6CLNRp52KGJhy18nKwg+9dJn5OtX4dfzo9DRKRbdV4fszIzUZiegCAXq2IDQVSFGiRn5yMx82EylJSVj4TMPMSk5RZ7vaVcpk2GnG0sYW9lAQcrBeytLGCrtCjWj7O6r/PG7ldU65Of1NRUzJ8/H76+vjh69CgcHBwAAE888QSaNWuGuXPnYv/+/UaO8qEtJ+9i/tufIl8jQaa0g8zaHjIrO8isbCGzsofM0rrcbWjyc6DOSkZhZjLUWSlQZyajMC0OBSlxKEiNhSY7DdflFtDUwNw1nB+HiOghY85NZmkhg5ejNbwc//cZIoRARl4hUnNUSM1WITWnQPv/hIySj5OWAFhbyqGUA+7j3kHbZz+DOjcDQpUDjSoPoiAPmoI8CFXuw38LVIDQAEINodH89/9C+38h1EDR/zUPy0GjgdCoAaGBq6sr7t26ZrQEqNYnP9u3b0dGRgbmz5+vTXwAwN/fH2PHjsX69esRHR1dIydkeXZcvg9Fi4FQ4OGJZmkhQWkhg1Iug7Lo/0U/8kd/l2CjkMPGUgZLuQxAcKn7KJq7hvPjEBHVHFObm0ySJDhaK+BorUCga/HnjOUXqpGeU4CMvEJk5hUgM68QGXkFyFGpkZWbD6VXQ8iU1ftsssL0BKP2K6r1yc/JkycBAKGhoSXWhYaGYv369QgPDzeJ5GfRgGBsWvAY3vj2Lzg42FfLvdeiuWs4Pw4RUc2rDddepYUc7g5yuDuUXJeWeB/LJwzFG5uPwtLWEYVqAZVagwK1BgVq8d9/NShUCwgAGiEgxMOWJg0AiEeWQUDz33XikeUFBQU4cfwQgLEGOR591PrkJybm4SRRvr6+JdYVLSsq82/x8fGIj48vsfz8+fMAgIiICANF+VBWVhYK0xMQd/08kqzKv8Wl1z7SUgAAUVfPwdbekfvgPmrNPmpqP9wH98F9lL+P2Gvnde5D/t+fqlDl5eKfQ+tx/vx02NkZNlEs+tzOzc0tu6Co5fr06SMAiNu3b5dYt2/fPgFAvP/++zpfu3TpUgGAP/zhD3/4wx/+1KGfjRs3lpk71PqWHxubh5Pw5eeX7MCVl5dXrMy/zZ49GyNGjCixPDU1FREREWjbti2sraunhcaYIiIiMHHiRGzcuBFNmzY1dji1GuvSsFifhsO6NCzWp+FUZ13m5uYiMjISAwcOLLNcrU9+Hr219e9KLOuWGAB4eXnBy8tL57q+ffsaMErT1LRpU7Rr187YYdQJrEvDYn0aDuvSsFifhlNdddm1a9dyy9T6Ge46deoEADh+/HiJdUXLOnbsWKMxERERkemq9cnPqFGjYG9vjzVr1hSblOnevXvYunUrevXqZRIjvYiIiMg01Prkx9nZGR999BFiYmLQtWtXfPnll/jkk0/Qo0cPSJKElStXGjtEIiIiMiG1PvkBHnZc3rp1K2xsbPDKK6/g7bffRsuWLXH06FG0bt3a2OERERGRCan1HZ6LjB07FmPHGm/CpNrEy8sLS5cuLbWzN1Uc69KwWJ+Gw7o0LNan4ZhCXUpCCGG0vRMRERHVsDpx24uIiIioopj8EBERkVlh8kNERERmhclPHafRaBASEgJJktCvX78S63NycvDaa68hMDAQSqUSgYGBWLRoEXJycowQrWkrqy6///57SJKk86dDhw5Giti0lFY/kiTh8uXLxcoWFhbiww8/ROPGjaFUKuHt7Y1nn30WycnJRore9FS0Pg8ePFhqOTc3NyMegWnJyMjAm2++iaZNm8La2houLi7o3LkzNm7cWKwcr5kVU5H6NOZ1s86M9iLdVq5ciStXruhcp1arMWTIEISFhWHSpEno0aMHLly4gI8++ginTp3Cnj17IJMxPy5SVl0Wef3110s8ZsXV1bU6w6pVunfvjlmzZpVY/u+JSKdNm4aNGzdi2LBhePnll3H37l2sXLkSR44cwYkTJ2Bra1tTIZu0itYnAMyaNQvdu3cvtszKyqraYqtNYmNj0bt3byQlJWHq1Klo3rw5srOzcePGDURFRWnL8ZpZMRWtzyJGuW4a4MHqZKJu374tbGxsxMqVKwUA0bdv32Lr165dKwCIF198sdjyjz/+WAAQ69evr8lwTVp5dblu3ToBQBw4cMA4AdYCAMSUKVPKLbdv3z4BQIwYMaLY8l9++UUAEG+99VY1RVi7VLQ+Dxw4IACIdevWVXtMtVWfPn2Ep6enuHfvXpnleM2smIrWpzGvm0xR67CZM2eiefPmePHFF3Wu37BhAwBgwYIFxZY/99xzsLa21q6n8uvyUVlZWcjPz6+BqGqngoICZGZmlrq+6LybP39+seVjxoxBYGAgz8t/Ka8+H5WTk4Pc3Nxqjqh2OXr0KPbv349XX30Vfn5+UKvVyMrK0lmW18zyVaY+H1XT100mP3XUmjVrcOjQIaxZs0ZnM6wQAuHh4fD29kZAQECxddbW1mjTpg3Cw8NrKlyTVl5dPmrkyJGwt7eHlZUVGjZsiBUrVqCwsLCGIjV9v/zyC6ytreHg4AAnJydMnDgRkZGRxcqcPHkSMpkMISEhJV7fpUsX3L59GykpKTUUsWmrSH0WmTt3LmxtbWFjYwM/Pz/2U/mvv//+GwAQHByMMWPGwNraGvb29vD29sY777wDtVoNgNfMiqpofT7KGNdN9vmpg+Li4rBw4UIsWLCg1Md7pKSkICcnBy1atNC53tfXF8ePH0dGRgYcHByqM1yTVpG6BAAbGxs88cQT6NevH7y8vBAbG4sffvgBr776Kg4fPozt27ebfV+ADh06YMyYMWjUqBHy8/Nx5MgRrFmzBjt37sTRo0fRpEkTAEBMTAzc3NygVCpLbMPX11dbxsXFpUbjNzUVrU+FQoGhQ4diyJAh8PPzQ2JiIn7//Xd88MEH2Lt3L8LCwmBjY2PkozGeiIgIAMCMGTMQFBSEb7/9FpIkYdWqVVi8eDHu3buHb775htfMCqpofQJGvm7W+I02qnYjRowQwcHBIicnR7sM/+qncu/ePQFAdO/eXec2Jk2aJACI+Pj4ao/XlFWkLkuj0WjEk08+KQCIzZs3V2eYtdaOHTsEADFw4EDtMplMJvz8/HSWX7x4sQAgjh8/XlMh1iq66rM0r776qgAg3n///RqIzHT17dtXABCBgYEiLy9Puzw/P18EBwcLSZLEtWvXeM2soIrWZ2lq6rpp3l9F66CffvoJf/zxB77++mtYW1uXWq7om15p91jz8vKKlTNHFa3L0kiShKVLlwIA/vrrL0OHVycMHjwYnTt3xr59+4qdczwv9aOrPkuzePFiyGQysz83i97b48ePL9baaGlpiQkTJkAIgQMHDvCaWUEVrc/S1NR1k8lPHZKfn485c+ZgwIABCAwMxK1bt7Q/AJCbm4tbt24hISEBLi4usLGxQUxMjM5txcTEwMHBwWybbytTl2UJCgoCADx48KDaY66tgoKCUFhYqO3H4+vri6SkJJ0fMkXna9HtLyrp3/VZGltbW7i7u5v9uVk0LYCuh2wWLUtJSeE1s4IqWp9lqYnrJpOfOiQ3NxeJiYnYvXs3GjZsWOwHAI4dO4aGDRti7ty52kmk4uLiSsy7kJubi/Pnz6Njx47GOAyTUJm6LMuNGzcAAJ6entUec21148YNKBQK7bwenTp1gkajwcmTJ0uUPX78OIKDg82+v09Z/l2fpcnIyEBCQoLZn5tFHeujo6NLrCta5uHhwWtmBVW0PstSI9fNaruhRjVOpVKJrVu36vwBIFq2bCm2bt0qTpw4IYQQYs2aNTrnrPjkk08EAPH9998b4zBMQmXrMikpqcQ2CgoKxNChQwUA8euvv9b0IZgUXfUjhBCbNm0SAMTw4cO1y/bs2aNznp9ff/1VABDLli2r1lhrg8rUp66yGo1GzJ49WwAQn3zySbXFWRukpaUJJycn4eXlJTIyMrTLMzMzhY+Pj1AoFNr5anjNLF9l6tOY101JCCGqL7UiUyFJEvr27Yu9e/dql6nVavTu3RuHDx/G5MmTtbOVrlq1Ct27d8fevXshl8uNGLVp0lWX3t7e6NatG1q2bAkvLy/ExcXhp59+QkREBJ588kls2rQJkiQZMWrjmjdvHo4ePYo+ffrA398fKpUKR48exa+//gpPT08cOXIE9evX15YfP348Nm/ejGHDhmHkyJG4e/cuPv30UwQFBeHkyZOws7Mz4tEYX2Xqs2PHjvDw8ECHDh3g6+uLxMREbN++HSdPnkSPHj2we/dunSPrzMmGDRswZcoUNG7cGDNmzIAkSfjuu+8QERGBd999F6+//joAXjMrqqL1adTrZrWlVWRSUMoIpczMTLFw4ULh7+8vFAqF8Pf3F6+88orIysoyQpS1g666XLBggejQoYNwdXUVFhYWwtHRUXTv3l2sW7dOaDQaI0VqOrZv3y4GDRokfH19hZWVlVAqlaJx48Zi/vz5IiEhoUR5lUol3nvvPdGwYUNhaWkpPD09xaxZs0RiYqIRojc9lanPDz74QHTt2lW4u7sLhUIh7OzsRMeOHcWnn34qVCqVkY7A9OzYsUP06NFD2NraCmtra9GpUyedo414zayYitSnMa+bbPkhIiIis8IOz0RERGRWmPwQERGRWWHyQ0RERGaFyQ8RERGZFSY/REREZFaY/BAREZFZYfJDREREZoXJDxEREZkVJj9ERERkVpj8EBnIwYMHIUkSli1bVmx5r169auy5Xrr2VVpcVLcY6m+/bNkySJKEgwcPGjZAIhPC5IfMmiRJ5f5Ux4dAbUpIvv/+e0iShO+//97YodS4qVOnQpIkREZGGjsUk1aTCT6RIVgYOwAiU7B06dJS1wUGBlZp2xs2bEBOTk6VtlEVnTp1QkREBNzc3IwWA1U/Y59nRLUJkx8ioFpbYPz9/att2xVhY2ODJk2aGDUGqn7GPs+IahPe9iKqpISEBMyYMQMeHh6wtrZGmzZtsH79+lLL//uWwNSpU9G7d28AwFtvvaXXLbaffvoJ7du3h7W1Ndzd3TFp0iTExcXpLFvaLbY7d+5g1qxZaNCgAaytreHi4oKWLVvimWeeQXJysjb2adOmAQCmTZtWLNaiW0FxcXFYvnw5unbtCk9PT1haWsLb2xvjx4/H1atXS8QTGRkJSZIwdepUREZG4sknn4SbmxusrKzQoUMH/PXXX6Ue95YtW9C3b1+4uLjAysoKgYGBeOqpp3D69OkSZTdv3ozevXvDyckJVlZWaNq0Kd555x3k5+dXpIohSZL27xoUFKQ97n+3BN68eROTJ0+Gj4+P9tgnT56MmzdvVmg/WVlZsLS0RNeuXYstz83NhZWVFSRJwg8//FBs3VdffQVJkvDdd99pl1X21tOZM2cwaNAg2Nvbw8HBAf369cPx48cr/Hrgf3/LsLAwAMVvI/fq1QtqtRp+fn5wcHBAVlaWzm28+OKLkCQJv/zyi3ZZ0evj4uIwadIkuLu7w9raGu3bt8emTZtKjWfXrl0YMmQI3NzcoFQqERwcjIULFyItLa1Sx0V1H1t+iCohKSkJoaGhuHPnDrp164Zu3bohPj4ezzzzDAYMGFChbYwaNQoAsH79evTs2RO9evXSrqvILbZPP/0U8+fPh5OTEyZPngwnJyfs2rULoaGhcHR0rFAM8fHx6NixIzIyMjBkyBCMGTMGeXl5uHv3Ln744Qe88MILcHV1xdSpU+Hk5ITt27dj5MiRaNOmjXYbTk5OAIBDhw7hgw8+QO/evTFmzBjY2dnh5s2b+OWXX/DHH3/g6NGjaN26dYkYoqKi0KlTJ9SvXx+TJk1CSkoKtmzZgpEjR2Lv3r3aBBEAhBCYNm0a1q9fDzc3N4wePRr16tVDTEwMDhw4gMaNG6NDhw7a8tOnT8e6devg6+uLMWPGwMnJCSdOnMDixYuxb98+7NmzBxYWZV/+li5dim3btuHChQuYO3eu9niL/gWA8PBw9OvXD5mZmRgxYgSaNWuGa9euYePGjdi+fTv27t2Ljh07lrkfOzs7dOrUCSdPnkRmZibs7e0BAEePHtUmavv27cOkSZO0r9m3bx8AoG/fvmVuuzTHjh1Dv379oFKpMHr0aDRo0ADnz59Hr1690KdPnwpvx8nJCUuXLsX333+PqKioYrePAwMDIZfLMXPmTCxduhSbN2/GzJkzi70+NzcXGzduhKenJ0aOHFlsXWpqKkJDQ+Hk5IRp06YhLS0NP//8MyZMmIDY2FgsXLiwWPm33noLy5Ytg4uLC4YNGwZ3d3dcvHgRH3/8MXbs2IHjx4/DwcFBj9qiOkkQmTEAAoBYunSpzp/333+/WPmZM2cKAOKll14qtjw8PFxYWFhot/Wonj17in+/1Q4cOKCzbHnu3r0rFAqFcHZ2Fnfv3tUuV6vVYvTo0drjKW9fn3/+uQAgVq5cWWIfWVlZIicnR/v7unXrBACxbt06nTElJCSIjIyMEsvPnz8vbG1txaBBg0ocQ1Gcy5YtK7bun3/+EQDE4MGDiy1fvXq1ACA6duwo0tLSiq0rLCwUcXFxJeJ97LHHih2HEEIsXbq01OPWZcqUKQJAsbouotFoRJMmTQQAsXHjxmLrfvrpJwFANG7cWKjV6nL3s3jxYgFA/PXXX9plr732mpDL5aJPnz7C19dXu1ytVgsXFxdRv379Ytuo6Hmm0WhE48aNBQCxbdu2YuVXrlyp/dscOHCg3LjL2neRuLg4YWFhIdq3b19iXdHf6vXXXy+2vCiGxx9/vFj93blzRzg7OwuFQiFu376tXb5//34BQHTp0kWkpqbq3Me/37Nk3pj8kFkrusiW9uPo6Kgtq1KphI2NjbC3ty/xASzE/z4oqzP5eeeddwQAsWTJkhLrbt++LWQyWaWSn9WrV5e7z/KSn7IMHz5cKJVKoVKptMuKkp+AgABRWFhY4jX+/v7C1dW12LIWLVoIAOLs2bPl7rNNmzbCwsKixIegEA8TJVdXV9GxY8cKxV9W8nPkyBHtB64u3bp1EwBEWFhYufs5ePCgACDmzZunXdaxY0fRqVMn8eWXXwoA4vr160IIIc6cOSMAiJkzZxbbRkXPs6K4e/ToUSKOwsJCERwcbNDkRwghxo4dKwCI06dPF1seEhIiZDJZifoFIORyubhz506JbRUlsI8mzqNGjRIAxOXLl3Xuv02bNqJevXoVPh6q+3jbiwgPb6uU59q1a8jJyUH37t113l7q1atXmX1/DOHs2bMAgJ49e5ZYV79+ffj5+SEqKqrc7YwYMQKvv/46nn/+eezatQsDBw5E165d0axZM72GLP/999/4+uuvcfr0aSQlJaGwsLDY+qSkJHh5eRVb1qZNG8jl8hLb8vPzK9b3JDs7G5cvX4aHhwfatm1bZhw5OTm4cOEC3NzcsHLlSp1llEolIiIiKnhkpSv6W5R2m6hPnz44cuQIzp07hx49epS5rS5dusDa2lp7Oys9PR1nz57FK6+8ot3+vn370KhRI+zfv7/M/VY0bl3nkFwuR7du3XD79u1iy1euXFmi38yoUaOK3QYty3PPPYdffvkFq1evxjfffAMAuHTpEk6cOIHBgwfrvN3r7++PoKCgEst79eqFt956C+fOndMuO378OBQKBbZu3YqtW7eWeI1KpUJiYiKSk5Ph6upaoZipbmPyQ1RB6enpAAAPDw+d6z09PU0ihookPwEBATh16hSWLVuGf/75B7/99huAh4nHyy+/jDlz5lQ4ps8++wwvvfQSnJ2d0b9/f/j7+8PGxgaSJGn7zOjqZPxo35lHWVhYQKPRaH8v+tD18fEpN5bU1FQIIZCYmIi33nqrwsegj6K/xb+TuiJFyyvS2dbS0hLdunXD3r17kZiYiGPHjkGtVqNv375o2rQpvLy8sG/fPjz77LPYt28fJEnSO/nR5zxeuXJlifMqMDCwwslP79690bRpU2zevBmffPIJ7O3ttUnQ7Nmzdb6mvPiKjgMAkpOTUVhYWO7fPCsri8kPAWDyQ1RhRa09CQkJOtffv3+/RmNo3rx5lWJo2rQptmzZgsLCQly4cAF79+7FF198gblz58LW1hYzZswodxuFhYVYtmwZPD09cfbs2RKJQGVHD+lSlCTFxsaWW7aoftq2batt4aguRfsqrc7j4+OLlStPnz59sGfPHuzbtw/Hjh2DlZWVdgRYnz59sHPnTuTn5+Pw4cNo3rw53N3dqxR3Zc5jQ0zy+Mwzz2Du3Ln48ccfMWXKFGzcuBE+Pj4YNmyYzvLlxfdovTo6OkKj0SAlJaXKcZJ54FB3ogpq0qQJbGxscP78+WLfOotUZiboots9arW6UjG0a9cOALRDix91584dREdHV2p7wMOWlvbt2+PVV1/F5s2bAQDbtm2rUKxJSUlIS0tDaGhoicQnKyvLIAmIra0tWrRogYSEhGK3OnSxs7ND8+bNceXKFYN8EJZ17EW34Er7ux84cADA//5m5SkaubVv3z7s378foaGhsLKy0q5LSUnBV199hezsbL1HeT0aj65zSK1W48iRI5XeZkXO5ylTpsDGxgbffPMNtmzZgrS0NMyYMUPnrU8AuHfvns6kq6i+H70FGhISgtTUVFy5cqXSsZN5YvJDVEEKhQITJkxAZmZmiTlzTp8+jR9//LHC2ypqer93716lYpgwYQIUCgW++OKLYh8MGo0GCxcuLHa7qCxnzpzRmcAVfdu2sbGpUKzu7u6wsbHBmTNnis3jUlBQgLlz5yIpKalC8ZSn6Dbc7NmzS8St0Wi0rSwAMH/+fKhUKkyfPl3nLafU1NQKJ2VlHXvXrl3RuHFjHDlypNgcNQDwyy+/4PDhw2jUqBG6detWoX21a9cOjo6O2L59O65cuVIswSm6xfX+++8X+10foaGhaNy4MQ4dOoTt27cXW/fll1+W6O9TERU5nx0dHTF+/HicO3cOb775pnYYfGnUajVeffXVYuf03bt38fnnn8PCwgITJ07ULp83bx4AYObMmTrnu8rOzsaJEycqfVxUd/G2FxHKnuH50Y6d7733Hvbt24eVK1fi9OnT2nl+tmzZgiFDhuCPP/6o0P4aN24MHx8f/PTTT1AoFAgICIAkSZg0aRICAgJKfV1gYCA++OADLFiwAG3btsW4cePg6OiIXbt2IS0tDa1atcLFixfL3f8PP/yA1atXo1u3bggODoazszNu376NP//8E0qlEi+99JK2bJcuXWBjY4OVK1ciOTlZ2+fixRdfhKOjI+bMmYMPPvgALVu2xMiRI6FSqXDgwAGkpKSgd+/e2haQqnj66adx+PBh/PDDD2jYsCFGjhyJevXqIS4uDvv378f06dO1f8Pp06fjzJkzWLVqFYKDgzFw4ED4+/sjJSUFd+/exaFDhzBt2jR8/fXX5e63b9+++OijjzBz5kyMGTMG9vb2cHJywgsvvKCdBLF///4YN24cRo4ciSZNmuD69evYtm0b7O3tsWHDBshkFfuOKZfL0atXL21C8mjyExAQgODgYNy+fRtyuVxnZ+WKkiQJa9euRf/+/TFmzJhi8/zs27cPgwYNwj///FOpbfbt2xdbt27F6NGjMWTIEFhbWyMgIKDY3ETAw47P3377LWJjYzF8+HD4+vqWus1WrVrh5MmTaN++PQYMGKCd5yctLQ0rVqxAcHBwsf1/8MEHWLRoERo2bIghQ4YgKCgIWVlZiIqKQlhYGLp161bp46I6zNjDzYiMCeUMdYeOId7x8fFi2rRpws3NTVhZWYnWrVuLdevWlTp8vbRhwKdOnRJ9+vQRDg4OQpKkSg0v3rRpk2jbtq1QKpXCzc1NTJgwQcTGxlZ4uPOJEyfEM888I1q1aiWcnZ2FlZWVCA4OFlOnThWXLl0qsb+dO3eKkJAQYWtrq62XouHJBQUF4pNPPhFNmzYVVlZWwsPDQ0ycOFFERkbqHCpeNNR9ypQpOo+trGHTGzduFD169BAODg5CqVSKwMBAMX78eHHmzJkSZf/8808xdOhQUa9ePaFQKISHh4fo2LGjeOONN0RERETZFfyITz75RDRp0kRYWlpqh+g/6tq1a2LixInC09NTWFhYCE9PTzFhwgRx7dq1Cu+jSNEUBA4ODiWmAZg1a5YAIDp16qTztZWdUuH06dNi4MCBws7OTtjZ2Ym+ffuKY8eOaYeSV2aoe2FhoVi0aJEICgrSznfVs2dPnWXbtGlTYk6jfyt6fWxsrJgwYYKoV6+eUCqVom3btuLHH38s9XWHDx8Wjz/+uPDy8hIKhUK4ubmJ1q1bi3nz5onw8PAKHw/VfZIQFRjjS0REVEWZmZnw9vaGi4sL7t69W2qrmCRJ6NmzZ6X60RFVBvv8EBFRjfjqq6+QlZWF5557rsK3A4mqA/v8EBFRtUlPT8dXX32F2NhYrFmzBl5eXnjuueeMHRaZOSY/RERUbVJTU7Fo0SIolUq0b98eX3zxhfbhrUTGwj4/REREZFZ405WIiIjMCpMfIiIiMitMfoiIiMisMPkhIiIis8Lkh4iIiMwKkx8iIiIyK0x+iIiIyKww+SEiIiKzwuSHiIiIzMr/A+FumsuX24URAAAAAElFTkSuQmCC\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Save the combined data to a new CSV file\n",
        "fname_sampled_seqs_df = f'output/{PDBID}_{chain}_ESMIF1_{temp}_{nsamples}_df.csv'\n",
        "sampled_seqs_df.to_csv(fname_sampled_seqs_df, index=False)"
      ],
      "metadata": {
        "id": "Rpbf_LhvS3VR"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}